<pre>Query         FD01848675_02448 RHS repeat family protein
</pre><pre>Match_columns 69
</pre><pre>No_of_seqs    119 out of 295
</pre><pre>Neff          6.21211
</pre><pre>Searched_HMMs 86581
</pre><pre>Date          Tue Feb 27 20:37:52 2024
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -o ../results/4214053.hhr -oa3m ../results/4214053.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 6SDK_A Stage 0 sporulation pro  94.1    0.29 3.3E-06   31.0   4.7   63    1-68      8-71  (199)
</pre><pre>  2 PF20188.2 ; DUF6551 ; Family o  94.0    0.22 2.6E-06   33.6   4.2   40   21-66     15-55  (242)
</pre><pre>  3 7BNR_B ParB family protein; DN  93.6    0.45 5.2E-06   30.6   5.0   43   21-68     28-71  (212)
</pre><pre>  4 1VZ0_A Chromosome-partitioning  91.9    0.72 8.3E-06   30.4   4.3   63    1-68     27-90  (230)
</pre><pre>  5 4UMK_B PROBABLE CHROMOSOME-PAR  91.8    0.73 8.4E-06   31.2   4.3   63    1-68     37-100 (240)
</pre><pre>  6 7NFU_B Nucleoid occlusion prot  91.0     1.2 1.4E-05   30.0   4.7   64    1-68     32-96  (252)
</pre><pre>  7 PF02195.22 ; ParBc ; ParB/Sulf  88.8     3.1 3.6E-05   22.3   4.5   61    1-66      2-64  (89)
</pre><pre>  8 PF08857.15 ; ParBc_2 ; Putativ  88.6    0.37 4.3E-06   31.6   0.8   62    3-68      1-67  (153)
</pre><pre>  9 7BM8_A Chromosome-partitioning  87.2     3.7 4.3E-05   28.1   5.0   43   21-68     58-104 (257)
</pre><pre> 10 6KY4_A Sulfiredoxin, chloropla  87.1     6.1   7E-05   23.2   5.3   43   21-68     42-85  (112)
</pre><pre> 11 6RYK_A ParB-like nuclease doma  86.4     4.7 5.5E-05   27.2   5.1   46   19-68     54-100 (229)
</pre><pre> 12 2HWJ_D Hypothetical protein At  84.9     5.1 5.8E-05   27.9   4.8   62    1-68     13-81  (205)
</pre><pre> 13 5K5D_B ParB domain protein nuc  78.8      12 0.00014   26.9   5.0   59    1-68     15-75  (300)
</pre><pre> 14 1XW3_A SULFIREDOXIN; retroredu  75.3      13 0.00015   21.4   3.7   29   39-68     50-84  (110)
</pre><pre> 15 6CKA_A Paratox; prophage, natu  73.4      20 0.00023   21.9   4.1   29   20-52      3-31  (68)
</pre><pre> 16 PF14805.10 ; THDPS_N_2 ; Tetra  48.5      78 0.00091   18.4   3.6   25   21-53     29-53  (67)
</pre><pre> 17 1OMH_A trwC protein; PROTEIN-D  48.2      48 0.00055   23.9   3.1   25   41-65    167-191 (293)
</pre><pre> 18 8A1B_A TraI; Relaxase, DNA BIN  46.8      31 0.00036   24.9   2.0   18   48-65    173-190 (306)
</pre><pre> 19 2K3U_A Chemotaxis inhibitory p  46.5      77 0.00089   20.2   3.4   33   22-55     13-45  (91)
</pre><pre> 20 PF11434.12 ; CHIPS ; Chemotaxi  46.5      77 0.00089   20.2   3.4   33   22-55     13-45  (91)
</pre><pre> 21 5VNY_A Lethal (2) giant discs   42.0      78  0.0009   17.7   2.8   20   21-40     41-61  (66)
</pre><pre> 22 8GLV_6S FAP339; cilia, microtu  40.2      50 0.00058   21.1   2.0   26   22-49     34-59  (87)
</pre><pre> 23 5UJE_A SbnI protein; staphylof  35.1 2.3E+02  0.0027   19.5   5.5   45   21-69     29-75  (242)
</pre><pre> 24 1VK1_A Conserved hypothetical   35.1 1.3E+02  0.0016   20.5   3.6   27   39-66     52-81  (242)
</pre><pre> 25 7ZSS_P de novo designed binder  34.6   1E+02  0.0012   18.1   2.6   21   20-40     45-66  (79)
</pre><pre> 26 6NR6_A Bifunctional transcript  31.8 2.8E+02  0.0032   19.3   5.3   42   21-66     27-70  (254)
</pre><pre> 27 3IMO_A Integron cassette prote  31.1      38 0.00043   23.1   0.4   41   19-60     47-92  (133)
</pre><pre> 28 PF07309.15 ; FlaF ; Flagellar   30.7      42 0.00048   21.3   0.6    8   55-62     45-52  (107)
</pre><pre> 29 PF21528.1 ; CC2D1A-B_DM14 ; Co  29.3 1.4E+02  0.0017   15.2   2.8   20   21-40     34-54  (56)
</pre><pre> 30 PF18315.5 ; VCH_CASS14 ; Integ  26.3   1E+02  0.0012   19.9   1.7   39   19-58     17-60  (97)
</pre><pre> 31 PF17455.6 ; LtuB ; Late transc  25.3 2.7E+02  0.0031   17.7   3.3   31   22-53     37-67  (79)
</pre><pre> 32 6T9K_B Transcription factor SP  24.4      48 0.00055   27.8   0.0   55    2-57    151-219 (604)
</pre><pre> 33 3ZGX_Z SEGREGATION AND CONDENS  22.7   2E+02  0.0023   17.8   2.5   21   19-39     13-33  (94)
</pre><pre> 34 PF02609.20 ; Exonuc_VII_S ; Ex  22.7 2.2E+02  0.0025   15.5   2.4   17   21-37      8-24  (52)
</pre><pre> 35 PF19702.3 ; DUF6200 ; Family o  22.5 1.5E+02  0.0018   18.5   2.0   25   21-45     44-72  (79)
</pre><pre> 36 6HIZ_DY mS72; mitoribosome, tr  22.5      99  0.0011   21.8   1.2   14   39-53     20-33  (163)
</pre><pre> 37 PF20162.3 ; Etd1 ; Septation p  22.4      47 0.00054   21.5  -0.3   28    2-29     42-70  (100)
</pre><pre> 38 1QV9_C F420-dependent methylen  21.5      87   0.001   24.1   0.9   33   11-43    149-183 (283)
</pre><pre> 39 7MJ2_F Pyridinium-3,5-biscarbo  21.3 1.9E+02  0.0022   20.8   2.5   19   19-37      2-20  (256)
</pre><pre>
</pre><pre>No 1
</pre><pre>>6SDK_A Stage 0 sporulation protein J; ParB, Chromosome segregation, Chromosome organization, DNA BINDING PROTEIN; HET: CDP, MSE; 1.81A {Bacillus subtilis (strain 168)} SCOP: d.268.1.0, a.4.14.0
</pre><pre>Probab=94.14  E-value=0.29  Score=31.05  Aligned_cols=63  Identities=27%  Similarity=0.322  Sum_probs=0.0  Template_Neff=11.400
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGEYTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      ++.+|..........+....+..+.+.+.....    +.||.|... +|.++-+| |+|+.|.+.++..
</pre><pre>T Consensus         8 ~~~~i~~~~~~~~~~~~~~~~~~l~~si~~~g~----~~pi~v~~~-~~~~~vi~G~~R~~a~~~~~~~   71 (199)
</pre><pre>T 6SDK_A            8 KIADLRPNPYQPRKHFDDEALAELKESVLQHGI----LQPLIVRKS-LKGYDIVAGERRFRAAKLAGLD   71 (199)
</pre><pre>T ss_dssp             EGGGEECCTTCCCCCCCHHHHHHHHHHHHHHCC----CSCEEEEEC-SSSEEEEECHHHHHHHHHTTCS
</pre><pre>T ss_pred             cHHHCCCCCCCCCccCCHHHHHHHHHHHHHcCC----CCCEEEEEC-CCcEEEEechhHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF20188.2 ; DUF6551 ; Family of unknown function (DUF6551)
</pre><pre>Probab=93.97  E-value=0.22  Score=33.62  Aligned_cols=40  Identities=20%  Similarity=0.051  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAG   66 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~   66 (69)
</pre><pre>                      +.++.+.+..+     -++||.|... +|++|-+| .+|+.|++.++
</pre><pre>T Consensus        15 v~~l~~~~~~~-----~~~pi~v~~~-~g~~~viDGqhR~~a~~~~~   55 (242)
</pre><pre>T D4K186_9FIRM/3   15 VELIARDFNEY-----TANEPKISFR-NGRYYVTDGQHTIEARILRN   55 (242)
</pre><pre>T ss_pred             HHHHHHhccHh-----hCCCCEEEEe-CCeEEEeCChhHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>7BNR_B ParB family protein; DNA-binding protein, CTP, Myxococcus, DNA-segregation, DNA BINDING PROTEIN; HET: UFQ, GOL; 1.7A {Myxococcus xanthus (strain DK 1622)}
</pre><pre>Probab=93.59  E-value=0.45  Score=30.63  Aligned_cols=43  Identities=28%  Similarity=0.303  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      +..+.+.+....    .++||.|... +|.++.+| ++|+.|++.++..
</pre><pre>T Consensus        28 ~~~l~~si~~~g----~~~pi~v~~~-~~~~~iidG~~R~~a~~~~~~~   71 (212)
</pre><pre>T 7BNR_B           28 LKELSESIKAQG----VLQPILVRKD-GDGYRIIAGERRWRASQAAGLK   71 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHC----CSSCEEEEEE-TTEEEEEECHHHHHHHHHHTCS
</pre><pre>T ss_pred             HHHHHHHHHHhC----CccCEEEEEc-CCceEEeecchHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>1VZ0_A Chromosome-partitioning protein Spo0J; NUCLEAR PROTEIN, CHROMOSOME SEGREGATION, DNA-BINDING, HELIX-TURN-HELIX; HET: CO; 2.3A {Thermus thermophilus HB27} SCOP: a.4.14.1, d.268.1.1
</pre><pre>Probab=91.95  E-value=0.72  Score=30.42  Aligned_cols=63  Identities=17%  Similarity=0.161  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGEYTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      ++++|.....-....+....+..+.+.|+...    -++||.|... +|+++-+| ++|+.|.+.++..
</pre><pre>T Consensus        27 ~i~~i~~~~~~~r~~~~~~~~~~l~~si~~~g----~~~pi~v~~~-~~~~~i~dG~~R~~a~~~~~~~   90 (230)
</pre><pre>T 1VZ0_A           27 PLASIRPNPRQPRKRFAEESLKELADSIREKG----LLQPLLVRPQ-GDGYELVAGERRYRAALMAGLQ   90 (230)
</pre><pre>T ss_dssp             EGGGEECCCCCHHHHHHHHHHHHHHHHHHHHC----CSSCEEEEEE-TTEEEEEECHHHHHHHHHHTCS
</pre><pre>T ss_pred             cHHHCCCCCCCCcccCCHHHHHHHHHHHHHhC----CCCCEEEEEc-CCeEEEEeChHHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>4UMK_B PROBABLE CHROMOSOME-PARTITIONING PROTEIN PARB; DNA BINDING PROTEIN-DNA COMPLEX; HET: SO4; 3.096A {HELICOBACTER PYLORI}
</pre><pre>Probab=91.82  E-value=0.73  Score=31.21  Aligned_cols=63  Identities=21%  Similarity=0.207  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGEYTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      ++..|..........+.+..+.++...+....    .++||.|... +|.++-+| ++|++|.+.++..
</pre><pre>T Consensus        37 ~l~~i~~~~~~~r~~~~~~~~~~l~~si~~~G----~~~Pi~v~~~-~~~~~iidG~~R~~a~~~~~~~  100 (240)
</pre><pre>T 4UMK_B           37 GIDEVMPNPYQPRKVFSEDSLEELAQSIKEHG----LLQPVLVVSE-NGRYHLIAGERRLRASKLAKMP  100 (240)
</pre><pre>T ss_dssp             --------CCCSSCCCCCCHHHHHSSTTTTTT----CCCCCCCEEE-ETTEEEESCGGGHHHHHTTSSS
</pre><pre>T ss_pred             cHHHcCCCCCCcccCCCHHHHHHHHHHHHHcC----CCcCEEEEee-CCEEEEEeChhHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>7NFU_B Nucleoid occlusion protein; CTP switch, chromosome segregation, protein-DNA recognition, peripheral membrane protein, DNA BINDING PROTEIN; HET: SO4; 2.5A {Geobacillus thermoleovorans CCB_US3_UF5}
</pre><pre>Probab=91.00  E-value=1.2  Score=30.04  Aligned_cols=64  Identities=16%  Similarity=0.104  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGEYTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      ++.+|..........+....+.++.+.|+...    -..||.|....+|.++-+| |+|+-|.+.++..
</pre><pre>T Consensus        32 ~~~~i~~~~~~~r~~~~~~~~~~l~~si~~~g----~~~Pi~v~~~~~~~y~vidG~~R~~a~~~~~~~   96 (252)
</pre><pre>T 7NFU_B           32 PVKSIIPNRFQPRTMFDEEKIDELALTIRTHG----IIQPIVVRECGNGRFEIIAGERRWRAVQKLGWT   96 (252)
</pre><pre>T ss_dssp             EGGGEECCTTSCCCCCCHHHHHHHHHHHHHHC----CSSCEEEEEETTTEEEEEECHHHHHHHHHTTCS
</pre><pre>T ss_pred             CHHHcCCCCCCCcccCCHHHHHHHHHHHHHcC----CCCCEEEEEcCCCcEEEecChhHHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>PF02195.22 ; ParBc ; ParB/Sulfiredoxin domain
</pre><pre>Probab=88.80  E-value=3.1  Score=22.35  Aligned_cols=61  Identities=16%  Similarity=0.076  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeEEEEeCC-CcEEeCC-chHHHHHHHcC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGEYTVLGNAEALLNGTLKPSALPPIKIWKDNL-GKIWTLD-HRRLAAFKLAG   66 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~-G~~~slD-NRRL~~fk~a~   66 (69)
</pre><pre>                      +..+|...+......+....+..+.+.++.....    +|+.|... + |.++.++ ++|+-|.+.++
</pre><pre>T Consensus         2 ~~~~i~~~~~~~~~~~~~~~~~~l~~~~~~~g~~----~pi~~~~~-~~~~~~v~~G~~r~~~~~~~~   64 (89)
</pre><pre>T PARB_PSEPK/39-    2 PVELVQRGKYQPRRDMDPEALEELAHSIRNHGVM----QPIVVRPI-GDNRYEIIAGERRWRATQQAG   64 (89)
</pre><pre>T ss_pred             CHHHccCCCCCCcccCCHHHHHHHHHHHHHcCCc----cCEEEEEc-CCCCEEEeeChhHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF08857.15 ; ParBc_2 ; Putative ParB-like nuclease
</pre><pre>Probab=88.58  E-value=0.37  Score=31.62  Aligned_cols=62  Identities=13%  Similarity=0.022  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             hHceechhhcchhhCC---CcHHHHHHHHHcCCCCHhHCCCeEEEE-eCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    3 KDVRYMQSSIKNQTGE---YTVLGNAEALLNGTLKPSALPPIKIWK-DNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         3 ~~IrftQ~sIs~~F~~---~~i~~~~~~l~~g~~~~~~ip~IrV~~-~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      .+|+=||..|+   .+   ..+........+....-..-.+|.|+. . +|++|-+| +.|+.|+..+|..
</pre><pre>T Consensus         1 ~~L~PtQ~~ig---~~~V~~K~~~~~~~~~~~~~~~~~~~~ipvv~~~-~g~~~liDGHHr~~a~~~~g~~   67 (153)
</pre><pre>T S5Y864_PARAH/7    1 IALRPTQMTVG---MKEVQRKRHDWREDSQSKEVNFLGHHLLPCVLGP-KKRYHLIDNHHLALALHLEGVT   67 (153)
</pre><pre>T ss_pred             CCCCCChHHcc---HHHHHHHHHHHHhcCHHHHHHHhhhCCcCEEECC-CCcEEEECCHHHHHHHHHccCc
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>7BM8_A Chromosome-partitioning protein ParB; chromosome segregation, CTP, molecular gates, protein-DNA recognition, DNA BINDING PROTEIN; HET: CTP; 2.73A {Caulobacter vibrioides (strain NA1000 / CB15N)} SCOP: d.268.1.0, a.4.14.0
</pre><pre>Probab=87.21  E-value=3.7  Score=28.15  Aligned_cols=43  Identities=23%  Similarity=0.294  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHCCCeEEEEeCC---CcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSALPPIKIWKDNL---GKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~ip~IrV~~~~~---G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      +.++.+.|....    .++||.|... +   |.+.-+| ++|+.|.+.++..
</pre><pre>T Consensus        58 ~~~L~~si~~~G----~~~Pi~v~~~-~~~~~~y~vidG~~R~~a~~~l~~~  104 (257)
</pre><pre>T 7BM8_A           58 LEDLSNSIREKG----VLQPILVRPS-PDTAGEYQIVAGERRWRAAQRAGLK  104 (257)
</pre><pre>T ss_dssp             HHHHHHHHHHHC----CCSCEEEEEC-SSSTTCEEEEECHHHHHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHhC----CCCCeEEEEC-CCCCCcEEEeeChhHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>6KY4_A Sulfiredoxin, chloroplastic/mitochondrial; sulfiredoxin, cysteine, sulfinic acid, peroxiredoxin repair protein, arabidopsis thaliana, antioxidant, oxidoreductase; HET: PO4, ADP; 3.2A {Arabidopsis thaliana}
</pre><pre>Probab=87.14  E-value=6.1  Score=23.23  Aligned_cols=43  Identities=19%  Similarity=0.198  Sum_probs=0.0  Template_Neff=10.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      +.++...++...    ..+|+.|... +|.++.++ ++|+-|.+.++..
</pre><pre>T Consensus        42 ~~~l~~s~~~~g----~~~pi~v~~~-~~~~~ii~G~~R~~aa~~~~~~   85 (112)
</pre><pre>T 6KY4_A           42 VKELMDSIRQIG----LQVPIDVIEV-DGTYYGFSGCHRYEAHQKLGLP   85 (112)
</pre><pre>T ss_dssp             HHHHHHHHHHHC----CCSCEEEEEE-TTEEECCSCHHHHHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHcC----CCCCeEEEEE-CCEEEEEcCchHHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6RYK_A ParB-like nuclease domain protein; ParABS, cytoskeleton, bactofilin, CTP, CELL CYCLE; HET: CTP; 1.7A {Myxococcus xanthus DK 1622}
</pre><pre>Probab=86.36  E-value=4.7  Score=27.22  Aligned_cols=46  Identities=22%  Similarity=0.206  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024   19 YTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus        19 ~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      ..+.++.+.++...    -+.|+.|....+|.++-++ ++|+.|.+.++..
</pre><pre>T Consensus        54 ~~~~~l~~si~~~g----~~~pi~v~~~~~~~y~lv~G~~R~~a~~~~~~~  100 (229)
</pre><pre>T 6RYK_A           54 GDVSGLATDIARLG----QLFPVDVRPAGEDRYQLVCGFRRVAALRFLKRD  100 (229)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHC----CCSCEEEEECSTTCEEEEECHHHHHHHHTTTCS
</pre><pre>T ss_pred             cchHHHHHHHHHhC----CCCCeEEEECCCCCEEEEeCcHHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>2HWJ_D Hypothetical protein Atu1540; Agrobacterium tumefaciens, Hypothetical protein, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE; 2.61A {Agrobacterium tumefaciens str. C58} SCOP: d.268.1.3
</pre><pre>Probab=84.94  E-value=5.1  Score=27.93  Aligned_cols=62  Identities=13%  Similarity=0.125  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCC-CcHHHHHHHHHcCCCCHh----HCCCeEEEE-eCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGE-YTVLGNAEALLNGTLKPS----ALPPIKIWK-DNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~-~~i~~~~~~l~~g~~~~~----~ip~IrV~~-~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      +..+++-||..|     + ..|...+..++......-    .-.+|.|+. . +|++|-+| +-|+.|+..+|..
</pre><pre>T Consensus        13 ~i~~L~PTQ~~v-----g~~~v~~~~~~~~~~~~~~~~~~l~~~~ipvvigp-~g~~yliDGHHr~~a~~~~G~~   81 (205)
</pre><pre>T 2HWJ_D           13 AIDKLRPTQIAV-----GFREVELKRKEWRETRKKDGDDFLGNHIVPVVAGP-KDRAYLIDHHHLVLALSKEGVE   81 (205)
</pre><pre>T ss_dssp             ETTTSBCSBSEE-----CHHHHHHHHHHHTTCC------CTTCCEEEEEECS-TTCEEECSCHHHHHHHHTTTCC
</pre><pre>T ss_pred             eHHHcCCCchhc-----CHHHHHHHHHHHHHhchhhccccccCceeeEEECC-CCCEEEeCChHHHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>5K5D_B ParB domain protein nuclease; ParB-N, pnob8, partition, HYDROLASE; HET: CIT, MSE; 2.45A {Sulfolobus solfataricus}
</pre><pre>Probab=78.81  E-value=12  Score=26.91  Aligned_cols=59  Identities=14%  Similarity=0.120  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             ChhHceechhhcchhhCC-CcHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024    1 NPKDVRYMQSSIKNQTGE-YTVLGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus         1 ~p~~IrftQ~sIs~~F~~-~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      +++.|.... .....|.+ ..+.++.+.|+...    .+.||.|    +|.+.-+| +||+.|.+.+|..
</pre><pre>T Consensus        15 ~i~~i~~~~-~~~~~~~~~~~~~~L~~sI~~~G----~~~Pi~v----~~~~~iidG~~R~~A~~~lg~~   75 (300)
</pre><pre>T 5K5D_B           15 SIEKLKEDD-EFKTMVPSNNSREDLEKSLREKS----QIFPLIA----DRNYVLIDGYTRLDIMKKLGFK   75 (300)
</pre><pre>T ss_dssp             EGGGCEECH-HHHHSSCCCTTHHHHHHHHHHHC----SCSCEEE----CTTCEEEECHHHHHHHHHTTCC
</pre><pre>T ss_pred             CHHHCCCCh-hHhccCCChhhHHHHHHHHHHhc----CCCCEEE----CCCCEEeCCHHHHHHHHHcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>1XW3_A SULFIREDOXIN; retroreduction, sulfinic acid, peroxiredoxin, ATP, OXIDOREDUCTASE; 1.65A {Homo sapiens} SCOP: l.1.1.1, d.268.1.4
</pre><pre>Probab=75.31  E-value=13  Score=21.38  Aligned_cols=29  Identities=24%  Similarity=0.457  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEEeCCCc-----EEeCC-chHHHHHHHcCCC
</pre><pre>Q FD01848675_024   39 PPIKIWKDNLGK-----IWTLD-HRRLAAFKLAGLD   68 (69)
</pre><pre>Q Consensus        39 p~IrV~~~~~G~-----~~slD-NRRL~~fk~a~~~   68 (69)
</pre><pre>                      |||.|... +|.     ++.++ ++|+-|.+.++..
</pre><pre>T Consensus        50 ~pi~v~~~-~~~~~~~~~~ii~G~~R~~a~~~~~~~   84 (110)
</pre><pre>T 1XW3_A           50 PPIDVLWI-KGAQGGDYFYSFGGCHRYAAYQQLQRE   84 (110)
</pre><pre>T ss_dssp             CCEEEEEE-ECTTSCEEEECCSCHHHHHHHHHTTCS
</pre><pre>T ss_pred             CCeEEEEE-eCCCCCceEEEEcchHHHHHHHHhccC
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>6CKA_A Paratox; prophage, natural competence, Streptococcus, protein binding, VIRAL PROTEIN; HET: MSE; 1.559A {Streptococcus pyogenes serotype M3}
</pre><pre>Probab=73.36  E-value=20  Score=21.88  Aligned_cols=29  Identities=24%  Similarity=0.239  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEE
</pre><pre>Q FD01848675_024   20 TVLGNAEALLNGTLKPSALPPIKIWKDNLGKIW   52 (69)
</pre><pre>Q Consensus        20 ~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~   52 (69)
</pre><pre>                      ++.++-+|+.+|-++   =..+.|++. ||++|
</pre><pre>T Consensus         3 ~~defkeAId~G~I~---gdtV~IVRk-nGkIf   31 (68)
</pre><pre>T 6CKA_A            3 YIDEFKEAIDKGYIL---GDTVAIVRK-NGKIF   31 (68)
</pre><pre>T ss_dssp             CHHHHHHHHHTTSSC---SSEEEEEEE-TTEEE
</pre><pre>T ss_pred             cHHHHHHHHHcCCCC---CCeEEEEEe-CCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>PF14805.10 ; THDPS_N_2 ; Tetrahydrodipicolinate N-succinyltransferase N-terminal
</pre><pre>Probab=48.53  E-value=78  Score=18.38  Aligned_cols=25  Identities=20%  Similarity=0.147  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEe
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSALPPIKIWKDNLGKIWT   53 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~s   53 (69)
</pre><pre>                      |.+++++|-+|.        |||....+|.|..
</pre><pre>T Consensus        29 i~~~i~~Ld~G~--------iRvA~k~~g~W~v   53 (67)
</pre><pre>T E1SU11_FERBD/3   29 VKSVLAMLDSGA--------ARVAEKQDGKWVV   53 (67)
</pre><pre>T ss_pred             HHHHHHHHHCCc--------ceEEEeeCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>1OMH_A trwC protein; PROTEIN-DNA COMPLEX, BACTERIAL CONJUGATION, RELAXASE, DNA REPLICATION, TRANSFERASE-DNA COMPLEX; HET: SO4, MSE; 1.95A {Escherichia coli} SCOP: d.89.1.5
</pre><pre>Probab=48.18  E-value=48  Score=23.85  Aligned_cols=25  Identities=12%  Similarity=0.128  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             eEEEEeCCCcEEeCCchHHHHHHHc
</pre><pre>Q FD01848675_024   41 IKIWKDNLGKIWTLDHRRLAAFKLA   65 (69)
</pre><pre>Q Consensus        41 IrV~~~~~G~~~slDNRRL~~fk~a   65 (69)
</pre><pre>                      +.++...+|+|.++|++.|+..+.+
</pre><pre>T Consensus       167 ~n~~~~~dG~~~al~~~~l~~~~~~  191 (293)
</pre><pre>T 1OMH_A          167 LNMTKRSDGQWRALKNDEIVKATRY  191 (293)
</pre><pre>T ss_dssp             ESCEECTTSCEECCBCHHHHHTHHH
</pre><pre>T ss_pred             EeeEECCCCCeeecccHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>8A1B_A TraI; Relaxase, DNA BINDING PROTEIN, Trans-esterase; HET: GOL; 1.7A {Escherichia coli}
</pre><pre>Probab=46.80  E-value=31  Score=24.87  Aligned_cols=18  Identities=22%  Similarity=0.222  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CCcEEeCCchHHHHHHHc
</pre><pre>Q FD01848675_024   48 LGKIWTLDHRRLAAFKLA   65 (69)
</pre><pre>Q Consensus        48 ~G~~~slDNRRL~~fk~a   65 (69)
</pre><pre>                      +|+|.++|++.|+..+.+
</pre><pre>T Consensus       173 dG~~~ald~~~l~~~~~~  190 (306)
</pre><pre>T 8A1B_A          173 DGQWRALKNDELMRNKMH  190 (306)
</pre><pre>T ss_dssp             TSCEECCBCHHHHTTHHH
</pre><pre>T ss_pred             CCCEEecccHHHHHhHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>2K3U_A Chemotaxis inhibitory protein; Chemotaxis Inhibitory Protein (CHIPS), sulfated tyrosine, GPCR membrane protein C5aR, anaphylotoxin C5a, Staphylococcus Aureus, complement cascade; HET: TYS, ACE, NH2; NMR {Staphylococcus aureus subsp. aureus str. Newman}
</pre><pre>Probab=46.50  E-value=77  Score=20.17  Aligned_cols=33  Identities=15%  Similarity=0.109  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC
</pre><pre>Q FD01848675_024   22 LGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD   55 (69)
</pre><pre>Q Consensus        22 ~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD   55 (69)
</pre><pre>                      ..+-..|+.|...-.++..|-+... .|.+||+|
</pre><pre>T Consensus        13 ~rLR~~iKkgtkn~~~F~KMVI~TE-~~~yYTv~   45 (91)
</pre><pre>T 2K3U_A           13 ERLRNYLKKGTKNSAQFEKMVILTE-NKGYYTVY   45 (91)
</pre><pre>T ss_dssp             HHHHHHHHHHCSCCTTEEEEEEECS-TTCEEEEE
</pre><pre>T ss_pred             HHHHHHHhhcCcChhHheeeEEEec-CCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF11434.12 ; CHIPS ; Chemotaxis-inhibiting protein CHIPS
</pre><pre>Probab=46.50  E-value=77  Score=20.17  Aligned_cols=33  Identities=15%  Similarity=0.109  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEeCC
</pre><pre>Q FD01848675_024   22 LGNAEALLNGTLKPSALPPIKIWKDNLGKIWTLD   55 (69)
</pre><pre>Q Consensus        22 ~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~slD   55 (69)
</pre><pre>                      ..+-..|+.|...-.++..|-+... .|.+||+|
</pre><pre>T Consensus        13 ~rLR~~iKkgtkn~~~F~KMVI~TE-~~~yYTv~   45 (91)
</pre><pre>T CHIPS_STAA8/59   13 ERLRNYLKKGTKNSAQFEKMVILTE-NKGYYTVY   45 (91)
</pre><pre>T ss_pred             HHHHHHHhhcCcChhHheeeEEEec-CCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>5VNY_A Lethal (2) giant discs 1, isoform B; ESCRT, ENDOCYTOSIS, PROTEIN BINDING; HET: MSE; 1.101A {Drosophila melanogaster}
</pre><pre>Probab=42.04  E-value=78  Score=17.66  Aligned_cols=20  Identities=15%  Similarity=0.132  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCHhHCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGT-LKPSALPP   40 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~-~~~~~ip~   40 (69)
</pre><pre>                      +.+.+.+++.|. ++..++|+
</pre><pre>T Consensus        41 ~~~~i~~~~~G~~i~~~~lP~   61 (66)
</pre><pre>T 5VNY_A           41 YEDAIKLYKAGKPVPYDELPV   61 (66)
</pre><pre>T ss_dssp             HHHHHHHHHTTCCCCGGGSCC
</pre><pre>T ss_pred             HHHHHHHHHCCCCCCHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>8GLV_6S FAP339; cilia, microtubule, dynein, MOTOR PROTEIN; HET: GDP, GTP, MG; 3.1A {Chlamydomonas reinhardtii}
</pre><pre>Probab=40.24  E-value=50  Score=21.14  Aligned_cols=26  Identities=15%  Similarity=0.088  Sum_probs=0.0  Template_Neff=3.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCCCHhHCCCeEEEEeCCC
</pre><pre>Q FD01848675_024   22 LGNAEALLNGTLKPSALPPIKIWKDNLG   49 (69)
</pre><pre>Q Consensus        22 ~~~~~~l~~g~~~~~~ip~IrV~~~~~G   49 (69)
</pre><pre>                      ...+++|.+| ..+--=-.+|||.| +|
</pre><pre>T Consensus        34 ~~ALQaLsNG-~~vPY~aTLR~V~H-eG   59 (87)
</pre><pre>T 8GLV_6S          34 FRALQVLANG-DEVPYEATLRNVIH-DG   59 (87)
</pre><pre>T ss_pred             HHHHHHHhCC-CCCCcHHheeeeee-cC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>5UJE_A SbnI protein; staphyloferrin B, heme, regulator, siderophore, GENE REGULATION; HET: GOL, MSE; 2.5A {Staphylococcus aureus}
</pre><pre>Probab=35.12  E-value=2.3e+02  Score=19.46  Aligned_cols=45  Identities=16%  Similarity=0.236  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHC-CCeEEEEeCCCcEEeCC-chHHHHHHHcCCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSAL-PPIKIWKDNLGKIWTLD-HRRLAAFKLAGLDS   69 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~i-p~IrV~~~~~G~~~slD-NRRL~~fk~a~~~~   69 (69)
</pre><pre>                      +.++.+.|....    -+ .||-|....++.++-+| ++|+.|++..+...
</pre><pre>T Consensus        29 ~~~l~~~i~~~G----~~~~Pi~v~~~~~~~~~i~dG~~R~~a~~~~~~~~   75 (242)
</pre><pre>T 5UJE_A           29 LEKTKSSIEADD----FIRHPILVTAMQHGRYMVIDGVHRYTSLKALGCKK   75 (242)
</pre><pre>T ss_dssp             HHHHHHHHHHHT----EESSCEEEEECSTTCEEEEECHHHHHHHHHTTCSE
</pre><pre>T ss_pred             HHHHHHHHHHcC----CCCCCeEEEEccCCeEEEcCCHHHHHHHHHcCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>1VK1_A Conserved hypothetical protein; reductive methylation, dimethyl lysine, Pyrococcus furiosus, conserved hypothetical protein, structural genomics, PSI, Protein Structure Initiative, Southeast; HET: MLY; 1.2A {Pyrococcus furiosus} SCOP: d.268.1.2
</pre><pre>Probab=35.11  E-value=1.3e+02  Score=20.51  Aligned_cols=27  Identities=26%  Similarity=0.369  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEEeCC--CcEEeCC-chHHHHHHHcC
</pre><pre>Q FD01848675_024   39 PPIKIWKDNL--GKIWTLD-HRRLAAFKLAG   66 (69)
</pre><pre>Q Consensus        39 p~IrV~~~~~--G~~~slD-NRRL~~fk~a~   66 (69)
</pre><pre>                      +||-|... .  |.++-+| ++|+.|++.++
</pre><pre>T Consensus        52 ~Pi~v~~~-~~~~~~~i~dG~hR~~a~~~~~   81 (242)
</pre><pre>T 1VK1_A           52 KPMLLAKI-PGTDEYLIVDGHHRWAGLQKLG   81 (242)
</pre><pre>T ss_dssp             SCEEEEEC-TTSSCEEEEECHHHHHHHHHHT
</pre><pre>T ss_pred             CCEEEEeC-CCCCeEEEcCChHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>7ZSS_P de novo designed binder; SARS-COV2, de novo, design binder, spike, RBD, receptor binding domain, ANTIVIRAL PROTEIN; HET: NAG;{Severe acute respiratory syndrome coronavirus 2}
</pre><pre>Probab=34.58  E-value=1e+02  Score=18.11  Aligned_cols=21  Identities=14%  Similarity=0.126  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             cHHHHHHHHHcCC-CCHhHCCC
</pre><pre>Q FD01848675_024   20 TVLGNAEALLNGT-LKPSALPP   40 (69)
</pre><pre>Q Consensus        20 ~i~~~~~~l~~g~-~~~~~ip~   40 (69)
</pre><pre>                      .+.+.+.+++.|. ++.+++|+
</pre><pre>T Consensus        45 ~~~~~i~~~~~G~~i~~~~lPp   66 (79)
</pre><pre>T 7ZSS_P           45 QYEDAIKLYKAGKPVPYDELPV   66 (79)
</pre><pre>T ss_dssp             HHHHHHHHHHHTCCCCTTTSCC
</pre><pre>T ss_pred             HHHHHHHHHHCCCCCCHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>6NR6_A Bifunctional transcriptional regulator/O-phospho-L-serine synthase SbnI; free serine kinase, siderophore, ParB/Srx family, staphyloferrin B, BIOSYNTHETIC PROTEIN; HET: ADP; 1.9A {Staphylococcus pseudintermedius}
</pre><pre>Probab=31.77  E-value=2.8e+02  Score=19.34  Aligned_cols=42  Identities=21%  Similarity=0.381  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhHC-CCeEEEEeCCCcEEeCC-chHHHHHHHcC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSAL-PPIKIWKDNLGKIWTLD-HRRLAAFKLAG   66 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~i-p~IrV~~~~~G~~~slD-NRRL~~fk~a~   66 (69)
</pre><pre>                      +..+.+.+....    -+ .||-|....++.++-+| +.|+.|++.++
</pre><pre>T Consensus        27 ~~~l~~~i~~~g----~~~~Pii~~~~~~~~~~i~DGhhR~~a~~~~~   70 (254)
</pre><pre>T 6NR6_A           27 LEKTKESIAKEQ----HLRHPVLVVKTLFGRYMVIDGVHRFMSLKALG   70 (254)
</pre><pre>T ss_dssp             HHHHHHHHHHHC----EESSCEEEEECGGGCEEEEECHHHHHHHHHTT
</pre><pre>T ss_pred             HHHHHHHHHHcC----CccCCeEEEEeCCCeEEEccCchHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>3IMO_A Integron cassette protein; Novel, Integron protein, Vibrio cholerae, Argentinean O139 strain, UNKNOWN FUNCTION; 1.8A {Vibrio cholerae O139}
</pre><pre>Probab=31.08  E-value=38  Score=23.07  Aligned_cols=41  Identities=24%  Similarity=0.049  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             CcHHHHHHHHHcCCCCHhHCCCeEEEEeCCC-----cEEeCCchHHH
</pre><pre>Q FD01848675_024   19 YTVLGNAEALLNGTLKPSALPPIKIWKDNLG-----KIWTLDHRRLA   60 (69)
</pre><pre>Q Consensus        19 ~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G-----~~~slDNRRL~   60 (69)
</pre><pre>                      ..+.+.+.+|..+-+.-.+-.+|+|... +|     .|+++..+|.+
</pre><pre>T Consensus        47 ~nV~~IilaLlG~Iiwr~d~~sI~v~~~-~G~~an~lW~ti~G~rYa   92 (133)
</pre><pre>T 3IMO_A           47 GNVEEIALALAGAILWRKDDTNIKVMAH-GADTKNVLWVTINGERYA   92 (133)
</pre><pre>T ss_dssp             GGGHHHHHHHHHHHHHHCCSSCCEEEES-SSTTSEEEEEEETTEEEE
</pre><pre>T ss_pred             ccHHHHHHHHHhHHHhcCCCCCEEEEEe-CCCcceEEEEEECCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>PF07309.15 ; FlaF ; Flagellar protein FlaF
</pre><pre>Probab=30.66  E-value=42  Score=21.26  Aligned_cols=8  Identities=13%  Similarity=0.206  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CchHHHHH
</pre><pre>Q FD01848675_024   55 DHRRLAAF   62 (69)
</pre><pre>Q Consensus        55 DNRRL~~f   62 (69)
</pre><pre>                      +|+|||..
</pre><pre>T Consensus        45 ~N~~LW~~   52 (107)
</pre><pre>T Q5FST4_GLUOX/4   45 ANLKLWSL   52 (107)
</pre><pre>T ss_pred             HHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF21528.1 ; CC2D1A-B_DM14 ; Coiled-coil and C2 domain-containing protein 1, DM14 domain
</pre><pre>Probab=29.29  E-value=1.4e+02  Score=15.20  Aligned_cols=20  Identities=25%  Similarity=0.612  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCC-CCHhHCCC
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGT-LKPSALPP   40 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~-~~~~~ip~   40 (69)
</pre><pre>                      +.+.+.++..|. ++...+|+
</pre><pre>T Consensus        34 ~~~~~~~~~~g~~~~~~~lP~   54 (56)
</pre><pre>T A0A6J1S960_FRA   34 LVDQIKSAQNGRPIKEEDIPP   54 (56)
</pre><pre>T ss_pred             HHHHHHHHHCCCCCCHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF18315.5 ; VCH_CASS14 ; Integron cassette protein VCH_CASS1 chain
</pre><pre>Probab=26.31  E-value=1e+02  Score=19.92  Aligned_cols=39  Identities=26%  Similarity=0.104  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             CcHHHHHHHHHcCCCCHhHCCCeEEEEeCCC-----cEEeCCchH
</pre><pre>Q FD01848675_024   19 YTVLGNAEALLNGTLKPSALPPIKIWKDNLG-----KIWTLDHRR   58 (69)
</pre><pre>Q Consensus        19 ~~i~~~~~~l~~g~~~~~~ip~IrV~~~~~G-----~~~slDNRR   58 (69)
</pre><pre>                      ..+.+.+.+|..+-+.-.+-.+|+|.+. +|     .|+++..+|
</pre><pre>T Consensus        17 ~~V~~I~lal~g~Iiwr~d~~~i~v~~~-~G~~an~lW~~i~g~r   60 (97)
</pre><pre>T A9GUG2_SORC5/1   17 GGVNEIALALAGAIIWKKDDEPIKVMVQ-DGDTKNVLWARLGGKR   60 (97)
</pre><pre>T ss_pred             cchHHHHHHHHhHHHhccCCCCEEEEEE-CCCcceEEEEEECCEE
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>PF17455.6 ; LtuB ; Late transcription unit B
</pre><pre>Probab=25.34  E-value=2.7e+02  Score=17.65  Aligned_cols=31  Identities=16%  Similarity=0.067  Sum_probs=0.0  Template_Neff=2.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHcCCCCHhHCCCeEEEEeCCCcEEe
</pre><pre>Q FD01848675_024   22 LGNAEALLNGTLKPSALPPIKIWKDNLGKIWT   53 (69)
</pre><pre>Q Consensus        22 ~~~~~~l~~g~~~~~~ip~IrV~~~~~G~~~s   53 (69)
</pre><pre>                      ..++..|.....++ +-..|-||.+++|=+||
</pre><pre>T Consensus        37 D~LV~sLL~~q~~~-d~~kiLIFnyq~GFV~t   67 (79)
</pre><pre>T LTUB_CHLTR/19-   37 DVLVHSLLDKHVPQ-NSDQVLIFTYQNGFVET   67 (79)
</pre><pre>T ss_pred             HHHHHHHHHccCCC-CCCeEEEEEcCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>6T9K_B Transcription factor SPT20; Coactivator, Transcription, Histone acetyltransferase, Histone deubiquitinase, GENE REGULATION; 3.3A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=24.36  E-value=48  Score=27.85  Aligned_cols=55  Identities=18%  Similarity=0.226  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             hhHceech---------hhcchhhCCCcHHHHHHHHHcCCCCHhHCCCeE---EEEeCCCcEE--eCCch
</pre><pre>Q FD01848675_024    2 PKDVRYMQ---------SSIKNQTGEYTVLGNAEALLNGTLKPSALPPIK---IWKDNLGKIW--TLDHR   57 (69)
</pre><pre>Q Consensus         2 p~~IrftQ---------~sIs~~F~~~~i~~~~~~l~~g~~~~~~ip~Ir---V~~~~~G~~~--slDNR   57 (69)
</pre><pre>                      |...||.|         ++.--...+..+.++++.|..|.+.++-+..++   |-.+ +|.+.  -.|+|
</pre><pre>T Consensus       151 Pan~Rlqqq~~~P~~~qDg~~l~knn~~~K~FLe~I~rg~IP~dLlEvL~d~~V~FY-DGCLIVEV~DHR  219 (604)
</pre><pre>T 6T9K_B          151 PANSRLQQQQKQPELTSDGLILTKNNETLKEFLEYVARGRIPDAIMEVLRDCNIQFY-EGNLILQVYDHT  219 (604)
</pre><pre>T ss_dssp             CC----------------CCCEECSCTTHHHHGGGTTTTBCCTTSHHHHHHHTCCCB-TTBEEEEEEEST
</pre><pre>T ss_pred             chhhhhhhccCCCccccCCcccCCCcHHHHHHHHHHHcCCCCHHHHHHHHHCCCCee-eCcEEEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>3ZGX_Z SEGREGATION AND CONDENSATION PROTEIN A; CELL CYCLE; 3.4A {BACILLUS SUBTILIS}
</pre><pre>Probab=22.67  E-value=2e+02  Score=17.81  Aligned_cols=21  Identities=5%  Similarity=-0.172  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CcHHHHHHHHHcCCCCHhHCC
</pre><pre>Q FD01848675_024   19 YTVLGNAEALLNGTLKPSALP   39 (69)
</pre><pre>Q Consensus        19 ~~i~~~~~~l~~g~~~~~~ip   39 (69)
</pre><pre>                      +++..+++.++.|+++|.+++
</pre><pre>T Consensus        13 gpl~~Ll~lv~~~~id~~~v~   33 (94)
</pre><pre>T 3ZGX_Z           13 GPLDLLLHLINRLEIDIYDIP   33 (94)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHTSSS
</pre><pre>T ss_pred             ChHHHHHHHHHhcCCCHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF02609.20 ; Exonuc_VII_S ; Exonuclease VII small subunit
</pre><pre>Probab=22.67  E-value=2.2e+02  Score=15.48  Aligned_cols=17  Identities=18%  Similarity=0.145  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHcCCCCHhH
</pre><pre>Q FD01848675_024   21 VLGNAEALLNGTLKPSA   37 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~~g~~~~~~   37 (69)
</pre><pre>                      |.+++..|.+|.++.++
</pre><pre>T Consensus         8 Le~iv~~Le~~~~~lde   24 (52)
</pre><pre>T D3E6M4_GEOS4/1    8 LETIVEQLEDGDVPLEK   24 (52)
</pre><pre>T ss_pred             HHHHHHHHHcCCCCHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>PF19702.3 ; DUF6200 ; Family of unknown function (DUF6200)
</pre><pre>Probab=22.53  E-value=1.5e+02  Score=18.50  Aligned_cols=25  Identities=12%  Similarity=0.247  Sum_probs=0.0  Template_Neff=4.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHH---cCCCCHhH-CCCeEEEE
</pre><pre>Q FD01848675_024   21 VLGNAEALL---NGTLKPSA-LPPIKIWK   45 (69)
</pre><pre>Q Consensus        21 i~~~~~~l~---~g~~~~~~-ip~IrV~~   45 (69)
</pre><pre>                      +.+++++|+   +|...... .|-|-|++
</pre><pre>T Consensus        44 V~~~i~~lk~~l~G~i~~~~lqPVVvvvr   72 (79)
</pre><pre>T A0A2P1PQV5_9GA   44 VARVLESVRTDLAAEAEGKILVPVVIVYR   72 (79)
</pre><pre>T ss_pred             HHHHHHHHHHHhhhhhcccccccEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>6HIZ_DY mS72; mitoribosome, translation, Trypanosoma, small ribosomal subunit, 9S rRNA, ribosomal protein, RIBOSOME; HET: SPD, GTP, UTP; 3.08A {Trypanosoma brucei brucei}
</pre><pre>Probab=22.46  E-value=99  Score=21.79  Aligned_cols=14  Identities=36%  Similarity=0.798  Sum_probs=0.0  Template_Neff=1.900
</pre><pre>
</pre><pre>Q ss_pred             CCeEEEEeCCCcEEe
</pre><pre>Q FD01848675_024   39 PPIKIWKDNLGKIWT   53 (69)
</pre><pre>Q Consensus        39 p~IrV~~~~~G~~~s   53 (69)
</pre><pre>                      |||.|+.. +-+||+
</pre><pre>T Consensus        20 pPisvIrs-Gpkwws   33 (163)
</pre><pre>T 6HIZ_DY          20 PPISVIRT-GPKWWA   33 (163)
</pre><pre>T ss_pred             CCceeeec-CCcccc
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF20162.3 ; Etd1 ; Septation protein etd1
</pre><pre>Probab=22.36  E-value=47  Score=21.49  Aligned_cols=28  Identities=21%  Similarity=0.362  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             hhHceechhhcchhhCC-CcHHHHHHHHH
</pre><pre>Q FD01848675_024    2 PKDVRYMQSSIKNQTGE-YTVLGNAEALL   29 (69)
</pre><pre>Q Consensus         2 p~~IrftQ~sIs~~F~~-~~i~~~~~~l~   29 (69)
</pre><pre>                      |..|.=.|.+|...+.. +.+...+++|+
</pre><pre>T Consensus        42 P~~I~~~Q~~v~~~l~~VReFal~VEdLK   70 (100)
</pre><pre>T A0A436ZWG1_9PE   42 PKSIKDAQATVMGHLGNVREFAKQVENLK   70 (100)
</pre><pre>T ss_pred             CHHHHHhHHHHHHhHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>1QV9_C F420-dependent methylenetetrahydromethanopterin dehydrogenase; monomer: alpha/beta domain, helix bundle  trimer of dimers, OXIDOREDUCTASE; HET: MSE; 1.54A {Methanopyrus kandleri} SCOP: c.127.1.1
</pre><pre>Probab=21.46  E-value=87  Score=24.09  Aligned_cols=33  Identities=12%  Similarity=0.038  Sum_probs=0.0  Template_Neff=3.300
</pre><pre>
</pre><pre>Q ss_pred             hcchhhCC--CcHHHHHHHHHcCCCCHhHCCCeEE
</pre><pre>Q FD01848675_024   11 SIKNQTGE--YTVLGNAEALLNGTLKPSALPPIKI   43 (69)
</pre><pre>Q Consensus        11 sIs~~F~~--~~i~~~~~~l~~g~~~~~~ip~IrV   43 (69)
</pre><pre>                      +|...|+-  ..|..++++|+.|+.+.-.+|.|-|
</pre><pre>T Consensus       149 a~TGv~r~i~~eiDkvI~~ik~g~~~~~~LP~ivv  183 (283)
</pre><pre>T 1QV9_C          149 AATGVFRVVQEAFDELIEKAKEDEISENDLPKLVI  183 (283)
</pre><pre>T ss_dssp             HHTSHHHHHHHHHHHHHHHHTTTCCCGGGSCEEEE
</pre><pre>T ss_pred             HHccHHHHHHHHHHHHHHHHHcCCCCCCCCCEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>7MJ2_F Pyridinium-3,5-biscarboxylic acid mononucleotide synthase; Carboxylase, Hydrolase, LYASE (CARBON-CARBON); 2.8A {Lactobacillus plantarum}
</pre><pre>Probab=21.26  E-value=1.9e+02  Score=20.76  Aligned_cols=19  Identities=26%  Similarity=0.323  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CcHHHHHHHHHcCCCCHhH
</pre><pre>Q FD01848675_024   19 YTVLGNAEALLNGTLKPSA   37 (69)
</pre><pre>Q Consensus        19 ~~i~~~~~~l~~g~~~~~~   37 (69)
</pre><pre>                      ..|.++++++++|++++++
</pre><pre>T Consensus         2 ~~l~~lL~~~~~g~is~~~   20 (256)
</pre><pre>T 7MJ2_F            2 ATTAEILQQVAAGQLSPTA   20 (256)
</pre><pre>T ss_dssp             -------------------
</pre><pre>T ss_pred             ccHHHHHHHHHcCCCCHHH
</pre><pre>
</pre><pre>
</pre>