<pre>Query         FD01845382_02687 PAAR domain-containing protein
</pre><pre>Match_columns 174
</pre><pre>No_of_seqs    107 out of 131
</pre><pre>Neff          7.12632
</pre><pre>Searched_HMMs 86581
</pre><pre>Date          Tue Feb 27 20:02:40 2024
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -o ../results/7263015.hhr -oa3m ../results/7263015.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 1CII_A COLICIN IA; COLICIN, BA  88.7      16 0.00018   35.7  11.9  136    3-171   430-565 (602)
</pre><pre>  2 PF01024.23 ; Colicin ; Colicin  88.3     3.2 3.7E-05   34.4   6.2  141    9-171     9-149 (185)
</pre><pre>  3 6THK_A Pyocin S5; Bacteriocin,  86.5      15 0.00017   35.4  10.0  136    3-171   323-458 (506)
</pre><pre>  4 8IOM_A Colicin 1B; pore-formin  86.3      30 0.00034   29.5  11.5  136    3-171    21-156 (205)
</pre><pre>  5 1COL_A COLICIN A; ANTIBACTERIA  86.2      25 0.00029   29.8  10.3  140    3-172    15-155 (204)
</pre><pre>  6 PF05120.16 ; GvpG ; Gas vesicl  85.0     8.7  0.0001   26.8   6.0   38   16-59     31-68  (72)
</pre><pre>  7 2I88_A Colicin-E1; protein-mem  78.6      59 0.00068   27.4  11.8  136    3-171    17-152 (191)
</pre><pre>  8 3FEW_X Colicin S4; Colicin S4,  78.1      47 0.00055   32.2  10.0  137   12-173   301-455 (505)
</pre><pre>  9 5TGY_A PS1; 4-helix bundle, co  75.9      26  0.0003   26.4   6.0   62   16-80     30-91  (109)
</pre><pre> 10 1A87_A COLICIN N; BACTERIOCIN,  75.5      99  0.0011   28.4  11.0  139   12-173   124-274 (321)
</pre><pre> 11 PF20848.1 ; ExoU_mid_dom ; Exo  72.6      22 0.00025   28.0   5.3   61   44-120    33-93  (126)
</pre><pre> 12 PF08649.14 ; DASH_Dad1 ; DASH   71.0      25 0.00029   23.3   4.7   46   45-112     2-47  (55)
</pre><pre> 13 7NFJ_A CC-Type2-(LaId)4-L28Y;   69.2      16 0.00018   21.1   2.9   20   11-30     11-30  (32)
</pre><pre> 14 8Q84_X DASH complex subunit DA  68.2      22 0.00025   26.5   4.3   55   36-112    11-65  (94)
</pre><pre> 15 3WXX_H AopB; Translocator, Mem  63.5      76 0.00088   27.3   7.2  109    9-135    26-137 (219)
</pre><pre> 16 6CFZ_F Dad1,Dad1; DASH, Dam1 c  63.4      29 0.00033   25.7   4.1   48   43-112     2-49  (92)
</pre><pre> 17 6DLC_A Designed protein DHD1:2  63.3      43 0.00049   24.9   4.9   45   43-87     11-55  (115)
</pre><pre> 18 7QWB_D CC-Type2-(Ue)4; aib, co  57.6      49 0.00057   19.2   3.5   21   11-31      4-24  (32)
</pre><pre> 19 PF09712.14 ; PHA_synth_III_E ;  55.8   2E+02  0.0023   25.4   8.7   76   18-117   223-298 (298)
</pre><pre> 20 7JJ8_D Zinc-binding lipoprotei  55.5 1.3E+02  0.0015   26.0   7.6  119    9-140   120-240 (299)
</pre><pre> 21 2F6M_D Vacuolar protein sortin  53.2 1.5E+02  0.0017   22.5   7.7   68   17-90     36-105 (109)
</pre><pre> 22 7R5K_W0 Nuclear pore complex p  50.0 3.6E+02  0.0041   27.1  10.3   87   13-117   603-690 (741)
</pre><pre> 23 7BUL_A General transcription f  49.2      80 0.00092   25.4   4.9   41   13-55    115-155 (161)
</pre><pre> 24 6VME_E Vacuolar protein sortin  48.8 1.2E+02  0.0014   23.4   5.6   41   17-63     44-84  (122)
</pre><pre> 25 7JJ9_A Zinc-binding lipoprotei  48.7 1.6E+02  0.0018   27.6   7.4  119    9-140   120-240 (475)
</pre><pre> 26 6S7O_E Dolichyl-diphosphooligo  47.6 4.2E+02  0.0048   25.9  10.2   87   11-117   482-569 (607)
</pre><pre> 27 2OTX_B Src kinase-associated p  46.7 1.6E+02  0.0019   22.3   6.2   44   16-62      4-47  (211)
</pre><pre> 28 2D4Y_B Flagellar hook-associat  46.7 1.5E+02  0.0017   27.9   6.9   55    7-61     94-150 (463)
</pre><pre> 29 2CAZ_B VACUOLAR PROTEIN SORTIN  46.6 2.5E+02  0.0029   23.1   7.5   61   17-83     53-115 (155)
</pre><pre> 30 7FIK_Y Nucleoporin Nup88A; CR,  46.1 4.5E+02  0.0052   26.3  10.3   87   13-117   590-677 (728)
</pre><pre> 31 PF10168.13 ; Nup88 ; Nuclear p  45.0 4.8E+02  0.0055   26.2  10.3   87   13-117   592-679 (729)
</pre><pre> 32 7JRQ_A MPP1; metalloprotein, p  44.8 1.3E+02  0.0015   23.6   5.0   63   16-80     47-110 (131)
</pre><pre> 33 6YP5_B Phosphoprotein; Phospho  44.4      72 0.00084   20.3   3.0   30    1-30      3-32  (45)
</pre><pre> 34 5XLN_B Threonine--tRNA ligase,  42.8      46 0.00054   20.5   2.2   17   43-59     22-38  (45)
</pre><pre> 35 PF10023.13 ; Aminopep ; Putati  42.7 3.9E+02  0.0045   24.2   9.5   98   11-119   211-310 (325)
</pre><pre> 36 4HKU_B TetR transcriptional re  42.4      60  0.0007   23.6   3.1   37   56-92     13-49  (178)
</pre><pre> 37 5C4Y_A Putative transcription   41.8      70  0.0008   22.9   3.3   38   56-93      7-44  (184)
</pre><pre> 38 6ZT1_G CC-Type2-(LaIdGe)4; coi  41.1 1.2E+02  0.0014   17.7   4.1   27    4-30      4-30  (32)
</pre><pre> 39 2KJG_A Archaeal protein SSO690  40.9 1.9E+02  0.0021   22.1   5.2   66   15-82     21-92  (99)
</pre><pre> 40 7QWE_H CC-Type2-(Ug)4; aib, co  40.9 1.2E+02  0.0014   17.7   3.5   21   11-31      4-24  (32)
</pre><pre> 41 PF03997.16 ; VPS28 ; VPS28 pro  40.5 1.7E+02  0.0019   24.2   5.6   40   17-63      6-49  (183)
</pre><pre> 42 4ME9_A Transcriptional regulat  40.2      67 0.00078   23.2   3.1   38   56-93     11-48  (189)
</pre><pre> 43 6YS8_D GldL; membrane protein   40.2 2.4E+02  0.0028   24.0   6.6   78    4-84    137-215 (215)
</pre><pre> 44 7WOO_K Nucleoporin NUP57; nucl  39.5 5.2E+02   0.006   24.7  11.0  101    8-117   369-474 (541)
</pre><pre> 45 4LUN_U Nonsense-mediated mRNA   39.4 2.1E+02  0.0024   25.7   6.5   70    9-90    244-313 (316)
</pre><pre> 46 1R8J_A KaiA; KaiA, Circadian C  39.1 3.9E+02  0.0046   23.2   9.9   98   16-120   152-257 (289)
</pre><pre> 47 8SNB_6R RIB35; sperm, doublet   39.0      64 0.00074   28.8   3.1   23   11-33    111-133 (310)
</pre><pre> 48 7ACM_B HTH-type transcriptiona  39.0   1E+02  0.0012   22.1   3.8   44   47-93      8-51  (178)
</pre><pre> 49 5XSJ_L Signal transduction his  38.9 2.8E+02  0.0032   21.4   6.8   97   14-117    21-122 (148)
</pre><pre> 50 3GEU_D Intercellular adhesion   38.7      83 0.00095   22.8   3.4   38   56-93      9-46  (189)
</pre><pre> 51 2EH3_A Transcriptional regulat  38.6      86 0.00099   22.5   3.4   37   56-92      8-44  (179)
</pre><pre> 52 1RKT_A protein yfiR; transcrip  38.1 1.2E+02  0.0014   22.3   4.3   54   36-92      1-54  (205)
</pre><pre> 53 6MJ1_A Probable HTH-type trans  37.9      81 0.00094   23.3   3.2   37   56-92     10-46  (207)
</pre><pre> 54 4ACI_A HTH-TYPE TRANSCRIPTIONA  37.5      78  0.0009   23.1   3.1   37   56-92     20-56  (191)
</pre><pre> 55 2NX4_C Transcriptional regulat  37.5      81 0.00094   23.0   3.2   52   38-92      1-52  (194)
</pre><pre> 56 2ZCM_B Biofilm operon icaABCD   37.5      82 0.00095   23.4   3.2   38   56-93     13-50  (192)
</pre><pre> 57 2HYJ_A Putative tetR-family tr  37.5 1.1E+02  0.0013   22.5   3.9   54   36-92      1-54  (200)
</pre><pre> 58 6AZH_B TetR family transcripti  37.4      90   0.001   22.5   3.4   38   56-93     12-49  (192)
</pre><pre> 59 3HE0_A Transcriptional regulat  37.2      86 0.00099   22.8   3.3   37   56-92     17-53  (196)
</pre><pre> 60 2Q24_B Putative tetR family tr  37.2 1.1E+02  0.0013   22.4   3.8   44   46-93     14-57  (194)
</pre><pre> 61 3LJL_A transcriptional regulat  36.9      81 0.00094   22.8   3.1   37   56-92     20-56  (156)
</pre><pre> 62 2FD5_A transcriptional regulat  36.9      85 0.00099   22.9   3.2   38   56-93     13-50  (180)
</pre><pre> 63 7SAZ_C GldL; type IX secretion  36.8 2.8E+02  0.0032   24.1   6.5   76    4-82    152-228 (228)
</pre><pre> 64 2PRS_A High-affinity zinc upta  36.6 2.6E+02  0.0031   23.9   6.5  116    9-137   123-240 (284)
</pre><pre> 65 3BJB_A Probable transcriptiona  36.5 1.3E+02  0.0015   22.4   4.1   50   41-93     16-65  (207)
</pre><pre> 66 2FBQ_A probable transcriptiona  36.3      83 0.00096   24.2   3.2   43   47-92      7-49  (235)
</pre><pre> 67 4OPC_A Conserved Archaeal prot  36.3 1.8E+02  0.0021   27.2   5.8   49   10-61    405-453 (453)
</pre><pre> 68 PF16312.9 ; Oberon_cc ; Coiled  36.2 2.6E+02   0.003   22.9   5.8   44   12-59     63-108 (148)
</pre><pre> 69 3QBM_B TetR transcriptional re  36.1      98  0.0011   22.7   3.4   38   56-93     13-50  (199)
</pre><pre> 70 5Y2G_A Maltose-binding peripla  35.9 5.2E+02   0.006   24.8   8.8   90    3-119   361-450 (595)
</pre><pre> 71 3DEW_A Transcriptional regulat  35.8      82 0.00095   23.3   3.0   37   56-92     14-50  (206)
</pre><pre> 72 PF20867.1 ; UVSSA_N ; UVSSA N-  35.7 1.4E+02  0.0016   23.4   4.3   62   49-117    76-140 (143)
</pre><pre> 73 2OV3_A Periplasmic binding pro  35.6 1.9E+02  0.0022   24.4   5.4   94    9-108   107-203 (260)
</pre><pre> 74 6O6O_A TetR family transcripti  35.5      86   0.001   22.8   3.0   38   56-93     15-52  (195)
</pre><pre> 75 5DAJ_G NalD; Repressor, Transc  35.5      97  0.0011   22.9   3.3   38   56-93     16-53  (212)
</pre><pre> 76 4W97_A HTH-type transcriptiona  35.4      91   0.001   22.7   3.1   38   56-93     16-53  (200)
</pre><pre> 77 3QKX_A Uncharacterized HTH-typ  35.3      88   0.001   22.6   3.0   37   56-92     14-50  (188)
</pre><pre> 78 2IBD_A Possible transcriptiona  34.9      96  0.0011   22.7   3.2   38   56-93     20-57  (204)
</pre><pre> 79 2G7S_A transcriptional regulat  34.7      88   0.001   22.8   3.0   37   56-92     14-50  (194)
</pre><pre> 80 1R5Q_A circadian oscillation r  34.7 3.3E+02  0.0038   21.0   6.1   64   46-120     4-72  (102)
</pre><pre> 81 3ANP_B Transcriptional repress  34.5      95  0.0011   22.9   3.1   38   56-93     15-52  (204)
</pre><pre> 82 3CCY_A Putative TetR-family tr  34.5      94  0.0011   22.7   3.1   38   56-93     20-57  (203)
</pre><pre> 83 5XBJ_A Flagellar hook-associat  34.4 3.6E+02  0.0041   25.8   7.4   54    7-60     92-146 (511)
</pre><pre> 84 3F1B_A TetR-like transcription  34.3      92  0.0011   22.9   3.0   38   56-93     20-57  (203)
</pre><pre> 85 PF20975.1 ; DGCcoil ; DGC-asso  34.0 6.3E+02  0.0073   24.3   8.9  101   11-117   391-491 (493)
</pre><pre> 86 3EUP_A Transcriptional regulat  33.9 1.1E+02  0.0012   22.7   3.3   38   56-93     17-54  (204)
</pre><pre> 87 1SGM_B Putative HTH-type trans  33.8   1E+02  0.0012   22.4   3.2   37   56-92     12-48  (191)
</pre><pre> 88 2JJ7_A HEMOLYSIN II REGULATORY  33.8      98  0.0011   22.4   3.1   38   56-93     13-50  (186)
</pre><pre> 89 2O7T_A Transcriptional regulat  33.8      92  0.0011   23.2   3.0   38   56-93     14-51  (199)
</pre><pre> 90 PF21643.1 ; T6SS_Tsi2-like ; T  33.7 1.7E+02   0.002   20.2   4.0   31   11-42     37-67  (71)
</pre><pre> 91 2DG7_A putative transcriptiona  33.6   1E+02  0.0012   22.5   3.2   37   56-92     13-49  (195)
</pre><pre> 92 3VPR_D Transcriptional regulat  33.5   1E+02  0.0012   22.2   3.1   38   56-93      9-46  (190)
</pre><pre> 93 2DII_A TFIIH basal transcripti  33.4 2.3E+02  0.0027   19.3   4.5   38   13-52     17-54  (61)
</pre><pre> 94 PF19095.4 ; DUF5783 ; Family o  33.4 1.1E+02  0.0013   23.4   3.2   40   16-55     11-50  (106)
</pre><pre> 95 5GP9_B Transcriptional regulat  33.4   1E+02  0.0012   22.4   3.1   37   56-92     12-48  (194)
</pre><pre> 96 6AYH_A TetR family transcripti  33.1 1.1E+02  0.0012   22.6   3.2   38   56-93     19-56  (201)
</pre><pre> 97 PF07688.16 ; KaiA ; KaiA C-ter  32.9 3.8E+02  0.0044   21.3   6.1   63   47-120    24-91  (122)
</pre><pre> 98 3ZK9_B MANGANESE ABC TRANSPORT  32.9 2.9E+02  0.0033   23.6   6.1  119    9-140   114-236 (278)
</pre><pre> 99 5ZTC_A Lmo2088 protein; TetR f  32.9 1.2E+02  0.0013   22.4   3.4   37   56-92      7-43  (204)
</pre><pre>100 2RAS_A Transcriptional regulat  32.7   1E+02  0.0012   22.9   3.1   38   56-93     17-54  (212)
</pre><pre>101 7W2Q_B Probable transcriptiona  32.6 1.1E+02  0.0013   22.8   3.2   38   56-93     19-56  (212)
</pre><pre>102 2HYT_A TetR-family transcripti  32.4 1.1E+02  0.0013   22.4   3.3   38   56-93     18-55  (197)
</pre><pre>103 1UI5_A A-factor receptor homol  32.4 1.2E+02  0.0014   22.7   3.4   38   56-93     15-52  (215)
</pre><pre>104 7KD8_A HTH-type transcriptiona  32.4 1.1E+02  0.0012   22.7   3.1   38   56-93     18-55  (211)
</pre><pre>105 6NSN_B Copper outer membrane r  32.4   1E+02  0.0012   22.7   3.0   37   56-92     18-54  (198)
</pre><pre>106 2GEN_A probable transcriptiona  32.2 1.1E+02  0.0013   22.3   3.2   37   56-92     13-49  (197)
</pre><pre>107 3ON4_G Transcriptional regulat  32.2 1.2E+02  0.0013   22.1   3.3   37   56-92     16-52  (191)
</pre><pre>108 3DPJ_G Transcription regulator  32.2 1.1E+02  0.0013   22.3   3.2   38   56-93     14-51  (194)
</pre><pre>109 5JG7_A Fur regulated Salmonell  32.1 2.8E+02  0.0033   21.2   5.5   48    9-61    108-155 (160)
</pre><pre>110 3CJD_A Transcriptional regulat  32.1   2E+02  0.0023   21.1   4.5   54   36-92      1-54  (198)
</pre><pre>111 4MK6_A Probable Dihydroxyaceto  32.0      89   0.001   23.3   2.6   37   56-92     15-51  (189)
</pre><pre>112 8GZU_tx Thioredoxin; Electron   31.9 2.7E+02  0.0032   21.3   5.4   49   16-65    118-166 (166)
</pre><pre>113 2ID6_A TRANSCRIPTIONAL REGULAT  31.9 1.3E+02  0.0015   22.1   3.5   44   47-93      5-48  (202)
</pre><pre>114 3RD3_B Probable transcriptiona  31.5 1.2E+02  0.0014   22.0   3.3   38   56-93     16-53  (197)
</pre><pre>115 3VUQ_A Transcriptional regulat  31.3 1.2E+02  0.0014   22.0   3.3   37   56-92     18-54  (187)
</pre><pre>116 6AYI_D HTH-type transcriptiona  31.3 1.1E+02  0.0013   22.4   3.1   38   56-93     20-57  (199)
</pre><pre>117 8T5Y_A DarR; cAMP, DarR, mycob  31.2 1.2E+02  0.0014   22.5   3.3   38   56-93     19-56  (212)
</pre><pre>118 3CDL_A Transcriptional regulat  31.1 1.1E+02  0.0012   22.6   3.0   38   56-93     15-52  (203)
</pre><pre>119 5JPD_A Manganese-binding lipop  31.1 4.1E+02  0.0047   22.8   6.7  119    9-140   126-248 (290)
</pre><pre>120 3ON2_C Probable transcriptiona  31.1 1.3E+02  0.0015   22.2   3.3   37   56-92     18-54  (199)
</pre><pre>121 3DCF_B transcriptional regulat  31.0 1.2E+02  0.0014   22.6   3.3   38   56-93     37-74  (218)
</pre><pre>122 4JYK_B HTH-type transcriptiona  31.0 1.2E+02  0.0014   22.6   3.3   38   56-93     24-61  (212)
</pre><pre>123 2QIB_A tetR-family transcripti  31.0 1.2E+02  0.0014   23.3   3.3   38   56-93     19-56  (231)
</pre><pre>124 3V6G_B PROBABLE TRANSCRIPTIONA  30.9 1.2E+02  0.0013   22.9   3.2   37   56-92     20-56  (208)
</pre><pre>125 2DG8_D putative tetR-family tr  30.8 1.5E+02  0.0018   21.7   3.8   43   47-92      9-51  (193)
</pre><pre>126 7AR9_d B14.5b; Complex-I, ELEC  30.7 2.9E+02  0.0033   20.4   4.8   41    3-54     39-81  (83)
</pre><pre>127 3MVP_B TetR/AcrR transcription  30.7   1E+02  0.0012   23.0   2.8   37   56-92     32-68  (217)
</pre><pre>128 2FQ4_A Transcriptional regulat  30.7 1.3E+02  0.0015   21.9   3.3   38   56-93     18-55  (192)
</pre><pre>129 6XY1_D 4-KE-4; coiled coil, te  30.6 1.5E+02  0.0017   17.3   2.7   16   11-26      4-19  (32)
</pre><pre>130 PF07964.15 ; Red1 ; Rec10 / Re  30.6 3.1E+02  0.0035   28.2   6.5   69    8-90    625-693 (699)
</pre><pre>131 4KWA_B Transcriptional regulat  30.6 1.2E+02  0.0014   21.9   3.1   37   56-92     10-46  (195)
</pre><pre>132 6EL2_B Transcriptional regulat  30.6 1.3E+02  0.0015   21.9   3.4   38   56-93     19-56  (196)
</pre><pre>133 3JSJ_C Putative TetR-family tr  30.5 1.2E+02  0.0013   22.2   3.0   37   56-93     15-51  (190)
</pre><pre>134 4UDS_A MBDR REGULATOR; TRANSCR  30.5 1.3E+02  0.0015   22.3   3.2   37   56-92     19-55  (214)
</pre><pre>135 2HKU_A A putative transcriptio  30.3 1.3E+02  0.0014   22.6   3.2   43   47-93     20-62  (215)
</pre><pre>136 3CWR_A Transcriptional regulat  30.3      99  0.0011   23.1   2.7   38   56-93     23-60  (208)
</pre><pre>137 2NP5_D Transcriptional regulat  30.3 1.3E+02  0.0015   22.2   3.3   38   56-93     15-52  (203)
</pre><pre>138 2HUJ_A Lin2004 protein; NP_471  30.3 4.4E+02  0.0051   21.1   8.5   87    8-110    16-102 (140)
</pre><pre>139 5K7F_A Transcriptional regulat  30.2 1.2E+02  0.0014   22.9   3.2   38   56-93     22-59  (231)
</pre><pre>140 3BRU_A Regulatory protein, Tet  30.2 1.3E+02  0.0015   22.8   3.3   38   56-93     36-73  (222)
</pre><pre>141 2REK_A Putative tetR-family tr  30.2 1.5E+02  0.0017   21.9   3.5   49   41-93     10-58  (199)
</pre><pre>142 5VL9_C Regulatory protein TetR  30.1 1.3E+02  0.0015   21.9   3.2   37   56-92     14-50  (192)
</pre><pre>143 2G3B_B putative TetR-family tr  30.1 1.3E+02  0.0015   22.2   3.2   38   56-93      9-46  (208)
</pre><pre>144 3FRQ_B Repressor protein MphR(  30.1 1.2E+02  0.0014   22.3   3.1   37   56-92     14-50  (195)
</pre><pre>145 2OER_B Probable transcriptiona  30.1 1.1E+02  0.0012   23.0   2.8   37   56-92     30-66  (214)
</pre><pre>146 5XAZ_C Gamma-butyrolactone rec  30.1 1.3E+02  0.0015   23.2   3.3   38   56-93     14-51  (228)
</pre><pre>147 3LHQ_B AcrAB operon repressor   30.1 1.2E+02  0.0014   22.7   3.0   38   56-93     20-57  (220)
</pre><pre>148 3COL_A Putative transcription   30.0 1.4E+02  0.0016   21.8   3.4   37   56-92     16-52  (196)
</pre><pre>149 3BQZ_A HTH-type transcriptiona  29.8 1.2E+02  0.0014   22.2   3.0   37   56-92      8-44  (194)
</pre><pre>150 1T33_B putative transcriptiona  29.7 1.3E+02  0.0015   23.1   3.2   42   47-92     12-53  (224)
</pre><pre>151 6OF0_A HTH-type transcriptiona  29.4 1.3E+02  0.0015   22.2   3.1   38   56-93     16-53  (210)
</pre><pre>152 2D6Y_B putative tetR family re  29.3 1.3E+02  0.0015   22.6   3.1   37   56-92     14-50  (202)
</pre><pre>153 2RAE_A Transcriptional regulat  29.3 1.3E+02  0.0015   22.2   3.1   38   56-93     23-60  (207)
</pre><pre>154 3BHQ_B Transcriptional regulat  29.3 1.3E+02  0.0015   22.4   3.1   37   56-92     18-54  (211)
</pre><pre>155 PF01297.21 ; ZnuA ; Zinc-uptak  29.2 5.4E+02  0.0062   22.0   7.1  120    9-141   118-239 (278)
</pre><pre>156 3LWJ_A Putative TetR-family tr  29.1 1.4E+02  0.0016   21.9   3.3   37   56-92     18-54  (202)
</pre><pre>157 3B81_A Transcriptional regulat  29.1 1.3E+02  0.0015   22.2   3.1   38   56-93     17-54  (203)
</pre><pre>158 6DKM_D DHD131_B; Computational  28.9 3.2E+02  0.0037   19.1   5.3   45   11-62     15-59  (79)
</pre><pre>159 PF05476.15 ; PET122 ; PET122    28.9      62 0.00072   28.7   1.5   64   24-91    116-180 (261)
</pre><pre>160 3NRG_B TetR family transcripti  28.7 1.2E+02  0.0014   22.6   2.9   38   56-93     19-56  (217)
</pre><pre>161 3PAS_A TetR family transcripti  28.6 1.3E+02  0.0015   21.8   3.1   38   56-93     14-51  (195)
</pre><pre>162 2F07_B YvdT; helix-turn-helix,  28.6 1.4E+02  0.0016   21.7   3.2   37   56-92     16-52  (197)
</pre><pre>163 3KNW_A Putative transcriptiona  28.4 1.4E+02  0.0017   22.1   3.2   37   56-92     20-56  (212)
</pre><pre>164 4JKZ_A Transcriptional regulat  28.4 1.6E+02  0.0018   21.8   3.4   38   56-93     24-61  (194)
</pre><pre>165 2GFN_B HTH-type transcriptiona  28.4 1.4E+02  0.0017   22.0   3.2   37   56-92     15-51  (209)
</pre><pre>166 8KE8_B NalC; NalC, TetR-type t  28.4 1.4E+02  0.0016   22.3   3.2   38   56-93     20-57  (213)
</pre><pre>167 8H7E_A De novo ferric enteroba  28.1 4.2E+02  0.0048   20.2   5.9   40   11-56      5-44  (102)
</pre><pre>168 3S5R_B Transcriptional regulat  28.1 1.4E+02  0.0016   22.2   3.1   38   56-93     16-53  (216)
</pre><pre>169 3E7Q_A transcriptional regulat  28.0 1.4E+02  0.0017   22.2   3.2   37   56-92     20-56  (215)
</pre><pre>170 6HS0_A Probable transcriptiona  27.9 2.5E+02  0.0028   21.2   4.5   55   36-93     23-77  (221)
</pre><pre>171 3CRJ_D Transcription regulator  27.9 1.5E+02  0.0017   21.8   3.3   37   56-92     20-56  (199)
</pre><pre>172 3HIM_A Probable transcriptiona  27.9 1.5E+02  0.0017   22.0   3.3   38   56-93     22-59  (211)
</pre><pre>173 4NNO_A Lipoprotein; Manganese   27.8   6E+02  0.0069   21.9   7.7  119    9-140   128-250 (291)
</pre><pre>174 2W53_A REPRESSOR; ANTIBIOTIC R  27.7 1.4E+02  0.0016   22.4   3.1   38   56-93     17-54  (219)
</pre><pre>175 3TU3_B ExoU; type III secretio  27.7 6.7E+02  0.0077   25.4   8.3  101    3-120   482-593 (711)
</pre><pre>176 4ZA6_B TetR family transcripti  27.6 2.1E+02  0.0025   21.0   4.0   42   46-92      4-45  (192)
</pre><pre>177 PF16271.9 ; DUF4924 ; Domain o  27.5 3.5E+02   0.004   22.7   5.4   45   14-59     75-119 (177)
</pre><pre>178 3PPB_A putative tetR family tr  27.5 1.5E+02  0.0018   21.5   3.2   37   56-92     15-51  (195)
</pre><pre>179 4ICH_B Transcriptional regulat  27.4 1.5E+02  0.0017   24.4   3.5   43   47-92    120-162 (311)
</pre><pre>180 5UA2_A HTH-type transcriptiona  27.3 1.3E+02  0.0015   22.7   3.0   37   56-92     47-83  (224)
</pre><pre>181 PF00440.27 ; TetR_N ; Bacteria  27.3 1.5E+02  0.0017   16.2   2.5   38   56-93      2-39  (47)
</pre><pre>182 4OXR_B Manganese ABC transport  27.2 4.6E+02  0.0053   22.5   6.4  119    9-140   122-244 (285)
</pre><pre>183 2YVE_A Transcriptional regulat  27.2 1.4E+02  0.0016   22.2   3.1   37   56-92     10-46  (185)
</pre><pre>184 6NSI_A Manganese-binding prote  27.2 5.9E+02  0.0068   22.1   7.1  120    9-141   108-243 (287)
</pre><pre>185 3CX3_B Lipoprotein; zinc-bindi  27.1 5.4E+02  0.0062   21.9   6.8  116    9-137   126-243 (284)
</pre><pre>186 6G8G_C TetR/AcrR family transc  27.1 1.4E+02  0.0016   22.2   3.0   38   56-93     27-64  (214)
</pre><pre>187 7KY1_A TetR family transcripti  27.1 1.6E+02  0.0018   22.1   3.3   38   56-93     35-72  (214)
</pre><pre>188 6KNY_B Protein Amuc_1100; Unch  27.1 6.6E+02  0.0076   22.1   9.1   91    9-116    36-134 (287)
</pre><pre>189 3NNR_A Transcriptional regulat  26.9 1.5E+02  0.0017   22.8   3.2   37   56-92     11-47  (228)
</pre><pre>190 3G7R_B Putative transcriptiona  26.9 1.2E+02  0.0014   23.0   2.7   38   56-93     41-78  (221)
</pre><pre>191 2I10_A Putative TetR transcrip  26.6 1.6E+02  0.0019   21.7   3.3   37   56-92     17-53  (202)
</pre><pre>192 5UXS_A Periplasmic chelated ir  26.6 4.8E+02  0.0056   23.1   6.5  119    9-140   147-269 (323)
</pre><pre>193 4LID_A A-100; VIRAL PROTEIN, H  26.4 4.8E+02  0.0055   20.3   6.2   43   17-63     56-98  (106)
</pre><pre>194 6S7O_E Dolichyl-diphosphooligo  26.4 6.9E+02  0.0079   24.5   8.0   52   11-65    551-602 (607)
</pre><pre>195 5AQC_A HTH-TYPE TRANSCRIPTIONA  26.4 1.4E+02  0.0016   22.3   2.9   37   56-92     22-58  (205)
</pre><pre>196 7NGG_C HTH-type transcriptiona  26.3 1.5E+02  0.0018   22.1   3.1   37   56-92     30-66  (216)
</pre><pre>197 5D18_A TetR family transcripti  26.3 1.6E+02  0.0018   22.1   3.2   37   56-92     20-56  (216)
</pre><pre>198 7QWA_C CC-Type2-(UgUe)4; aib,   26.2 2.1E+02  0.0024   16.7   2.7   21   11-31      4-24  (32)
</pre><pre>199 1TOA_B PROTEIN (PERIPLASMIC BI  26.2 4.1E+02  0.0047   23.3   6.0  116    9-137   144-263 (313)
</pre><pre>200 1VI0_B transcriptional regulat  26.2 1.5E+02  0.0018   21.9   3.0   38   56-93     14-51  (206)
</pre><pre>201 5W56_A Periplasmic solute bind  26.1 6.4E+02  0.0074   21.6   7.1  116    9-137   121-240 (286)
</pre><pre>202 3GZI_A Transcriptional regulat  26.1 1.3E+02  0.0015   22.5   2.7   37   56-92     23-59  (218)
</pre><pre>203 2QWT_A Transcriptional regulat  26.1 1.5E+02  0.0018   21.9   3.0   36   56-92     19-54  (196)
</pre><pre>204 PF20865.1 ; FAM186A-B_C ; FAM1  26.0 2.8E+02  0.0032   24.2   4.8   46   16-63     86-131 (236)
</pre><pre>205 6YB0_I CC-Type2-(TaSd)2; alpha  26.0 1.9E+02  0.0022   17.6   2.7   18   17-34      3-20  (39)
</pre><pre>206 7EQE_A TetR/AcrR family transc  25.8 1.7E+02   0.002   21.8   3.2   38   56-93     25-62  (200)
</pre><pre>207 3QQA_A CmeR; Alpha-helical, He  25.8 1.7E+02  0.0019   22.3   3.2   37   56-92     25-61  (216)
</pre><pre>208 4C18_B TSI2; TOXIN, IMMUNITY P  25.8 4.5E+02  0.0052   19.8   5.9   74    1-76     19-92  (98)
</pre><pre>209 PF04099.16 ; Sybindin ; Sybind  25.7 2.1E+02  0.0025   20.7   3.7   40   16-56     87-132 (132)
</pre><pre>210 5MQQ_D AmtR protein; transcrip  25.5 1.7E+02   0.002   21.7   3.3   38   56-93     11-48  (208)
</pre><pre>211 4YZE_C HTH-type transcriptiona  25.5 1.8E+02  0.0021   21.4   3.3   37   56-92     16-52  (201)
</pre><pre>212 2OF7_A Putative tetR-family tr  25.4 1.6E+02  0.0019   23.3   3.2   37   56-92     54-90  (260)
</pre><pre>213 2YM0_B CELL INVASION PROTEIN S  25.2 2.9E+02  0.0034   23.9   4.8   57    1-66     11-67  (233)
</pre><pre>214 6O6P_A TetR family transcripti  25.1   2E+02  0.0023   22.6   3.7   43   47-92     59-101 (248)
</pre><pre>215 2WUI_A TRANSCRIPTIONAL REGULAT  24.9 1.8E+02  0.0021   21.8   3.3   38   56-93     17-54  (210)
</pre><pre>216 5JWR_G Circadian clock protein  24.6   6E+02   0.007   20.9   6.0   63   47-120    46-113 (145)
</pre><pre>217 3NI7_A Bacterial regulatory pr  24.5 1.9E+02  0.0022   22.0   3.4   44   46-92      6-49  (213)
</pre><pre>218 7WOO_L Nucleoporin NSP1; nucle  24.4 1.2E+03   0.014   24.1   9.7  107    4-122   693-807 (823)
</pre><pre>219 3F0C_A Transcriptional regulat  24.4 1.8E+02  0.0021   21.6   3.2   37   56-92     17-53  (216)
</pre><pre>220 3BNI_A Putative TetR-family tr  24.2 1.6E+02  0.0019   22.5   2.9   38   56-93     49-86  (229)
</pre><pre>221 4ETP_B Spindle pole body-assoc  24.2   6E+02   0.007   22.5   6.7   44    9-58     13-56  (333)
</pre><pre>222 7C7X_E NAP1-related protein 1;  24.2 3.6E+02  0.0042   22.7   5.2   64   13-81     15-79  (239)
</pre><pre>223 6YJ2_A Probable transcriptiona  24.1 1.8E+02  0.0021   21.9   3.1   37   56-92     31-67  (215)
</pre><pre>224 3EGQ_A TetR family Transcripti  24.0 1.6E+02  0.0019   21.2   2.8   38   56-93     10-47  (170)
</pre><pre>225 5FGL_B NicR; co-crystal derepr  24.0 1.9E+02  0.0021   21.6   3.2   37   56-92      5-41  (208)
</pre><pre>226 1K0F_A Periplasmic zinc-bindin  23.9 6.5E+02  0.0075   21.4   6.7  119    9-140   108-230 (277)
</pre><pre>227 PF17402.6 ; DUF5408 ; Family o  23.8 2.3E+02  0.0027   19.8   3.2   25    6-30     39-63  (63)
</pre><pre>228 3RH2_A Hypothetical TetR-like   23.8 1.9E+02  0.0022   21.7   3.2   38   56-93      9-46  (212)
</pre><pre>229 4MO7_A Transcriptional regulat  23.8 1.9E+02  0.0022   21.5   3.2   38   56-93     35-72  (212)
</pre><pre>230 6SIH_C Flagellar hook-associat  23.7 8.8E+02    0.01   24.4   8.3   56    4-65    605-660 (663)
</pre><pre>231 5OVY_B Putative transcriptiona  23.7 1.3E+02  0.0014   22.9   2.2   36   56-92     24-59  (222)
</pre><pre>232 4UT1_A FLAGELLAR HOOK-ASSOCIAT  23.7 4.4E+02  0.0051   26.5   6.3   54    7-61    165-219 (673)
</pre><pre>233 6KO8_A Putative regulatory pro  23.5 1.9E+02  0.0022   22.7   3.2   36   56-92     15-50  (194)
</pre><pre>234 7WKK_H IL4I1 protein; nuclear   23.4 1.1E+03   0.013   23.6  10.2  126    9-153   419-545 (547)
</pre><pre>235 5HAW_A Nucleoid occlusion fact  23.3 1.7E+02   0.002   21.2   2.9   37   56-92     14-51  (196)
</pre><pre>236 2QKO_B Possible transcriptiona  23.3   2E+02  0.0023   21.8   3.2   37   56-92     34-70  (215)
</pre><pre>237 6WP9_D AvaR1; Transcriptional   23.2   2E+02  0.0023   22.4   3.2   37   56-92     26-62  (245)
</pre><pre>238 7UZK_S Effector SidK; V-ATPase  23.0 8.5E+02  0.0098   22.0   7.3   92    9-117    40-141 (280)
</pre><pre>239 4H0F_B Laminin-binding surface  23.0 5.8E+02  0.0067   21.6   6.2  116    9-137   106-223 (264)
</pre><pre>240 5T51_A KLLA0F02343p; cell cycl  23.0 5.2E+02   0.006   19.5   9.0   66    7-73     24-97  (120)
</pre><pre>241 4J2C_C Syntaxin-6; Tethering c  22.9 4.6E+02  0.0053   18.9   7.8   60    4-63     43-102 (110)
</pre><pre>242 5D1W_C Rv3249c transcriptional  22.9 1.7E+02  0.0019   22.4   2.7   37   56-92     29-65  (217)
</pre><pre>243 3VRC_B Cytochrome c'; c-type c  22.9 4.9E+02  0.0057   19.2   7.2  101   41-141     4-123 (131)
</pre><pre>244 1V2Z_A circadian clock protein  22.7 5.8E+02  0.0067   20.0   6.0   63   47-120    12-79  (111)
</pre><pre>245 3C2B_A Transcriptional regulat  22.7 1.5E+02  0.0017   22.4   2.4   37   56-92     21-57  (221)
</pre><pre>246 PF03568.21 ; Peptidase_C50 ; P  22.6 4.4E+02   0.005   24.6   5.8   58   11-69     39-96  (460)
</pre><pre>247 2QTQ_C Transcriptional regulat  22.6 1.9E+02  0.0022   21.6   3.0   37   56-92     22-58  (213)
</pre><pre>248 2IM8_B Hypothetical protein yp  22.4   6E+02  0.0069   20.0   7.5   84   11-110     3-86  (131)
</pre><pre>249 5ZVM_b pan-CoV inhibitory pept  22.3 2.6E+02   0.003   18.2   2.9   22   11-32     18-39  (44)
</pre><pre>250 6SID_A Activity-regulated cyto  22.2 3.9E+02  0.0045   18.6   4.3   78   26-117     8-88  (90)
</pre><pre>
</pre><pre>No 1
</pre><pre>>1CII_A COLICIN IA; COLICIN, BACTERIOCIN, ION CHANNEL FORMATION, TRANSMEMBRANE PROTEIN; 3.0A {Escherichia coli} SCOP: f.1.1.1, h.4.3.1
</pre><pre>Probab=88.72  E-value=16  Score=35.72  Aligned_cols=136  Identities=15%  Similarity=0.156  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      ++..++++.....+.+.+...+|.+-..+.++. .--.+.++-.+.-.++...++..|+.-        +...|.++|  
</pre><pre>T Consensus       430 a~~~~~~~~~~~~~~~g~~~~~~a~~la~~~~G-k~ir~~~~A~~~~~k~~~~~~~k~~~~--------Dr~ai~~Al--  498 (602)
</pre><pre>T 1CII_A          430 AINFTTEFLKSVSEKYGAKAEQLAREMAGQAKG-KKIRNVEEALKTYEKYRADINKKINAK--------DRAAIAAAL--  498 (602)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCTHHHHHHHHHHHTTTT-CBCCCHHHHHHHHHHTHHHHTTSSCSH--------HHHHHHHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHcC-ccCCCHHHHHHHHHHHHhchhhccCHH--------HHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchhHH
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGFKQ  162 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g~~  162 (174)
</pre><pre>                                    ..++ ..+.-.++.+++|.   .|+++.++++..-..++.+++.+|+...    .+.|.-+++.|.
</pre><pre>T Consensus       499 --------------~~~~-~~~~a~~l~~~~k~---fg~~~~~~~~~~~~~~~~~~~~tg~W~p----~~~~~e~~~~g~  556 (602)
</pre><pre>T 1CII_A          499 --------------ESVK-LSDISSNLNRFSRG---LGYAGKFTSLADWITEFGKAVRTENWRP----LFVKTETIIAGN  556 (602)
</pre><pre>T ss_dssp             --------------HTCC-HHHHHHHHHHHHHH---HTCCSCBCCHHHHHHHHHHHHHSTTCHH----HHHHHTTTBTTC
</pre><pre>T ss_pred             --------------HHCC-HHHHHHHHHHHHHH---hchHHHHccHHHHHHHHHHHHhcCCcHH----HhHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHh
</pre><pre>Q FD01845382_026  163 TGSMLAGLV  171 (174)
</pre><pre>Q Consensus       163 ~gG~~gg~~  171 (174)
</pre><pre>                      +++.+.+.+
</pre><pre>T Consensus       557 ~a~~~~~~~  565 (602)
</pre><pre>T 1CII_A          557 AATALVALV  565 (602)
</pre><pre>T ss_dssp             BHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF01024.23 ; Colicin ; Colicin pore forming domain
</pre><pre>Probab=88.30  E-value=3.2  Score=34.43  Aligned_cols=141  Identities=15%  Similarity=0.170  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhh
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIM   88 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~v   88 (174)
</pre><pre>                      .......+-+.+..++|-+.|-+.+..    +-.+--..-+-+-.+..+..+..+-+      .-.++...++..-+..+
</pre><pre>T Consensus         9 ~~i~~a~~~~~df~k~i~~k~G~k~~~----~AkelA~~~kGKkIRn~~dAl~~f~k------~~~~~~~k~s~~Dr~Ai   78 (185)
</pre><pre>T Q9I4Y4_PSEAE/3    9 QALQDAIKFTADFYKEVTEKFGARTSE----MARQLAEGARGKNIRSSAEAIKSFEK------HKDALNKKLSLKDRQAI   78 (185)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH----HHHHHHHHhcCccCCCHHHHHHHHHH------HHHHHhhcCCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchhHHHHHHHH
</pre><pre>Q FD01845382_026   89 HQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGFKQTGSMLA  168 (174)
</pre><pre>Q Consensus        89 h~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g~~~gG~~g  168 (174)
</pre><pre>                      -++=+.  -+.+.+++++.+++|.      .|+++.+++...-...+.+++++|+...    .+.|.-+++.|..++.+.
</pre><pre>T Consensus        79 ~~Al~s--~d~~~~a~nl~~~sK~------fg~~~~~~~~~~l~~~~~~g~~TgnW~p----~~l~~Esi~aG~~a~~~~  146 (185)
</pre><pre>T Q9I4Y4_PSEAE/3   79 AKAFDS--LDKQMMAKSLEKFSKG------FGVVGKAIDAASLYQEFKISTETGDWKP----FFVKIETLAAGAAASWLV  146 (185)
</pre><pre>T ss_pred             HHHHHH--CCHHHHHHHHHHHHHH------HhHHHHHHcHHHHHHHHHHHHhhCCcHH----HHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHh
</pre><pre>Q FD01845382_026  169 GLV  171 (174)
</pre><pre>Q Consensus       169 g~~  171 (174)
</pre><pre>                      +.+
</pre><pre>T Consensus       147 a~~  149 (185)
</pre><pre>T Q9I4Y4_PSEAE/3  147 GIA  149 (185)
</pre><pre>T ss_pred             HHH
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>6THK_A Pyocin S5; Bacteriocin, Pyocin, Antibiotic, Ionophore, ANTIMICROBIAL PROTEIN; 2.2A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=86.50  E-value=15  Score=35.38  Aligned_cols=136  Identities=16%  Similarity=0.173  Sum_probs=0.0  Template_Neff=5.100
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      ++..+++......+.+-.....+.+-..+..+.    .+...|-..++.+ .+....+    ...+.-.+...+.++|  
</pre><pre>T Consensus       323 Av~~~adfgK~IseK~G~k~s~iAkeLAd~iKG----KKIRn~ddA~kaf-eK~k~np----~~K~s~aDr~AIanAL--  391 (506)
</pre><pre>T 6THK_A          323 AIKFTADFYKEVTEKFGARTSEMARQLAEGARG----KNIRSSAEAIKSF-EKHKDAL----NKKLSLKDRQAIAKAF--  391 (506)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHTT----CBCCCHHHHHHHH-HHHHHHH----HHHSCHHHHHHHHHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhHHHHHHHHHHHHhhcC----CCCCCHHHHHHHH-HHHhhhh----chhcchhhHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchhHH
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGFKQ  162 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g~~  162 (174)
</pre><pre>                                    ..++ +.+.-.++.+++|.   .|+++.++++..-...+.++..+|+...    .+.|.-+++.|+
</pre><pre>T Consensus       392 --------------~sld-~~~~A~nL~~~sK~---Fg~~g~~i~~~~v~~~~~~g~~TGnWkP----l~le~Esi~aG~  449 (506)
</pre><pre>T 6THK_A          392 --------------DSLD-KQMMAKSLEKFSKG---FGVVGKAIDAASLYQEFKISTETGDWKP----FFVKIETLAAGA  449 (506)
</pre><pre>T ss_dssp             --------------HTCC-HHHHHHHHHHHHHH---TTCCCCCCCHHHHHHHHHHHHHSSCCHH----HHHHHHHHHTTC
</pre><pre>T ss_pred             --------------HHCC-HHHHHHHHHHHHHH---HhHHHHHhcHHHHHHHHHHHHhhCCcHH----HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHh
</pre><pre>Q FD01845382_026  163 TGSMLAGLV  171 (174)
</pre><pre>Q Consensus       163 ~gG~~gg~~  171 (174)
</pre><pre>                      ++|.+.+.+
</pre><pre>T Consensus       450 ~A~~~~a~~  458 (506)
</pre><pre>T 6THK_A          450 AASWLVGIA  458 (506)
</pre><pre>T ss_dssp             BHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>8IOM_A Colicin 1B; pore-forming toxin, antibacterial protein, colicin, ANTIMICROBIAL PROTEIN; 3.0A {Shigella flexneri}
</pre><pre>Probab=86.31  E-value=30  Score=29.45  Aligned_cols=136  Identities=18%  Similarity=0.187  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      ++..+++.+....+.+.....++.+...+..+. .--.+.++-.+.-.+....++..++.-        +...|.+.|  
</pre><pre>T Consensus        21 av~~~sdf~k~i~~k~G~k~~~~AkelA~~~kG-KkIr~~~dAl~~feK~~~~~~~k~s~~--------Dr~Ai~nAl--   89 (205)
</pre><pre>T 8IOM_A           21 AIKLTSDFYRTIYDEFGKQASELAKELASVSQG-KQIKSVDDALNAFDKFRNNLNKKYNIQ--------DRMAISKAL--   89 (205)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHTT-CBCCCHHHHHHHHHHHHHHHTTTCCHH--------HHHHHHHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCC-CCCCCHHHHHHHHHHHHHchhhhcCHH--------hHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchhHH
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGFKQ  162 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g~~  162 (174)
</pre><pre>                                    ..++ +.+.-.++.+++|.   .|+++.++++..-..++.++..+|+...    .+.|.-+++.|.
</pre><pre>T Consensus        90 --------------~s~d-~~~~A~nL~~~sK~---Fg~~g~~~~~~~l~~~~~~g~~TGnW~P----~~le~Es~~~g~  147 (205)
</pre><pre>T 8IOM_A           90 --------------EAIN-QVHMAENFKLFSKA---FGFTGKVIDRYDVAVELQKAVKTDNWRP----FFVKLESLAAGR  147 (205)
</pre><pre>T ss_dssp             --------------TTCC-HHHHHHHHHHHHHH---TTCCSCCCCCHHHHHHHHHHHHHCCCHH----HHHHHHHHSTTS
</pre><pre>T ss_pred             --------------HHCc-HHHHHHHHHHHHHH---HhhHHHHhcHHHHHHHHHHHHHcCCcHH----HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHh
</pre><pre>Q FD01845382_026  163 TGSMLAGLV  171 (174)
</pre><pre>Q Consensus       163 ~gG~~gg~~  171 (174)
</pre><pre>                      +.+.+.+.+
</pre><pre>T Consensus       148 ~As~~~a~~  156 (205)
</pre><pre>T 8IOM_A          148 AASAVTAWA  156 (205)
</pre><pre>T ss_dssp             BHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>1COL_A COLICIN A; ANTIBACTERIAL PROTEIN; 2.4A {Escherichia coli} SCOP: f.1.1.1
</pre><pre>Probab=86.22  E-value=25  Score=29.81  Aligned_cols=140  Identities=11%  Similarity=0.112  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      .+...++...+..+++-+...++.+-..+..+.-+|+..+.  +.+--+.|.+..+.++......    +...|.+.|  
</pre><pre>T Consensus        15 a~~~~~d~~~~i~~k~G~k~~~~AkelA~~~kn~~GKkIrn--~~dAl~~feK~~~~~~~k~s~~----Dr~Ai~~Al--   86 (204)
</pre><pre>T 1COL_A           15 TSELIAGMGDKIGEHLGDKYKAIAKDIADNIKNFQGKTIRS--FDDAMASLNKITANPAMKINKA----DRDALVNAW--   86 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHCHHHHHHHHHHHHHHHTCTTCCCCC--HHHHHHHHHHHHTSGGGCCCHH----HHHHHHHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhcCCCCCCC--HHHHHHHHHHHhhCcchhcCHH----HHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccch-hH
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFG-FK  161 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~-g~  161 (174)
</pre><pre>                                    ..++ +.+.-.++.+++|.   .|+++.++++..-..++.+++++|+...    .++|.-+++ ++
</pre><pre>T Consensus        87 --------------~s~d-~~~~a~nl~~~sK~---fg~~~~~i~~~~l~~~~~~g~~TgnW~p----~~lk~Es~~~~g  144 (204)
</pre><pre>T 1COL_A           87 --------------KHVD-AQDMANKLGNLSKA---FKVADVVMKVEKVREKSIEGYETGNWGP----LMLEVESWVLSG  144 (204)
</pre><pre>T ss_dssp             --------------HTCC-HHHHHHHHHHHCGG---GCCTTHHHHHHHHHHHHHHHHHHCCCHH----HHHHHHHHHHTT
</pre><pre>T ss_pred             --------------HhCC-HHHHHHHHHHHHHH---hcHHHHhhhHHHHHHHHHHHHhhCCcHH----HHHHHHHHHHhH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhc
</pre><pre>Q FD01845382_026  162 QTGSMLAGLVS  172 (174)
</pre><pre>Q Consensus       162 ~~gG~~gg~~~  172 (174)
</pre><pre>                      ++++...+.++
</pre><pre>T Consensus       145 ~aa~~a~~~~~  155 (204)
</pre><pre>T 1COL_A          145 IASSVALGIFS  155 (204)
</pre><pre>T ss_dssp             CCHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>PF05120.16 ; GvpG ; Gas vesicle protein G
</pre><pre>Probab=85.04  E-value=8.7  Score=26.81  Aligned_cols=38  Identities=21%  Similarity=0.344  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTI   59 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~   59 (174)
</pre><pre>                      ..|.+-|.+++.+|.      .|.+..++|..+++.|+.+|+..
</pre><pre>T Consensus        31 ~~i~~~L~eL~~~~e------~GeIseeEy~~~e~~Ll~rl~~~   68 (72)
</pre><pre>T Q2JJS5_SYNJB/2   31 ENLQKELTALQIQLD------LGEIDEETYARREEEILLALEAL   68 (72)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHH------cCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>2I88_A Colicin-E1; protein-membrane interactions, toxin-membrane interactions, toxin structure, voltage-gated channel, colicin, MEMBRANE PROTEIN; 2.5A {Escherichia coli}
</pre><pre>Probab=78.58  E-value=59  Score=27.38  Aligned_cols=136  Identities=13%  Similarity=0.089  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      ++..+++...+..+++-....++.+...+.... .--.+.++-.+.-++....++..++.-        +...|.++|  
</pre><pre>T Consensus        17 a~~~~~df~k~i~~k~G~k~~~~AkelA~~~kG-Kkir~~~~A~~~f~K~~~n~~~K~s~~--------Dr~Ai~nAl--   85 (191)
</pre><pre>T 2I88_A           17 AVDATVSFYQTLTEKYGEKYSKMAQELADKSKG-KKIGNVNEALAAFEKYKDVLNKKFSKA--------DRDAIFNAL--   85 (191)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHSCHHHHHHHHHHHHHHTT-CCCCCHHHHHHHHHHHHHHHHHSSCHH--------HHHHHHHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCC-CCCCCHHHHHHHHHHHHHHHhhhcCHH--------HHHHHHHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchhHH
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGFKQ  162 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g~~  162 (174)
</pre><pre>                                    ..++ +.+.-.++.+++|.   .|+++.++++..-..++.+++++|+...    -+.|.-+++.|.
</pre><pre>T Consensus        86 --------------~s~d-~~~~a~nl~~~sK~---Fg~~~~~~~~~~v~~~~~~g~~TGnW~p----l~l~~Esi~~g~  143 (191)
</pre><pre>T 2I88_A           86 --------------ASVK-YDDWAKHLDQFAKY---LKITGHVSFGYDVVSDILKIKDTGDWKP----LFLTLEKKAADA  143 (191)
</pre><pre>T ss_dssp             --------------TTCC-THHHHTTHHHHHHH---TTCCSCCCTTCCHHHHHHHHHHHCCCHH----HHHHHHHHHGGG
</pre><pre>T ss_pred             --------------HhCC-HHHHHHHHHHHHHH---hhHHHHHhcHHHHHHHHHHHHhcCCcHH----HHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHh
</pre><pre>Q FD01845382_026  163 TGSMLAGLV  171 (174)
</pre><pre>Q Consensus       163 ~gG~~gg~~  171 (174)
</pre><pre>                      +.+.+.+.+
</pre><pre>T Consensus       144 ~a~~~va~~  152 (191)
</pre><pre>T 2I88_A          144 GVSYVVALL  152 (191)
</pre><pre>T ss_dssp             THHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3FEW_X Colicin S4; Colicin S4, cell lysis, Y-ray, cell killing, IMMUNE SYSTEM; 2.45A {Escherichia coli}
</pre><pre>Probab=78.09  E-value=47  Score=32.16  Aligned_cols=137  Identities=9%  Similarity=0.150  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHH--------------HHHHHHHhH-hcCCC--ccc
</pre><pre>Q FD01845382_026   12 KSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLD--------------TILNKFSKT-HLNLE--EYN   74 (174)
</pre><pre>Q Consensus        12 ~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld--------------~~l~~~~~~-~~~~~--~~~   74 (174)
</pre><pre>                      ....+.+.++.+-+.+.|++...+    + .+.+...-+.|-..++              ..|+++... ...+.  +..
</pre><pre>T Consensus       301 ~~e~~~l~~a~~~~~d~~k~i~~k----~-G~k~~~~AkelA~~~kN~qGKkIRn~~dA~~af~K~~~n~~~K~s~~Dr~  375 (505)
</pre><pre>T 3FEW_X          301 RIQSDVLNKAAEVISDIGNKVGDY----L-GDAYKSLAREIADDVKNFQGKTIRSYDDAMASLNKVLSNPGFKFNRADSD  375 (505)
</pre><pre>T ss_dssp             HHCCCCCHHHHHHHHHHHHHHHHH----H-CHHHHHHHHHHHHHHHTCTTCCCCCHHHHHHHHHHHHTCTTCCCCHHHHH
</pre><pre>T ss_pred             HhcHHHHHHHHHHHHHHHHHHHHH----H-HHHHHHHHHHHHHHHHhcCCCcCCCHHHHHHHHHHHhcCccccCCHHhHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCCh-HHcceeeee
</pre><pre>Q FD01845382_026   75 SIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRE-SACRKAAIT  153 (174)
</pre><pre>Q Consensus        75 ~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~-~~C~k~~~~  153 (174)
</pre><pre>                      .|-++|                ..++ ..+.-.++.+++|.   .|++..++++..-..+..+++.+|+. ..--++...
</pre><pre>T Consensus       376 AI~nAl----------------~s~d-~~~~A~nl~~~sK~---Fg~~~~~~~~~~l~~~~~~g~~TGnW~P~~le~Es~  435 (505)
</pre><pre>T 3FEW_X          376 ALANVW----------------RSID-AQDMANKLGNISKA---FKFADVVMKVEKVREKSIEGYETGNWGPLMLEVESW  435 (505)
</pre><pre>T ss_dssp             HHHHHH----------------HHCC-HHHHHHHHTTTCGG---GCSSCCHHHHHHHHHHHHHHHHHSCCHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHH----------------HhCC-HHHHHHHHHHHHHH---hchHHHhhHHHHHHHHHhhhhhcCCcHHHhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ecccchhHHHHHHHHHHhcc
</pre><pre>Q FD01845382_026  154 EYSKFGFKQTGSMLAGLVSG  173 (174)
</pre><pre>Q Consensus       154 E~~~~~g~~~gG~~gg~~~~  173 (174)
</pre><pre>                      =.++++++++.+++.+.++.
</pre><pre>T Consensus       436 ~~~g~a~~~a~~~~~~~~~~  455 (505)
</pre><pre>T 3FEW_X          436 VLSGIASAVALGVFSATLGA  455 (505)
</pre><pre>T ss_dssp             HHTTCCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HhHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>5TGY_A PS1; 4-helix bundle, coiled-coil, holoprotein, UNKNOWN FUNCTION; HET: 7BU; NMR {Escherichia coli}
</pre><pre>Probab=75.88  E-value=26  Score=26.43  Aligned_cols=62  Identities=16%  Similarity=0.207  Sum_probs=0.0  Template_Neff=1.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHh
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKAL   80 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~L   80 (174)
</pre><pre>                      +.+++.|.+|+++|+.+..  -.....+....+...+-.++-..|++|-. .+..++..++...|
</pre><pre>T Consensus        30 dsleQ~Leele~l~qkh~e--L~~~~~e~~~~Ea~~~~de~vqlFqr~re-a~dkGDdDS~Eq~L   91 (109)
</pre><pre>T 5TGY_A           30 DSLEQVLEEIEELIQKHRQ--LFDNRQEAADTEAAKQGDQWVQLFQRFRE-AIDKGDKDSLEQLL   91 (109)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHH--HHHHSCCCSCHHHHHHHHHHHHHHHHHHH-HHHTTCHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHHhCCccHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>1A87_A COLICIN N; BACTERIOCIN, TOXIN, PORE-FORMING ACTIVITY; 3.1A {Escherichia coli K12} SCOP: f.1.1.1
</pre><pre>Probab=75.48  E-value=99  Score=28.36  Aligned_cols=139  Identities=12%  Similarity=0.162  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHH-----------HHHHHHHHHhHhcCCCccchHHHHh
</pre><pre>Q FD01845382_026   12 KSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKK-----------LDTILNKFSKTHLNLEEYNSIKKAL   80 (174)
</pre><pre>Q Consensus        12 ~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~-----------Ld~~l~~~~~~~~~~~~~~~lk~~L   80 (174)
</pre><pre>                      .+..+.++++.+-+.+.|+....+    + .+.+...-+.|-..           .+..|..|-+..-.+...-+-.+  
</pre><pre>T Consensus       124 k~e~~~l~~a~~~~~d~~k~i~~k----~-G~k~~~lAkelA~~~kn~qGKkIRn~~dAlaaf~K~~~n~~~K~saaD--  196 (321)
</pre><pre>T 1A87_A          124 KNEKEALLKASELVSGMGDKLGEY----L-GVKYKNVAKEVANDIKNFHGRNIRSYNEAMASLNKVLANPKMKVNKSD--  196 (321)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHH----H-CHHHHHHHHHHHHHHHTCCGGGCCCHHHHHHHHHHHHTCTTCCCCHHH--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHH----H-HHHHHHHHHHHHHHHHccccCCCCCHHHHHHHHHHHhhCccccCCHHH--
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeeeecccchh
</pre><pre>Q FD01845382_026   81 GLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAITEYSKFGF  160 (174)
</pre><pre>Q Consensus        81 glStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~E~~~~~g  160 (174)
</pre><pre>                         ...|...|..   -+.+.+++++.+++|.      .|+++.++++..-..++.+++.+|+...    .+.|.-+++.
</pre><pre>T Consensus       197 ---r~AI~nAlks---ld~~~~A~nL~~~sKa------Fg~~~~~i~~~~v~~~~~~g~~TGnW~P----l~levEs~~l  260 (321)
</pre><pre>T 1A87_A          197 ---KDAIVNAWKQ---VNAKDMANKIGNLGKA------FKVADLAIKVEKIREKSIEGYNTGNWGP----LLLEVESWII  260 (321)
</pre><pre>T ss_dssp             ---HHHHHHHHHT---CCHHHHHHHHHTTCGG------GCCTTHHHHHHHHHHHHHHHHTTCCCHH----HHHHHHHHHH
</pre><pre>T ss_pred             ---HHHHHHHHHh---CCHHHHHHHHHHHHHH------HHHHHHHhhHHHHHHHHhHhhhcCCcHH----HHHhhHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             -HHHHHHHHHHhcc
</pre><pre>Q FD01845382_026  161 -KQTGSMLAGLVSG  173 (174)
</pre><pre>Q Consensus       161 -~~~gG~~gg~~~~  173 (174)
</pre><pre>                       +++++++.|.++.
</pre><pre>T Consensus       261 sg~a~~~ala~~aa  274 (321)
</pre><pre>T 1A87_A          261 GGVVAGVAISLFGA  274 (321)
</pre><pre>T ss_dssp             TTCCHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>PF20848.1 ; ExoU_mid_dom ; ExoU toxin, middle helical domain
</pre><pre>Probab=72.59  E-value=22  Score=27.96  Aligned_cols=61  Identities=18%  Similarity=0.175  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   44 EFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        44 ~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      .|+.+-++.+..|+..+...-...-.+...++|...|                ..++.++.+=+++..++|.|..++
</pre><pre>T Consensus        33 ~FR~~A~qaL~~L~~AI~~an~~~~~L~~T~~l~~Al----------------~~LD~La~~p~~~eWLa~~LN~~~   93 (126)
</pre><pre>T G8Q2K0_PSEO1/3   33 AFRRQAQQALPQLKQAIQEANKTFTALEPTPQMHMAI----------------WALDQLADQPGKQEWLAKRLNHDN   93 (126)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhcCCCCCCCHHHHHHH----------------HHHHHHccCHHHHHHHHHHhCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF08649.14 ; DASH_Dad1 ; DASH complex subunit Dad1
</pre><pre>Probab=70.97  E-value=25  Score=23.28  Aligned_cols=46  Identities=15%  Similarity=0.268  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   45 FFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKL  112 (174)
</pre><pre>Q Consensus        45 F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~  112 (174)
</pre><pre>                      |.++|.+|..++...|..+..      .-..|.|.|                ..+-+..+.++.++.+
</pre><pre>T Consensus         2 Fe~eR~~Li~eI~~~l~~~~~------nln~LNrnL----------------E~~~~vG~ef~~v~~L   47 (55)
</pre><pre>T G1WZT5_ARTOA/1    2 FETQREALIEEICKSFDQVLL------NLNHLNRSL----------------EGLAGVGQEFTEIEAL   47 (55)
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHHH------HHHHHhHHH----------------HHHHHhhhcHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>7NFJ_A CC-Type2-(LaId)4-L28Y; Coiled coil, synthetic peptide, homomeric assembly, DE NOVO PROTEIN; HET: ACE; 1.19A {N/A}
</pre><pre>Probab=69.20  E-value=16  Score=21.10  Aligned_cols=20  Identities=35%  Similarity=0.302  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYK   30 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~   30 (174)
</pre><pre>                      ..+.+++|.=.|++|.++|+
</pre><pre>T Consensus        11 iakalkeiawalkeiaqayk   30 (32)
</pre><pre>T 7NFJ_A           11 IAKALKEIAWALKEIAQAYK   30 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>8Q84_X DASH complex subunit DAD1; Kinetochore, microtubule, error correction, chromosome segregation, CELL CYCLE;{Saccharomyces cerevisiae}
</pre><pre>Probab=68.25  E-value=22  Score=26.48  Aligned_cols=55  Identities=13%  Similarity=0.209  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   36 NKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKL  112 (174)
</pre><pre>Q Consensus        36 ~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~  112 (174)
</pre><pre>                      +........|..+|.+|+.++...|..+..      .-..|.|.|                ..+-.....++.++.+
</pre><pre>T Consensus        11 ~s~~~~~s~Fe~ER~~Li~eIa~~le~vl~------nln~LNRnL----------------E~v~~VG~eF~~va~L   65 (94)
</pre><pre>T 8Q84_X           11 KLISTTDKYFIEQRNIVLQEINETMNSILN------GLNGLNISL----------------ESSIAVGREFQSVSDL   65 (94)
</pre><pre>T ss_dssp             --CCHHHHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHH----------------HHHHHHHTTHHHHHHH
</pre><pre>T ss_pred             cCCcCCcHHHHHHHHHHHHHHHHHHHHHHH------HHHHhcHHH----------------HHHHHhccchHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>3WXX_H AopB; Translocator, Membrane protein-Chaperone complex; HET: MSE; 2.7A {Aeromonas hydrophila}
</pre><pre>Probab=63.53  E-value=76  Score=27.30  Aligned_cols=109  Identities=10%  Similarity=0.008  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC---ccchHHHHhCCCcc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE---EYNSIKKALGLSTS   85 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~---~~~~lk~~LglStk   85 (174)
</pre><pre>                      ..+.+....+...+..+-+.+-... + ...+..++|...-..|..+|+....+..+......   ....+++..     
</pre><pre>T Consensus        26 ~~~~~~~~~~~~~~~~~~~~~p~L~-~-~~~~s~~~l~~~L~~l~~kl~~~q~~~~~~~I~~~~~~~~~~~~e~~-----   98 (219)
</pre><pre>T 3WXX_H           26 DQLRKTAQLGTANAAKLLGSSTLLN-K-LAFASPEEFEIELSKMTSELEQTQKKLKLADLERIRAENLKKIDENQ-----   98 (219)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHTSSSCCCS-S-CSSSCCCCCCC----------------CHHHHHHHHHHTHHHHHHHH-----
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhcccccc-c-ccCCCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-----
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHH
</pre><pre>Q FD01845382_026   86 SIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNV  135 (174)
</pre><pre>Q Consensus        86 s~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I  135 (174)
</pre><pre>                                 ..|+...+..++..+....-|+.||++.++....+...|
</pre><pre>T Consensus        99 -----------~Ki~Ea~eka~kAkk~g~~~KIfgwI~a~~svV~ga~~i  137 (219)
</pre><pre>T 3WXX_H           99 -----------TKMKEASEAADKAKKSGLASKIFGWISAIASMVIGAILI  137 (219)
</pre><pre>T ss_dssp             -----------HHHHHHHHHHHHHHHSSCHHHHSTTSCHHHHHHHHHHHH
</pre><pre>T ss_pred             -----------HHHHHHHHHHHHHHHccHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>6CFZ_F Dad1,Dad1; DASH, Dam1 complex, Ask1, Dad1, Dad2, Dad3, Dad4, Dam1, Duo1, Hsk3, Spc19, Spc34, kinetochore, kinetochore-microtubule interface;{Chaetomium thermophilum}
</pre><pre>Probab=63.41  E-value=29  Score=25.72  Aligned_cols=48  Identities=15%  Similarity=0.234  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   43 EEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKL  112 (174)
</pre><pre>Q Consensus        43 ~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~  112 (174)
</pre><pre>                      ..|..+|.+|+.++...|..+..      .-..|.+.|                ..+......++.++.+
</pre><pre>T Consensus         2 s~Fe~eR~~Li~eI~~~le~vl~------nln~LNRnL----------------E~v~~VG~eF~~vs~L   49 (92)
</pre><pre>T 6CFZ_F            2 SYFEQQRQALIEEIAMNFEHVLA------NINKLNRSL----------------EAVIAVGNEFSSVEAL   49 (92)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHH----------------HHHHHHHCCCHHHHHH
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHHH------HHHHHhhhH----------------HHHHHhhhcchhHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>6DLC_A Designed protein DHD1:234_A; Computational Design, Heterodimer, Coiled-coil, DE NOVO PROTEIN; 3.261A {synthetic construct}
</pre><pre>Probab=63.30  E-value=43  Score=24.90  Aligned_cols=45  Identities=22%  Similarity=0.348  Sum_probs=0.0  Template_Neff=1.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchh
</pre><pre>Q FD01845382_026   43 EEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSI   87 (174)
</pre><pre>Q Consensus        43 ~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~   87 (174)
</pre><pre>                      ..|...-++-+.+||..+.++-.--...|....+|..+.+|.+|+
</pre><pre>T Consensus        11 ktflenlrrhldrldkhikqlrdilsenpedervkd~idlsersv   55 (115)
</pre><pre>T 6DLC_A           11 KTFLENLRRHLDRLDKHIKQLRDILSENPEDERVKDAIDLSERSV   55 (115)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHCTTCHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhhCCCchHHHHHhhhhHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>7QWB_D CC-Type2-(Ue)4; aib, coiled coil, hexamer, antiparallel, 2-Aminoisobutyric acid, DE NOVO PROTEIN; HET: AIB, 4BF; 1.64A {N/A}
</pre><pre>Probab=57.61  E-value=49  Score=19.18  Aligned_cols=21  Identities=33%  Similarity=0.272  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKN   31 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~   31 (174)
</pre><pre>                      ..+.+++|.+.|++|..+.+.
</pre><pre>T Consensus         4 i~qalkei~kalkei~~alke   24 (32)
</pre><pre>T 7QWB_D            4 IXQALKEIXKALKEIXXALKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF09712.14 ; PHA_synth_III_E ; Poly(R)-hydroxyalkanoic acid synthase subunit (PHA_synth_III_E)
</pre><pre>Probab=55.84  E-value=2e+02  Score=25.38  Aligned_cols=76  Identities=12%  Similarity=0.191  Sum_probs=0.0  Template_Neff=8.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccccCcc
</pre><pre>Q FD01845382_026   18 INKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDKTGVD   97 (174)
</pre><pre>Q Consensus        18 i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~~~~g   97 (174)
</pre><pre>                      +..-++.+++.|...+.       +++|-.--.++.+.+-..-..+-+      ....+-+.++|.++           .
</pre><pre>T Consensus       223 y~~W~~~~e~~~~~~~~-------s~~f~~~~~~~~~a~~~~k~~~~~------~~e~~l~~~~lPtr-----------~  278 (298)
</pre><pre>T L0GYM6_9GAMM/1  223 YDAWIGCCEAVYAEEVS-------SSEYARLHGRLVNAQMALKQRMSI------VVDEVLGAMNMPTR-----------S  278 (298)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhC-------CHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHhCCCCCH-----------H
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   98 NIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        98 ~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      +|+.+.++|..+.+-.+.|+
</pre><pre>T Consensus       279 e~d~l~~~l~~L~r~v~~L~  298 (298)
</pre><pre>T L0GYM6_9GAMM/1  279 ELRTLQDRLQESRRESKRQR  298 (298)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>7JJ8_D Zinc-binding lipoprotein AdcA; AdcA, ZnuA, SBP, ATP-binding cassette transporter, Zn acquisition, metal binding protein; 2.03A {Streptococcus pneumoniae (strain ATCC BAA-255 / R6)} SCOP: c.92.2.0
</pre><pre>Probab=55.54  E-value=1.3e+02  Score=25.97  Aligned_cols=119  Identities=13%  Similarity=0.059  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     -+...|..+-..|.+.++..+..+..+.+-.-  .+.-+-+.+|+....
</pre><pre>T Consensus       120 ~~~~~~a~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~l~~~~~~~v~~~h~~~~yf~~~~gl~~~~  194 (299)
</pre><pre>T 7JJ8_D          120 VRAIKLVEHIRDSLSADYPDKKETFE-----KNAAAYIEKLQSLDKAYAEGLSQAKQKSFVTQHAAFNYLALDYGLKQVA  194 (299)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHTTCSCCEEEESSCCCHHHHHHHTCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHH-----HHHHHHHHHHHHHHHHHHHHhHhhCCCeEEeechHHHHHHHHhCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      ++        ..-++.....+.+.++.+.++..+.-.|-.+.......+..-..
</pre><pre>T Consensus       195 ~~--------~~~~~~~~s~~~l~~~~~~~~~~~v~~i~~~~~~~~~~~~~l~~  240 (299)
</pre><pre>T 7JJ8_D          195 IS--------GLSPDAEPSAARLAELTEYVKKNKIAYIYFEENASQALANTLSK  240 (299)
</pre><pre>T ss_dssp             CC--------SCCTTCCCCHHHHHHHHHHHHHHTCCEEEESSCCCHHHHHHCCC
</pre><pre>T ss_pred             cc--------cCCCCCCCCHHHHHHHHHHHHHcCCCEEEECCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>2F6M_D Vacuolar protein sorting-associated protein VPS28; endosomes, trafficking complex, Vps23, Vps28, Vacuole Protein Sorting, ESCRT protein complexes, Endosomal Sorting Complex Required for; HET: DDQ; 2.1A {Saccharomyces cerevisiae} SCOP: a.2.17.2
</pre><pre>Probab=53.25  E-value=1.5e+02  Score=22.49  Aligned_cols=68  Identities=15%  Similarity=0.290  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH--HhHhcCCCccchHHHHhCCCcchhhhc
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF--SKTHLNLEEYNSIKKALGLSTSSIMHQ   90 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~--~~~~~~~~~~~~lk~~LglStks~vh~   90 (174)
</pre><pre>                      .|-.+++.||++|.      .+.+..++|..+-.+|+.+.+..+..+  -...-.+|.....-+..++.-..-+|.
</pre><pre>T Consensus        36 sii~tle~LEka~~------rd~I~~~eY~~~C~~Li~qyk~~~~~l~~~~~~~~~~~i~~F~~~y~l~cp~A~~R  105 (109)
</pre><pre>T 2F6M_D           36 SIVITLDHVEKAYL------KDSIDDTQYTNTVDKLLKQFKVYLNSQNKEEINKHFQSIEAFADTYNITASNAITR  105 (109)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHH------TTSSCHHHHHHHHHHHHHHHHHHHTSTTHHHHHHHSCSSTHHHHHTTCCCHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHh------ccCCCHHHhHHHHHHHHHHHHHHHHHCChhHHHHhcCCHHHHHHHhCCCChHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>7R5K_W0 Nuclear pore complex protein Nup88; multiprotein complex, nucleocytoplasmic transport, TRANSPORT PROTEIN; 12.0A {Homo sapiens}
</pre><pre>Probab=50.01  E-value=3.6e+02  Score=27.12  Aligned_cols=87  Identities=14%  Similarity=0.143  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCCCcchhhhcc
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGLSTSSIMHQW   91 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~LglStks~vh~w   91 (174)
</pre><pre>                      +.++++.+.++++.+.+.....+      .++...+-+.|.++++..+.++.+....+. ....+++.|    +.+-   
</pre><pre>T Consensus       603 ~~l~~l~~~~~~l~~~~~~l~~r------~e~~~~kq~~L~~Rl~~vl~~l~~~~p~LS~aE~~~~~eL----~~i~---  669 (741)
</pre><pre>T 7R5K_W0         603 EDLSYCREERKSLREMAERLADK------YEEAKEKQEDIMNRMKKLLHSFHSELPVLSDSERDMKKEL----QLIP---  669 (741)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHHhcCCCCCHHHHHHHHHH----HHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   92 DKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        92 ~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                           ..+.+|..+++++..-.++.+
</pre><pre>T Consensus       670 -----~~l~~~~~~l~~l~~~~~~~~  690 (741)
</pre><pre>T 7R5K_W0         670 -----DQLRHLGNAIKQVTMKKDYQQ  690 (741)
</pre><pre>T ss_pred             -----HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>7BUL_A General transcription factor IIH subunit 1; DNA repair factor, general transcription factor, nucleotide excision repair, NUCLEAR PROTEIN; NMR {Homo sapiens}
</pre><pre>Probab=49.16  E-value=80  Score=25.40  Aligned_cols=41  Identities=22%  Similarity=0.272  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKK   55 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~   55 (174)
</pre><pre>                      ..-..+-..-.+|.++|++...  .|.+..++|-..|..|+.+
</pre><pre>T Consensus       115 ~~r~~lL~~~~~L~~l~~~LV~--~g~lse~EFW~~r~~ll~~  155 (161)
</pre><pre>T 7BUL_A          115 EEKNRMLQEDPVLFQLYKDLVV--SQVISAEEFWANRLNVNAT  155 (161)
</pre><pre>T ss_dssp             HHHHHHHHHCHHHHHHCCCCCT--TTSSCHHHHHHHHHTCCCH
</pre><pre>T ss_pred             HHHHHHHhhCHHHHHHHHHHhh--cCCCCHHHHHhHHhHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>6VME_E Vacuolar protein sorting-associated protein 28 homolog; Endosomal transport, HIV release, macroautophagy, PROTEIN TRANSPORT; 2.19A {Homo sapiens} SCOP: a.2.17.0
</pre><pre>Probab=48.80  E-value=1.2e+02  Score=23.43  Aligned_cols=41  Identities=17%  Similarity=0.335  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF   63 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~   63 (174)
</pre><pre>                      .|-.+++.+|++|.      .+.+..++|..+-.+|++|.+..+..+
</pre><pre>T Consensus        44 sIi~tle~LEka~~------rd~I~~~eY~~~C~~Li~q~k~~~~~l   84 (122)
</pre><pre>T 6VME_E           44 AVVKTMQALEKAYI------KDCVSPSEYTAACSRLLVQYKAAFRQV   84 (122)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHH------TTCSCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHhhh------ccCCCHHHhhHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>7JJ9_A Zinc-binding lipoprotein AdcA; AdcA, SBP, ATP-binding cassette transporter, Zn acquisition, METAL BINDING PROTEIN; 1.58A {Streptococcus pneumoniae (strain ATCC BAA-255 / R6)}
</pre><pre>Probab=48.73  E-value=1.6e+02  Score=27.62  Aligned_cols=119  Identities=13%  Similarity=0.059  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     .+...|..+-..|.++++..+..+-.+.+-.-  ...-+-+.+||....
</pre><pre>T Consensus       120 ~~~~~~a~~Ia~~L~~~dP~~~~~y~-----~n~~~~~~~L~~l~~~~~~~~~~~~~~~vv~~h~a~~Yf~~~~gl~~~~  194 (475)
</pre><pre>T 7JJ9_A          120 VRAIKLVEHIRDSLSADYPDKKETFE-----KNAAAYIEKLQSLDKAYAEGLSQAKQKSFVTQHAAFNYLALDYGLKQVA  194 (475)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHTTCSCCEEEESSCCCHHHHHHHTCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHH-----HHHHHHHHHHHHHHHHHHHHHHhhcCCeEEEechHHHHHHHHhCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      ++        ..-|+-....+.+.++.+.++-.+.-.|-++.......+..-..
</pre><pre>T Consensus       195 ~~--------~~~~~~~~s~~~l~~l~~~i~~~~v~~i~~~~~~~~~~~~~la~  240 (475)
</pre><pre>T 7JJ9_A          195 IS--------GLSPDAEPSAARLAELTEYVKKNKIAYIYFEENASQALANTLSK  240 (475)
</pre><pre>T ss_dssp             CC--------SCCTTCCCCHHHHHHHHHHHHHHTCCEEEESSCCCHHHHHHHHH
</pre><pre>T ss_pred             ee--------cCCCCCCCCHHHHHHHHHHHHHCCCcEEEEcCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>6S7O_E Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1; N-glycosylation, Oligosaccharyltransferase, OSTA, TRANSFERASE; HET: NAG, KZE, MAN, EGY, BMA, KZB; 3.5A {Homo sapiens}
</pre><pre>Probab=47.62  E-value=4.2e+02  Score=25.93  Aligned_cols=87  Identities=5%  Similarity=-0.006  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCCCcchhhh
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGLSTSSIMH   89 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~LglStks~vh   89 (174)
</pre><pre>                      +-+..++....+.+|+++..+..++    ....+|..+|+.|-++|.....++....-.+. ..+.+.+.+         
</pre><pre>T Consensus       482 ~~~~yeek~~l~~~l~~l~~~~~~~----ki~~~y~~~~~~l~~~l~~l~~~l~~~~~~l~~~~~~~~~~i---------  548 (607)
</pre><pre>T 6S7O_E          482 VLTLVNKRIGLYRHFDETVNRYKQS----RDISTLNSGKKSLETEHKALTSEIALLQSRLKTEGSDLCDRV---------  548 (607)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHCSSCCT----TTHHHHHHHHHHHHHHHHHHHHHHHHHTSSSCCSHHHHHHHH---------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHhcc----CCHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccchHHHHH---------
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   90 QWDKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        90 ~w~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                             ..|......++.+.+-..-+.
</pre><pre>T Consensus       549 -------~~le~~e~e~~~~~~~l~~~~  569 (607)
</pre><pre>T 6S7O_E          549 -------SEMQKLDAQVKELVLKSAVEA  569 (607)
</pre><pre>T ss_dssp             -------HHHHHHHHHHHHHHHHTTSSS
</pre><pre>T ss_pred             -------HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>2OTX_B Src kinase-associated phosphoprotein 2; NOVEL DIMERIZATION DOMAIN, PH DOMAIN, SIGNALING PROTEIN; 2.6A {Mus musculus} SCOP: b.55.1.1
</pre><pre>Probab=46.74  E-value=1.6e+02  Score=22.28  Aligned_cols=44  Identities=20%  Similarity=0.353  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNK   62 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~   62 (174)
</pre><pre>                      +++++.|.+|+.-..+.+.  ..++ ...-..+|+.|+++|+.....
</pre><pre>T Consensus         4 ~~~~~ll~~~~~fl~~~l~--~e~l-s~~~~~~~~~l~~~~~~~~~~   47 (211)
</pre><pre>T 2OTX_B            4 EEIRNLLADVETFVADTLK--GENL-SKKAKEKRESLIKKIKDVKSV   47 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHCCCCCCTT--TCBC-CHHHHHHHHHHHHHHHHHGGG
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhC--CCCc-cHHHHHHHHHHHHHHHhHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>2D4Y_B Flagellar hook-associated protein 1; Multi-domain protein, alpha-helical bundle, complex all-beta folds, STRUCTURAL PROTEIN; 2.1A {Salmonella typhimurium}
</pre><pre>Probab=46.69  E-value=1.5e+02  Score=27.92  Aligned_cols=55  Identities=20%  Similarity=0.232  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhcccc--CCCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    7 ISEVGKSYYEEINKILIEIQELYKNTYTYNK--GIISGEEFFGQRARLFKKLDTILN   61 (174)
</pre><pre>Q Consensus         7 ~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~--g~l~~~~F~~~R~~Lf~~Ld~~l~   61 (174)
</pre><pre>                      ....+...+++|...|++|.++=++......  .......++.||..|+.+|-..++
</pre><pre>T Consensus        94 ~~~~i~~~v~~iN~ll~~Ia~LN~~I~~~~~~~~g~~~ndL~DqRd~ll~~LS~li~  150 (463)
</pre><pre>T 2D4Y_B           94 VNIAIGSSVAQINNYAKQIANLNDQISRMTGVGAGASPNDLLDQRDQLVSELNKIVG  150 (463)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHC---------CHHHHHHHHHHHHHHHHSC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHhhhcCCCCCCcchHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>2CAZ_B VACUOLAR PROTEIN SORTING-ASSOCIATED PROTEIN VPS28; PROTEIN TRANSPORT, ESCRT, MVB, MULTIVESICULAR BODIES, ENDOSOME, LYSOSOME, PH DOMAIN, PROTEIN SORTING, VESICLE TRAFFICKING, UBIQUITIN, PHOSPHOINOSITIDE; 3.6A {SACCHAROMYCES CEREVISIAE} SCOP: a.2.17.2
</pre><pre>Probab=46.59  E-value=2.5e+02  Score=23.05  Aligned_cols=61  Identities=15%  Similarity=0.274  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH--HhHhcCCCccchHHHHhCCC
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF--SKTHLNLEEYNSIKKALGLS   83 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~--~~~~~~~~~~~~lk~~LglS   83 (174)
</pre><pre>                      .|-.+|+.||++|.      .+.+..++|..+-.+|+.+.+..+..+  -...-.+|.....-+..++.
</pre><pre>T Consensus        53 SII~tlE~LEkAyv------rd~I~~~eYt~~C~rLI~Qykt~~~~l~~~~~~~~f~sle~F~~~Y~ld  115 (155)
</pre><pre>T 2CAZ_B           53 SIVITLDHVEKAYL------KDSIDDTQYTNTVDKLLKQFKVYLNSQNKEEINKHFQSIEAFCDTYNIT  115 (155)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHT------TTSSCHHHHHHHHHHHHHHHHHHHTSSSCHHHHHHTCSHHHHHHHTTCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHh------ccCCCHHHhHHHHHHHHHHHHHHHHhcChhHHHHhcCCHHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>7FIK_Y Nucleoporin Nup88A; CR, NPC, nucleoporin, Nup358, Nup93, Y complex, STRUCTURAL PROTEIN; 3.7A {Xenopus laevis}
</pre><pre>Probab=46.11  E-value=4.5e+02  Score=26.31  Aligned_cols=87  Identities=16%  Similarity=0.146  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCCCcchhhhcc
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGLSTSSIMHQW   91 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~LglStks~vh~w   91 (174)
</pre><pre>                      ..++++.+.++++.+.+.....+      .++...+-..|.++++..+.++.+....+. ....+++.|    +.+-   
</pre><pre>T Consensus       590 ~~l~~L~~~~~~l~~~~~~l~~r------~e~~~~kq~~L~~Rl~~vl~~l~~~~~~LS~aE~~~~~eL----~~~~---  656 (728)
</pre><pre>T 7FIK_Y          590 EDLRYCREERKSLTETAERLAEK------FEEAKEKQEDLINRLKRILRSFHTQLPVLSESERDMKKEL----QATN---  656 (728)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHHhcCCCCCHHHHHHHHHH----HHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   92 DKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        92 ~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                           ..+.+|..+++.+..-.++.+
</pre><pre>T Consensus       657 -----~~l~~l~~~~~~l~~~~~~~~  677 (728)
</pre><pre>T 7FIK_Y          657 -----QQLQQLGNSINQVNRKMSYQE  677 (728)
</pre><pre>T ss_dssp             --------------------------
</pre><pre>T ss_pred             -----HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>PF10168.13 ; Nup88 ; Nuclear pore component
</pre><pre>Probab=45.02  E-value=4.8e+02  Score=26.19  Aligned_cols=87  Identities=14%  Similarity=0.148  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCCCcchhhhcc
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGLSTSSIMHQW   91 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~LglStks~vh~w   91 (174)
</pre><pre>                      +.++++++.++++.+-+.....+      .++...+-+.|.++++..+.++.+....+. ...++++.|    +.+-   
</pre><pre>T Consensus       592 ~~l~~l~~~~~~l~~~~~~l~~r------~~~~~~kq~~L~~R~~~vl~~l~~~~p~LS~aE~~~~~eL----~~i~---  658 (729)
</pre><pre>T Q3SZ56_BOVIN/1  592 EDLNYCREERKSLREMAERLADK------YEEAKEKQEDIMNRMKKVLHSFHSQLPVLSDSERDMKKEL----QLIP---  658 (729)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHHhcCCCCCHHHHHHHHHH----HHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   92 DKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        92 ~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                           ..+.+|..+++.+..-.++.+
</pre><pre>T Consensus       659 -----~~l~~l~~~l~~l~~~~~~~~  679 (729)
</pre><pre>T Q3SZ56_BOVIN/1  659 -----DQLRHLGNAIKQVTMKKDYQQ  679 (729)
</pre><pre>T ss_pred             -----HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>7JRQ_A MPP1; metalloprotein, porphyrin, oxidation, sulfoxidation, high-talent, helical bundle, DE NOVO PROTEIN; HET: SMU; 1.75A {Escherichia coli}
</pre><pre>Probab=44.78  E-value=1.3e+02  Score=23.62  Aligned_cols=63  Identities=11%  Similarity=0.217  Sum_probs=0.0  Template_Neff=1.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCC-CCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHh
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGI-ISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKAL   80 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~-l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~L   80 (174)
</pre><pre>                      +.++++|.+|+++|+....- ... +.....-.+...+-.++-..|++|-. .+..++..++...|
</pre><pre>T Consensus        47 dsleQ~Leele~l~qkh~eL-~~~~~~k~~~~kEAa~~~de~vqlFqr~re-afdkGDdDS~Eq~L  110 (131)
</pre><pre>T 7JRQ_A           47 DSFEQVLEELEEALQKHRQL-ADQGRKKGLLTSEAAKQGDQFVQLFQRFRE-AWDKGDKDSLEQIL  110 (131)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHH-HHHHHHTTCCCCHHHHHHHHHHHHHHHHHH-HHHHTCHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHH-HHHHHHHhHhcHHHHHHHHHHHHHHHHHHH-HHHcCCcchHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>6YP5_B Phosphoprotein; Phosphoprotein, RNA dependent RNA polymerase cofactor, Tetramer, Coiled-coil, VIRAL PROTEIN; NMR {Respiratory syncytial virus}
</pre><pre>Probab=44.42  E-value=72  Score=20.26  Aligned_cols=30  Identities=23%  Similarity=0.348  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             CcchHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    1 GDGLGYISEVGKSYYEEINKILIEIQELYK   30 (174)
</pre><pre>Q Consensus         1 s~~~G~~~~~~~~~l~~i~~~L~~ie~ly~   30 (174)
</pre><pre>                      ..+.|..+..++.+++.|++.|..|--+.+
</pre><pre>T Consensus         3 gsgsgs~~~sieaRL~~IeEkLsmIlGlL~   32 (45)
</pre><pre>T 6YP5_B            3 GSGSGSTNDNITARLDRIDEKLSEILGMLH   32 (45)
</pre><pre>T ss_dssp             ----CCCHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCCccchhHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>5XLN_B Threonine--tRNA ligase, cytoplasmic; TRS, 4EHP, Complex, RNA BINDING PROTEIN-LIGASE complex; 1.9A {Homo sapiens}
</pre><pre>Probab=42.79  E-value=46  Score=20.55  Aligned_cols=17  Identities=18%  Similarity=0.356  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   43 EEFFGQRARLFKKLDTI   59 (174)
</pre><pre>Q Consensus        43 ~~F~~~R~~Lf~~Ld~~   59 (174)
</pre><pre>                      +.|..+|-+||.+|...
</pre><pre>T Consensus        22 P~fi~eRl~lfe~lk~~   38 (45)
</pre><pre>T 5XLN_B           22 PEYIYTRLEMYNILKAE   38 (45)
</pre><pre>T ss_dssp             CSCCCCHHHHHHHHHHH
</pre><pre>T ss_pred             cchHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>PF10023.13 ; Aminopep ; Putative aminopeptidase
</pre><pre>Probab=42.71  E-value=3.9e+02  Score=24.19  Aligned_cols=98  Identities=8%  Similarity=0.146  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHh-HhcCCCccchHHHHhCCCcchhhh
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSK-THLNLEEYNSIKKALGLSTSSIMH   89 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~-~~~~~~~~~~lk~~LglStks~vh   89 (174)
</pre><pre>                      ....-..+.+..++|+.+|.      ++ +..++-.++|+++|..+...+..+.. .....+.+..+.+.  +..-.+++
</pre><pre>T Consensus       211 ~~~f~~~l~~~~~~L~~lY~------~~-~~~~ekl~~K~~i~~~~r~~~~~~~~~~~~~~~~y~~~~~~--~NNA~l~~  281 (325)
</pre><pre>T I0HMF1_RUBGI/2  211 RRAFRALVGETREALRTLYA------SD-ADEATKRARKAELMAAMRARHAALKAGEWAGYAGYDAWFDN--ANNAALGV  281 (325)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhc------CC-CCHHHHHHHHHHHHHHHHHHHHHHhcccccccCcchhHHHh--cccceeHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cccccCcccchHHHHHHHHHH-HHHHHHhhh
</pre><pre>Q FD01845382_026   90 QWDKTGVDNIERYATYIENSA-KLMKLMKTV  119 (174)
</pre><pre>Q Consensus        90 ~w~~~~~g~Ip~~~~~i~~~~-k~sk~lk~g  119 (174)
</pre><pre>                      .....  ...|.+.+-+++.. .+.+.+...
</pre><pre>T Consensus       282 ~~~Y~--~~~~~~~~l~~~~g~dl~~fl~~l  310 (325)
</pre><pre>T I0HMF1_RUBGI/2  282 QAAYD--EWVPAFERLFEAEGADFKRFYAEV  310 (325)
</pre><pre>T ss_pred             Hhchh--cccHHHHHHHHHcCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>4HKU_B TetR transcriptional regulator; structural genomics, PSI-Biology, protein structure initiative, midwest center for structural genomics, MCSG, TRANSCRIPTION; HET: MSE; 2.302A {Listeria monocytogenes}
</pre><pre>Probab=42.36  E-value=60  Score=23.62  Aligned_cols=37  Identities=8%  Similarity=0.195  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+.+|+|..+++|++.
</pre><pre>T Consensus        13 l~a~~~~~~~~g~~~~s~~~ia~~~gvs~~~iy~~F~   49 (178)
</pre><pre>T 4HKU_B           13 LNMAEKIIYEKGMEKTTLYDIASNLNVTHAALYKHYR   49 (178)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCSTTTTTTCS
</pre><pre>T ss_pred             HHHHHHHHHHHCCccccHHHHHHHcCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>5C4Y_A Putative transcription regulator Lmo0852; Structural Genomics, PSI-Biology, Midwest Center for Structural Genomics, MCSG, Transcriptional factor, Listeria monocytogenes, TRANSCRIPTION; HET: MSE; 1.77A {Listeria monocytogenes serovar 1/2a (strain ATCC BAA-679 / EGD-e)}
</pre><pre>Probab=41.81  E-value=70  Score=22.90  Aligned_cols=38  Identities=21%  Similarity=0.361  Sum_probs=0.0  Template_Neff=13.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus         7 l~aa~~~~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   44 (184)
</pre><pre>T 5C4Y_A            7 KEVALTLFAEKGYDGTALSEIAKAVGIKTPSLYAHFAS   44 (184)
</pre><pre>T ss_dssp             --------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>6ZT1_G CC-Type2-(LaIdGe)4; coiled coil, de novo peptide, multi-state, DE NOVO PROTEIN; HET: 4BF, ACE; 2.01A {N/A}
</pre><pre>Probab=41.09  E-value=1.2e+02  Score=17.71  Aligned_cols=27  Identities=26%  Similarity=0.212  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYK   30 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~   30 (174)
</pre><pre>                      +|.+---..+.+++|...|++|-++.+
</pre><pre>T Consensus         4 igqalkeigkalkeig~alkeigqalk   30 (32)
</pre><pre>T 6ZT1_G            4 IGQALKEIGKALKEIGXALKEIGQALK   30 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>2KJG_A Archaeal protein SSO6904; hypothetical protein, helical protein, METAL BINDING PROTEIN; NMR {Sulfolobus solfataricus}
</pre><pre>Probab=40.95  E-value=1.9e+02  Score=22.05  Aligned_cols=66  Identities=18%  Similarity=0.266  Sum_probs=0.0  Template_Neff=2.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhccccC-----CCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCC
</pre><pre>Q FD01845382_026   15 YEEINKILIEIQELYKNTYTYNKG-----IISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGL   82 (174)
</pre><pre>Q Consensus        15 l~~i~~~L~~ie~ly~~~~~~~~g-----~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~Lgl   82 (174)
</pre><pre>                      .+.++++|++|+.-|...-.- ..     .-+.-+...+.+.+|.-+-..|.++.. .+|++ ...-+...|.+
</pre><pre>T Consensus        21 ~~~V~~IL~eie~d~~ks~~i-kssiIfvYanh~~~Vk~NKe~y~lI~~ILEKys~-KiGiENV~~LIlnsL~~   92 (99)
</pre><pre>T 2KJG_A           21 FNLVMQILDEIELDLRGSDNI-KTSIIYVYSSHLDEIRKNKEFYDMIAEILQRYYK-KIGIENVNQLILTTIKL   92 (99)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHSSCH-HHHHHHHHHHTHHHHHHTHHHHHHHHHHHHHHHT-TTCHHHHHHHHHHHHCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhcCCCc-cHHHeehhhHhHHHHHhCHHHHHHHHHHHHHHHH-hhChhHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>7QWE_H CC-Type2-(Ug)4; aib, coiled coil, pentamer, 2-Aminoisobutyric acid, DE NOVO PROTEIN; HET: AIB, 4BF; 1.34A {N/A}
</pre><pre>Probab=40.93  E-value=1.2e+02  Score=17.70  Aligned_cols=21  Identities=33%  Similarity=0.325  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKN   31 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~   31 (174)
</pre><pre>                      ..+.+++|.+.|++|....+.
</pre><pre>T Consensus         4 iaq~lkeiak~lkeia~~lke   24 (32)
</pre><pre>T 7QWE_H            4 IAQXLKEIAKXLKEIAXXLKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>PF03997.16 ; VPS28 ; VPS28 protein
</pre><pre>Probab=40.53  E-value=1.7e+02  Score=24.16  Aligned_cols=40  Identities=13%  Similarity=0.306  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhccccCCCCHHH-HHHHHHHHHHHHHHHH---HHH
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYTYNKGIISGEE-FFGQRARLFKKLDTIL---NKF   63 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~~~~g~l~~~~-F~~~R~~Lf~~Ld~~l---~~~   63 (174)
</pre><pre>                      .|-.+++.||++|.      .|.+. ++ |..+-++|+++.+..+   +.+
</pre><pre>T Consensus         6 aiI~tle~LE~a~~------~~~i~-~e~Y~~~c~~Li~q~k~~~~~~~~l   49 (183)
</pre><pre>T A4HQ33_LEIBR/1    6 AVIIAIEQIEKANR------RDLID-QDQYSSTVRRLLEKYKNTVAHLQSA   49 (183)
</pre><pre>T ss_pred             HHHHHHHHHHHHhh------cccCC-HHHhHHHHHHHHHHHHHHHHhHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>4ME9_A Transcriptional regulator, TetR family; TetR_N domain, PF00440 family, tetracyclin repressor-like, C-terminal domain fold, Structural Genomics, Joint Center for Structural Genomics; HET: MSE; 2.5A {Bacillus cereus}
</pre><pre>Probab=40.21  E-value=67  Score=23.19  Aligned_cols=38  Identities=11%  Similarity=0.237  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        11 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   48 (189)
</pre><pre>T 4ME9_A           11 FDAAMQLFAERGYDGTTIPMIAEKAKVGAGTIYRYFEN   48 (189)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCcccHHHHHHHhCCChHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>6YS8_D GldL; membrane protein complex ion-driven motor, MOTOR PROTEIN; 3.9A {Flavobacterium johnsoniae}
</pre><pre>Probab=40.16  E-value=2.4e+02  Score=24.01  Aligned_cols=78  Identities=10%  Similarity=0.009  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCC
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~Lgl   82 (174)
</pre><pre>                      +...+..+...++.+.+.|..|++.|..+.+  .-.-....+...... -+++..++.++.+.--.+. .|.++=.++..
</pre><pre>T Consensus       137 ~s~~t~~Y~eqm~~lnknLs~LN~~Ye~qlk--~~~~~~~~~~~~~~~-t~~~~ee~~~L~~nl~~LN~VYg~MLsAM~~  213 (215)
</pre><pre>T 6YS8_D          137 SIAGQKKYAEEMSMAAAQMESLNSLYKVQLE--SASRNAQANSEIAEN-AAKLKEQMASMTANIASLNSVYGGMLSAMSN  213 (215)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--HHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Cc
</pre><pre>Q FD01845382_026   83 ST   84 (174)
</pre><pre>Q Consensus        83 St   84 (174)
</pre><pre>                      ..
</pre><pre>T Consensus       214 ~~  215 (215)
</pre><pre>T 6YS8_D          214 KG  215 (215)
</pre><pre>T ss_dssp             --
</pre><pre>T ss_pred             CC
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>7WOO_K Nucleoporin NUP57; nuclear pore complex, inner ring, protomer, Saccharomyces cerevisiae, TRANSPORT PROTEIN; 3.71A {Saccharomyces cerevisiae}
</pre><pre>Probab=39.47  E-value=5.2e+02  Score=24.72  Aligned_cols=101  Identities=6%  Similarity=0.008  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHH---HHhHhcCCC-ccchHHHHhCCC
</pre><pre>Q FD01845382_026    8 SEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNK---FSKTHLNLE-EYNSIKKALGLS   83 (174)
</pre><pre>Q Consensus         8 ~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~---~~~~~~~~~-~~~~lk~~LglS   83 (174)
</pre><pre>                      .......++++.+.+++|.+.+.....     ....+++.+...|..+|...+.+   +-.+.+.+. ....|++.|   
</pre><pre>T Consensus       369 ~~~~~~~l~~l~~~~~~l~~~~~~~~~-----~~i~~~~~~~~~l~~rll~l~~~~~~l~~~g~~l~~~ee~L~~~l---  440 (541)
</pre><pre>T 7WOO_K          369 VAQARIILNHILEKSTQLQQKHELDTA-----SRILKAQSRNVEIEKRILKLGTQLATLKNRGLPLGIAEEKMWSQF---  440 (541)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHCCCCCST-----TTHHHHHHHHHHHHHHHHHHHHHHHHTSSCSSSCSTTHHHHHHHH---
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHHHHHHHcCCCCCHhHHHHHHHH---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cchhhhcccccCc-ccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   84 TSSIMHQWDKTGV-DNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        84 tks~vh~w~~~~~-g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                       ..+..+-+.... +.+..+..++..+..-.+.++
</pre><pre>T Consensus       441 -~~l~~~l~~p~~~~~~~el~a~l~~l~~~~~~l~  474 (541)
</pre><pre>T 7WOO_K          441 -QTLLQRSEDPAGLGKTNELWARLAILKERAKNIS  474 (541)
</pre><pre>T ss_dssp             -HHHHHTTSCSSSSSSSSTHHHHHHSHHHHHHHHH
</pre><pre>T ss_pred             -HHHHHHccCccchhhHHHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>4LUN_U Nonsense-mediated mRNA decay protein 2; HEAT-repeat, mRNA decay, RNA BINDING PROTEIN; 1.641A {Saccharomyces cerevisiae}
</pre><pre>Probab=39.35  E-value=2.1e+02  Score=25.72  Aligned_cols=70  Identities=13%  Similarity=0.153  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhh
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIM   88 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~v   88 (174)
</pre><pre>                      ..+..++.+..+.|.++++..+..+..      ..+...+|+.-+.++...+.++..      ....|.+.||...-.+.
</pre><pre>T Consensus       244 ~~~~~~l~~~~~~l~~~e~~n~~~~~~------~Gev~ee~~~~~e~~~~~~ekL~~------~~~~Lae~L~~~~P~~~  311 (316)
</pre><pre>T 4LUN_U          244 DATFARATELHKKVNKLQREHQKCQIR------TGKLRDEYVEEYDKLLPIFIRFKT------SAITLGEFFKLEIPELH  311 (316)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHH------HCCSCCHHHHHHHHHHHHHHHHHH------HHHHHHHHHTCCCCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH------HCCCCHHHHHHHHHHHHHHHHHHH------HHHHHHHHcCCCccccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hc
</pre><pre>Q FD01845382_026   89 HQ   90 (174)
</pre><pre>Q Consensus        89 h~   90 (174)
</pre><pre>                      ||
</pre><pre>T Consensus       312 ~~  313 (316)
</pre><pre>T 4LUN_U          312 HH  313 (316)
</pre><pre>T ss_dssp             CC
</pre><pre>T ss_pred             CC
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>1R8J_A KaiA; KaiA, Circadian Clock Protein; 2.03A {Synechococcus elongatus PCC 7942} SCOP: a.186.1.1, l.1.1.1, c.23.1.5
</pre><pre>Probab=39.07  E-value=3.9e+02  Score=23.22  Aligned_cols=98  Identities=10%  Similarity=0.155  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccC---CCCHHHH-----HHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchh
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKG---IISGEEF-----FGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSI   87 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g---~l~~~~F-----~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~   87 (174)
</pre><pre>                      ..+...-..+.+.+++.+.- -|   +.+...|     ..+|+.|+++|+.....+...-|+-+.  +|+..+    ..+
</pre><pre>T Consensus       152 ~~L~~qq~rL~~kL~erLgy-lGv~ykr~~~~f~r~l~~~er~ell~~lk~~y~~i~l~yf~~~~--~ln~~I----d~~  224 (289)
</pre><pre>T 1R8J_A          152 PELSSQQRDLAQRLQERLGY-LGVYYKRDPDRFLRNLPAYESQKLHQAMQTSYREIVLSYFSPNS--NLNQSI----DNF  224 (289)
</pre><pre>T ss_dssp             TTGGGHHHHHHHHHHHHHEE-EEEEECCCGGGSTTTSCHHHHHHHHHHHHHHHHHHHHHHTSTTC--CHHHHH----HHH
</pre><pre>T ss_pred             hhhhHHHHhHHHHHHHHhcc-cccccCCChhhchhcCCHHHHHHHHHHHHHHHHHHHHHhhCCCC--chhHHH----HHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhcccccCcccchHHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   88 MHQWDKTGVDNIERYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        88 vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      |...-++....=.-+.=||+-+-.++|.||.=|
</pre><pre>T Consensus       225 ~~~~~~~~~~~~~~~~~h~~~~~~~~~~l~~~~  257 (289)
</pre><pre>T 1R8J_A          225 VNMAFFADVPVTKVVEIHMELMDEFAKKLRVEG  257 (289)
</pre><pre>T ss_dssp             HHHHHHTTCCHHHHHHHHHHHHHHHHHHHHTTT
</pre><pre>T ss_pred             HHHHHhCCCCHHHHHHHHHHHHHHHHHHhccCC
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>8SNB_6R RIB35; sperm, doublet microtubule, DMT, microtubule inner protein, cilia, flagella, FAP, CFAP, MIP, MAP, cytoskeleton, tublin, sea; HET: GDP, MG, GTP; 3.3A {Strongylocentrotus purpuratus}
</pre><pre>Probab=39.04  E-value=64  Score=28.78  Aligned_cols=23  Identities=22%  Similarity=0.163  Sum_probs=0.0  Template_Neff=2.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTY   33 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~   33 (174)
</pre><pre>                      ..+|+++|+.+|++||.-|++..
</pre><pre>T Consensus       111 ~~qH~~dI~svleeLe~~yk~tv  133 (310)
</pre><pre>T 8SNB_6R         111 LLQHVEDIRTSLEELETYHQQTL  133 (310)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHccc
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>7ACM_B HTH-type transcriptional regulator RcdA; transcription factor, HTH motif, DNA-binding protein, GENE REGULATION; HET: TMO; 1.763A {Escherichia coli}
</pre><pre>Probab=39.04  E-value=1e+02  Score=22.15  Aligned_cols=44  Identities=9%  Similarity=0.174  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ..|++++...-..+..-.-....+.   .|-+..|+|..+++|+|..
</pre><pre>T Consensus         8 ~~~~~i~~aa~~l~~~~g~~~~t~~---~ia~~agvs~~~~y~~f~~   51 (178)
</pre><pre>T 7ACM_B            8 QRREKIIQATLEAVKLYGIHAVTHR---KIATLAGVPLGSMTYYFSG   51 (178)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTSCGGGGGGTSSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCHhhccHH---HHHHHHCCChHHHHhhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>5XSJ_L Signal transduction histidine kinase, LytS; Two component system, histidine kinase, signal transmission across the membrane, D-xylose uptake, SUGAR BINDING PROTEIN; HET: XYP; 2.202A {Clostridium beijerinckii (strain ATCC 51743 / NCIMB 8052)}
</pre><pre>Probab=38.91  E-value=2.8e+02  Score=21.38  Aligned_cols=97  Identities=9%  Similarity=0.020  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   14 YYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        14 ~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      .++++...++++.......... ...-...+|...++.+-..++..-.. ......+ ...+|+.++    .++....+.
</pre><pre>T Consensus        21 ~ln~~~~~~~~~~~~l~~Yl~~-~s~~~l~~y~~~~~~l~~~~~~L~~~-~~~e~~~-~~~~i~nm~----~sy~~~~~~   93 (148)
</pre><pre>T 5XSJ_L           21 ITNEIKQHVDSSLDNFNQYILN-GTPSKKESYNNEVILAKQKIGNLKKN-SDDVNQY-ILRDLDNTL----DSYIESSKN   93 (148)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHH-CCHHHHHHHHHHHHHHHHHHHHHHHT-CCTTHHH-HHHHHHHHH----HHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHc-CChhHHHHHHHHHHHHHHHHHhhhcC-CCchhHH-HHHHHHHHH----HHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             cCc----ccc-hHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   94 TGV----DNI-ERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        94 ~~~----g~I-p~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      +-.    +.+ +.|...+..+.++..++.
</pre><pre>T Consensus        94 ~i~~~~~~~~~~~y~~~y~e~~~i~~yI~  122 (148)
</pre><pre>T 5XSJ_L           94 TISAYENKEGYVFYYDDFVAAKNIASYCD  122 (148)
</pre><pre>T ss_dssp             HHHHHHTTCCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHcCCchhhHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>3GEU_D Intercellular adhesion protein R; TetR family, intercellular adhesion regulator, IDP00851, DNA-binding, Repressor, Transcription, Transcription regulation, Structural Genomics, CSGID, Center for; HET: FMT, MSE; 1.9A {Staphylococcus aureus}
</pre><pre>Probab=38.72  E-value=83  Score=22.79  Aligned_cols=38  Identities=18%  Similarity=0.389  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+.+.++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus         9 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   46 (189)
</pre><pre>T 3GEU_D            9 IDNAITLFSEKGYDGTTLDDIAKSVNIKKASLYYHFDS   46 (189)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCGGGGGGTCSC
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHcCCChHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>2EH3_A Transcriptional regulator; All alpha proteins, Tetracyclin repressor-like, C-terminal domain, Homeodomain-like, DNA/RNA-binding 3-helical bundle, Helix-turn-helix, Structural Genomics, NPPSFA, National; 1.55A {Aquifex aeolicus}
</pre><pre>Probab=38.62  E-value=86  Score=22.46  Aligned_cols=37  Identities=16%  Similarity=0.182  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++||+.
</pre><pre>T Consensus         8 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   44 (179)
</pre><pre>T 2EH3_A            8 LEVSKELFFEKGYQGTSVEEIVKRANLSKGAFYFHFK   44 (179)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCChHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>1RKT_A protein yfiR; transcription regulator, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, transcription; HET: UNX, MSE; 1.95A {Bacillus subtilis} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=38.12  E-value=1.2e+02  Score=22.29  Aligned_cols=54  Identities=9%  Similarity=0.159  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   36 NKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        36 ~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      +......+.-...|++++...-..+..-.-....+.   .|-+..|+|..+++||+.
</pre><pre>T Consensus         1 m~~~~~~~~~~~~~~~il~aa~~l~~~~g~~~~t~~---~ia~~agvs~~~~y~~F~   54 (205)
</pre><pre>T 1RKT_A            1 MSPKVTKEHKDKRQAEILEAAKTVFKRKGFELTTMK---DVVEESGFSRGGVYLYFS   54 (205)
</pre><pre>T ss_dssp             -CCTTHHHHHHHHHHHHHHHHHHHHHHHCSTTCCHH---HHHHHHTSCHHHHHTTCS
</pre><pre>T ss_pred             CCcccChhHHHHHHHHHHHHHHHHHHHhCcccCCHH---HHHHHHCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>6MJ1_A Probable HTH-type transcriptional regulator YttP; TetR family, cell division, FtsZ regulator, DNA BINDING PROTEIN; 2.6A {Bacillus subtilis (strain 168)}
</pre><pre>Probab=37.87  E-value=81  Score=23.33  Aligned_cols=37  Identities=11%  Similarity=0.209  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|.++++|++.
</pre><pre>T Consensus        10 l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   46 (207)
</pre><pre>T 6MJ1_A           10 IESAVMLFNQKGFSGTSVREIAKSADVNVAHISYYFK   46 (207)
</pre><pre>T ss_dssp             HHHHHHHHHHSCTTTCCHHHHHHHHTSCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHcCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>4ACI_A HTH-TYPE TRANSCRIPTIONAL REPRESSOR ACNR; TRANSCRIPTION, ACONITASE, CITRATE, TETR SUPERFAMILY; HET: CIT, GOL; 1.65A {CORYNEBACTERIUM GLUTAMICUM}
</pre><pre>Probab=37.54  E-value=78  Score=23.05  Aligned_cols=37  Identities=19%  Similarity=0.307  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++||+.
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~s~~~ia~~ag~s~~~~y~~f~   56 (191)
</pre><pre>T 4ACI_A           20 LEGARRCFAEHGYEGATVRRLEEATGKSRGAIFHHFG   56 (191)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHhCCcchhHHHHHHHHCCChhHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>2NX4_C Transcriptional regulator, TetR family protein; tetR, HTH DNA binding motif, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; 1.7A {Rhodococcus sp. RHA1}
</pre><pre>Probab=37.48  E-value=81  Score=23.03  Aligned_cols=52  Identities=12%  Similarity=0.155  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   38 GIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        38 g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |......-...|++++...-..+..-.-....+.   .|-+..|+|.++++||+.
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~~i~~a~~~l~~~~g~~~~s~~---~ia~~ag~s~~~~y~~f~   52 (194)
</pre><pre>T 2NX4_C            1 GVPKLVDHDERRRSITAAAWRLIAARGIEAANMR---DIATEAGYTNGALSHYFA   52 (194)
</pre><pre>T ss_dssp             -------CCHHHHHHHHHHHHHHHHHCTTTCCHH---HHHHHHTCCHHHHHTTCS
</pre><pre>T ss_pred             CCCCCCCHHHHHHHHHHHHHHHHHHHCcccccHH---HHHHHhCCCHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>2ZCM_B Biofilm operon icaABCD HTH-type negative transcriptional regulator icaR; helix-turn-helix, biofilm, TetR family, repressor, polysaccharide intercellular adhesin (PIA), DNA-binding, Transcription, Transcription regulation; HET: MSE; 1.33A {Staphylococcus epidermidis}
</pre><pre>Probab=37.46  E-value=82  Score=23.42  Aligned_cols=38  Identities=18%  Similarity=0.404  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        13 ~~~a~~l~~~~g~~~~t~~~Ia~~agvs~~t~y~~F~~   50 (192)
</pre><pre>T 2ZCM_B           13 IDNAITLFSEKGYDGTTLDDISKSVNIKKASLYYHYDN   50 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHTTCCGGGGGGTCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>2HYJ_A Putative tetR-family transcriptional regulator; tetR, HTH DNA binding motif, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; 2.19A {Streptomyces coelicolor} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=37.45  E-value=1.1e+02  Score=22.54  Aligned_cols=54  Identities=15%  Similarity=0.205  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   36 NKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        36 ~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      +......+.....|+++++..-..+..-.-..+.++   .|-+..|+|..+++|+|.
</pre><pre>T Consensus         1 ~~~~~~~~~~~~~r~~i~~aa~~l~~~~g~~~~t~~---~ia~~agvs~~~~y~~f~   54 (200)
</pre><pre>T 2HYJ_A            1 MSPRRSAAEAQATRGRILGRAAEIASEEGLDGITIG---RLAEELEMSKSGVHKHFG   54 (200)
</pre><pre>T ss_dssp             -------CTHHHHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTCCHHHHHTTCS
</pre><pre>T ss_pred             CCCCcchHHHHHHHHHHHHHHHHHHHHcCCccccHH---HHHHHhCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>6AZH_B TetR family transcriptional regulator; Regulator, TRANSCRIPTION; HET: MSE; 1.75A {Clostridium perfringens}
</pre><pre>Probab=37.45  E-value=90  Score=22.50  Aligned_cols=38  Identities=11%  Similarity=0.275  Sum_probs=0.0  Template_Neff=13.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        12 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   49 (192)
</pre><pre>T 6AZH_B           12 FEAAINVFATSGYNGSTVDEIASKANVAKGTLYYNFKS   49 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCChHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>3HE0_A Transcriptional regulator, TetR family; TetR, AcrR, transcriptional regulator, Vibrio parahaemolyticus, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural; HET: SO4, MSE, GOL; 2.2A {Vibrio parahaemolyticus}
</pre><pre>Probab=37.19  E-value=86  Score=22.78  Aligned_cols=37  Identities=8%  Similarity=0.180  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        17 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   53 (196)
</pre><pre>T 3HE0_A           17 LAAAEQLIAESGFQGLSMQKLANEAGVAAGTIYRYFS   53 (196)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCccCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>2Q24_B Putative tetR family transcriptional regulator; TRANSCRIPTIONAL REGULATOR, STRUCTURAL GENOMICS, PSI, PROTEIN STRUCTURE INITIATIVE, MIDWEST CENTER FOR STRUCTURAL GENOMICS, MCSG, TRANSCRIPTION; HET: MSE; 1.8A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=37.19  E-value=1.1e+02  Score=22.40  Aligned_cols=44  Identities=11%  Similarity=0.262  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   46 FGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        46 ~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ...|++++...-..+..-.-... +.   .|-+..|+|..+++|++..
</pre><pre>T Consensus        14 ~~~r~~i~~aa~~l~~~~g~~~~-~~---~ia~~agvs~~~~y~~f~s   57 (194)
</pre><pre>T 2Q24_B           14 QRNRDKILAAAVRVFSEEGLDAH-LE---RIAREAGVGSGTLYRNFPT   57 (194)
</pre><pre>T ss_dssp             --CHHHHHHHHHHHHHHHGGGCC-HH---HHHHHTTCCHHHHHHHCCS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhcHHHH-HH---HHHHHHCCChHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>3LJL_A transcriptional regulator LuxT; structural genomics, PSI-2, protein structure initiative, midwest center for structural genomics, MCSG, DNA-binding, transcription regulator; HET: MES, MSE; 3.2A {Vibrio parahaemolyticus}
</pre><pre>Probab=36.90  E-value=81  Score=22.77  Aligned_cols=37  Identities=16%  Similarity=0.475  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+.+|++..+++||+.
</pre><pre>T Consensus        20 ~~a~~~~~~~~g~~~~t~~~ia~~~gv~~~~~y~~f~   56 (156)
</pre><pre>T 3LJL_A           20 MDAVVDQLLRLGYDKMSYTTLSQQTGVSRTGISHHFP   56 (156)
</pre><pre>T ss_dssp             HHHHHHHHHHTHHHHCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCCccccHHHHHHHHCCCchHHhhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>2FD5_A transcriptional regulator; Transcriptional regulator, DNA-binding protein, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; 1.7A {Pseudomonas aeruginosa} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=36.87  E-value=85  Score=22.88  Aligned_cols=38  Identities=11%  Similarity=0.142  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        13 ~~aa~~~~~~~G~~~~t~~~ia~~agvs~~~~y~~F~~   50 (180)
</pre><pre>T 2FD5_A           13 LGAATQALLERGAVEPSVGEVMGAAGLTVGGFYAHFQS   50 (180)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTTSCCHHHHHHHTTCCGGGGGGTCSC
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCCHHHHHHHhCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>7SAZ_C GldL; type IX secretion system, MOTOR PROTEIN; 3.0A {Capnocytophaga canimorsus (strain 5)}
</pre><pre>Probab=36.78  E-value=2.8e+02  Score=24.13  Aligned_cols=76  Identities=14%  Similarity=0.101  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccchHHHHhCC
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~lk~~Lgl   82 (174)
</pre><pre>                      +...+..+...++.+.+.|..|++.|..+..  .-.-..+.+....+. -+++..++.++.+.--.+. .|.++=.+++.
</pre><pre>T Consensus       152 ~~~~t~~y~eq~~~l~~nL~~lN~~Ye~~l~--~~~~~~~~~~~~~~~-~~~~~~e~~~L~~nl~~LN~vYg~ML~AM~~  228 (228)
</pre><pre>T 7SAZ_C          152 AMVSTHKYGEELSMAAAHLESLNSLYKLQLE--RTENQVSAQAGVVDN-LNSLNEQMMSFKDNLKSLNSVYGGMLSAMGK  228 (228)
</pre><pre>T ss_dssp             --------------------------------------------------------------------------------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH--HHHHHHHHHHHHHHH-HHHHHHHHHHHHHHHHHHHHHHHHHHHHccC
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>2PRS_A High-affinity zinc uptake system protein znuA; protein consists of two (beta/alfa)4 domains, METAL TRANSPORT; 1.7A {Escherichia coli} SCOP: c.92.2.0
</pre><pre>Probab=36.56  E-value=2.6e+02  Score=23.87  Aligned_cols=116  Identities=11%  Similarity=0.061  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH--hcCCCccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT--HLNLEEYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~--~~~~~~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     -+...|..+-+.|..++...+..+-.+  -..-|.+.-+-+.+||..-.
</pre><pre>T Consensus       123 ~~~~~~~~~Ia~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~~~~~~~~~y~~~~~gl~~~~  197 (284)
</pre><pre>T 2PRS_A          123 EIARATAVAIHGKLVELMPQSRAKLD-----ANLKDFEAQLASTETQVGNELAPLKGKGYFVFHDAYGYFEKQFGLTPLG  197 (284)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHGGGTTCCEEEEESCCHHHHHHHTCCCCE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHH-----HHHHHHHHHHHHHHHHHHHhccccCCCeeEEeehHHHHHHHHhCCCcce
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYD  137 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~  137 (174)
</pre><pre>                      ++        ..-++-...-+.+.++.+.++..+.-.|-.+.......+..
</pre><pre>T Consensus       198 ~~--------~~~~~~~~s~~~l~~l~~~~~~~~v~~i~~~~~~~~~~~~~  240 (284)
</pre><pre>T 2PRS_A          198 HF--------TVNPEIQPGAQRLHEIRTQLVEQKATCVFAEPQFRPAVVES  240 (284)
</pre><pre>T ss_dssp             EE--------ESSTTSCCCHHHHHHHHHHHHHTTCCEEEECTTSCSHHHHH
</pre><pre>T ss_pred             ee--------ecCCCCCCCHHHHHHHHHHHHHCCCcEEEECCCCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>3BJB_A Probable transcriptional regulator, TetR family protein; APC7331, TetR, Rhodococcus sp. RHA1, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: SO4, MSE; 2.5A {Rhodococcus sp.}
</pre><pre>Probab=36.53  E-value=1.3e+02  Score=22.37  Aligned_cols=50  Identities=12%  Similarity=0.245  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   41 SGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        41 ~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ..+.-...|++++...-..+..-.-..+.+.   .|-+..|+|..+++||+..
</pre><pre>T Consensus        16 ~~~~~~~~r~~i~~aa~~~~~~~g~~~~t~~---~ia~~agvs~~~~y~~F~~   65 (207)
</pre><pre>T 3BJB_A           16 SSEEQRARHVRMLEAAIELATEKELARVQMH---EVAKRAGVAIGTLYRYFPS   65 (207)
</pre><pre>T ss_dssp             -CCHHHHHHHHHHHHHHHHHHHSCGGGCCHH---HHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             ChHHHHHHHHHHHHHHHHHHHhcChhhCCHH---HHHHHhCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>2FBQ_A probable transcriptional regulator; PA3006, APC5893, transcriptional regulator, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; 1.8A {Pseudomonas aeruginosa} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=36.32  E-value=83  Score=24.17  Aligned_cols=43  Identities=9%  Similarity=0.212  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ..|++++...-..+..-.-..+.+.   .|-+..|+|..+++|++.
</pre><pre>T Consensus         7 ~~r~~il~aa~~l~~~~g~~~~t~~---~ia~~agvs~~~~y~~F~   49 (235)
</pre><pre>T 2FBQ_A            7 ETVERILDAAEQLFAEKGFAETSLR---LITSKAGVNLAAVNYHFG   49 (235)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCSTTCCHH---HHHHHHTSCHHHHHHHTC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCCccCCHH---HHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>4OPC_A Conserved Archaeal protein; Rossmann Fold, Oxidoreductase, Archaeal Protein; HET: PGT, FDA; 1.4A {Sulfolobus acidocaldarius}
</pre><pre>Probab=36.29  E-value=1.8e+02  Score=27.23  Aligned_cols=49  Identities=16%  Similarity=0.212  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   10 VGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILN   61 (174)
</pre><pre>Q Consensus        10 ~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~   61 (174)
</pre><pre>                      .+-..|..+.+.++++.++|.+ |+  ...-..++|..+...+|++....++
</pre><pre>T Consensus       405 ~~l~~l~~~~~~~~~~~~~y~~-yp--~~~~~~~~w~~~~~~~~~~~~~~~~  453 (453)
</pre><pre>T 4OPC_A          405 SLLFKLKAVAESMKKIKELYLN-YP--RSPSSLGSWRREVDNVLTEFNKSLS  453 (453)
</pre><pre>T ss_dssp             CGGGGHHHHHHHHHHHHHHHHT-CC--SSGGGHHHHHHHHHHHHHHHHHHHC
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHHh-CC--CChhHHHHHHHHHHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>PF16312.9 ; Oberon_cc ; Coiled-coil region of Oberon
</pre><pre>Probab=36.19  E-value=2.6e+02  Score=22.91  Aligned_cols=44  Identities=16%  Similarity=0.127  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHH--HHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   12 KSYYEEINKI--LIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTI   59 (174)
</pre><pre>Q Consensus        12 ~~~l~~i~~~--L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~   59 (174)
</pre><pre>                      ...+..|..+  ..++++-|...|.+    +...+..++|+.+|.+|+..
</pre><pre>T Consensus        63 AE~L~~ia~a~Ks~K~EeeYa~~~lK----lrL~EaE~~rr~~~eelk~~  108 (148)
</pre><pre>T K3ZHQ6_SETIT/4   63 AERLRSVALAKKSEEAGQDYASMYLK----RRLEEAEAEKQYIFEKIKLQ  108 (148)
</pre><pre>T ss_pred             HHHHHHHHHHhcchHHHHHHHHHHHH----hHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>3QBM_B TetR transcriptional regulator; DNA/RNA-BINDING THREE-HELICAL BUNDLE, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, TRANSCRIPTION; HET: PEG, MSE, PGE; 1.8A {Chloroflexus aurantiacus}
</pre><pre>Probab=36.09  E-value=98  Score=22.72  Aligned_cols=38  Identities=16%  Similarity=0.232  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        13 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   50 (199)
</pre><pre>T 3QBM_B           13 VAQAAALFNVSGYAGTAISDIMAATGLEKGGIYRHFES   50 (199)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTSCHHHHHHHTTCTTSCGGGTCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCChhHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>5Y2G_A Maltose-binding periplasmic protein,Protein B; Camp factor, pore forming toxin, TOXIN; HET: SO4, GLC; 3.0A {Escherichia coli}
</pre><pre>Probab=35.92  E-value=5.2e+02  Score=24.81  Aligned_cols=90  Identities=6%  Similarity=0.135  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGL   82 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~Lgl   82 (174)
</pre><pre>                      .+......+...+++..+.++++++.+...... ........|.+-++.++..|+..+                 ..+++
</pre><pre>T Consensus       361 al~~~~~~~~~~l~~~~~~~~~l~~~~~~~~~~-~~~~~~~~~~~~~~~~~~~l~~~~-----------------~~~~~  422 (595)
</pre><pre>T 5Y2G_A          361 ALAAAQTNAVDSNNQAQQMAQKLDQDSIQLRNI-KDNVQGTDYEKPVNEAITSVEKLK-----------------TSLRA  422 (595)
</pre><pre>T ss_dssp             HHHHHHHHHHTCCCHHHHHHHHHHHHHHHHHHH-HHHSCSSTTHHHHHHHHHHHHHHH-----------------HHTTT
</pre><pre>T ss_pred             HHHHHHHHHHhhhHHHHHHHHHHHHHHHHHHHH-HHhcCCCChHHHHHHHHHHHHHHH-----------------Hhccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhh
</pre><pre>Q FD01845382_026   83 STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTV  119 (174)
</pre><pre>Q Consensus        83 Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~g  119 (174)
</pre><pre>                      ...+++         .+.....+++-+.++.+.++..
</pre><pre>T Consensus       423 ~~~~~~---------~~~~~~~r~~~~~~~~~~~~~~  450 (595)
</pre><pre>T 5Y2G_A          423 NSETVY---------DLNSIGSRVEALTDVIEAITFS  450 (595)
</pre><pre>T ss_dssp             CCCCSS---------CSHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCcee---------ccccHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>3DEW_A Transcriptional regulator, TetR family; TetR, transcriptional regulator, Geobacter sulfurreducens, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE, EDO; 1.75A {Geobacter sulfurreducens}
</pre><pre>Probab=35.78  E-value=82  Score=23.29  Aligned_cols=37  Identities=16%  Similarity=0.209  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|.++++|++.
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~f~   50 (206)
</pre><pre>T 3DEW_A           14 MEVATELFAQKGFYGVSIRELAQAAGASISMISYHFG   50 (206)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHSC
</pre><pre>T ss_pred             HHHHHHHHHhcCCccCCHHHHHHHhCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>PF20867.1 ; UVSSA_N ; UVSSA N-terminal domain
</pre><pre>Probab=35.68  E-value=1.4e+02  Score=23.35  Aligned_cols=62  Identities=16%  Similarity=0.258  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhHh---cCCCccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   49 RARLFKKLDTILNKFSKTH---LNLEEYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        49 R~~Lf~~Ld~~l~~~~~~~---~~~~~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      |..+..+|+.-+.......   -.+|+.....+.|.--.=.++..|...       |...+.++.-+.++|+
</pre><pre>T Consensus        76 R~~v~~~l~~~l~l~~~~~~~~~~LP~P~~~a~~Lr~~~l~~i~~W~~~-------fg~~y~~L~~a~~~L~  140 (143)
</pre><pre>T W5MYM5_LEPOC/2   76 RTLLVNNFQQFLELTVETDPSDQPLPPPKEVAQKLKTMAIKAVQEWQSS-------YGEAYKKLALGYHFLK  140 (143)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCCCCCCCCCCcHHHHHHHHHHHHHHHHHHHHH-------HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>2OV3_A Periplasmic binding protein component of an ABC type zinc uptake transporter; ABC Transporter, solute binding domain, zinc transport, TRANSPORT PROTEIN; 2.4A {Synechocystis sp.} SCOP: c.92.2.2
</pre><pre>Probab=35.64  E-value=1.9e+02  Score=24.38  Aligned_cols=94  Identities=15%  Similarity=0.179  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH--hcCCCccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT--HLNLEEYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~--~~~~~~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     -+...|..+-..|..+++..+..+-..  ...-|.+.-+-+.+|+..-.
</pre><pre>T Consensus       107 ~~~~~~~~~i~~~L~~~~p~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~v~~~~~~~~y~~~~~gl~~~~  181 (260)
</pre><pre>T 2OV3_A          107 TLVKRQATTIAKELAELDPDNRDQYE-----ANLAAFLAELERLNQELGQILQPLPQRKFIVFHPSWAYFARDYNLVQIP  181 (260)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHTTCSCCEEEESSCCCHHHHHHTTCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHH-----HHHHHHHHHHHHHHHHHHHHhccCCCCEEEEechhHHHHHHHcCCEEeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCc-ccchHHHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGV-DNIERYATYIEN  108 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~-g~Ip~~~~~i~~  108 (174)
</pre><pre>                      +. ....... ..+..+.+.+++
</pre><pre>T Consensus       182 ~~-~~~~~~s~~~l~~l~~~~~~  203 (260)
</pre><pre>T 2OV3_A          182 IE-VEGQEPSAQELKQLIDTAKE  203 (260)
</pre><pre>T ss_dssp             SC-CSSSCCCTTHHHHHHHHHHH
</pre><pre>T ss_pred             ee-eCCCCCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>6O6O_A TetR family transcriptional regulator; TetR-like transcription factor, fatty acid biosynthesis regulation, TRANSCRIPTION; HET: MYR; 1.63A {Mycobacterium tuberculosis}
</pre><pre>Probab=35.51  E-value=86  Score=22.75  Aligned_cols=38  Identities=13%  Similarity=0.235  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   52 (195)
</pre><pre>T 6O6O_A           15 LVVASDVFVDRGYHAAGMDEIADRAGVSKPVLYQHFSS   52 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCcccCCHHHHHHHHCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>5DAJ_G NalD; Repressor, Transcriptional regulator, TetR family, TRANSCRIPTION REGULATOR; HET: MSE; 2.65A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=35.49  E-value=97  Score=22.86  Aligned_cols=38  Identities=13%  Similarity=0.264  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        16 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   53 (212)
</pre><pre>T 5DAJ_G           16 LLAAEELFLEKGVSHTSLEQIARAAGVTRGAVYWHFQN   53 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHHCCCCCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>4W97_A HTH-type transcriptional repressor KstR2; cholesterol, catabolism, KstR2, transcription regulator, TetR, helix-turn-helix, structural genomics, Midwest Center for Structural Genomics, MCSG, PSI-Biology; HET: UCA; 1.6A {Mycobacterium tuberculosis}
</pre><pre>Probab=35.43  E-value=91  Score=22.68  Aligned_cols=38  Identities=18%  Similarity=0.352  Sum_probs=0.0  Template_Neff=13.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~F~~   53 (200)
</pre><pre>T 4W97_A           16 LELAAAMFAERGLRATTVRDIADGAGILSGSLYHHFAS   53 (200)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHTTTCSC
</pre><pre>T ss_pred             HHHHHHHHHHcCcccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>3QKX_A Uncharacterized HTH-type transcriptional regulator HI_0893; STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, TRANSCRIPTION, TRANSCRIPTION REGULATOR; HET: EDO, MSE, SO4; 2.35A {Haemophilus influenzae}
</pre><pre>Probab=35.33  E-value=88  Score=22.62  Aligned_cols=37  Identities=11%  Similarity=0.143  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++|++.
</pre><pre>T Consensus        14 ~~a~~~l~~~~g~~~~t~~~ia~~a~is~~~~y~~f~   50 (188)
</pre><pre>T 3QKX_A           14 FSATDRLMAREGLNQLSMLKLAKEANVAAGTIYLYFK   50 (188)
</pre><pre>T ss_dssp             HHHHHHHHHHSCSTTCCHHHHHHHHTCCHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhChhhccHHHHHHHhCCCchHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>2IBD_A Possible transcriptional regulator; Probable transcriptional regulatory protein, Rhodococcus sp. RHA1, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for; HET: MSE; 1.5A {Rhodococcus sp.}
</pre><pre>Probab=34.93  E-value=96  Score=22.72  Aligned_cols=38  Identities=26%  Similarity=0.437  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        20 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~F~~   57 (204)
</pre><pre>T 2IBD_A           20 LDIAATLFAERGLRATTVRDIADAAGILSGSLYHHFDS   57 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHTTSCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHcCcccCCHHHHHHHhCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>2G7S_A transcriptional regulator, TetR family; transcription, APC5906, TetR, PSI, Protein structure initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE; 1.4A {Agrobacterium tumefaciens str.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=34.72  E-value=88  Score=22.77  Aligned_cols=37  Identities=22%  Similarity=0.348  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|.++++||+.
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   50 (194)
</pre><pre>T 2G7S_A           14 LQCARTLIIRGGYNSFSYADISQVVGIRNASIHHHFP   50 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHCCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCccccCHHHHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>1R5Q_A circadian oscillation regulator; FOUR-HELIX-BUNDLE, GENE REGULATION; 2.0A {Nostoc sp.} SCOP: a.186.1.1
</pre><pre>Probab=34.66  E-value=3.3e+02  Score=20.95  Aligned_cols=64  Identities=14%  Similarity=0.108  Sum_probs=0.0  Template_Neff=3.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcchhhhcccccCcccch---HHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   46 FGQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSSIMHQWDKTGVDNIE---RYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        46 ~~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks~vh~w~~~~~g~Ip---~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      ..+|+.|+++|+.....+.-.-|.-+  ....|.+..+..=-           .+||   -..=||+-+-.+||.||.=|
</pre><pre>T Consensus         4 ~~e~~~ll~~Lk~~Yr~Iil~YF~~~~~~n~~ID~FVn~aFf-----------adi~v~qivEIHMeLmDefskQLKLEG   72 (102)
</pre><pre>T 1R5Q_A            4 EVDQQILLQQLKSDYRQILLSYFTTDKALKEKIDKFINAVFC-----------ANIPVPEIIEIHMELIDEFSKQLRLEG   72 (102)
</pre><pre>T ss_dssp             CCCTHHHHHHHHHHHHHHHHHTTSCC--CHHHHHHHHHHHHH-----------TTCCHHHHHHHHHHHHHHHHHHTTC--
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhhCCChHHHHHHHHHHHHHhh-----------CCCCHHHHHHHHHHHHHHHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>3ANP_B Transcriptional repressor, TetR family; All Alpha Protein, Transcriptional Repressor, DNA, Acyl-CoA, Transcription; HET: DAO, DCC, MSE; 1.95A {Thermus thermophilus}
</pre><pre>Probab=34.54  E-value=95  Score=22.90  Aligned_cols=38  Identities=13%  Similarity=0.204  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        15 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   52 (204)
</pre><pre>T 3ANP_B           15 FRAAMELFRNRGFQETTATEIAKAAHVSRGTFFNYYPY   52 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHhcCCCcCCHHHHHHHcCCChHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>3CCY_A Putative TetR-family transcriptional regulator; APC88698, TetR, Bordetella parapertussis 12822, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE; 2.01A {Bordetella parapertussis 12822}
</pre><pre>Probab=34.48  E-value=94  Score=22.73  Aligned_cols=38  Identities=18%  Similarity=0.299  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   57 (203)
</pre><pre>T 3CCY_A           20 IERAAAMFARQGYSETSIGDIARACECSKSRLYHYFDS   57 (203)
</pre><pre>T ss_dssp             HHHHHHHHHHTCTTTSCHHHHHHHTTCCGGGGTTTCSC
</pre><pre>T ss_pred             HHHHHHHHHHcCCCccCHHHHHHHcCCChhHhHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>5XBJ_A Flagellar hook-associated protein FlgK; Bacterial Flagellum, Hook, Filament, Junction protein, BIOSYNTHETIC PROTEIN; 2.448A {Campylobacter jejuni}
</pre><pre>Probab=34.44  E-value=3.6e+02  Score=25.76  Aligned_cols=54  Identities=24%  Similarity=0.273  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhcccc-CCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    7 ISEVGKSYYEEINKILIEIQELYKNTYTYNK-GIISGEEFFGQRARLFKKLDTIL   60 (174)
</pre><pre>Q Consensus         7 ~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~-g~l~~~~F~~~R~~Lf~~Ld~~l   60 (174)
</pre><pre>                      ....+....++|...|++|.++=++-..... +.-...+.+.||..|+.+|-..+
</pre><pre>T Consensus        92 ~~~~i~~~v~~iN~ll~~Ia~LN~~I~~~~~~~g~~~ndL~DqRd~ll~eLS~~v  146 (511)
</pre><pre>T 5XBJ_A           92 VNDDIKNTVDEINRIGEEIATINKQIYGQEALPTEHANELRDRRDELELTLSKLV  146 (511)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHTSSSCCCHHHHHHHHHHHHHHHHHHHTTS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHchhcCCCCChhHHHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>3F1B_A TetR-like transcriptional regulator; APC5888, TetR, Rhodococcus sp. RHA1, structural genomics, PSI-2, protein structure initiative, midwest center for structural genomics; HET: SO4, MSE; 2.4A {Rhodococcus}
</pre><pre>Probab=34.35  E-value=92  Score=22.87  Aligned_cols=38  Identities=16%  Similarity=0.271  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   57 (203)
</pre><pre>T 3F1B_A           20 LDAAVDVFSDRGFHETSMDAIAAKAEISKPMLYLYYGS   57 (203)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHTTSCHHHHHHHCCS
</pre><pre>T ss_pred             HHHHHHHHHHcCCccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>PF20975.1 ; DGCcoil ; DGC-associated coil
</pre><pre>Probab=33.99  E-value=6.3e+02  Score=24.29  Aligned_cols=101  Identities=12%  Similarity=0.118  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhc
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQ   90 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~   90 (174)
</pre><pre>                      +..++.++...|.+|+..-......      ..+....++.|=..++..+..+-...-.-.+-..||..+.=.-..|..+
</pre><pre>T Consensus       391 ~e~FL~~L~~rL~~l~~~l~~s~~~------~~~~~~~~~~l~~~v~~qv~~i~~~v~~a~dL~~LK~~V~~~L~~I~~~  464 (493)
</pre><pre>T DGCP_PSEAE/1-4  391 FEEYLRQLNERLESFLGHLGDAHAG------YTDVLDNARGFDQSLREQVSGLQASVQQATDLNSLKLAVDSRLNGLLAS  464 (493)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHh------HHHHHHHHHHHHHHHHHHHHHHHHHHhhCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   91 WDKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        91 w~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      ++.--...-.-.....+++..+..-|+
</pre><pre>T Consensus       465 l~~~~~~e~~r~~~~~~~l~~L~~rL~  491 (493)
</pre><pre>T DGCP_PSEAE/1-4  465 MDEHQREQAEHEQEVSGRLQALMERVN  491 (493)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>3EUP_A Transcriptional regulator, TetR family; Transcriptional regulator, TetR family, Structural genomics, PSI2, MCSG, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: SO4; 1.99A {Cytophaga hutchinsonii}
</pre><pre>Probab=33.90  E-value=1.1e+02  Score=22.70  Aligned_cols=38  Identities=16%  Similarity=0.281  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        17 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   54 (204)
</pre><pre>T 3EUP_A           17 IESTAPVFNVKGLAGTSLTDLTEATNLTKGSIYGNFEN   54 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHTTTSSS
</pre><pre>T ss_pred             HHHHHHHHHHhcccCCCHHHHHHHcCCChhHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>1SGM_B Putative HTH-type transcriptional regulator yxaF; TRANSCRIPTION, Structural Genomics, PSI, Protein Structure Initiative, New York SGX Research Center for Structural Genomics, NYSGXRC; 2.0A {Bacillus subtilis} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=33.83  E-value=1e+02  Score=22.42  Aligned_cols=37  Identities=22%  Similarity=0.182  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        12 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   48 (191)
</pre><pre>T 1SGM_B           12 LHTASRLSQLQGYHATGLNQIVKESGAPKGSLYHFFP   48 (191)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCSSCHHHHSGG
</pre><pre>T ss_pred             HHHHHHHHHHhCccccCHHHHHHHHCCChHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 88
</pre><pre>>2JJ7_A HEMOLYSIN II REGULATORY PROTEIN; DNA-BINDING PROTEIN, TRANSCRIPTION REGULATION, DNA-BINDING, TETR FAMILY, TRANSCRIPTION, TRANSCRIPTIONAL REGULATOR; 2.1A {BACILLUS CEREUS} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=33.79  E-value=98  Score=22.39  Aligned_cols=38  Identities=13%  Similarity=0.138  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++|++..
</pre><pre>T Consensus        13 ~~aa~~l~~~~g~~~~s~~~ia~~ag~s~~~~y~~F~~   50 (186)
</pre><pre>T 2JJ7_A           13 LKAAKKKFGERGYEGTSIQEIAKEAKVNVAMASYYFNG   50 (186)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTSCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHhcCCCcCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 89
</pre><pre>>2O7T_A Transcriptional regulator; Transcription regulator, dna/rna-binding 3-helical bundle fold, helix turn helix motif, hth motif, transcription regulation, structural genomics; HET: UNL, MSE; 2.1A {Corynebacterium glutamicum} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=33.79  E-value=92  Score=23.17  Aligned_cols=38  Identities=13%  Similarity=0.305  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|||..
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   51 (199)
</pre><pre>T 2O7T_A           14 ITTTCNLYRTHHHDSLTMENIAEQAGVGVATLYRNFPD   51 (199)
</pre><pre>T ss_dssp             HHHHHHHHHHSCGGGCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHCCcccCCHHHHHHHhCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 90
</pre><pre>>PF21643.1 ; T6SS_Tsi2-like ; T6SS, Immune protein Tsi2-like
</pre><pre>Probab=33.73  E-value=1.7e+02  Score=20.24  Aligned_cols=31  Identities=3%  Similarity=-0.024  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISG   42 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~   42 (174)
</pre><pre>                      ...++-.+.+++.++..+|+...+. ..++.+
</pre><pre>T Consensus        37 ~ee~l~~~~~a~~~L~~~Y~~~~~~-~~nlP~   67 (71)
</pre><pre>T TSI2_PSEAE/1-7   37 LEQLLVGYDLAADDLKNAYEQALGQ-YSGLPP   67 (71)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHh-CCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 91
</pre><pre>>2DG7_A putative transcriptional regulator; helix-turn-helix motif, TetR family, GENE REGULATION; HET: MSE; 2.3A {Streptomyces coelicolor}
</pre><pre>Probab=33.57  E-value=1e+02  Score=22.46  Aligned_cols=37  Identities=16%  Similarity=0.215  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++|++.
</pre><pre>T Consensus        13 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~f~   49 (195)
</pre><pre>T 2DG7_A           13 KRAALELYSEHGYDNVTVTDIAERAGLTRRSYFRYFP   49 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHSCGGGCCHHHHHHHTTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcCCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 92
</pre><pre>>3VPR_D Transcriptional regulator, TetR family; All Alpha, Helix-Turn-Helix, Transcriptional Repressor, DNA BINDING PROTEIN; HET: MSE; 2.27A {Thermus thermophilus}
</pre><pre>Probab=33.49  E-value=1e+02  Score=22.25  Aligned_cols=38  Identities=21%  Similarity=0.445  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+.+.++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus         9 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   46 (190)
</pre><pre>T 3VPR_D            9 LEEAAKLFTEKGYEATSVQDLAQALGLSKAALYHHFGS   46 (190)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHTTCCHHHHHHHTSS
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHhCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 93
</pre><pre>>2DII_A TFIIH basal transcription factor complex p62 subunit; BTF2-p62, General transcription factor IIH polypeptide 1, Nuclear protein, Transcription regulation, structural genomics, NPPSFA, National Project; NMR {Homo sapiens} SCOP: a.240.1.1, l.1.1.1
</pre><pre>Probab=33.44  E-value=2.3e+02  Score=19.33  Aligned_cols=38  Identities=24%  Similarity=0.323  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHH
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARL   52 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~L   52 (174)
</pre><pre>                      ..-..+-+.-.++..+|++...  .|.+..++|-..|+.+
</pre><pre>T Consensus        17 e~r~kLL~~n~~L~~L~~~LV~--~g~iseeEFW~~r~~~   54 (61)
</pre><pre>T 2DII_A           17 EEKNRMLQEDPVLFQLYKDLVV--SQVISAEEFWANRLNV   54 (61)
</pre><pre>T ss_dssp             HHHHHHHHHCHHHHHHHCCCCC--TTSSCHHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHhCHHHHHHHHHHHh--cCCCCHHHHHhhhhhh
</pre><pre>
</pre><pre>
</pre><pre>No 94
</pre><pre>>PF19095.4 ; DUF5783 ; Family of unknown function (DUF5783)
</pre><pre>Probab=33.39  E-value=1.1e+02  Score=23.37  Aligned_cols=40  Identities=13%  Similarity=0.231  Sum_probs=0.0  Template_Neff=2.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKK   55 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~   55 (174)
</pre><pre>                      +++...+.+|++||++.|..+++...+.-..+-=++++++
</pre><pre>T Consensus        11 ~KY~~YF~eLQ~AYknAf~~mn~~yDS~lvhaIDqqVL~E   50 (106)
</pre><pre>T A0A343TG48_9EU   11 DKYANYFPELQRAYKSAFETMNDEYDSELIHAIDQAVLNE   50 (106)
</pre><pre>T ss_pred             HHhhhhHHHHHHHHHHHHHHHHHHhchHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 95
</pre><pre>>5GP9_B Transcriptional regulator (TetR/AcrR family); fadR, transcriptional regulator, TRANSCRIPTION, DNA BINDING PROTEIN; HET: PLM, GOL; 1.755A {Bacillus halodurans (strain ATCC BAA-125 / DSM 18197 / FERM 7344 / JCM 9153 / C-125)} SCOP: a.4.1.0, a.121.1.0
</pre><pre>Probab=33.35  E-value=1e+02  Score=22.40  Aligned_cols=37  Identities=16%  Similarity=0.328  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        12 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~   48 (194)
</pre><pre>T 5GP9_B           12 IDAAVQVIAEHGYHQAQVSKIAKAAGVADGTIYLYFN   48 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCcccccHHHHHHHhCCChhHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 96
</pre><pre>>6AYH_A TetR family transcriptional regulator; Transcriptional Repressor Protein, Glucuronide Binding Protein, DNA Binding Protein, TRANSCRIPTION; HET: C3G, GOL, MSE; 2.05A {Salmonella choleraesuis}
</pre><pre>Probab=33.06  E-value=1.1e+02  Score=22.61  Aligned_cols=38  Identities=18%  Similarity=0.313  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        19 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   56 (201)
</pre><pre>T 6AYH_A           19 LRAARQCFAENGFHSTSMKTICKASDMSPGTLYHHFPS   56 (201)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHcCCChHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 97
</pre><pre>>PF07688.16 ; KaiA ; KaiA C-terminal domain
</pre><pre>Probab=32.94  E-value=3.8e+02  Score=21.27  Aligned_cols=63  Identities=11%  Similarity=0.142  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcchhhhcccccCcccch---HHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSSIMHQWDKTGVDNIE---RYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks~vh~w~~~~~g~Ip---~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      .+|+.|+++|......+.-.-|.-+  ....|.+..+..=-           .+||   -..=||+-+-.+||.||.=|
</pre><pre>T Consensus        24 ~e~~~lL~~Lk~~Yr~I~l~YF~~~~~~n~~ID~FVn~aFf-----------adipV~qivEIHMeLmDefskQLKLEG   91 (122)
</pre><pre>T K9VYJ2_9CYAN/1   24 NEKTLFLDNLKSKYCEIILTYFDNDPKLNQEIDDFVNLGFF-----------ADLSVSQIVEIHMELMEQFSQQLKIEG   91 (122)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhhCCChhhHHHHHHHHHHHHc-----------CCCCHHHHHHHHHHHHHHHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 98
</pre><pre>>3ZK9_B MANGANESE ABC TRANSPORTER SUBSTRATE-BINDING LIPOPROTEIN; METAL TRANSPORT, ATP-BINDING CASSETTE TRANSPORTERS, BACTERIAL ADHESION, CARRIER PROTEINS, LIPOPROTEINS, MEMBRANE TRANSPORT PROTEINS; HET: TRS; 1.45A {STREPTOCOCCUS PNEUMONIAE} SCOP: c.92.2.2
</pre><pre>Probab=32.93  E-value=2.9e+02  Score=23.55  Aligned_cols=119  Identities=16%  Similarity=0.131  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     -+...|..+-+.|..+++..+..+   .+. ...-|.+.-+-+.+|+..
</pre><pre>T Consensus       114 ~~~~~~a~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~l~~~~~~~~~v~~~h~~~~yl~~~~gl~~  188 (278)
</pre><pre>T 3ZK9_B          114 ENGIIFAKNIAKQLSAKDPNNKEFYE-----KNLKEYTDKLDKLDKESKDKFNKIPAEKKLIVTSEGAFKYFSKAYGVPS  188 (278)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHSCGGGCCEEEEESCCHHHHHHHTCCE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhCcccHHHHH-----HHHHHHHHHHHHHHHHHHHHHhcCChhhCEEEEecchHHHHHHHHCCCE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      ..++        ..-++.....+.++++...++-.+.-.|-.+.......+...+.
</pre><pre>T Consensus       189 ~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~i~~~~~~~~~~~~~l~~  236 (278)
</pre><pre>T 3ZK9_B          189 AYIW--------EINTEEEGTPEQIKTLVEKLRQTKVPSLFVESSVDDRPMKTVSQ  236 (278)
</pre><pre>T ss_dssp             EEEE--------CTTCSSCSCHHHHHHHHHHHTTSCCCCEEEETTSCSHHHHHHHH
</pre><pre>T ss_pred             EEEe--------cCCCCCCCCHHHHHHHHHHHHHcCCCEEEECCCCCCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 99
</pre><pre>>5ZTC_A Lmo2088 protein; TetR family, TRANSCRIPTION; HET: 1PE; 1.7A {Listeria monocytogenes serovar 1/2a (strain ATCC BAA-679 / EGD-e)}
</pre><pre>Probab=32.92  E-value=1.2e+02  Score=22.37  Aligned_cols=37  Identities=19%  Similarity=0.310  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus         7 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   43 (204)
</pre><pre>T 5ZTC_A            7 KQAALTLFANNGFEGTSLADIAGVVGLKKQSIYSHFK   43 (204)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHHCCChHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 100
</pre><pre>>2RAS_A Transcriptional regulator, TetR family; Bacterial regulatory proteins, tetr family, dna-binding, dna/rna-binding 3-helical bundle fold, helix turn helix motif, hth motif; HET: EDO, MSE; 1.8A {Novosphingobium aromaticivorans}
</pre><pre>Probab=32.67  E-value=1e+02  Score=22.85  Aligned_cols=38  Identities=8%  Similarity=0.100  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        17 l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   54 (212)
</pre><pre>T 2RAS_A           17 VDVAQAIVEERGGAGLTLSELAARAGISQANLSRYFET   54 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHTSSCCCHHHHHHHHTSCHHHHTTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCccccCHHHHHHHhCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 101
</pre><pre>>7W2Q_B Probable transcriptional regulator; TetR, transcriptinal regulator, DNA BINDING PROTEIN; 1.8A {Pseudomonas aeruginosa (strain ATCC 15692 / DSM 22644 / CIP 104116 / JCM 14847 / LMG 12228 / 1C / PRS 101 / PAO1)}
</pre><pre>Probab=32.61  E-value=1.1e+02  Score=22.76  Aligned_cols=38  Identities=24%  Similarity=0.365  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++|++..
</pre><pre>T Consensus        19 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   56 (212)
</pre><pre>T 7W2Q_B           19 LDTALKLFLEQGEDSVTVEMIADAVGIGKGTIYKHFKS   56 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 102
</pre><pre>>2HYT_A TetR-family transcriptional regulator; TETR-FAMILY TRANSCRIPTIONAL REGULATOR, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-2, TRANSCRIPTION; HET: MSE, EDO; 1.64A {Pectobacterium atrosepticum}
</pre><pre>Probab=32.44  E-value=1.1e+02  Score=22.38  Aligned_cols=38  Identities=16%  Similarity=0.280  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        18 ~~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   55 (197)
</pre><pre>T 2HYT_A           18 LATARKVFSERGYADTSMDDLTAQASLTRGALYHHFGD   55 (197)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCTTHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCChHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 103
</pre><pre>>1UI5_A A-factor receptor homolog; helix-turn-helix, alpha-helix-bundle, ANTIBIOTIC; HET: MSE; 2.4A {Streptomyces coelicolor} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=32.38  E-value=1.2e+02  Score=22.73  Aligned_cols=38  Identities=8%  Similarity=0.246  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        15 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   52 (215)
</pre><pre>T 1UI5_A           15 IGAAADLFDRRGYESTTLSEIVAHAGVTKGALYFHFAA   52 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHHCCCcCCHHHHHHHHCCChhHHHHhcCc
</pre><pre>
</pre><pre>
</pre><pre>No 104
</pre><pre>>7KD8_A HTH-type transcriptional regulator TtgR; TtgR, TetR family, epistasis, resveratrol, TRANSCRIPTION; HET: STL; 1.71A {Pseudomonas putida}
</pre><pre>Probab=32.36  E-value=1.1e+02  Score=22.71  Aligned_cols=38  Identities=13%  Similarity=0.289  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        18 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   55 (211)
</pre><pre>T 7KD8_A           18 IEAAERAFYKRGVARTTLADIAELAGVTRGAIYWHFNN   55 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCTTHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHhhCccccCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 105
</pre><pre>>6NSN_B Copper outer membrane regulator; Repressor, DNA-binding complex, Epoxide sensing, TRANSCRIPTION-DNA complex; 2.6A {Pseudomonas aeruginosa}
</pre><pre>Probab=32.35  E-value=1e+02  Score=22.66  Aligned_cols=37  Identities=11%  Similarity=0.220  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        18 ~~aa~~~~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   54 (198)
</pre><pre>T 6NSN_B           18 LQRAMDVFWVRGYEGASLAALTEAMEIRPPSLYAAFG   54 (198)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHSCCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHhcccccCHHHHHHHhCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 106
</pre><pre>>2GEN_A probable transcriptional regulator; APC6095, Pseudomonas aeruginosa PA01, TetR family, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for Structural; HET: MSE; 1.7A {Pseudomonas aeruginosa PAO1} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=32.22  E-value=1.1e+02  Score=22.31  Aligned_cols=37  Identities=24%  Similarity=0.391  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        13 ~~aa~~l~~~~g~~~~s~~~ia~~ag~s~~~~y~~f~   49 (197)
</pre><pre>T 2GEN_A           13 LQAALACFSEHGVDATTIEMIRDRSGASIGSLYHHFG   49 (197)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHCCCHHHHHHHTC
</pre><pre>T ss_pred             HHHHHHHHHHhCcccccHHHHHHHhCCCHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 107
</pre><pre>>3ON4_G Transcriptional regulator, TetR family; Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, transcriptional regulator, TRANSCRIPTION; HET: MSE; 1.85A {Legionella pneumophila subsp. pneumophila str. Philadelphia 1}
</pre><pre>Probab=32.20  E-value=1.2e+02  Score=22.13  Aligned_cols=37  Identities=22%  Similarity=0.259  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++|++.
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   52 (191)
</pre><pre>T 3ON4_G           16 LAVAEALIQKDGYNAFSFKDIATAINIKTASIHYHFP   52 (191)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHcCCChhHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 108
</pre><pre>>3DPJ_G Transcription regulator, TetR family; APC88616, TetR, Silicibacter pomeroyi DSS, structural genomics, PSI-2, protein structure initiative, midwest center for structural genomics; HET: MES, EDO, MSE; 1.9A {Silicibacter pomeroyi}
</pre><pre>Probab=32.17  E-value=1.1e+02  Score=22.28  Aligned_cols=38  Identities=13%  Similarity=0.271  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   51 (194)
</pre><pre>T 3DPJ_G           14 VAAADELFYRQGFAQTSFVDISAAVGISRGNFYYHFKT   51 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCCC
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCCHHHHHHHHCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 109
</pre><pre>>5JG7_A Fur regulated Salmonella iron transporter; structural genomics, CSGID, periplasmic binding protein, Center for Structural Genomics of Infectious Diseases, METAL TRANSPORT; HET: MSE, GOL; 1.7A {Salmonella typhimurium (strain LT2 / SGSC1412 / ATCC 700720)}
</pre><pre>Probab=32.12  E-value=2.8e+02  Score=21.21  Aligned_cols=48  Identities=17%  Similarity=0.246  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILN   61 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~   61 (174)
</pre><pre>                      ..+....+.+.+.|.++.-.+...|.     .+.+.|..+-..+.++++..|.
</pre><pre>T Consensus       108 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~l~  155 (160)
</pre><pre>T 5JG7_A          108 ENALIYVDNIRDALVKYDPDNAQIYK-----QNAERYKAKIRQMADPLRAELE  155 (160)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHTCHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhhHHHHH-----HHHHHHHHHHHHHHHHHHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 110
</pre><pre>>3CJD_A Transcriptional regulator, TetR family; YP_510936.1, Putative TetR Transcriptional Regulator, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative; HET: MSE, STE; 1.79A {Jannaschia sp.}
</pre><pre>Probab=32.08  E-value=2e+02  Score=21.07  Aligned_cols=54  Identities=7%  Similarity=0.075  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   36 NKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        36 ~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |.-......-...|++++...-..+..-.-....+.   .|-+..|+|..+++|++.
</pre><pre>T Consensus         1 m~~~~~~~~~~~~~~~i~~aa~~l~~~~g~~~~t~~---~ia~~ag~s~~~~y~~f~   54 (198)
</pre><pre>T 3CJD_A            1 GMAGKVEARKAALREKLIDLAEAQIEAEGLASLRAR---ELARQADCAVGAIYTHFQ   54 (198)
</pre><pre>T ss_dssp             ---------CHHHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTSCHHHHHHHCS
</pre><pre>T ss_pred             CCccchhHHHHHHHHHHHHHHHHHHHHhChhhcCHH---HHHHHcCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 111
</pre><pre>>4MK6_A Probable Dihydroxyacetone Kinase Regulator DHSK_reg; Structural Genomics, PSI-Biology, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, Alpha-beta-fold, TetR-fold, TRANSCRIPTION REGULATOR; HET: MSE; 2.35A {Listeria monocytogenes}
</pre><pre>Probab=31.98  E-value=89  Score=23.31  Aligned_cols=37  Identities=8%  Similarity=0.179  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+.+.++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~f~   51 (189)
</pre><pre>T 4MK6_A           15 AGGLMELCQHKRFEKISIADITNICGLNRQTFYYHFT   51 (189)
</pre><pre>T ss_dssp             HHHHHHHTTTSCTTTCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHhcCCcccccHHHHHHHhCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 112
</pre><pre>>8GZU_tx Thioredoxin; Electron transport chain, supercomplex, membrane protein, Tetrahymena thermophila, ELECTRON TRANSPORT; HET: FES, FMN, CU, U10, F3S, HEA, SF4, ADP, HEC, ZN, TPO, FAD, NDP, SEP, HEM; 4.18A {Tetrahymena thermophila SB210}
</pre><pre>Probab=31.93  E-value=2.7e+02  Score=21.32  Aligned_cols=49  Identities=24%  Similarity=0.332  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSK   65 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~   65 (174)
</pre><pre>                      +.+.+.++++.+........ .+.-..++|..++......+|..+..+.|
</pre><pre>T Consensus       118 ~~l~~~l~~~~~~~~~~~~~-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  166 (166)
</pre><pre>T 8GZU_tx         118 RNLEHRLFKVQEYVKGLNFP-AGMDTYEEFHEQYMAEHTQLDKEINEWAR  166 (166)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHcCCCCC-CCcccHHHHHHHHHHHhhHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 113
</pre><pre>>2ID6_A TRANSCRIPTIONAL REGULATOR, TetR FAMILY; transcriptional regulator  TetR Family, TRANSCRIPTION; HET: MSE; 1.746A {Thermotoga maritima} SCOP: a.4.1.9, l.1.1.1, a.121.1.1
</pre><pre>Probab=31.91  E-value=1.3e+02  Score=22.09  Aligned_cols=44  Identities=16%  Similarity=0.306  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ..|+++++..-..+..-.-....+.   .|-+..|+|..+++|++..
</pre><pre>T Consensus         5 ~~r~~i~~aa~~~~~~~g~~~~t~~---~ia~~agvs~~~~y~~F~~   48 (202)
</pre><pre>T 2ID6_A            5 SKRDAILKAAVEVFGKKGYDRATTD---EIAEKAGVAKGLIFHYFKN   48 (202)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHCCHH---HHHHHHTCCTHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhChhhCCHH---HHHHHHCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 114
</pre><pre>>3RD3_B Probable transcriptional regulator; transcriptional regulator, TRANSCRIPTION; 2.4A {Pseudomonas aeruginosa}
</pre><pre>Probab=31.50  E-value=1.2e+02  Score=22.05  Aligned_cols=38  Identities=21%  Similarity=0.252  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        16 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   53 (197)
</pre><pre>T 3RD3_B           16 LDTGYRIMAVKGFSGVGLNEILQSAGVPKGSFYHYFKS   53 (197)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSSSSCHHHHHHTTTCCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 115
</pre><pre>>3VUQ_A Transcriptional regulator (TetR/AcrR family); helix-turn-helix, transcriptional regulator, DNA Binding, TRANSCRIPTION REGULATOR; 2.05A {Thermus thermophilus}
</pre><pre>Probab=31.29  E-value=1.2e+02  Score=21.95  Aligned_cols=37  Identities=11%  Similarity=0.186  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        18 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   54 (187)
</pre><pre>T 3VUQ_A           18 LEATLAVLRERGLSGLKMEEVARRAEVGKGTIYLYFR   54 (187)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTSCHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCChHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 116
</pre><pre>>6AYI_D HTH-type transcriptional regulator UidR; Transcriptional Repressor Protein, Glucuronide Binding Protein, DNA Binding Protein, TRANSCRIPTION; HET: C3G; 2.09A {Escherichia coli O157:H7}
</pre><pre>Probab=31.26  E-value=1.1e+02  Score=22.44  Aligned_cols=38  Identities=18%  Similarity=0.320  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   57 (199)
</pre><pre>T 6AYI_D           20 LNAAREIFSENGFHSASMKAICKSCAISPGTLYHHFIS   57 (199)
</pre><pre>T ss_dssp             -------------------------------------C
</pre><pre>T ss_pred             HHHHHHHHHHhCCccccHHHHHHHcCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 117
</pre><pre>>8T5Y_A DarR; cAMP, DarR, mycobacteria, TetR, TFR, transcription; HET: CMP; 1.44A {Rhodococcus sp. USK13}
</pre><pre>Probab=31.18  E-value=1.2e+02  Score=22.50  Aligned_cols=38  Identities=16%  Similarity=0.257  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        19 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   56 (212)
</pre><pre>T 8T5Y_A           19 LDAAADAFMLRGFANTTIDDIADDVGATAGLIYYHFRS   56 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 118
</pre><pre>>3CDL_A Transcriptional regulator AefR; APC88582, TetR, Pseudomonas syringae pv. tomato str. DC3000, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center; HET: MSE; 2.36A {Pseudomonas syringae pv. tomato}
</pre><pre>Probab=31.11  E-value=1.1e+02  Score=22.57  Aligned_cols=38  Identities=8%  Similarity=0.303  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        15 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   52 (203)
</pre><pre>T 3CDL_A           15 VQAAIAEFGDRGFEITSMDRIAARAEVSKRTVYNHFPS   52 (203)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHTTSCHHHHHTTSSS
</pre><pre>T ss_pred             HHHHHHHHHhcCccccCHHHHHHHhCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 119
</pre><pre>>5JPD_A Manganese-binding lipoprotein MntA; ABC transporter, cadmium, structural genomics, IDP02508, Center for Structural Genomics of Infectious Diseases, CSGID, TRANSPORT PROTEIN; HET: CD, MSE; 1.72A {Listeria monocytogenes serovar 1/2a} SCOP: c.92.2.0
</pre><pre>Probab=31.08  E-value=4.1e+02  Score=22.85  Aligned_cols=119  Identities=11%  Similarity=0.140  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     .+..+|..+-..+..+++..+..+   ... -..-|.+.-+-+.+||..
</pre><pre>T Consensus       126 ~~~~~~a~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~L~~l~~~~~~~~~~~~~~~~~v~~~~~~~~y~~~~~gl~~  200 (290)
</pre><pre>T 5JPD_A          126 HNGIIYTENVRDALVKADPDNADFYK-----ENAKKYIDKLATLDKEAKQKFADLPENQKTLVTSEGAFKYFAARYGLKA  200 (290)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHTTTSCTTCCEEEEEECTTHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHHhcCChhhCEEEEechHHHHHHHHhCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      ..++        ..-++....-+.+.++.+.++..+.-.|-++.......+..-+.
</pre><pre>T Consensus       201 ~~~~--------~~~~~~~~s~~~l~~~~~~~~~~~v~~i~~~~~~~~~~~~~l~~  248 (290)
</pre><pre>T 5JPD_A          201 AYIW--------EINTESQGTPDQMKQIVGIVEKEKVPNLFVETSVDPRSMESVSK  248 (290)
</pre><pre>T ss_dssp             EEEE--------SSTTSCSSCHHHHHHHHHHHHHTTCSCEEEETTSCTHHHHHHHH
</pre><pre>T ss_pred             EEEe--------ccCCCCCCCHHHHHHHHHHHHHhCCCEEEEcCCCChHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 120
</pre><pre>>3ON2_C Probable transcriptional regulator; Protein Structure Initiative, PSI-2, Structural Genomics, Midwest Center for Structural Genomics, MCSG, unknown function; HET: SO4, PG6, MSE; 1.96A {Rhodococcus jostii}
</pre><pre>Probab=31.07  E-value=1.3e+02  Score=22.19  Aligned_cols=37  Identities=11%  Similarity=0.181  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        18 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~   54 (199)
</pre><pre>T 3ON2_C           18 LARAESTLEKDGVDGLSLRQLAREAGVSHAAPSKHFR   54 (199)
</pre><pre>T ss_dssp             HHHHHHHHHHHTGGGCCHHHHHHHTTCCSSSCCCSSS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccccHHHHHHHhCCCchHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 121
</pre><pre>>3DCF_B transcriptional regulator of the TetR/AcrR family; YP_290855.1, transcriptional regulator of the TetR/AcrR family, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein; HET: MSE; 2.5A {Thermobifida fusca YX}
</pre><pre>Probab=30.99  E-value=1.2e+02  Score=22.56  Aligned_cols=38  Identities=11%  Similarity=0.256  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        37 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   74 (218)
</pre><pre>T 3DCF_B           37 IKVATELFREKGYYATSLDDIADRIGFTKPAIYYYFKS   74 (218)
</pre><pre>T ss_dssp             HHHHHHHHHHTCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 122
</pre><pre>>4JYK_B HTH-type transcriptional regulator RutR; Structural Genomics, PSI-Biology, New York Structural Genomics Research Consortium, NYSGRC, Helix-turn-Helix, Transcriptional Regulator, TRANSCRIPTION REGULATOR; HET: SO4, URA; 1.7A {Escherichia coli} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=30.98  E-value=1.2e+02  Score=22.57  Aligned_cols=38  Identities=18%  Similarity=0.336  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        24 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   61 (212)
</pre><pre>T 4JYK_B           24 LSAALDTFSQFGFHGTRLEQIAELAGVSKTNLLYYFPS   61 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcCCHHHHHHHHCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 123
</pre><pre>>2QIB_A tetR-family transcriptional regulator; tetR-family transcriptional regulator, HTH DNA binding, Structural genomics, MCSG, PSI-2, Protein Structure Initiative, Midwest Center for; HET: P6G, MSE; 1.7A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=30.96  E-value=1.2e+02  Score=23.29  Aligned_cols=38  Identities=21%  Similarity=0.338  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|||..
</pre><pre>T Consensus        19 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~f~s   56 (231)
</pre><pre>T 2QIB_A           19 IGVALDLFSRRSPDEVSIDEIASAAGISRPLVYHYFPG   56 (231)
</pre><pre>T ss_dssp             HHHHHHHHHHSCGGGCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHhhCCCcCCHHHHHHHhCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 124
</pre><pre>>3V6G_B PROBABLE TRANSCRIPTIONAL REGULATORY PROTEIN (PROBABLY DEOR-FAMILY); helix-turn-helix DNA binding domain, TRANSCRIPTION; 1.821A {Mycobacterium tuberculosis}
</pre><pre>Probab=30.94  E-value=1.2e+02  Score=22.89  Aligned_cols=37  Identities=5%  Similarity=0.055  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        20 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   56 (208)
</pre><pre>T 3V6G_B           20 VEAAERVIARQGLGGLSHRRVAAEANVPVGSTTYYFN   56 (208)
</pre><pre>T ss_dssp             HHHHHHHHHHHTGGGCCHHHHHHHHTSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccccHHHHHHHhCCCHhHhheecC
</pre><pre>
</pre><pre>
</pre><pre>No 125
</pre><pre>>2DG8_D putative tetR-family transcriptional regulatory protein; helix-turn-helix motif, TetR family, GENE REGULATION; 2.21A {Streptomyces coelicolor}
</pre><pre>Probab=30.83  E-value=1.5e+02  Score=21.75  Aligned_cols=43  Identities=12%  Similarity=0.231  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ..|+++++.....+..-.-....+.   .|-+..|+|..+++|++.
</pre><pre>T Consensus         9 ~~r~~i~~a~~~l~~~~g~~~~t~~---~ia~~ag~s~~~~y~~f~   51 (193)
</pre><pre>T 2DG8_D            9 QRRERILAATLDLIAEEGIARVSHR---RIAQRAGVPLGSMTYHFT   51 (193)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCHhhccHH---HHHHHHCCCHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 126
</pre><pre>>7AR9_d B14.5b; Complex-I, ELECTRON TRANSPORT; HET: PTY, CDL, PC7, 8Q1; 2.97A {Polytomella sp. Pringsheim 198.80}
</pre><pre>Probab=30.71  E-value=2.9e+02  Score=20.44  Aligned_cols=41  Identities=12%  Similarity=0.031  Sum_probs=0.0  Template_Neff=3.900
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHHHHHHHHHHHHHHH--HHhccccCCCCHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEEINKILIEIQELYK--NTYTYNKGIISGEEFFGQRARLFK   54 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~i~~~L~~ie~ly~--~~~~~~~g~l~~~~F~~~R~~Lf~   54 (174)
</pre><pre>                      ++|..+-.+....+--++..++|++.+.  ..-.+           .+|+.+|.
</pre><pre>T Consensus        39 ~mg~Ga~~~n~l~~we~~~~~dl~~~~~~~~l~~~-----------~~~k~~~~   81 (83)
</pre><pre>T 7AR9_d           39 SVTLSYLGSKLLDEVQDTYYLEHLKRVERKGLQVT-----------EEHKKLFS   81 (83)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTCBCC-----------HHHHHHCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhHHHHHHHHHHHHHHhcccCc-----------HHHHhhhc
</pre><pre>
</pre><pre>
</pre><pre>No 127
</pre><pre>>3MVP_B TetR/AcrR transcriptional regulator; PSI, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure Initiative, transcription regulator; HET: MSE; 1.85A {Streptococcus mutans}
</pre><pre>Probab=30.69  E-value=1e+02  Score=23.00  Aligned_cols=37  Identities=19%  Similarity=0.184  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        32 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   68 (217)
</pre><pre>T 3MVP_B           32 LQVAKDLFSDKTYFNVTTNEIAKKADVSVGTLYAYFA   68 (217)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHTTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHhhcCcccCCHHHHHHHcCCCHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 128
</pre><pre>>2FQ4_A Transcriptional regulator, TetR family; Transcriptional regulator, TetR, DNA-binding protein, Bacillus cereus, Structural Genomics, PSI, Protein Structure Initiative, Midwest Center for; 1.79A {Bacillus cereus} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=30.68  E-value=1.3e+02  Score=21.93  Aligned_cols=38  Identities=13%  Similarity=0.253  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++|++..
</pre><pre>T Consensus        18 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   55 (192)
</pre><pre>T 2FQ4_A           18 LSASYELLLESGFKAVTVDKIAERAKVSKATIYKWWPN   55 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHhCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 129
</pre><pre>>6XY1_D 4-KE-4; coiled coil, tetramer, parallel, de novo protein; HET: ACE; 1.5A {N/A}
</pre><pre>Probab=30.62  E-value=1.5e+02  Score=17.27  Aligned_cols=16  Identities=44%  Similarity=0.385  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQ   26 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie   26 (174)
</pre><pre>                      ..+.+++|++.|++|.
</pre><pre>T Consensus         4 iqkqlkeiqkqlkeiq   19 (32)
</pre><pre>T 6XY1_D            4 IQKQLKEIQKQLKEIQ   19 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 130
</pre><pre>>PF07964.15 ; Red1 ; Rec10 / Red1
</pre><pre>Probab=30.61  E-value=3.1e+02  Score=28.15  Aligned_cols=69  Identities=16%  Similarity=0.287  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchh
</pre><pre>Q FD01845382_026    8 SEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSI   87 (174)
</pre><pre>Q Consensus         8 ~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~   87 (174)
</pre><pre>                      ........+.|+.+..+|+.---..+.            ++.+.||++|+..+.+=.+...++  +..|++.|.+...+|
</pre><pre>T Consensus       625 ~~f~~~~~~ki~iin~ein~~i~~~l~------------~ky~~l~~~l~~~f~~d~~~~~~f--~~~i~~~L~l~e~~L  690 (699)
</pre><pre>T RED1_YEAST/124  625 NHFSNELVRKISIINQELNKKILKELS------------EKYQKLFSELQDNFQNDTNEMLKF--MGEIKDMMNLPEDQL  690 (699)
</pre><pre>T ss_pred             hhcCHHHHHHHHHHHHHHHHHHHHHHH------------HHHHHHHHHHHHHHHHHHHHHHHH--HHHHHHHhcCCHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhc
</pre><pre>Q FD01845382_026   88 MHQ   90 (174)
</pre><pre>Q Consensus        88 vh~   90 (174)
</pre><pre>                      ||.
</pre><pre>T Consensus       691 ~~~  693 (699)
</pre><pre>T RED1_YEAST/124  691 VHA  693 (699)
</pre><pre>T ss_pred             HHH
</pre><pre>
</pre><pre>
</pre><pre>No 131
</pre><pre>>4KWA_B Transcriptional regulator; Structural Genomics, PSI-Biology, Midwest Center for Structural Genomics, MCSG, TetR family, TRANSCRIPTION REGULATOR; HET: MSE, CHT; 1.8A {Saccharomonospora viridis}
</pre><pre>Probab=30.56  E-value=1.2e+02  Score=21.87  Aligned_cols=37  Identities=19%  Similarity=0.269  Sum_probs=0.0  Template_Neff=13.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        10 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~F~   46 (195)
</pre><pre>T 4KWA_B           10 LETAWRLIARRGYHNVRIHDIASELGTSNATIHYHFP   46 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHSCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccccHHHHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 132
</pre><pre>>6EL2_B Transcriptional regulator TetR family; Transcription factor, DNA binding, Archea, TRANSCRIPTION; HET: DCC; 1.9A {Sulfolobus acidocaldarius}
</pre><pre>Probab=30.55  E-value=1.3e+02  Score=21.93  Aligned_cols=38  Identities=13%  Similarity=0.192  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        19 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   56 (196)
</pre><pre>T 6EL2_B           19 LDASVELIADKGFLSTSINDITSKAGVAYGLFYFYFKS   56 (196)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCTTHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCChHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 133
</pre><pre>>3JSJ_C Putative TetR-family transcriptional regulator; Dna-binding, transcription regulation, bacterial regulatory proteins, dna/rna-binding 3-helical bundle fold, helix turn helix motif, hth motif; HET: EDO, MSE; 2.1A {Streptomyces avermitilis MA-4680}
</pre><pre>Probab=30.52  E-value=1.2e+02  Score=22.20  Aligned_cols=37  Identities=16%  Similarity=0.228  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++. -....|-+..|+|..+++|+|..
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~-~t~~~ia~~ag~s~~~~y~~f~~   51 (190)
</pre><pre>T 3JSJ_C           15 LEAAAALTYRDGVG-IGVEALCKAAGVSKRSMYQLFES   51 (190)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTT-CCHHHHHHHHTCCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHhCcc-ccHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 134
</pre><pre>>4UDS_A MBDR REGULATOR; TRANSCRIPTION, TETR FAMILY; HET: MSE; 1.76A {AZOARCUS SP. CIB}
</pre><pre>Probab=30.47  E-value=1.3e+02  Score=22.30  Aligned_cols=37  Identities=22%  Similarity=0.320  Sum_probs=0.0  Template_Neff=13.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        19 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   55 (214)
</pre><pre>T 4UDS_A           19 LGCALDLFVKNGYRATTIDMIAARAGLTKGAIYFYFK   55 (214)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTC--CTGGGTCS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHhCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 135
</pre><pre>>2HKU_A A putative transcriptional regulator; Structural Genomics, APC6040, transcriptional regulator, TetR family, Rhodococcus sp. RHA1, PSI-2, Protein Structure Initiative, Midwest Center; HET: EDO, MSE, PG4; 2.0A {Rhodococcus sp.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=30.33  E-value=1.3e+02  Score=22.64  Aligned_cols=43  Identities=14%  Similarity=0.227  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ..|++++...-..+..-. ....+.   .|-+..|+|..+++|++..
</pre><pre>T Consensus        20 ~~r~~i~~aa~~l~~~~g-~~~t~~---~ia~~agvs~~~~y~~f~~   62 (215)
</pre><pre>T 2HKU_A           20 QTRDALFTAATELFLEHG-EGVPIT---QICAAAGAHPNQVTYYYGS   62 (215)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHC-TTSCHH---HHHHHHTCCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhC-CCCCHH---HHHHHhCCChhHHhHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 136
</pre><pre>>3CWR_A Transcriptional regulator, TetR family; YP_425770.1, transcriptional regulator of TetR family, Bacterial regulatory proteins, tetR family, Structural Genomics, Joint Center for; HET: MSE, SO4; 1.5A {Rhodospirillum rubrum ATCC 11170}
</pre><pre>Probab=30.32  E-value=99  Score=23.05  Aligned_cols=38  Identities=5%  Similarity=0.060  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++||+..
</pre><pre>T Consensus        23 l~aa~~l~~~~g~~~~t~~~ia~~ag~s~~~~y~~F~~   60 (208)
</pre><pre>T 3CWR_A           23 VGAAQRLLSSGGAAAMTMEGVASEAGIAKKTLYRFASG   60 (208)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHcCChhhCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 137
</pre><pre>>2NP5_D Transcriptional regulator; tetR family, transcriptional regulator, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: LMT, NDS, MSE; 1.8A {Rhodococcus sp.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=30.28  E-value=1.3e+02  Score=22.17  Aligned_cols=38  Identities=13%  Similarity=0.297  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~f~~   52 (203)
</pre><pre>T 2NP5_D           15 AAALFDVAAESGLEGASVREVAKRAGVSIGAVQHHFST   52 (203)
</pre><pre>T ss_dssp             HHHHHHHHHHHCC-------------------------
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 138
</pre><pre>>2HUJ_A Lin2004 protein; NP_471338.1, hypothetical protein, Structural Genomics, PSI-2, Protein Structure Initiative, Joint Center for Structural Genomics, JCSG, UNKNOWN; HET: MSE; 1.74A {Listeria innocua} SCOP: a.24.26.1, l.1.1.1
</pre><pre>Probab=30.25  E-value=4.4e+02  Score=21.13  Aligned_cols=87  Identities=6%  Similarity=0.072  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchh
</pre><pre>Q FD01845382_026    8 SEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSI   87 (174)
</pre><pre>Q Consensus         8 ~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~   87 (174)
</pre><pre>                      .......++..++.|..+++++...... ...-...+||.+-+-...+.+..+..|..         ....-+.--....
</pre><pre>T Consensus        16 ~~~~~~L~~~T~~Ll~~~~~~~~~y~~~-k~~~~~~DF~~eVKPFad~v~~~~~~W~~---------~a~~wi~~~~pky   85 (140)
</pre><pre>T 2HUJ_A           16 YFQGMELLIRTEQLLLQNEKNWELYLSN-REEEKPFDFYKDMKPFVDEAKRCADDFLE---------LAIPWVNTERPPY   85 (140)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHH-HHSCCCCCCCCCCHHHHHHHHHHHHHHHH---------HHHHHHHHHCCTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhc-cccCCCCchHHhhHHHHHHHHHHHHHHHH---------HHHHHHHccCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhcccccCcccchHHHHHHHHHH
</pre><pre>Q FD01845382_026   88 MHQWDKTGVDNIERYATYIENSA  110 (174)
</pre><pre>Q Consensus        88 vh~w~~~~~g~Ip~~~~~i~~~~  110 (174)
</pre><pre>                      +|.      ..|....++|+.++
</pre><pre>T Consensus        86 lh~------~QIe~t~eNie~ls  102 (140)
</pre><pre>T 2HUJ_A           86 LGE------LQLRQACDNVQMTA  102 (140)
</pre><pre>T ss_dssp             CCH------HHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCH------HHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 139
</pre><pre>>5K7F_A Transcriptional regulator, TetR family; TetR like regulator, isovalerate, regulation, transcription; 1.7A {Myxococcus xanthus (strain DK 1622)}
</pre><pre>Probab=30.21  E-value=1.2e+02  Score=22.94  Aligned_cols=38  Identities=13%  Similarity=0.190  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        22 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~s   59 (231)
</pre><pre>T 5K7F_A           22 LETAARLICDRGYEGTSMQEIAAACRMTKAGLYHHIQN   59 (231)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHTTCCHHHHHHHCCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcccHHHHHHHcCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 140
</pre><pre>>3BRU_A Regulatory protein, TetR family; structural genomics, APC88928, TetR, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, DNA-binding, Transcription; HET: MSE; 2.3A {Rhodobacter sphaeroides 2.4.1}
</pre><pre>Probab=30.17  E-value=1.3e+02  Score=22.83  Aligned_cols=38  Identities=16%  Similarity=0.215  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        36 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   73 (222)
</pre><pre>T 3BRU_A           36 IRAGLEHLTEKGYSSVGVDEILKAARVPKGSFYHYFRN   73 (222)
</pre><pre>T ss_dssp             HHHHHHHHHHSCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHcCCCchHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 141
</pre><pre>>2REK_A Putative tetR-family transcriptional regulator; tetR-family, transcriptional regulator, Sulfur, SAD, Cr, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural; 1.86A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=30.15  E-value=1.5e+02  Score=21.92  Aligned_cols=49  Identities=10%  Similarity=0.249  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   41 SGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        41 ~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ....-...|++++...-..+..-. ..+.+.   .|-+..|+|..+++|++..
</pre><pre>T Consensus        10 ~~~~~~~~~~~i~~aa~~l~~~~g-~~~t~~---~ia~~agvs~~~~y~~f~~   58 (199)
</pre><pre>T 2REK_A           10 KRADARRNYDRIIEAAAAEVARHG-ADASLE---EIARRAGVGSATLHRHFPS   58 (199)
</pre><pre>T ss_dssp             --CHHHHHHHHHHHHHHHHHHHHG-GGCCHH---HHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             hhhhHHHHHHHHHHHHHHHHHHhC-CCCCHH---HHHHHHCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 142
</pre><pre>>5VL9_C Regulatory protein TetR; transcription factor, protein-DNA complex, TRANSCRIPTION-DNA complex; HET: HEZ, MSE; 2.16A {Enterobacter lignolyticus}
</pre><pre>Probab=30.13  E-value=1.3e+02  Score=21.88  Aligned_cols=37  Identities=11%  Similarity=0.217  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        14 ~~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   50 (192)
</pre><pre>T 5VL9_C           14 MQAAMRVALDQGFTGMTVRNIATAAGVAAGQVHHHFT   50 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTSCTHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcccHHHHHHHhCCCHHHhHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 143
</pre><pre>>2G3B_B putative TetR-family transcriptional regulator; transcription regulator, TetR-family, structural genomics, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSCRIPTION; HET: MSE, GOL; 2.0A {Rhodococcus sp.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=30.11  E-value=1.3e+02  Score=22.19  Aligned_cols=38  Identities=11%  Similarity=0.192  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus         9 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   46 (208)
</pre><pre>T 2G3B_B            9 LKASATAIAQRGIRGLRVNDVAEVAGVSPGLLYYHFKD   46 (208)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHhCccccCHHHHHHHHCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 144
</pre><pre>>3FRQ_B Repressor protein MphR(A); macrolide antibiotic. repressor, biosensor, erythromycin, Strptomyces, natural products, biosynthesis, DNA-binding, Transcription, Transcription regulation, DNA BINDING PROTEIN; HET: GOL, ERY; 1.76A {Escherichia coli}
</pre><pre>Probab=30.09  E-value=1.2e+02  Score=22.32  Aligned_cols=37  Identities=14%  Similarity=0.251  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        14 ~~a~~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   50 (195)
</pre><pre>T 3FRQ_B           14 LEAATVVLKRCGPIEFTLSGVAKEVGLSRAALIQRFT   50 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHHhCccccCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 145
</pre><pre>>2OER_B Probable transcriptional regulator; helix-turn-helix, alpha-beta, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSCRIPTION; HET: MSE; 2.0A {Pseudomonas aeruginosa}
</pre><pre>Probab=30.09  E-value=1.1e+02  Score=22.96  Aligned_cols=37  Identities=11%  Similarity=0.178  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|+|.
</pre><pre>T Consensus        30 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   66 (214)
</pre><pre>T 2OER_B           30 LEAAVQVLASEGAQRFTTARVAERAGVSIGSLYQYFP   66 (214)
</pre><pre>T ss_dssp             HHHHHHHHC--CCSSCCHHHHHHTTTCCHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHhCCcccccHHHHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 146
</pre><pre>>5XAZ_C Gamma-butyrolactone receptor protein; TetR Family, Transcription Regulator, Antibiotic regulation, HTH DNA binding domain, DNA BINDING PROTEIN; 2.3A {Streptomyces fradiae}
</pre><pre>Probab=30.07  E-value=1.3e+02  Score=23.24  Aligned_cols=38  Identities=11%  Similarity=0.213  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        14 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   51 (228)
</pre><pre>T 5XAZ_C           14 VAAAAAVFDELGYEATTIAEILKRSGVTKGALYFHFTS   51 (228)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHCCCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHHCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 147
</pre><pre>>3LHQ_B AcrAB operon repressor (TetR/AcrR family); structural genomics, IDP02616, CSGID, DNA-binding, transcription, repressor, AcrR, Transcription regulation, Center for Structural Genomics of Infectious; HET: PEG; 1.56A {Salmonella enterica subsp. enterica serovar Typhimurium}
</pre><pre>Probab=30.07  E-value=1.2e+02  Score=22.65  Aligned_cols=38  Identities=24%  Similarity=0.412  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        20 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   57 (220)
</pre><pre>T 3LHQ_B           20 LDVALRLFSQQGVSATSLAEIANAAGVTRGAIYWHFKN   57 (220)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCcccHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 148
</pre><pre>>3COL_A Putative transcription regulator; Transcription regulator, Lactobacillus plantarum, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE; 2.1A {Lactobacillus plantarum WCFS1}
</pre><pre>Probab=30.01  E-value=1.4e+02  Score=21.80  Aligned_cols=37  Identities=11%  Similarity=0.131  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++|++.
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   52 (196)
</pre><pre>T 3COL_A           16 QDAVAAIILAEGPAGVSTTKVAKRVGIAQSNVYLYFK   52 (196)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTSCHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHhCCChhHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 149
</pre><pre>>3BQZ_A HTH-type transcriptional regulator qacR; QacR, multidrug resistance, TetR, Malachite Green, DNA-binding, Plasmid, Repressor, Transcription, Transcription regulation; HET: SO4, IMD, MGR; 2.17A {Staphylococcus aureus}
</pre><pre>Probab=29.81  E-value=1.2e+02  Score=22.20  Aligned_cols=37  Identities=19%  Similarity=0.157  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus         8 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   44 (194)
</pre><pre>T 3BQZ_A            8 LGVAKELFIKNGYNATTTGEIVKLSESSKGNLYYHFK   44 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHcCCccCCHHHHHHHHccchHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 150
</pre><pre>>1T33_B putative transcriptional repressor (TetR/AcrR family); Structural genomics, transcriptional repressor, TetR/CcrR family, Helix turn helix DNA binding domain, PSI, Protein Structure Initiative; 2.2A {Salmonella typhimurium} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=29.67  E-value=1.3e+02  Score=23.13  Aligned_cols=42  Identities=12%  Similarity=0.208  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ..|++|+...-..+.... ....+.   .|-+..|+|..+++||+.
</pre><pre>T Consensus        12 ~~r~~il~aa~~l~~~~G-~~~t~~---~ia~~agvs~~~iy~~F~   53 (224)
</pre><pre>T 1T33_B           12 QAKSQLIAAALAQFGEYG-LHATTR---DIAALAGQNIAAITYYFG   53 (224)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHG-GGSCHH---HHHHHHTSCTHHHHHHTC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhc-CcccHH---HHHHHhCCCHHHhhhhcC
</pre><pre>
</pre><pre>
</pre><pre>No 151
</pre><pre>>6OF0_A HTH-type transcriptional regulator MtrR; TetR family, Helix-turn-helix (HTH), transcription regulator, DNA BINDING PROTEIN; HET: PO4; 2.0A {Neisseria gonorrhoeae}
</pre><pre>Probab=29.38  E-value=1.3e+02  Score=22.24  Aligned_cols=38  Identities=16%  Similarity=0.351  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   53 (210)
</pre><pre>T 6OF0_A           16 MLAALETFYRKGIARTSLNEIAQAAGVTRGALYWHFKN   53 (210)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCCHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 152
</pre><pre>>2D6Y_B putative tetR family regulatory protein; Helix-turn-Helix, GENE REGULATION; HET: TLA, MSE; 2.3A {Streptomyces coelicolor} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=29.34  E-value=1.3e+02  Score=22.56  Aligned_cols=37  Identities=8%  Similarity=0.187  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|.++++|++.
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~f~   50 (202)
</pre><pre>T 2D6Y_B           14 FEAAVAEFARHGIAGARIDRIAAEARANKQLIYAYYG   50 (202)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTTSCCHHHHHHHHTCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 153
</pre><pre>>2RAE_A Transcriptional regulator, AcrR family protein; TetR/AcrR family transcriptional regulator, structural genomics, PSI-2, RHA08332, MCSG, Protein Structure Initiative, Midwest Center for Structural; 2.2A {Rhodococcus sp.}
</pre><pre>Probab=29.32  E-value=1.3e+02  Score=22.25  Aligned_cols=38  Identities=8%  Similarity=0.263  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        23 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   60 (207)
</pre><pre>T 2RAE_A           23 STVGIELFTEQGFDATSVDEVAEASGIARRTLFRYFPS   60 (207)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTSCHHHHHHHTTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCCccCCHHHHHHHhCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 154
</pre><pre>>3BHQ_B Transcriptional regulator; PUTATIVE TETR-FAMILY TRANSCRIPTIONAL REGULATOR, BACTERIAL REGULATORY PROTEINS, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN; HET: MSE, EDO; 1.54A {Mesorhizobium loti}
</pre><pre>Probab=29.31  E-value=1.3e+02  Score=22.41  Aligned_cols=37  Identities=11%  Similarity=0.219  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+....-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        18 l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   54 (211)
</pre><pre>T 3BHQ_B           18 IQAATAAFISKGYDGTSMEEIATKAGASKQTVYKHFT   54 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 155
</pre><pre>>PF01297.21 ; ZnuA ; Zinc-uptake complex component A periplasmic
</pre><pre>Probab=29.15  E-value=5.4e+02  Score=21.98  Aligned_cols=120  Identities=11%  Similarity=0.074  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH--hcCCCccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT--HLNLEEYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~--~~~~~~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+.+.+.     .+...|..+-+.|..++...+...-..  -..-+.+.-+-+.+|+..-.
</pre><pre>T Consensus       118 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~~~~h~a~~y~~~~~gl~~~~  192 (278)
</pre><pre>T C0ZHG4_BREBN/4  118 MQLKAQAEKVKNTLVQKDQAHAADYE-----KNYTQLATDLEQLDKDFKDMVTQAPKKEFMVSHSAFSYMAKRYGLEQVA  192 (278)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCHHhHHHHH-----HHHHHHHHHHHHHHHHHHHHHhcCCCCeeEEechhHHHHHHHhCCEEEe
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHc
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSK  141 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~  141 (174)
</pre><pre>                      +.        +.=++.....+.+.++.+.+|-.+...|-.+.......+..-.+.
</pre><pre>T Consensus       193 ~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~i~~~~~~~~~~~~~l~~~  239 (278)
</pre><pre>T C0ZHG4_BREBN/4  193 IS--------GVNPSDEPSTTELKSLVEHVKEHNISYVLFETLVSPKVAEVIAKE  239 (278)
</pre><pre>T ss_pred             cc--------cCCCCCCCCHHHHHHHHHHHHHCCCCEEEEcCCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 156
</pre><pre>>3LWJ_A Putative TetR-family transcriptional regulator; Putative TetR-family transcriptional regulator, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2; HET: MSE; 2.07A {Syntrophomonas wolfei subsp. wolfei}
</pre><pre>Probab=29.15  E-value=1.4e+02  Score=21.91  Aligned_cols=37  Identities=14%  Similarity=0.108  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        18 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   54 (202)
</pre><pre>T 3LWJ_A           18 LTCSLDLFIEKGYYNTSIRDIIALSEVGTGTFYNYFV   54 (202)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHCSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCChhHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 157
</pre><pre>>3B81_A Transcriptional regulator, AcrR family; NP_350189.1, predicted DNA-binding transcriptional regulator of TetR/AcrR family, Bacterial regulatory proteins, tetR family, Structural Genomics, Joint; HET: MSE, EDO; 2.1A {Clostridium acetobutylicum ATCC 824}
</pre><pre>Probab=29.07  E-value=1.3e+02  Score=22.18  Aligned_cols=38  Identities=18%  Similarity=0.296  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        17 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~f~~   54 (203)
</pre><pre>T 3B81_A           17 ANKIWDIFIANGYENTTLAFIINKLGISKGALYHYFSS   54 (203)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHHTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCcccCHHHHHHHHCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 158
</pre><pre>>6DKM_D DHD131_B; Computational Design, Heterodimer, Coiled-coil, DE NOVO PROTEIN; 2.38A {synthetic construct}
</pre><pre>Probab=28.91  E-value=3.2e+02  Score=19.13  Aligned_cols=45  Identities=27%  Similarity=0.422  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNK   62 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~   62 (174)
</pre><pre>                      .-+.+.++++.|+++..+|...-+       .+--..+-.+++.+++..|.+
</pre><pre>T Consensus        15 yhrvlreyeklleelrrlyeeykr-------gevseeesdrilreikeildk   59 (79)
</pre><pre>T 6DKM_D           15 YHRVLREYEKLLEELRRLYEEYKR-------GEVSEEESDRILREIKEILDK   59 (79)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHTT-------SCCCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHc-------CCCCHHHHHHHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 159
</pre><pre>>PF05476.15 ; PET122 ; PET122
</pre><pre>Probab=28.91  E-value=62  Score=28.73  Aligned_cols=64  Identities=11%  Similarity=0.222  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHH-hCCCcchhhhcc
</pre><pre>Q FD01845382_026   24 EIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKA-LGLSTSSIMHQW   91 (174)
</pre><pre>Q Consensus        24 ~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~-LglStks~vh~w   91 (174)
</pre><pre>                      ++-..+++.++  .|......|.++=+.+++++|+.|..  ...+...+.|.|-+. ++-.....+..|
</pre><pre>T Consensus       116 eLlRikVEsfA--k~t~~~TsFrEKWKvFLediD~~lp~--~~~~~vrDFP~Lt~S~l~~~~~~~l~~~  180 (261)
</pre><pre>T H2AQ87_KAZAF/2  116 ELLRSKVEAFA--RGTGMKTEFSEKWKVFLEDIENELPK--NVVYRIRDFPHLTKGVKTKKDRDTLQDL  180 (261)
</pre><pre>T ss_pred             HHHHHHHHHHh--cCCCCCCcHHHHHHHHHHHHhhcCCC--CCeeEeecChhhccccccCCCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 160
</pre><pre>>3NRG_B TetR family transcriptional regulator; Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, TRANSCRIPTION; HET: MSE; 2.56A {Chloroflexus aurantiacus}
</pre><pre>Probab=28.70  E-value=1.2e+02  Score=22.61  Aligned_cols=38  Identities=18%  Similarity=0.442  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..++++++..
</pre><pre>T Consensus        19 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~~   56 (217)
</pre><pre>T 3NRG_B           19 IDVLLDEFAQNDYDSVSINRITERAGIAKGSFYQYFAD   56 (217)
</pre><pre>T ss_dssp             HHHHHHHHHHSCGGGCCHHHHHHHHTCCTTGGGGTCSS
</pre><pre>T ss_pred             HHHHHHHHHhCCcccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 161
</pre><pre>>3PAS_A TetR family transcription regulator; STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, TRANSCRIPTION REGULATION, TRANSCRIPTION REGULATOR; HET: ACT, MSE, PGO; 1.9A {Marinobacter aquaeolei}
</pre><pre>Probab=28.62  E-value=1.3e+02  Score=21.84  Aligned_cols=38  Identities=18%  Similarity=0.358  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   51 (195)
</pre><pre>T 3PAS_A           14 LEATVREVADHGFSATSVGKIAKAAGLSPATLYIYYED   51 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCcccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 162
</pre><pre>>2F07_B YvdT; helix-turn-helix, TRANSCRIPTION; HET: BTB; 2.3A {Bacillus subtilis subsp. subtilis}
</pre><pre>Probab=28.55  E-value=1.4e+02  Score=21.74  Aligned_cols=37  Identities=16%  Similarity=0.257  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   52 (197)
</pre><pre>T 2F07_B           16 LQAAIEVISEKGLDKASISDIVKKAGTAQGTFYLYFS   52 (197)
</pre><pre>T ss_dssp             HHHHHHHHHHTCSBTBCHHHHHHHHTSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHCccccCHHHHHHHhCCChHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 163
</pre><pre>>3KNW_A Putative transcriptional regulator (TetR/AcrR family); TetR-like protein, MCSG, PSI, Structural Genomics, Protein Structure Initiative, Midwest Center for Structural Genomics, DNA-binding, Transcription; HET: MSE; 2.45A {Acinetobacter sp.}
</pre><pre>Probab=28.44  E-value=1.4e+02  Score=22.11  Aligned_cols=37  Identities=19%  Similarity=0.239  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        20 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~   56 (212)
</pre><pre>T 3KNW_A           20 LDSGFHLVLRKGFVGVGLQEILKTSGVPKGSFYHYFE   56 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcccHHHHHHHHCCChHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 164
</pre><pre>>4JKZ_A Transcriptional regulator, TetR family; transcriptional regulator, TRANSCRIPTION; 1.8A {Mycobacterium smegmatis}
</pre><pre>Probab=28.37  E-value=1.6e+02  Score=21.82  Aligned_cols=38  Identities=8%  Similarity=0.217  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        24 ~~aa~~l~~~~g~~~~s~~~ia~~ag~s~~~~y~~f~~   61 (194)
</pre><pre>T 4JKZ_A           24 VEATRALLLERGFDGLSIEAVAAKAGVGKQTIYRWWPS   61 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTTTCCHHHHHHHHTCCHHHHTTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHhCCChHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 165
</pre><pre>>2GFN_B HTH-type transcriptional regulator pksA related protein; hth-type transcriptional regulator, transcriptional regulator tetr, PSI-2, Regulatory protein, Structural Genomics, Protein Structure Initiative, Midwest Center; HET: MSE; 1.9A {Rhodococcus sp.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=28.36  E-value=1.4e+02  Score=22.03  Aligned_cols=37  Identities=16%  Similarity=0.294  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..++++++.
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~   51 (209)
</pre><pre>T 2GFN_B           15 ADAVLALIAREGISAVTTRAVAEESGWSTGVLNHYFG   51 (209)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHCCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 166
</pre><pre>>8KE8_B NalC; NalC, TetR-type transcriptional factor, Multi-drug efflux system, DNA BINDING PROTEIN; 2.05A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=28.35  E-value=1.4e+02  Score=22.34  Aligned_cols=38  Identities=13%  Similarity=0.201  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        20 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   57 (213)
</pre><pre>T 8KE8_B           20 LDAATQAFLEHGFEGTTLDMVIERAGGSRGTLYSSFGG   57 (213)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTTTCCHHHHHHHHCSCHHHHHHHHSS
</pre><pre>T ss_pred             HHHHHHHHHHcccCCCCHHHHHHHhCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 167
</pre><pre>>8H7E_A De novo ferric enterobactin esterase Syn-F4; Binary patterned library, De novo enzyme, Dimeric 4-helix bundle, Ferric enterobactin esterase, DE NOVO PROTEIN; 2.0A {synthetic construct}
</pre><pre>Probab=28.13  E-value=4.2e+02  Score=20.17  Aligned_cols=40  Identities=13%  Similarity=0.293  Sum_probs=0.0  Template_Neff=2.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKL   56 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~L   56 (174)
</pre><pre>                      +.+.++.++++|+++++-|.      .+.-+--.|-.+-+.+++.+
</pre><pre>T Consensus         5 LN~L~e~l~Evl~~lnknWh------~~~~nlhd~~~~l~~li~~i   44 (102)
</pre><pre>T 8H7E_A            5 LNQLFHNLNEIVEDLNKNWH------RERRTLHDFADELHQLVKHV   44 (102)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhH------hccccHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 168
</pre><pre>>3S5R_B Transcriptional regulator TetR family; DNA/RNA-binding 3-helical bundle, Tetracyclin repressor-like, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative; HET: PEG, MSE; 2.6A {Syntrophus aciditrophicus}
</pre><pre>Probab=28.11  E-value=1.4e+02  Score=22.21  Aligned_cols=38  Identities=16%  Similarity=0.287  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   53 (216)
</pre><pre>T 3S5R_B           16 LDAATTLFAEQGIAATTMAEIAASVGVNPAMIHYYFKT   53 (216)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHTTTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCccccCHHHHHHHhCCCHHHHHHHccC
</pre><pre>
</pre><pre>
</pre><pre>No 169
</pre><pre>>3E7Q_A transcriptional regulator; transcriptional regulator, structural genomics, PSI, MCSG, Protein Structure Initiative, Midwest Center for Structural Genomics, DNA-binding, Transcription; 2.2A {Pseudomonas aeruginosa}
</pre><pre>Probab=27.97  E-value=1.4e+02  Score=22.22  Aligned_cols=37  Identities=19%  Similarity=0.304  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   56 (215)
</pre><pre>T 3E7Q_A           20 IEATLACLKRHGFQGASVRKICAEAGVSVGLINHHYD   56 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCcHHHHHHHHCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 170
</pre><pre>>6HS0_A Probable transcriptional regulatory protein; HELIX-TURN-HELIX, TETR-FAMILY, COMPLEX, INHIBITOR, DRUG DESIGN, TUBERCULOSIS, ETHIONAMIDE, DNA BINDING PROTEIN; HET: GNW; 1.45A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)}
</pre><pre>Probab=27.95  E-value=2.5e+02  Score=21.16  Aligned_cols=55  Identities=15%  Similarity=0.169  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             ccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   36 NKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        36 ~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      +........-...|++++...-..+..-.-..+.+.   .|-+..|+|..+++|++..
</pre><pre>T Consensus        23 ~~~~~~~~~~~~~r~~il~aa~~l~~~~g~~~~s~~---~ia~~agvs~~t~y~~F~~   77 (221)
</pre><pre>T 6HS0_A           23 IKRRTQEERSAATREALITGARKLWGLRGYAEVGTP---EIATEAGVTRGAMYHQFAD   77 (221)
</pre><pre>T ss_dssp             ----CHHHHHHHHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             cccccHHHHHHHHHHHHHHHHHHHHHHhCHHhcCHH---HHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 171
</pre><pre>>3CRJ_D Transcription regulator; APC88200, TetR, Haloarcula marismortui ATCC 43049, structural genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural; HET: MSE; 2.6A {Haloarcula marismortui ATCC 43049}
</pre><pre>Probab=27.94  E-value=1.5e+02  Score=21.79  Aligned_cols=37  Identities=14%  Similarity=0.179  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        20 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   56 (199)
</pre><pre>T 3CRJ_D           20 MQATYRALREHGYADLTIQRIADEYGKSTAAVHYYYD   56 (199)
</pre><pre>T ss_dssp             HHHHHHHHTTTHHHHCCHHHHHHHHTSCHHHHHTTCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 172
</pre><pre>>3HIM_A Probable transcriptional regulator; TetR, Bacterial, Rhodococcus, RHA1, PSI-2, MCSG, Structural Genomics, Midwest Center for Structural Genomics, Protein Structure Initiative; 2.2A {Rhodococcus jostii}
</pre><pre>Probab=27.94  E-value=1.5e+02  Score=21.96  Aligned_cols=38  Identities=11%  Similarity=0.230  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        22 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   59 (211)
</pre><pre>T 3HIM_A           22 RAAAIEVFAAKGYGATTTREIAASLDMSPGAVYPHYKT   59 (211)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHTTCCTTSSTTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHHCCCCCCHHHHHHHcCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 173
</pre><pre>>4NNO_A Lipoprotein; Manganese transporter, MRSA, ABC superfamily ATP binding cassette transporter, TRANSPORT PROTEIN; 1.174A {Staphylococcus aureus subsp. aureus} SCOP: c.92.2.0
</pre><pre>Probab=27.83  E-value=6e+02  Score=21.89  Aligned_cols=119  Identities=13%  Similarity=0.188  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...+.     .+..+|.++-..|.++++..+...   .+. -..-|.+.-+-+.+|+..
</pre><pre>T Consensus       128 ~~~~~~~~~i~~~L~~~~p~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~~vi~~~~~~~yl~~~~gl~~  202 (291)
</pre><pre>T 4NNO_A          128 DNGIKYVKTIQQTFIDNDKKHKADYE-----KQGNKYIAQLEKLNNDSKDKFNDIPKEQRAMITSEGAFKYFSKQYGITP  202 (291)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHTTTTTTSCGGGCEEEEEECCCHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCHHhHHHHH-----HHHHHHHHHHHHHHHHHHHHHhcCCHHHCeEEEecchHHHHHHHhCCeE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      -.++        ..-++.....+.+.++.+.++-.+.-.|..+.......+..-..
</pre><pre>T Consensus       203 ~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~ii~~~~~~~~~~~~la~  250 (291)
</pre><pre>T 4NNO_A          203 GYIW--------EINTEKQGTPEQMRQAIEFVKKHKLKHLLVETSVDKKAMESLSE  250 (291)
</pre><pre>T ss_dssp             EEEE--------SSTTSCSSCHHHHHHHHHHHHHTTCSCEEEESSSCSHHHHHHHH
</pre><pre>T ss_pred             EEee--------ccCCCCCCCHHHHHHHHHHHHHCCCCEEEEeCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 174
</pre><pre>>2W53_A REPRESSOR; ANTIBIOTIC RESISTANCE, MULTI-DRUG EFFLUX PUMP, TRANSCRIPTION REGULATION, TRANSCRIPTIONAL REPRESSOR, DNA BINDING, TETR FAMILY, DNA-BINDING, TRANSCRIPTION; HET: SO4; 2.0A {STENOTROPHOMONAS MALTOPHILIA}
</pre><pre>Probab=27.72  E-value=1.4e+02  Score=22.36  Aligned_cols=38  Identities=13%  Similarity=0.228  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..++++++..
</pre><pre>T Consensus        17 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   54 (219)
</pre><pre>T 2W53_A           17 LDAAEACFHEHGVARTTLEMIGARAGYTRGAVYWHFKN   54 (219)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCHHHHHTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 175
</pre><pre>>3TU3_B ExoU; type III secretion system, Pseudomonas aeruginosa, ExoU, SpcU, Infectious Diseases, Structural Genomics, Center for Structural Genomics; 1.92A {Pseudomonas aeruginosa}
</pre><pre>Probab=27.69  E-value=6.7e+02  Score=25.43  Aligned_cols=101  Identities=12%  Similarity=0.103  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             chHHHHHHHHHHHHH--------HHHHHHHHHHHHHHHhccccC---CCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC
</pre><pre>Q FD01845382_026    3 GLGYISEVGKSYYEE--------INKILIEIQELYKNTYTYNKG---IISGEEFFGQRARLFKKLDTILNKFSKTHLNLE   71 (174)
</pre><pre>Q Consensus         3 ~~G~~~~~~~~~l~~--------i~~~L~~ie~ly~~~~~~~~g---~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~   71 (174)
</pre><pre>                      +.-.+...+.++...        ++..|..+++...+.... .+   .-...+|+.+.+..+..|+..+..+......++
</pre><pre>T Consensus       482 G~~aa~~~L~~~~~~~~~~~f~s~~~al~~l~~~~~~~~~~-~~~~~~~~~~~~r~~a~~~l~~L~~ai~~~~~~~~~l~  560 (711)
</pre><pre>T 3TU3_B          482 LQERVGEHLEKRLQASERHTFASLDEALLALDDSMLTSVAQ-QNPEITDGAVAFRQKARDAFTELTVAIVSANGLAGRLK  560 (711)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTCEEEEEESSHHHHHHHSCHHHHHHHHH-HCHHHHHHHHHHHHHHHHHHHHHHHHHHHHTCSCSSSC
</pre><pre>T ss_pred             HHHHHHHHHHHHhHhhcceeccCHHHHHHhCCHHHHHHHHh-cCCcccHHHHHHHHHHHHHHHHHHHHHHHhhccCcccc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccchHHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   72 EYNSIKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        72 ~~~~lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      ..+.|...|                ..++.++.+=+++..+++.|...+
</pre><pre>T Consensus       561 ~~~~l~~~~----------------~~ld~l~~~~~~~~~l~~~l~~~~  593 (711)
</pre><pre>T 3TU3_B          561 LDEAMRSAL----------------QRLDALADTPERLAWLAAELNHAD  593 (711)
</pre><pre>T ss_dssp             SCHHHHHHH----------------HHHHHHCCSHHHHHHHHHHHTCTT
</pre><pre>T ss_pred             CCHHHHHHH----------------HHHHHHccCHHHHHHHHHHhcCCC
</pre><pre>
</pre><pre>
</pre><pre>No 176
</pre><pre>>4ZA6_B TetR family transcriptional regulator; Quorum-sensing, NAHL degradation, qsdA transcriptional regulator, Rhodococcus erythropolis R138, signaling protein; HET: MPD; 1.97A {Rhodococcus erythropolis}
</pre><pre>Probab=27.55  E-value=2.1e+02  Score=20.96  Aligned_cols=42  Identities=12%  Similarity=0.145  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   46 FGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        46 ~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ...|++++...-..+..-.    ++ ....|-+..|+|.++++|++.
</pre><pre>T Consensus         4 ~~~r~~i~~aa~~l~~~~g----~~-s~~~ia~~ag~s~~t~y~~F~   45 (192)
</pre><pre>T 4ZA6_B            4 DLERRRAIDTAASMYLAEE----PL-DMSLLAERLGVGRATLYRWVG   45 (192)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHTTC----CC-CHHHHHHHHTCCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhC----CC-CHHHHHHHHCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 177
</pre><pre>>PF16271.9 ; DUF4924 ; Domain of unknown function (DUF4924)
</pre><pre>Probab=27.52  E-value=3.5e+02  Score=22.71  Aligned_cols=45  Identities=13%  Similarity=0.076  Sum_probs=0.0  Template_Neff=5.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   14 YYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTI   59 (174)
</pre><pre>Q Consensus        14 ~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~   59 (174)
</pre><pre>                      |+..+.+++.+++.+|...... .-...+.+-|.+-.-...+|...
</pre><pre>T Consensus        75 HL~~~~~~i~eL~~lH~~LL~~-~~d~~Y~~~y~~a~p~I~e~~~k  119 (177)
</pre><pre>T R5B4P6_9BACE/1   75 HLQLNKNIIIQLVELHQALLKD-PRFADYTAEFYRTLPFIVELRAK  119 (177)
</pre><pre>T ss_pred             cHHHHHHHHHHHHHHHHHHHcC-cCCHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 178
</pre><pre>>3PPB_A putative tetR family transcription regulator; DNA-BINDING, HELIX-TURN-HELIX MOTIF, HTH MOTIF, DNA/RNA-BINDING 3- HELICAL BUNDLE FOLD, TETRACYCLIN REPRESSOR-LIKE FOLD, TRANSCRIPTION, TRANSCRIPTION REGULATION; HET: PEG, EDO, MSE, PG4; 2.1A {Shewanella loihica}
</pre><pre>Probab=27.46  E-value=1.5e+02  Score=21.49  Aligned_cols=37  Identities=22%  Similarity=0.428  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        15 ~~aa~~l~~~~g~~~~t~~~ia~~ag~s~~t~y~~f~   51 (195)
</pre><pre>T 3PPB_A           15 LETALQLFVSQGFHGTSTATIAREAGVATGTLFHHFP   51 (195)
</pre><pre>T ss_dssp             HHHHHHHHHHTCSTTSCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCccHHHHHHHHCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 179
</pre><pre>>4ICH_B Transcriptional regulator; Structural Genomics, PSI-Biology, Midwest Center for Structural Genomics, MCSG, transcriptional regulator, Transcription Regulator; HET: MSE, B3P; 1.95A {Saccharomonospora viridis}
</pre><pre>Probab=27.38  E-value=1.5e+02  Score=24.43  Aligned_cols=43  Identities=16%  Similarity=0.312  Sum_probs=0.0  Template_Neff=11.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ..|++|+..+...+..-.-....+.   .|-+..|+|..+++|+|.
</pre><pre>T Consensus       120 ~~~~~i~~a~~~l~~~~g~~~~ti~---~ia~~agvs~~~~y~~f~  162 (311)
</pre><pre>T 4ICH_B          120 EARRRILETAWRLIARRGYHNVRIH---DIASELGTSNATIHYHFP  162 (311)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGCCHH---HHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccCCHH---HHHHHHCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 180
</pre><pre>>5UA2_A HTH-type transcriptional repressor KstR; KstR, DNA, complex, transcriptional regulator, TetR family transcriptional repressor, Structural Genomics, PSI-2, Protein Structure Initiative, TB; 2.9002A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)}
</pre><pre>Probab=27.35  E-value=1.3e+02  Score=22.74  Aligned_cols=37  Identities=11%  Similarity=0.192  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        47 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   83 (224)
</pre><pre>T 5UA2_A           47 LDATMAIASKGGYEAVQMRAVADRADVAVGTLYRYFP   83 (224)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHTTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHhcCccccCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 181
</pre><pre>>PF00440.27 ; TetR_N ; Bacterial regulatory proteins, tetR family
</pre><pre>Probab=27.28  E-value=1.5e+02  Score=16.16  Aligned_cols=38  Identities=16%  Similarity=0.186  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+..+....+.-.....+-+..+++..++.++|..
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~~~~~~~i~~~~~~~~~~~~~~~~~   39 (47)
</pre><pre>T P95100_MYCTU/2    2 LRAARELLEETPFAELSVRAISLRAGVARSGFYFYFDS   39 (47)
</pre><pre>T ss_pred             HHHHHHHHHhCChhHCCHHHHHHHHCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 182
</pre><pre>>4OXR_B Manganese ABC transporter, periplasmic-binding protein SitA; Manganese binding protein, SBP, ABC transporter, METAL BINDING PROTEIN; 2.0A {Staphylococcus pseudintermedius} SCOP: c.92.2.0
</pre><pre>Probab=27.24  E-value=4.6e+02  Score=22.49  Aligned_cols=119  Identities=12%  Similarity=0.148  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+......|.+.|.+++-.+...|.     -+...|..+-++|..+++..+..+   .+. -..-|.+.-+-+.+|+..
</pre><pre>T Consensus       122 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~l~~~~~~~~~vv~~h~~~~y~~~~~gl~~  196 (285)
</pre><pre>T 4OXR_B          122 GNGIEYVKTIKSALENADKTHAKDYD-----KQGTEYLSKLEKLNKESKDKFNDIPKEKRVMITSEGAFKYFAQQFDVKP  196 (285)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCSTTHHHHH-----HHHHHHHHHHHHHHHHHTTGGGGSCGGGCEEEEEECCCHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHhhcCChhhCeEEEechhHHHHHHHcCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      -.++        ..-|+-....+.+..+.+.++--+.-.|-.+.......+..-+.
</pre><pre>T Consensus       197 ~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~v~~i~~~~~~~~~~~~~l~~  244 (285)
</pre><pre>T 4OXR_B          197 GYIW--------EINTENQGTPEQMKQAVDFVKENHIKNLLLETSVSDKSMKSLGE  244 (285)
</pre><pre>T ss_dssp             EEEE--------SSTTSCSSCHHHHHHHHHHHHHTTCCEEEEETTSCSHHHHHHHH
</pre><pre>T ss_pred             EEee--------cCCCCCCCCHHHHHHHHHHHHHCCCCEEEECCCCChHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 183
</pre><pre>>2YVE_A Transcriptional regulator; transcriptional regulator, helix-turn-helix, TetR-family, TRANSCRIPTION; HET: GOL, MBT; 1.4A {Corynebacterium glutamicum} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=27.24  E-value=1.4e+02  Score=22.23  Aligned_cols=37  Identities=27%  Similarity=0.446  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.-.....+-+..|+|..+++||+.
</pre><pre>T Consensus        10 ~~a~~~~~~~~g~~~~t~~~ia~~agvs~~t~y~~f~   46 (185)
</pre><pre>T 2YVE_A           10 LRTAIDYIGEYSLETLSYDSLAEATGLSKSGLIYHFP   46 (185)
</pre><pre>T ss_dssp             HHHHHHHHHHSCSTTCCHHHHHHHHCCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCchhCCHHHHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 184
</pre><pre>>6NSI_A Manganese-binding protein; YtgA, solute-binding protein, Chlamydia trachomatis, iron acquisition, ABC transporter, METAL BINDING PROTEIN; 2.00006345565A {Chlamydia trachomatis} SCOP: c.92.2.0
</pre><pre>Probab=27.16  E-value=5.9e+02  Score=22.14  Aligned_cols=120  Identities=9%  Similarity=0.055  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+.+..+.|.+.|.++.-.+...+.     .+..+|..+-+.|..++...+..+   .+. -..-|.+.-+-+.+|+..
</pre><pre>T Consensus       108 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~L~~l~~~~~~~~~~~~~~~~~iv~~h~~~~y~~~~~gl~~  182 (287)
</pre><pre>T 6NSI_A          108 RVWGAAVKEMAAALIQQFPQYEEDFQ-----KNADQILSEMEELDRWAARSLSTIPEKNRYLVTGHNAFSYFTRRYLSSD  182 (287)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHTTSCGGGCCEEBSSSCCHHHHHHHSSCH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHHcCCChhhcEEEeechhHHHHHHHcCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ch------------hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHc
</pre><pre>Q FD01845382_026   85 SS------------IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSK  141 (174)
</pre><pre>Q Consensus        85 ks------------~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~  141 (174)
</pre><pre>                      -.            ++        +.-++-....+.+.++.+.++-.+.-.|-++.......+..-.+.
</pre><pre>T Consensus       183 ~~~~~~~~~~~~~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~i~~~~~~~~~~~~~l~~~  243 (287)
</pre><pre>T 6NSI_A          183 AERVSGEWRSRCISPE--------GLSPEAQISIRDIMRVVEYISANDVEVVFLEDTLNQDALRKIVSC  243 (287)
</pre><pre>T ss_dssp             HHHHHSGGGGGEECSS--------CSSTTSCCCHHHHHHHHHHHHHHTCCEEBCBTTSCCHHHHHHHHT
</pre><pre>T ss_pred             EeccCCcccccccCCC--------CCCcccCCCHHHHHHHHHHHHHCCCCEEEEcCCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 185
</pre><pre>>3CX3_B Lipoprotein; zinc-binding, Lipoprotein, Transport, LIPID BINDING PROTEIN, METAL BINDING PROTEIN; HET: MSE; 2.4A {Streptococcus pneumoniae} SCOP: c.92.2.0
</pre><pre>Probab=27.14  E-value=5.4e+02  Score=21.88  Aligned_cols=116  Identities=16%  Similarity=0.153  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH--hcCCCccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT--HLNLEEYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~--~~~~~~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...+.     .+..+|..+-+.|..++...+.+.-..  -..-|.+.-+-+.+|+.--.
</pre><pre>T Consensus       126 ~~~~~~~~~i~~~L~~~~p~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~v~~~~~~~y~~~~~gl~~~~  200 (284)
</pre><pre>T 3CX3_B          126 EKAGEEAQIIADKLSEVDSEHKETYQ-----KNAQAFIKKAQELTKKFQPKFEKATQKTFVTQHTAFSYLAKRFGLNQLG  200 (284)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTTTSSTTSTTEEH-----HHHHHHHHHHHHHHHHHHHHTTTCSCCEEEESSSCCHHHHHHTTCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHhhChhhHHHHH-----HHHHHHHHHHHHHHHHHHHHHhhccCCeEEEechHHHHHHHHhCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYD  137 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~  137 (174)
</pre><pre>                      ++        ..-++....-+.+.++.+.++-.+.-.|..+.......+..
</pre><pre>T Consensus       201 ~~--------~~~~~~~~s~~~l~~~~~~l~~~~v~~i~~~~~~~~~~~~~  243 (284)
</pre><pre>T 3CX3_B          201 IA--------GISPEQEPSPRQLTEIQEFVKTYKVKTIFTESNASSKVAET  243 (284)
</pre><pre>T ss_dssp             SS--------CSSTTCCCCHHHHHHHHHHHHHHCCCEEEESSCCC--CTTT
</pre><pre>T ss_pred             ec--------cCCCCCCCCHHHHHHHHHHHHHcCCCEEEECCCCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 186
</pre><pre>>6G8G_C TetR/AcrR family transcriptional regulator; Flavonoids, repressor, TetR-family, TRANSCRIPTION; HET: GEN, NHE, MSE; 2.6A {Bradyrhizobium diazoefficiens}
</pre><pre>Probab=27.12  E-value=1.4e+02  Score=22.18  Aligned_cols=38  Identities=16%  Similarity=0.301  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        27 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~~   64 (214)
</pre><pre>T 6G8G_C           27 IRAAWQLFLEQGFSATSMDAIAKAAGVSKATLYAYFPS   64 (214)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTCCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHcCcccccHHHHHHHhCCCHHHHHHHCCC
</pre><pre>
</pre><pre>
</pre><pre>No 187
</pre><pre>>7KY1_A TetR family transcriptional regulator; antibiotic, streptomyces, tetR-family transcriptional regulator, signalling, microbial natural product; 1.50003347687A {Streptomyces coelicolor}
</pre><pre>Probab=27.09  E-value=1.6e+02  Score=22.05  Aligned_cols=38  Identities=8%  Similarity=0.271  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        35 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   72 (214)
</pre><pre>T 7KY1_A           35 LEAASEIFASRGYRGASVKDVAERVGMTKGAVYFHFPS   72 (214)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 188
</pre><pre>>6KNY_B Protein Amuc_1100; Uncharacterized, protein, UNKNOWN FUNCTION; 2.1A {Akkermansia muciniphila (strain ATCC BAA-835 / Muc)}
</pre><pre>Probab=27.05  E-value=6.6e+02  Score=22.12  Aligned_cols=91  Identities=11%  Similarity=0.069  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhc--cccCCCCHHHHHHHHHHHHHHHHHHHHH-----HH-hHhcCCCccchHHHHh
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYT--YNKGIISGEEFFGQRARLFKKLDTILNK-----FS-KTHLNLEEYNSIKKAL   80 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~--~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~-----~~-~~~~~~~~~~~lk~~L   80 (174)
</pre><pre>                      ....+.++++++.+.++.+.+.....  . .-.+...+|..+......+|......     -- ...+|++.+     .=
</pre><pre>T Consensus        36 ~~~~~~~~~l~~~~~~l~~~l~~~~~~~~-~~~~~~~~F~~~L~~~~~~L~~~A~~~~V~lp~~~f~fGF~~y-----~~  109 (287)
</pre><pre>T 6KNY_B           36 EELEKELNRYAKAVGSLETAYKPFLASSA-LVPTTPTAFQNELKTFRDSLISSCKKKNILITDTSSWLGFQVY-----ST  109 (287)
</pre><pre>T ss_dssp             --------------CCHHHHHHHHHHHHC-CCCCCHHHHHHHHHHHHHHHHHHHHHHTCEECHHHHTTTCGGG-----TT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhhhhccccC-CCCCChhHHHHHHHHHHHHHHHHHHHCCCeeCCCceecchhHh-----cC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCCcchhhhcccccCcccchHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   81 GLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLM  116 (174)
</pre><pre>Q Consensus        81 glStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~l  116 (174)
</pre><pre>                      .+...           ..||.+...+.-|..+...|
</pre><pre>T Consensus       110 ~~P~~-----------~~i~~L~~QL~~i~~l~~~L  134 (287)
</pre><pre>T 6KNY_B          110 QAPSV-----------QAASTLGFELKAINSLVNKL  134 (287)
</pre><pre>T ss_dssp             SCCCG-----------GGHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCH-----------HHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 189
</pre><pre>>3NNR_A Transcriptional regulator, TetR family; TETR-FAMILY TRANSCRIPTIONAL REGULATOR, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, TRANSCRIPTION; HET: SO4, MSE; 2.49A {Marinobacter aquaeolei}
</pre><pre>Probab=26.93  E-value=1.5e+02  Score=22.76  Aligned_cols=37  Identities=19%  Similarity=0.193  Sum_probs=0.0  Template_Neff=12.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        11 l~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~F~   47 (228)
</pre><pre>T 3NNR_A           11 LLSSLELFNDKGERNITTNHIAAHLAISPGNLYYHFR   47 (228)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHhCCCHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 190
</pre><pre>>3G7R_B Putative transcriptional regulator; TetR, transcriptional regulator, Streptomyces coelicolor, all-helical, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural; HET: EDO, PEG, MSE; 1.38A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=26.91  E-value=1.2e+02  Score=23.02  Aligned_cols=38  Identities=11%  Similarity=0.186  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        41 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~iy~~F~~   78 (221)
</pre><pre>T 3G7R_B           41 LGTATRIFYAEGIHSVGIDRITAEAQVTRATLYRHFSG   78 (221)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTSCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCHhhcCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 191
</pre><pre>>2I10_A Putative TetR transcriptional regulator; structural genomics, APC5890, TetR family, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSCRIPTION; HET: MSE, NPO, PGE; 2.05A {Rhodococcus sp.} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=26.62  E-value=1.6e+02  Score=21.66  Aligned_cols=37  Identities=22%  Similarity=0.370  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        17 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   53 (202)
</pre><pre>T 2I10_A           17 LQTAMELFWRQGYEGTSITDLTKALGINPPSLYAAFG   53 (202)
</pre><pre>T ss_dssp             HHHHHHHHHHHTTTTCCHHHHHHHHTCCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHHhCCCCCCHHHHHHHHCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 192
</pre><pre>>5UXS_A Periplasmic chelated iron-binding protein YfeA; polyspecific, transition metal, transport, METAL TRANSPORT; 1.421A {Yersinia pestis}
</pre><pre>Probab=26.58  E-value=4.8e+02  Score=23.07  Aligned_cols=119  Identities=12%  Similarity=0.126  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHh---cCCC-ccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTH---LNLE-EYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~---~~~~-~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+.+..+.+.+.|.++.-.+...+.     .+...|..+-++|..++...+..+-...   +-.- .+.-+-+.+|+..
</pre><pre>T Consensus       147 ~~~~~~a~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~L~~l~~~~~~~l~~~~~~~~~vv~~h~~~~yf~~~~gl~~  221 (323)
</pre><pre>T 5UXS_A          147 SNALIYIENIRKALVEHDPAHAETYN-----RNAQAYAEKIKALDAPLRERLSRIPAEQRWLVTSEGAFSYLAKDYGFKE  221 (323)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHTSCGGGCEEEEEESTTHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCHHhHHHHH-----HHHHHHHHHHHHhHHHHHHHHcCCCcccceEEEechHHHHHHHHhCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      -.++        +.-|+.....+.+.++.+.++-.+.-.|-.+.......+..-..
</pre><pre>T Consensus       222 ~~~~--------~~~~~~~~s~~~l~~l~~~i~~~~v~~ii~~~~~~~~~~~~l~~  269 (323)
</pre><pre>T 5UXS_A          222 VYLW--------PINAEQQGIPQQVRHVIDIIRENKIPVVFSESTISDKPAKQVSK  269 (323)
</pre><pre>T ss_dssp             EEEE--------SSSSSCSSCTTHHHHHHHHHHHHTCCEEEEETTSCSHHHHHHHH
</pre><pre>T ss_pred             EEEe--------cCCCCCCCCHHHHHHHHHHHHHCCCCEEEECCCCCcHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 193
</pre><pre>>4LID_A A-100; VIRAL PROTEIN, Hyperthermophilic viral protein, SSV1, Sulfolobus spindle-shape virus 1, Archaea, A100, DNA binding scaffold, A; 3.0A {Sulfolobus spindle-shaped virus 1}
</pre><pre>Probab=26.45  E-value=4.8e+02  Score=20.28  Aligned_cols=43  Identities=14%  Similarity=0.209  Sum_probs=0.0  Template_Neff=3.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF   63 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~   63 (174)
</pre><pre>                      ++.+.|++|.++|+-....    -+.+.=..-|+.++..+-..|.+|
</pre><pre>T Consensus        56 ~~~~iL~kI~kLY~vy~~~----~nek~qegirkEILklIA~Ll~~~   98 (106)
</pre><pre>T 4LID_A           56 KLENELRQLKRLYEVYQSV----EIDDAQKAIQKEALLTIAKILSVF   98 (106)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHTTC----CCTTHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHc----chHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 194
</pre><pre>>6S7O_E Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1; N-glycosylation, Oligosaccharyltransferase, OSTA, TRANSFERASE; HET: NAG, KZE, MAN, EGY, BMA, KZB; 3.5A {Homo sapiens}
</pre><pre>Probab=26.44  E-value=6.9e+02  Score=24.51  Aligned_cols=52  Identities=6%  Similarity=0.170  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSK   65 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~   65 (174)
</pre><pre>                      +.....++++.++++.+++. .|.  .|.+...+|...++.+.++++...+++-+
</pre><pre>T Consensus       551 le~~e~e~~~~~~~l~~~~~-~y~--~gkisr~~y~kl~~~~~k~i~k~~~~i~~  602 (607)
</pre><pre>T 6S7O_E          551 MQKLDAQVKELVLKSAVEAE-RLV--AGKLKKDTYIENEKLISGKRQELVTKIDH  602 (607)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTTSSSC-CSH--HHHHHHHHHHHHHHHHHHHCC--------
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHH-HHH--cCCCCHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 195
</pre><pre>>5AQC_A HTH-TYPE TRANSCRIPTIONAL REPRESSOR KSTR; TRANSCRIPTION; HET: MPD, 5JB; 1.66A {MYCOBACTERIUM TUBERCULOSIS}
</pre><pre>Probab=26.39  E-value=1.4e+02  Score=22.25  Aligned_cols=37  Identities=11%  Similarity=0.192  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        22 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~   58 (205)
</pre><pre>T 5AQC_A           22 LDATMAIASKGGYEAVQMRAVADRADVAVGTLYRYFP   58 (205)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTCCHHHHHHHCC
</pre><pre>T ss_pred             HHHHHHHHHHhChhhcCHHHHHHHcCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 196
</pre><pre>>7NGG_C HTH-type transcriptional regulator EthR; ETHR, Tuberculosis, Inhibition, TRANSCRIPTION; HET: U9W; 1.17A {Mycobacterium tuberculosis}
</pre><pre>Probab=26.35  E-value=1.5e+02  Score=22.11  Aligned_cols=37  Identities=19%  Similarity=0.223  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        30 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   66 (216)
</pre><pre>T 7NGG_C           30 LATAENLLEDRPLADISVDDLAKGAGISRPTFYFYFP   66 (216)
</pre><pre>T ss_dssp             HHHHHHHHTTSCGGGCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHhCCcccCCHHHHHHHhCCCchHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 197
</pre><pre>>5D18_A TetR family transcriptional regulator; Mycobacterium tuberculosis, Transcriptional regulator, TRANSCRIPTION; HET: IPA, EPE; 2.04A {Mycobacterium tuberculosis}
</pre><pre>Probab=26.25  E-value=1.6e+02  Score=22.10  Aligned_cols=37  Identities=24%  Similarity=0.336  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        20 l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   56 (216)
</pre><pre>T 5D18_A           20 LDATERLMATKGYAATSISDIRDACGLAPSSIYWHFG   56 (216)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHCCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHHhCccCCCHHHHHHHHCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 198
</pre><pre>>7QWA_C CC-Type2-(UgUe)4; aib, coiled coil, hexamer, 2-Aminoisobutyric acid, DE NOVO PROTEIN; HET: AIB, 4BF; 1.54A {N/A}
</pre><pre>Probab=26.25  E-value=2.1e+02  Score=16.73  Aligned_cols=21  Identities=33%  Similarity=0.339  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKN   31 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~   31 (174)
</pre><pre>                      ..+.+++|.+.|++|....+.
</pre><pre>T Consensus         4 i~q~lkei~k~lkei~~~lke   24 (32)
</pre><pre>T 7QWA_C            4 IXQXLKEIXKXLKEIXXXLKE   24 (32)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHchhhhHH
</pre><pre>
</pre><pre>
</pre><pre>No 199
</pre><pre>>1TOA_B PROTEIN (PERIPLASMIC BINDING PROTEIN TROA); PERIPLASMIC BINDING PROTEIN, ZINC BINDING PROTEIN, ABC TRANSPORTER, BINDING PROTEIN; HET: GOL; 1.8A {Treponema pallidum} SCOP: c.92.2.2
</pre><pre>Probab=26.18  E-value=4.1e+02  Score=23.27  Aligned_cols=116  Identities=9%  Similarity=0.064  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhHhcCCC-ccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKTHLNLE-EYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~~~~~~-~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+.+.|.     .+..+|..+-..+-++++..+...   .+.-+-.- .+.-+-+.+|+..
</pre><pre>T Consensus       144 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~L~~l~~~~~~~~~~~~~~~~~~i~~h~~~~yl~~~~gl~~  218 (313)
</pre><pre>T 1TOA_B          144 KLWSYSVKAVYESLCKLLPGKTREFT-----QRYQAYQQQLDKLDAYVRRKAQSLPAERRVLVTAHDAFGYFSRAYGFEV  218 (313)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHTSCGGGCEEEEEESCCHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHHhCCChhcCEEEEecchHHHHHHHhCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYD  137 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~  137 (174)
</pre><pre>                      ..++        +.-++.....+.+.++.+.++-.+--.|-.+.......+..
</pre><pre>T Consensus       219 ~~~~--------~~~~~~~ps~~~l~~l~~~i~~~~~~~i~~~~~~~~~~~~~  263 (313)
</pre><pre>T 1TOA_B          219 KGLQ--------GVSTASEASAHDMQELAAFIAQRKLPAIFIESSIPHKNVEA  263 (313)
</pre><pre>T ss_dssp             EEEE--------CSSSSSCCCHHHHHHHHHHHHHTTCSEEEEETTSCTHHHHH
</pre><pre>T ss_pred             EEee--------CCCCCCCCCHHHHHHHHHHHHHCCCCEEEEcCCCChHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 200
</pre><pre>>1VI0_B transcriptional regulator; structural genomics, TRANSCRIPTION; HET: MSE, DCC; 1.65A {Bacillus subtilis} SCOP: a.4.1.9, a.121.1.1
</pre><pre>Probab=26.15  E-value=1.5e+02  Score=21.91  Aligned_cols=38  Identities=13%  Similarity=0.300  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        14 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~F~~   51 (206)
</pre><pre>T 1VI0_B           14 IDAAVEVIAENGYHQSQVSKIAKQAGVADGTIYLYFKN   51 (206)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHcCCccccHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 201
</pre><pre>>5W56_A Periplasmic solute binding protein; Solute binding protein, Zinc, ABC Transporter, METAL BINDING PROTEIN; HET: GOL; 2.03A {Paracoccus denitrificans (strain Pd 1222)} SCOP: c.92.2.0
</pre><pre>Probab=26.10  E-value=6.4e+02  Score=21.62  Aligned_cols=116  Identities=12%  Similarity=0.118  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...|.     -+...|..+-+.+-.++...+...   .+. ..--|.+.-+-+.+|+..
</pre><pre>T Consensus       121 ~~~~~~~~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~~v~~~h~~~~y~~~~~gl~~  195 (286)
</pre><pre>T 5W56_A          121 PNAKVYVQNIAAAFCAADAEGCAAYQ-----ANAARYIGELDALDTEIRAAIAALPQDRRTVVVAHNAFRYFEAAYGVHF  195 (286)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHTSCTTCCEEEESSCCCHHHHHHSSCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHHHCCCcccCEEEEechhHHHHHHHhCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYD  137 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~  137 (174)
</pre><pre>                      -.++        +.=++.....+.+.++.+.+|-.+.-.|-++.......+..
</pre><pre>T Consensus       196 ~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~i~~~~~~~~~~~~~  240 (286)
</pre><pre>T 5W56_A          196 LSPQ--------GVSTESEAAAADVAGLIREIRARNASAIFAENISDTRLLEQ  240 (286)
</pre><pre>T ss_dssp             ECCC------------------CTHHHHHHHHHHHTCCEEEECSSSCSHHHHH
</pre><pre>T ss_pred             EccC--------CCCCcccCCHHHHHHHHHHHHHcCCCEEEeCCCCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 202
</pre><pre>>3GZI_A Transcriptional regulator, TetR family; TETR FAMILY TRANSCRIPTIONAL REGULATOR, STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-2; HET: MSE, EDO; 2.05A {Shewanella loihica PV-4}
</pre><pre>Probab=26.08  E-value=1.3e+02  Score=22.51  Aligned_cols=37  Identities=14%  Similarity=0.065  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        23 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   59 (218)
</pre><pre>T 3GZI_A           23 ILAARNLFIERPYAQVSIREIASLAGTDPGLIRYYFG   59 (218)
</pre><pre>T ss_dssp             HHHHHHHHHTSCCSCCCHHHHHHHHTSCTHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHhhCChhhCCHHHHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 203
</pre><pre>>2QWT_A Transcriptional regulator, TetR family; STRUCTURAL GENOMICS, TRANSCRIPTION, PSI-2, Protein Structure Initiative, New York SGX Research Center for Structural Genomics, NYSGXRC; 2.3A {Mycobacterium vanbaalenii PYR-1}
</pre><pre>Probab=26.07  E-value=1.5e+02  Score=21.92  Aligned_cols=36  Identities=14%  Similarity=0.346  Sum_probs=0.0  Template_Neff=12.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++ -.....|-+..|+|..+++|++.
</pre><pre>T Consensus        19 ~~aa~~~~~~~g~-~~t~~~ia~~agvs~~t~y~~f~   54 (196)
</pre><pre>T 2QWT_A           19 LEVAYDTFAAEGL-GVPMDEIARRAGVGAGTVYRHFP   54 (196)
</pre><pre>T ss_dssp             HHHHHHHHHHTCT-TSCHHHHHHHTTSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCC-CCCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 204
</pre><pre>>PF20865.1 ; FAM186A-B_C ; FAM186A/B, C-terminal
</pre><pre>Probab=25.99  E-value=2.8e+02  Score=24.22  Aligned_cols=46  Identities=13%  Similarity=0.140  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF   63 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~   63 (174)
</pre><pre>                      .++...|+.|+......+.  .-.-....+.++|+.++.+|...|+.+
</pre><pre>T Consensus        86 ~~ly~~Lermd~~q~~~lq--~W~~Kq~~lee~R~~~l~~m~~lF~ql  131 (236)
</pre><pre>T F1SHB0_PIG/652   86 QNLYLFLENIDHLQNVQLQ--AWTNKQKDLEEKRQECLSSMVTTFPKL  131 (236)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHH--HHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 205
</pre><pre>>6YB0_I CC-Type2-(TaSd)2; alpha-helical barrel, de novo protein, de novo peptide, coiled coil, designed peptide, designed protein; 1.86A {N/A}
</pre><pre>Probab=25.98  E-value=1.9e+02  Score=17.58  Aligned_cols=18  Identities=33%  Similarity=0.396  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01845382_026   17 EINKILIEIQELYKNTYT   34 (174)
</pre><pre>Q Consensus        17 ~i~~~L~~ie~ly~~~~~   34 (174)
</pre><pre>                      +|.+.|++|.++.+...+
</pre><pre>T Consensus         3 eiaqalkeiaqalkesak   20 (39)
</pre><pre>T 6YB0_I            3 EIAQALKEIAQALKESAK   20 (39)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 206
</pre><pre>>7EQE_A TetR/AcrR family transcriptional regulator; Transcription factor, TRANSCRIPTION; HET: MSE; 2.399A {Streptomyces griseoluteus}
</pre><pre>Probab=25.81  E-value=1.7e+02  Score=21.83  Aligned_cols=38  Identities=13%  Similarity=0.288  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+....-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        25 l~aa~~~~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   62 (200)
</pre><pre>T 7EQE_A           25 QSVALELFIERGYEKTSMREIAEGLGITKAALYYHFKA   62 (200)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 207
</pre><pre>>3QQA_A CmeR; Alpha-helical, Helix-Turn-Helix, DNA-binding, Transcription regulation, Transcription repressor, drug binding, TRANSCRIPTION; HET: TCH; 2.2A {Campylobacter jejuni}
</pre><pre>Probab=25.81  E-value=1.7e+02  Score=22.35  Aligned_cols=37  Identities=22%  Similarity=0.159  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        25 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~iy~~F~   61 (216)
</pre><pre>T 3QQA_A           25 KAVALELFLTKGYQETSLSDIIKLSGGSYSNIYDGFK   61 (216)
</pre><pre>T ss_dssp             HHHHHHHHHHTCTTTCCHHHHHHHHTTSCCSSSCSCC
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHhCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 208
</pre><pre>>4C18_B TSI2; TOXIN, IMMUNITY PROTEIN, T6SS, ANTI-TOXIN; 1.49A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=25.79  E-value=4.5e+02  Score=19.81  Aligned_cols=74  Identities=8%  Similarity=0.008  Sum_probs=0.0  Template_Neff=5.600
</pre><pre>
</pre><pre>Q ss_pred             CcchHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchH
</pre><pre>Q FD01845382_026    1 GDGLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSI   76 (174)
</pre><pre>Q Consensus         1 s~~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~l   76 (174)
</pre><pre>                      +.........+.-.+..+...+..++.+-+..-.  .-.-..++......+++.+|........+...++|++..|
</pre><pre>T Consensus        19 ~~M~eIS~~tL~laIqAV~~eI~~le~~l~s~~~--~d~~d~eE~L~~y~~aa~eLk~~Y~~~~~~~~nLPpyeeL   92 (98)
</pre><pre>T 4C18_B           19 SHMMNLKPQTLMVAIQCVAARTRELDAQLQNDDP--QNAAELEQLLVGYDLAADDLKNAYEQALGQYSGLPPYDRL   92 (98)
</pre><pre>T ss_dssp             -CCCCCCHHHHHHHHHHHHHHHHHHHHHTTSSCC--TTHHHHHHHHHHHHHHHHHHHHHHHHHHHHSSSCCCHHHH
</pre><pre>T ss_pred             CcccCCCHHHHHHHHHHHHHHHHHHHHHHcCCCc--cCHHHHHHHHHHHHHHHHHHHHHHHHHHHhCCCCCCHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 209
</pre><pre>>PF04099.16 ; Sybindin ; Sybindin-like family
</pre><pre>Probab=25.70  E-value=2.1e+02  Score=20.71  Aligned_cols=40  Identities=20%  Similarity=0.119  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhccccC------CCCHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   16 EEINKILIEIQELYKNTYTYNKG------IISGEEFFGQRARLFKKL   56 (174)
</pre><pre>Q Consensus        16 ~~i~~~L~~ie~ly~~~~~~~~g------~l~~~~F~~~R~~Lf~~L   56 (174)
</pre><pre>                      ..+...|++|.++|.+.... +-      .+..+.|..+.+++++++
</pre><pre>T Consensus        87 ~~i~~~l~~i~~~y~~~~~~-NPf~~~~~~i~~~~F~~~~~~~~~~~  132 (132)
</pre><pre>T TRS23_SCHPO/1-   87 TNARLQLQKFYELYSDYVLK-NPFYTLEMPIKCQLFDEQLKRYIDSH  132 (132)
</pre><pre>T ss_pred             cCHHHHHHHHHHHHHHHHhh-CCCCCCCCCcCCHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 210
</pre><pre>>5MQQ_D AmtR protein; transcription regulator, nitrogen regulation, transcription; HET: SO4; 2.093A {Corynebacterium glutamicum}
</pre><pre>Probab=25.48  E-value=1.7e+02  Score=21.70  Aligned_cols=38  Identities=18%  Similarity=0.355  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus        11 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~s   48 (208)
</pre><pre>T 5MQQ_D           11 LDASAELFTRQGFATTSTHQIADAVGIRQASLYYHFPS   48 (208)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSC
</pre><pre>T ss_pred             HHHHHHHHHHhCcccCCHHHHHHHHCCCHHHHHHhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 211
</pre><pre>>4YZE_C HTH-type transcriptional repressor NemR; transcription factor, cystein-lysine sulfenamide thiol, TRANSCRIPTION; 2.2A {Escherichia coli}
</pre><pre>Probab=25.46  E-value=1.8e+02  Score=21.45  Aligned_cols=37  Identities=14%  Similarity=0.045  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        16 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   52 (201)
</pre><pre>T 4YZE_C           16 LATGEQLSLQRGFTGMGLSELLKTAEVPKGSFYHYFR   52 (201)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcCCCCCCHHHHHHHcCCCccHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 212
</pre><pre>>2OF7_A Putative tetR-family transcriptional regulator; APC7240, tetR-family transcriptional regulator, Streptomyces coelicolor A3, structural genomics, PSI-2, protein structure initiative, midwest center for; HET: MSE; 2.3A {Streptomyces coelicolor}
</pre><pre>Probab=25.38  E-value=1.6e+02  Score=23.33  Aligned_cols=37  Identities=8%  Similarity=0.127  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        54 l~aa~~l~~~~G~~~~t~~~Ia~~agvs~~t~y~~F~   90 (260)
</pre><pre>T 2OF7_A           54 RAATYGLIRQQGYEATTVEQIAERAEVSPSTVLRYFP   90 (260)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSTTCCHHHHHHHHTSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcCCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 213
</pre><pre>>2YM0_B CELL INVASION PROTEIN SIPD; CELL INVASION, TYPE 3 SECRETION, T3SS, NEEDLE TIP PROTEIN, HOST PATHOGEN INTERACTION, BACTERIAL PATHOGENESIS, TIP COMPLEX; 3.0A {SALMONELLA ENTERICA SUBSP. ENTERICA SEROVAR TYPHIMURIUM} SCOP: a.250.1.0, l.1.1.1
</pre><pre>Probab=25.22  E-value=2.9e+02  Score=23.85  Aligned_cols=57  Identities=14%  Similarity=0.198  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CcchHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH
</pre><pre>Q FD01845382_026    1 GDGLGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT   66 (174)
</pre><pre>Q Consensus         1 s~~~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~   66 (174)
</pre><pre>                      +.+.+..........++|.+.+..|++-|.+.         +++...+...+++++...+.++..+
</pre><pre>T Consensus        11 ~~~~~~~~~S~~e~~~~i~~~I~~i~~~yl~~---------y~~~~~ky~~fy~~~s~i~~~l~~~   67 (233)
</pre><pre>T 2YM0_B           11 SSGLVPRGSHMAEIWDMVSQNISAIGDSYLGV---------YENVVAVYTDFYQAFSDILSKMGGW   67 (233)
</pre><pre>T ss_dssp             ----------CCHHHHHHHHHHHHHHHHHHHH---------HHHHHHHHHHHHHHHHHHHHTTTTT
</pre><pre>T ss_pred             cCCCCCCCCCHHHHHHHHHHHHHHHchhhHHH---------HHHHHHHHHHHHHHHHHHHHHhhhh
</pre><pre>
</pre><pre>
</pre><pre>No 214
</pre><pre>>6O6P_A TetR family transcriptional regulator; TetR-like transcription factor, fatty acid biosynthesis regulation, TRANSCRIPTION-DNA complex; 3.851A {Mycobacterium tuberculosis}
</pre><pre>Probab=25.10  E-value=2e+02  Score=22.59  Aligned_cols=43  Identities=12%  Similarity=0.258  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ..|++++...-..+..-.-..+.+.   .|-+..|+|..+++|++.
</pre><pre>T Consensus        59 ~~r~~Il~aa~~l~~~~g~~~~t~~---~Ia~~agvs~~t~y~~F~  101 (248)
</pre><pre>T 6O6P_A           59 ERRGQLLVVASDVFVDRGYHAAGMD---EIADRAGVSKPVLYQHFS  101 (248)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHCTTTCCHH---HHHHHHTSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCHhhCCHH---HHHHHHCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 215
</pre><pre>>2WUI_A TRANSCRIPTIONAL REGULATOR; GENE REGULATION, TRANSCRIPTION REGULATION, TETR, DNA-BINDING, TRANSCRIPTION; 2.9A {PSEUDOMONAS AERUGINOSA}
</pre><pre>Probab=24.87  E-value=1.8e+02  Score=21.84  Aligned_cols=38  Identities=16%  Similarity=0.300  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.-.....|-+..|+|..+++|++..
</pre><pre>T Consensus        17 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~s   54 (210)
</pre><pre>T 2WUI_A           17 LDAAERVFLEKGVGTTAMADLADAAGVSRGAVYGHYKN   54 (210)
</pre><pre>T ss_dssp             HHHHHHHHHHSCTTTCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHHCCccCCHHHHHHHHCCCHHHHhHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 216
</pre><pre>>5JWR_G Circadian clock protein KaiA; Transcription Regulator, foldswitch; HET: PO4; 2.61A {Thermosynechococcus elongatus}
</pre><pre>Probab=24.65  E-value=6e+02  Score=20.86  Aligned_cols=63  Identities=16%  Similarity=0.205  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcchhhhcccccCcccch---HHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSSIMHQWDKTGVDNIE---RYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks~vh~w~~~~~g~Ip---~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      .+|+.|+++|......+.-.-|.-+  ...+|.+..+..=-           .+||   -..=||+-+-.+||.||.=|
</pre><pre>T Consensus        46 ~e~~~lL~~Lk~~Yr~Ill~YF~~~~~ln~~ID~Fvn~aFf-----------adipV~qIvEIHMeLmDefskQLKLEG  113 (145)
</pre><pre>T 5JWR_G           46 ADKRKLLDELRSIYRTIVLEYFNTDAKVNERIDEFVSKAFF-----------ADISVSQVLEIHVELMDTFSKQLKLEG  113 (145)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHTTCTTSCHHHHHHHHHHHHHH-----------TTCCHHHHHHHHHHHHHHHHHHHHTTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhhCCChhHHHHHHHHHHHHhh-----------CCCCHHHHHHHHHHHHHHHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 217
</pre><pre>>3NI7_A Bacterial regulatory proteins, TetR family; TetR, transcriptional regulator, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSCRIPTION; HET: MSE; 2.78A {Nitrosomonas europaea}
</pre><pre>Probab=24.54  E-value=1.9e+02  Score=22.03  Aligned_cols=44  Identities=16%  Similarity=0.077  Sum_probs=0.0  Template_Neff=12.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   46 FGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        46 ~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ...|++|++..-..+..-.-..+.+.   .|-+..|+|..+++||+.
</pre><pre>T Consensus         6 ~~~~~~i~~a~~~l~~~~g~~~~tv~---~ia~~agvsr~t~y~~F~   49 (213)
</pre><pre>T 3NI7_A            6 DPMRDAIVDTAVELAAHTSWEAVRLY---DIAARLAVSLDEIRLYFR   49 (213)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHSCSTTCCHH---HHHHHTTSCHHHHHHHCS
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHhcChhhCCHH---HHHHHHCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 218
</pre><pre>>7WOO_L Nucleoporin NSP1; nuclear pore complex, inner ring, protomer, Saccharomyces cerevisiae, TRANSPORT PROTEIN; 3.71A {Saccharomyces cerevisiae}
</pre><pre>Probab=24.38  E-value=1.2e+03  Score=24.09  Aligned_cols=107  Identities=14%  Similarity=0.080  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccC-------CCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC-ccch
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYKNTYTYNKG-------IISGEEFFGQRARLFKKLDTILNKFSKTHLNLE-EYNS   75 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g-------~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~-~~~~   75 (174)
</pre><pre>                      +...-..+..+-+++++.|++||+.....+.. ..       ......=+.+.-+|-..||.+|..|...-..+- +...
</pre><pre>T Consensus       693 l~~~Ld~ie~qQ~el~~~L~~lE~~~~~~~~~-~~~~~~~~~~~~~d~~R~~~y~la~~l~~qL~~l~~~L~~li~~ln~  771 (823)
</pre><pre>T 7WOO_L          693 IDQSLQYIERQQDELENFLDNFETKTEALLSD-VVSTSSGAAANNNDQKRQQAYKTAQTLDENLNSLSSNLSSLIVEINN  771 (823)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH-HBCCSSSSCBTTSCCTTCHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHh-hhccCCccccchHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHhCCCcchhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcch
</pre><pre>Q FD01845382_026   76 IKKALGLSTSSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFV  122 (174)
</pre><pre>Q Consensus        76 lk~~LglStks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v  122 (174)
</pre><pre>                      +...+.-...           .+++.-....+-+..+-.+|..+-||
</pre><pre>T Consensus       772 ~~~~~~~~~~-----------~~~~~~d~~~qi~~iLn~h~~sL~wi  807 (823)
</pre><pre>T 7WOO_L          772 VSNTFNKTTN-----------IDINNEDENIQLIKILNSHFDALRSL  807 (823)
</pre><pre>T ss_dssp             HHHHHHSSSS-----------SSSCSSCSCCTTHHHHHCCCCCCHHH
</pre><pre>T ss_pred             HhhhhccccC-----------CCcCCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 219
</pre><pre>>3F0C_A Transcriptional regulator; TETR, MCSG, PSI, SAD, Structural Genomics, Protein Structure Initiative, Midwest Center for Structural Genomics, DNA-binding, Transcription; HET: SO4, MSE; 2.96A {Cytophaga hutchinsonii}
</pre><pre>Probab=24.38  E-value=1.8e+02  Score=21.60  Aligned_cols=37  Identities=16%  Similarity=0.324  Sum_probs=0.0  Template_Neff=13.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..++++++.
</pre><pre>T Consensus        17 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   53 (216)
</pre><pre>T 3F0C_A           17 INAAQKRFAHYGLCKTTMNEIASDVGMGKASLYYYFP   53 (216)
</pre><pre>T ss_dssp             HHHHHHHHHHHCSSSCCHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCCccCCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 220
</pre><pre>>3BNI_A Putative TetR-family transcriptional regulator; structural genomics, APC7281, TetR transcriptional regulator, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: PG4; 2.3A {Streptomyces coelicolor A3(2)}
</pre><pre>Probab=24.23  E-value=1.6e+02  Score=22.54  Aligned_cols=38  Identities=8%  Similarity=0.141  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|..+++||+..
</pre><pre>T Consensus        49 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~~   86 (229)
</pre><pre>T 3BNI_A           49 LDACADLLDEVGYDALSTRAVALRADVPIGSVYRFFGN   86 (229)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCchhcCHHHHHHHcCCCHHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 221
</pre><pre>>4ETP_B Spindle pole body-associated protein VIK1; Kinesin Motor Protein, Kinesin Motor Homology Domain, karyogamy, mitosis, microtubules, internal Vik1 crosslink with N, N-Ethylenebis(iodoacetamide); HET: EBC, GOL, ADP; 2.3A {Saccharomyces cerevisiae}
</pre><pre>Probab=24.19  E-value=6e+02  Score=22.51  Aligned_cols=44  Identities=14%  Similarity=0.128  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDT   58 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~   58 (174)
</pre><pre>                      ......++++++.+.+++..|.....+      ......+|+.|..++..
</pre><pre>T Consensus        13 ~~~~~~~~~~~~~~~~~~~~~~~~~~~------~~~~~~~rr~L~n~i~e   56 (333)
</pre><pre>T 4ETP_B           13 AALEKEIAALEKEISKQEKFYNDTYNT------VCKELLRSRRLENSIIE   56 (333)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 222
</pre><pre>>7C7X_E NAP1-related protein 1; complex, Histone, PLANT PROTEIN, CHAPERONE, TRANSCRIPTION-CHAPERONE complex; HET: GOL; 3.0A {Arabidopsis thaliana} SCOP: d.305.1.0, l.1.1.1
</pre><pre>Probab=24.17  E-value=3.6e+02  Score=22.74  Aligned_cols=64  Identities=13%  Similarity=0.145  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH-hcCCCccchHHHHhC
</pre><pre>Q FD01845382_026   13 SYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT-HLNLEEYNSIKKALG   81 (174)
</pre><pre>Q Consensus        13 ~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~-~~~~~~~~~lk~~Lg   81 (174)
</pre><pre>                      +.|..++.-+.+++..+......    + ...|..+++.||.+-...+.++-.. ..-+..++.+...+.
</pre><pre>T Consensus        15 ~~L~~~q~~~~~l~~~~~~~~~~----l-~~~~~~~~~pl~~~R~~ii~~ip~FW~~al~n~~~~~~~i~   79 (239)
</pre><pre>T 7C7X_E           15 EKLQEIQDDLEKINEKASDEVLE----V-EQKYNVIRKPVYDKRNEVIQSIPGFWMTAFLSHPALGDLLT   79 (239)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHH----H-HHHHHHHHHHHHHHHHHHHTTSTTHHHHHHHHSHHHHTTCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHH----H-HHHHHHHhHHHHHHHHHHHhhCCChHHHHHHhCHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 223
</pre><pre>>6YJ2_A Probable transcriptional regulatory protein (Probably TetR-family); Tuberculosis, Branch-chain amino acid catabolism, TetR Family, DNA BINDING PROTEIN; HET: GOL; 2.0A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)}
</pre><pre>Probab=24.08  E-value=1.8e+02  Score=21.86  Aligned_cols=37  Identities=19%  Similarity=0.249  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        31 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   67 (215)
</pre><pre>T 6YJ2_A           31 LAAAERLFAERGFLAVRLEDIGAAAGVSGPAIYRHFP   67 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTCCHHHHTTTCS
</pre><pre>T ss_pred             HHHHHHHHHhcCcccCCHHHHHHHhCCChhHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 224
</pre><pre>>3EGQ_A TetR family Transcriptional regulator; Dna-binding, transcription regulation, bacterial regulatory proteins, dna/rna-binding 3-helical bundle fold, helix turn helix motif, hth motif; HET: PE8, MSE; 2.55A {Archaeoglobus fulgidus}
</pre><pre>Probab=24.03  E-value=1.6e+02  Score=21.20  Aligned_cols=38  Identities=16%  Similarity=0.242  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|.++++|++..
</pre><pre>T Consensus        10 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~~~y~~f~~   47 (170)
</pre><pre>T 3EGQ_A           10 IEAALRLYMKKPPHEVSIEEIAREAKVSKSLIFYHFES   47 (170)
</pre><pre>T ss_dssp             HHHHHHHHTTSCGGGCCHHHHHHHHTSCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHhhCCcccCCHHHHHHHhCCCHHHHHHHcCC
</pre><pre>
</pre><pre>
</pre><pre>No 225
</pre><pre>>5FGL_B NicR; co-crystal derepression, Nicotine regulator 2, APOPTOSIS; HET: 6HY; 2.4A {Pseudomonas putida (strain S16)}
</pre><pre>Probab=24.01  E-value=1.9e+02  Score=21.64  Aligned_cols=37  Identities=11%  Similarity=0.103  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+.+.++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus         5 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   41 (208)
</pre><pre>T 5FGL_B            5 LNEAEKVFAMHGFLGATLKQIAQNSNVTQALITYYYG   41 (208)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHCCHHHHHHHHTSCHHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHHhcccCCCHHHHHHHcCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 226
</pre><pre>>1K0F_A Periplasmic zinc-binding protein troA; apo protein, helix backbone, closed conformation, TRANSPORT PROTEIN; 2.5A {Treponema pallidum} SCOP: c.92.2.2
</pre><pre>Probab=23.88  E-value=6.5e+02  Score=21.39  Aligned_cols=119  Identities=8%  Similarity=0.047  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH---HhH-hcCCCccchHHHHhCCCc
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF---SKT-HLNLEEYNSIKKALGLST   84 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~---~~~-~~~~~~~~~lk~~LglSt   84 (174)
</pre><pre>                      ..+....+.|.+.|.+++-.+.+.|.     .+...|..+-..+-++++..+...   .+. -..-|.+.-+-+.+|+..
</pre><pre>T Consensus       108 ~~~~~~a~~i~~~L~~~~P~~~~~~~-----~n~~~~~~~l~~l~~~~~~~~~~~~~~~~~vi~~h~~~~yl~~~~gl~v  182 (277)
</pre><pre>T 1K0F_A          108 KLWSYSVKAVYESLCKLLPGKTREFT-----QRYQAYQQQLDKLDAYVRRKAQSLPAERRVLVTAHDAFGYFSRAYGFEV  182 (277)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHHSCGGGGEEEEEESCCHHHHHHHTCEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHCcccHHHHH-----HHHHHHHHHHHHHHHHHHHHHhcCChhhCEEEeechhHHHHHHHhCCEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             chhhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHH
</pre><pre>Q FD01845382_026   85 SSIMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACS  140 (174)
</pre><pre>Q Consensus        85 ks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~  140 (174)
</pre><pre>                      ..+.        +.-++.......+.++.+.++-.+.-.|-++.......+..-..
</pre><pre>T Consensus       183 ~~~~--------~~~~~~~~s~~~l~~l~~~~~~~~~~~ii~~~~~~~~~~~~l~~  230 (277)
</pre><pre>T 1K0F_A          183 KGLQ--------GVSTASEASAHDMQELAAFIAQRKLPAIFIESSIPHKNVEALRD  230 (277)
</pre><pre>T ss_dssp             EEEE--------CSCSSSCCCHHHHHHHHHHHHHTTCSEEEEETTSCGGGHHHHHH
</pre><pre>T ss_pred             Eeec--------CCCCCCCCCHHHHHHHHHHHHHcCCCEEEECCCCChHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 227
</pre><pre>>PF17402.6 ; DUF5408 ; Family of unknown function (DUF5408)
</pre><pre>Probab=23.83  E-value=2.3e+02  Score=19.80  Aligned_cols=25  Identities=4%  Similarity=-0.005  Sum_probs=0.0  Template_Neff=3.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    6 YISEVGKSYYEEINKILIEIQELYK   30 (174)
</pre><pre>Q Consensus         6 ~~~~~~~~~l~~i~~~L~~ie~ly~   30 (174)
</pre><pre>                      ++++.+++.++.+++-++++++.+.
</pre><pre>T Consensus        39 tAtA~mskkik~le~kl~~le~~~~   63 (63)
</pre><pre>T C5ZXC1_9HELI/7   39 TATANLSKNQKILEQKILILEKQQK   63 (63)
</pre><pre>T ss_pred             HHHhhHHHHHHHHHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 228
</pre><pre>>3RH2_A Hypothetical TetR-like transcriptional regulator; DNA/RNA-binding 3-helical bundle, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-Biology, DNA; HET: GOL, PO4, MSE; 2.42A {Shewanella amazonensis}
</pre><pre>Probab=23.77  E-value=1.9e+02  Score=21.67  Aligned_cols=38  Identities=16%  Similarity=0.233  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++..
</pre><pre>T Consensus         9 ~~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~f~~   46 (212)
</pre><pre>T 3RH2_A            9 IQASLELFNEHGERTITTNHIAAHLDISPGNLYYHFRN   46 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHCSS
</pre><pre>T ss_pred             HHHHHHHHHHhCCCcCCHHHHHHHcCCCHHHHHHhcCC
</pre><pre>
</pre><pre>
</pre><pre>No 229
</pre><pre>>4MO7_A Transcriptional regulator I2; TetR, Superantigen, Transcription; HET: CME; 1.701A {Pseudomonas fluorescens A506}
</pre><pre>Probab=23.75  E-value=1.9e+02  Score=21.52  Aligned_cols=38  Identities=21%  Similarity=0.426  Sum_probs=0.0  Template_Neff=12.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhcccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWDK   93 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~~   93 (174)
</pre><pre>                      ++..+.-+...++.--....|-+..|+|.++++|++..
</pre><pre>T Consensus        35 ~~aa~~l~~~~g~~~~s~~~ia~~agvs~~t~y~~F~~   72 (212)
</pre><pre>T 4MO7_A           35 LQTAAHLFRNKGFERTTVRDLASAVGIQSGSIFHHFKS   72 (212)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHTSCGGGGGGTSSS
</pre><pre>T ss_pred             HHHHHHHHHhcCcccCCHHHHHHHhCCCHHHHHhhCCC
</pre><pre>
</pre><pre>
</pre><pre>No 230
</pre><pre>>6SIH_C Flagellar hook-associated protein 2; Flagellum, Flagella, FliD, HAP2, Campylobacter jejuni, C.jejuni, transport protein;{Campylobacter jejuni}
</pre><pre>Probab=23.73  E-value=8.8e+02  Score=24.37  Aligned_cols=56  Identities=7%  Similarity=0.068  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSK   65 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~   65 (174)
</pre><pre>                      .......+.+.++++++-|+..++.|.++|..      .++...+-+.....|..+|..+..
</pre><pre>T Consensus       605 l~~~i~~l~~~i~~~~~rl~~~~~rl~~qF~a------mE~~ls~l~sq~~~L~~~l~~~~~  660 (663)
</pre><pre>T 6SIH_C          605 LTNDIKSLNTSKDSTQAMIDTRYDTMANQWLQ------YESILNKLNQQLNTVTNMINAANN  660 (663)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHC--
</pre><pre>T ss_pred             HHHHHHHHHhhHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 231
</pre><pre>>5OVY_B Putative transcriptional regulator, TetR family; TetR, efflux pump regulation, antibiotic resistance, DNA BINDING PROTEIN; 1.9A {Mycobacterium abscessus (strain ATCC 19977 / DSM 44196 / CIP 104536 / JCM 13569 / NCTC 13031 / TMC 1543)}
</pre><pre>Probab=23.73  E-value=1.3e+02  Score=22.85  Aligned_cols=36  Identities=14%  Similarity=0.302  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.- ....|-+..|+|..+++|++.
</pre><pre>T Consensus        24 ~~aa~~l~~~~g~~~-t~~~ia~~agvs~~~~y~~f~   59 (222)
</pre><pre>T 5OVY_B           24 IAAARELFAVHGLES-TLNEVAHHAGLGVGTVYRRFP   59 (222)
</pre><pre>T ss_dssp             HHHHHHHHHHHGGGC-CHHHHHHHHTCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHhCHHH-HHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 232
</pre><pre>>4UT1_A FLAGELLAR HOOK-ASSOCIATED PROTEIN; MOTOR PROTEIN, ANTIGEN, EPITOPE DISCOVERY; HET: EDO; 1.8A {BURKHOLDERIA PSEUDOMALLEI K96243}
</pre><pre>Probab=23.68  E-value=4.4e+02  Score=26.49  Aligned_cols=54  Identities=15%  Similarity=0.160  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhccccCC-CCHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    7 ISEVGKSYYEEINKILIEIQELYKNTYTYNKGI-ISGEEFFGQRARLFKKLDTILN   61 (174)
</pre><pre>Q Consensus         7 ~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~-l~~~~F~~~R~~Lf~~Ld~~l~   61 (174)
</pre><pre>                      ....+....++|...+++|.++=++.... .+. -...+.+.+|.+|+.+|-..++
</pre><pre>T Consensus       165 ~~~~i~~~v~~iN~l~~~Ia~LN~~I~~~-~~~g~~~ndL~DqRd~ll~eLS~~v~  219 (673)
</pre><pre>T 4UT1_A          165 VNSQLTDTVTQINSYTSQIAQLNEQIASA-SSQGQPPNQLLDQRDLAVSKLSQLAG  219 (673)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHH-TC----CHHHHHHHHHHHHHHHTTSC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHH-hcCCCCCchHHHHHHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 233
</pre><pre>>6KO8_A Putative regulatory protein; multisite binding pocket, HTH-MOTIF, DNA BINDING PROTEIN; HET: CHD; 1.55A {Salmonella enterica subsp. enterica serovar Typhimurium str. 14028S}
</pre><pre>Probab=23.49  E-value=1.9e+02  Score=22.66  Aligned_cols=36  Identities=8%  Similarity=0.284  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      ++..+.-+...++ -.....|-+..|+|.++++||+.
</pre><pre>T Consensus        15 l~aa~~ll~~~g~-~~s~~~ia~~agvs~~~~y~~F~   50 (194)
</pre><pre>T 6KO8_A           15 LEAATQAIAQSGI-AASTAVIARNAGVAEGTLFRYFA   50 (194)
</pre><pre>T ss_dssp             HHHHHHHHHHHCT-TCCHHHHHHHSSSCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHCC-CCCHHHHHHHhCCCHHHHHHHCC
</pre><pre>
</pre><pre>
</pre><pre>No 234
</pre><pre>>7WKK_H IL4I1 protein; nuclear pore complex, inner ring, Nup205, Nup93, Nup188, Nup155, NDC1, Aladin, STRUCTURAL PROTEIN; 4.2A {Xenopus laevis}
</pre><pre>Probab=23.38  E-value=1.1e+03  Score=23.56  Aligned_cols=126  Identities=13%  Similarity=0.138  Sum_probs=0.0  Template_Neff=3.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhh
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIM   88 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~v   88 (174)
</pre><pre>                      +.+..+-+++++.|.-+|+.+++.... --....++=++++=+|-+.+|.+|++|..      +...|-+.|+-+...--
</pre><pre>T Consensus       419 d~IlsqQkELed~Lt~lE~svk~qsgs-~~lq~~deErE~tYkLAEnIdaqLKrMaq------DLkdIIEhLNt~~~pad  491 (547)
</pre><pre>T 7WKK_H          419 DFILSQQKELEDLLTPLEESVKEQSGT-IYLQHADEEREKTYKLAENIDAQLKRMAQ------DLKEVIEHLNTSAGPGD  491 (547)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHH-HTSSSSCCCCCTTHHHHHHHHHHHHHHHH------HHHHHHHHHHHHC----
</pre><pre>T ss_pred             HHHHHchHHHHHHHhhhHHHHHHhhcc-eeccCchHHHHHHHHHHHHHHHHHHHHHH------HHHHHHHHHHhcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcccccCc-ccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHcCChHHcceeeee
</pre><pre>Q FD01845382_026   89 HQWDKTGV-DNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSKGRESACRKAAIT  153 (174)
</pre><pre>Q Consensus        89 h~w~~~~~-g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~g~~~~C~k~~~~  153 (174)
</pre><pre>                      ..+.-.-. .-++++-+.++-|-+=+..|.+            ....|.+.|+.-+.++-+..+..
</pre><pre>T Consensus       492 tsdPLqQIcKILNAHmdSLQWIDQnsalLQr------------KVEevtkv~E~rrkeqer~~ri~  545 (547)
</pre><pre>T 7WKK_H          492 ASNPLQQICKILNAHMDSLQWIDQNSALLQR------------KVEQVTKECESRRKEQERGFSIA  545 (547)
</pre><pre>T ss_dssp             --CCHHHHHHHHHHHHHHHHHHHHHHHHHHH------------HHHHC------------------
</pre><pre>T ss_pred             CCCHHHHHHHHHHHHHHHHHHHHHhHHHHHH------------HHHHHHHHHHHHHHHHhccceec
</pre><pre>
</pre><pre>
</pre><pre>No 235
</pre><pre>>5HAW_A Nucleoid occlusion factor SlmA; SlmA, nucleoid occlusion, FTsZ, cytokinesis, CELL CYCLE-DNA complex; 1.89A {Vibrio cholerae serotype O1 (strain ATCC 39315 / El Tor Inaba N16961)}
</pre><pre>Probab=23.33  E-value=1.7e+02  Score=21.23  Aligned_cols=37  Identities=11%  Similarity=0.252  Sum_probs=0.0  Template_Neff=13.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhH-hcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKT-HLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~-~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+... ++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        14 ~~aa~~l~~~~~g~~~~s~~~ia~~agvs~~~~y~~f~   51 (196)
</pre><pre>T 5HAW_A           14 LQALAEMLESNEGASRITTAKLAKQVGVSEAALYRHFP   51 (196)
</pre><pre>T ss_dssp             HHHHHHHHHSGGGGSCCCHHHHHHHHTSCHHHHTTTCS
</pre><pre>T ss_pred             HHHHHHHHHhCCCcccccHHHHHHHhCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 236
</pre><pre>>2QKO_B Possible transcriptional regulator, TetR family protein; Rhodococcus sp., tetR family protein, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE; 2.35A {Rhodococcus sp.}
</pre><pre>Probab=23.29  E-value=2e+02  Score=21.77  Aligned_cols=37  Identities=0%  Similarity=0.127  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.-.....|-+..|+|..+++|++.
</pre><pre>T Consensus        34 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   70 (215)
</pre><pre>T 2QKO_B           34 VNAAIEVLAREGARGLTFRAVDVEANVPKGTASNYFP   70 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHTCTTTCCHHHHHHHSSSTTTCHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHcccccCCHHHHHHHcCCChhHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 237
</pre><pre>>6WP9_D AvaR1; Transcriptional Repressor bound ligand, DNA BINDING PROTEIN, DNA BINDING PROTEIN-INHIBITOR complex, DNA BINDING PROTEIN-HORMONE complex; HET: XVR; 2.0A {Streptomyces avermitilis}
</pre><pre>Probab=23.22  E-value=2e+02  Score=22.36  Aligned_cols=37  Identities=11%  Similarity=0.243  Sum_probs=0.0  Template_Neff=12.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        26 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   62 (245)
</pre><pre>T 6WP9_D           26 LVAAAEVFDEVGYEAATISDVLKRSGVTKGALYFHFT   62 (245)
</pre><pre>T ss_dssp             HHHHHHHHHHHCTTTCCHHHHHHHHCCCHHHHHHHCS
</pre><pre>T ss_pred             HHHHHHHHHHHCcccCCHHHHHHHhCCCHHHHHHhCC
</pre><pre>
</pre><pre>
</pre><pre>No 238
</pre><pre>>7UZK_S Effector SidK; V-ATPase, Complex, Membrane protein, PROTON TRANSPORT; HET: ADP; 3.0A {Rattus norvegicus}
</pre><pre>Probab=23.04  E-value=8.5e+02  Score=22.00  Aligned_cols=92  Identities=17%  Similarity=0.321  Sum_probs=0.0  Template_Neff=2.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHH----HHhccccCC-CCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCC----ccchHHHH
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYK----NTYTYNKGI-ISGEEFFGQRARLFKKLDTILNKFSKTHLNLE----EYNSIKKA   79 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~----~~~~~~~g~-l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~----~~~~lk~~   79 (174)
</pre><pre>                      +.....-+++++.+.-+++.|+    -+..+ .|+ ++.-+-..+|.-.+++|+....-+-.+.+...    .|.++|+.
</pre><pre>T Consensus        40 ~~~q~IR~DYq~~i~W~~k~~~F~~I~eASK-sGKCP~DlDtLsrkaLilQEL~rLVslI~PF~mk~e~ies~Y~kmK~h  118 (280)
</pre><pre>T 7UZK_S           40 NNLKKIREDYQDVLIWAEKNYRFEEILEASK-SGKCPNDLDALSRRSLILQELLRLVSSISPFKMKLDLIESQYEKMKQH  118 (280)
</pre><pre>T ss_dssp             SCCHHHHHHTHHHHTTSSSCCCHHHHHHHHH-TTSSCCCHHHHHHHHHHHHHHHHHHHHSCSSCCCHHHHHHHHHHHHTC
</pre><pre>T ss_pred             hhHHHHHHHHHHHHHHHhhccCHHHHHHHHh-cCCCCCchHHhhHHHHHHHHHHHHHHhcCCcccCHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hCC-CcchhhhcccccCcccchHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026   80 LGL-STSSIMHQWDKTGVDNIERYATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus        80 Lgl-Stks~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      -+| |+.                |-.+...+..+.++++
</pre><pre>T Consensus       119 ~NLW~S~----------------Yh~~LN~L~qL~~~~~  141 (280)
</pre><pre>T 7UZK_S          119 VNLWKSD----------------YHVKLNQLNQLTDYLK  141 (280)
</pre><pre>T ss_dssp             SCGGGSH----------------HHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             cccchhH----------------HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 239
</pre><pre>>4H0F_B Laminin-binding surface protein; Adhesin, Human Laminin, CELL ADHESION; 2.4A {Streptococcus agalactiae} SCOP: c.92.2.0
</pre><pre>Probab=23.02  E-value=5.8e+02  Score=21.57  Aligned_cols=116  Identities=13%  Similarity=0.077  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhH--hcCCCccchHHHHhCCCcch
</pre><pre>Q FD01845382_026    9 EVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKT--HLNLEEYNSIKKALGLSTSS   86 (174)
</pre><pre>Q Consensus         9 ~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~--~~~~~~~~~lk~~LglStks   86 (174)
</pre><pre>                      ..+....+.|.+.|.++.-.+...+.     -+...|..+-+.|..+++..+..+-.+  -..-|.+.-+-+.+|+....
</pre><pre>T Consensus       106 ~~~~~~~~~i~~~L~~~~p~~~~~~~-----~n~~~~~~~l~~l~~~~~~~l~~~~~~~~~~~h~~~~y~~~~~gl~v~~  180 (264)
</pre><pre>T 4H0F_B          106 VLAGEEAVNIAKELGHLDPKHKDSYT-----KKAKAFKKEAEQLTEEYTQKFKKVRSKTFVTQHTAFSYLAKRFGLKQLG  180 (264)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHCGGGHHHHH-----HHHHHHHHHHHHHHHHHHHHHHTCSCCEEEEEESCCHHHHHHTTCEEEE
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhhhChhcHHHHH-----HHHHHHHHHHHHHHHHHHHHhcccCCCeEEeecchHHHHHHHhCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hhhcccccCcccchHHHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHH
</pre><pre>Q FD01845382_026   87 IMHQWDKTGVDNIERYATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYD  137 (174)
</pre><pre>Q Consensus        87 ~vh~w~~~~~g~Ip~~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~  137 (174)
</pre><pre>                      ++        ..-++....-+.+..+.+.++--+--.|-.+..........
</pre><pre>T Consensus       181 ~~--------~~~~~~~~s~~~l~~l~~~~~~~~v~~i~~~~~~~~~~~~~  223 (264)
</pre><pre>T 4H0F_B          181 IS--------GISPEQEPSPRQLKEIQDFVKEYNVKTIFAEDNVNPKIAHA  223 (264)
</pre><pre>T ss_dssp             EE--------CSSTTSCCCHHHHHHHHHHHHHTTCCEEEECTTSCCHHHHH
</pre><pre>T ss_pred             Ee--------cCCCCCCCCHHHHHHHHHHHHHcCCCEEEECCCCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 240
</pre><pre>>5T51_A KLLA0F02343p; cell cycle, kinetochore, complex, chromosome, segregation, MIND, Mis12; 2.2007A {Kluyveromyces lactis (strain ATCC 8585 / CBS 2359 / DSM 70799 / NBRC 1267 / NRRL Y-1140 / WM37)}
</pre><pre>Probab=22.95  E-value=5.2e+02  Score=19.54  Aligned_cols=66  Identities=12%  Similarity=0.069  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHH--------HHhHhcCCCcc
</pre><pre>Q FD01845382_026    7 ISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNK--------FSKTHLNLEEY   73 (174)
</pre><pre>Q Consensus         7 ~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~--------~~~~~~~~~~~   73 (174)
</pre><pre>                      .........+.+.+.+..+++.-...+.. .+.....+...-...|+..|...+.+        +++.-+.+|+.
</pre><pre>T Consensus        24 iddv~n~v~~~~~d~~d~le~~l~~~~~~-~~~~~~~~l~~g~~~l~~~l~~~~dk~~d~fE~y~l~niF~IP~~   97 (120)
</pre><pre>T 5T51_A           24 VDDIINAVNEIMYKCTNAMEKYLMQRNII-GKKDFSDEIKIGTAKLESLLENSVDKNFDKLELYVLRNILSIPSD   97 (120)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHTCEE-TTEECHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTSCCCTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHhCCC-CCCCcHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhCccCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 241
</pre><pre>>4J2C_C Syntaxin-6; Tethering complex, PROTEIN TRANSPORT; 1.801A {Homo sapiens}
</pre><pre>Probab=22.94  E-value=4.6e+02  Score=18.85  Aligned_cols=60  Identities=7%  Similarity=0.069  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026    4 LGYISEVGKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKF   63 (174)
</pre><pre>Q Consensus         4 ~G~~~~~~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~   63 (174)
</pre><pre>                      ......-+...++.++..|.+|++.-.-..+.+.-.-..++-..+|+.+..++...+..|
</pre><pre>T Consensus        43 ~~~~~~~l~~~~~~l~~~l~dL~~tv~~v~~np~kf~i~~~El~~Rr~fv~~~~~~l~~i  102 (110)
</pre><pre>T 4J2C_C           43 IDWTTNELRNNLRSIEWDLEDLDETISIVEANPRKFNLDATELSIRKAFITSTRQVVRDM  102 (110)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHSHHHHCCCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHhhHHHHHHHHHHCHHHcCCCHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 242
</pre><pre>>5D1W_C Rv3249c transcriptional regulator; Mycobacterium tuberculosis, transcriptional regulator, TRANSCRIPTION; HET: PLM; 3.59A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)}
</pre><pre>Probab=22.90  E-value=1.7e+02  Score=22.37  Aligned_cols=37  Identities=16%  Similarity=0.340  Sum_probs=0.0  Template_Neff=12.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|+|.
</pre><pre>T Consensus        29 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   65 (217)
</pre><pre>T 5D1W_C           29 LDAMRDLLLTRDWSAITLSDVARAAGISRQTIYNEFG   65 (217)
</pre><pre>T ss_dssp             HHHHHHHHTTSCTTSCCHHHHHTTTTSCSTTHHHHHC
</pre><pre>T ss_pred             HHHHHHHHHHcChhhCCHHHHHHHhCCCHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 243
</pre><pre>>3VRC_B Cytochrome c'; c-type cytpchrome, ELECTRON TRANSPORT; HET: PG4, HEC; 1.0A {Thermochromatium tepidum} SCOP: a.24.3.2
</pre><pre>Probab=22.89  E-value=4.9e+02  Score=19.22  Aligned_cols=101  Identities=13%  Similarity=0.088  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHhHhcCC-C--ccchHHHHhCCCcchhhhcc-cccCc-c--------------cchH
</pre><pre>Q FD01845382_026   41 SGEEFFGQRARLFKKLDTILNKFSKTHLNL-E--EYNSIKKALGLSTSSIMHQW-DKTGV-D--------------NIER  101 (174)
</pre><pre>Q Consensus        41 ~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~-~--~~~~lk~~LglStks~vh~w-~~~~~-g--------------~Ip~  101 (174)
</pre><pre>                      ..++....|+.+++.|...+..+.....+. .  .....-+.|.-....+.+.| ..... +              .-+.
</pre><pre>T Consensus         4 ~~~~~v~~r~~~M~~~~~~~~~l~~~~~~~~d~~~~~~~a~~l~~~~~~~~~~~~~~~~~~~~~~~~~~a~~~iw~~~~~   83 (131)
</pre><pre>T 3VRC_B            4 SPEEQIETRQAGYAFMAWNMGKIKANLEGEYNADQVRAAANVVAAIANSGMGALYGPGTDKNVGAVKTRAKPELFQNLED   83 (131)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHHHHHHHHHHHHHSSCCHHHHHHHHHHHHHHHTSCGGGGCCTTTTSCBTTBCCCBCHHHHTCHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHHhhchhhcCCCCCCCCCCccccCCCHHHHhCHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhhhcchheEeehhhhHHHHHHHHHc
</pre><pre>Q FD01845382_026  102 YATYIENSAKLMKLMKTVGFVGIGLDLAGYTSNVYDACSK  141 (174)
</pre><pre>Q Consensus       102 ~~~~i~~~~k~sk~lk~gG~v~i~ld~~~~~~~I~~aC~~  141 (174)
</pre><pre>                      |...+......+..|...---.=--........|.+.|..
</pre><pre>T Consensus        84 f~~~~~~~~~~~~~l~~a~~~~d~~~~~~~~~~v~~~C~~  123 (131)
</pre><pre>T 3VRC_B           84 VGKLARDLGTAANALAAAAATGEANAVKSAFADVGAACKA  123 (131)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHTTCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHcCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 244
</pre><pre>>1V2Z_A circadian clock protein KaiA homolog; ALL ALPHA, RIKEN Structural Genomics/Proteomics Initiative, RSGI, Structural Genomics, circadian clock protein; 1.8A {Thermosynechococcus elongatus} SCOP: a.186.1.1, l.1.1.1
</pre><pre>Probab=22.73  E-value=5.8e+02  Score=19.99  Aligned_cols=63  Identities=16%  Similarity=0.205  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhHhcCCC--ccchHHHHhCCCcchhhhcccccCcccch---HHHHHHHHHHHHHHHHhhhc
</pre><pre>Q FD01845382_026   47 GQRARLFKKLDTILNKFSKTHLNLE--EYNSIKKALGLSTSSIMHQWDKTGVDNIE---RYATYIENSAKLMKLMKTVG  120 (174)
</pre><pre>Q Consensus        47 ~~R~~Lf~~Ld~~l~~~~~~~~~~~--~~~~lk~~LglStks~vh~w~~~~~g~Ip---~~~~~i~~~~k~sk~lk~gG  120 (174)
</pre><pre>                      .+|+.++++|......+.-.-|.-+  ....|.+..+..=-           .+||   -..=||+-+-.+||.||.=|
</pre><pre>T Consensus        12 ~e~~~ll~~Lk~~Yr~Iil~YF~~~~~~n~~ID~Fvn~aFf-----------adi~v~qivEIHMeLmDefskQLKLEG   79 (111)
</pre><pre>T 1V2Z_A           12 ADKRKLLDELRSIYRTIVLEYFNTDAKVNERIDEFVSKAFF-----------ADISVSQVLEIHVELMDTFSKQLKLEG   79 (111)
</pre><pre>T ss_dssp             SCCHHHHHHHHHHHHHHHHHTTCTTSCHHHHHHHHHHHHHH-----------HTCCHHHHHHHHHHHHHHHHHHTTCCG
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHhhCCChhHHHHHHHHHHHHHh-----------CCCCHHHHHHHHHHHHHHHHHHHhhCC
</pre><pre>
</pre><pre>
</pre><pre>No 245
</pre><pre>>3C2B_A Transcriptional regulator, TetR family; structural genomics, APC5923, TetR, transcription, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, TRANSCRIPTION; HET: MSE; 2.1A {Agrobacterium tumefaciens str.}
</pre><pre>Probab=22.69  E-value=1.5e+02  Score=22.45  Aligned_cols=37  Identities=16%  Similarity=0.157  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++||+.
</pre><pre>T Consensus        21 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~t~y~~F~   57 (221)
</pre><pre>T 3C2B_A           21 LDQALRLLVEGGEKALTTSGLARAANCSKESLYKWFG   57 (221)
</pre><pre>T ss_dssp             HHHHHHHHHHHCGGGCCHHHHHHHHTCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHhcCHhhCCHHHHHHHhCCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 246
</pre><pre>>PF03568.21 ; Peptidase_C50 ; Peptidase family C50
</pre><pre>Probab=22.59  E-value=4.4e+02  Score=24.64  Aligned_cols=58  Identities=12%  Similarity=0.171  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcC
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLN   69 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~   69 (174)
</pre><pre>                      +...++++++++++.++.-...... ...-...+|-.+|+.|=++|+..|..|-..-++
</pre><pre>T Consensus        39 ~~~~~~~~~~Ii~~~~~~~~~~~~~-~~~~~k~~wW~~R~~lD~~l~~ll~~ie~~wlg   96 (460)
</pre><pre>T G4UQF5_NEUT9/1   39 YQQGRAELLEIIRLANETCHDARDF-TVKGAKTAWWADREALDNRMKDLLTNIEQIWLG   96 (460)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHhhccCCcC-CchHHHHHHHHHHHHHHHHHHHHHHHHHHHchh
</pre><pre>
</pre><pre>
</pre><pre>No 247
</pre><pre>>2QTQ_C Transcriptional regulator, TetR family; Transcription regulator, dna/rna-binding 3-helical bundle fold, helix turn helix motif, hth motif, transcription regulation, structural genomics; HET: EDO, MSE, NO3; 1.85A {Novosphingobium aromaticivorans}
</pre><pre>Probab=22.56  E-value=1.9e+02  Score=21.64  Aligned_cols=37  Identities=14%  Similarity=0.127  Sum_probs=0.0  Template_Neff=12.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhccc
</pre><pre>Q FD01845382_026   56 LDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQWD   92 (174)
</pre><pre>Q Consensus        56 Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~w~   92 (174)
</pre><pre>                      |+..+.-+...++.--....|-+..|+|..+++|++.
</pre><pre>T Consensus        22 l~aa~~l~~~~g~~~~t~~~ia~~agvs~~~~y~~F~   58 (213)
</pre><pre>T 2QTQ_C           22 LQTASNIMREGDVVDISLSELSLRSGLNSALVKYYFG   58 (213)
</pre><pre>T ss_dssp             HHHHHHHHHHHTCSCCCHHHHHHHHTCCHHHHHHHHS
</pre><pre>T ss_pred             HHHHHHHHHhcCCCCCCHHHHHHHcCCCHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 248
</pre><pre>>2IM8_B Hypothetical protein yppE; SR213, yppE, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, UNKNOWN FUNCTION; HET: MSE; 2.0A {Bacillus subtilis} SCOP: a.24.26.1
</pre><pre>Probab=22.45  E-value=6e+02  Score=20.03  Aligned_cols=84  Identities=10%  Similarity=0.128  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHHhccccCCCCHHHHHHHHHHHHHHHHHHHHHHHhHhcCCCccchHHHHhCCCcchhhhc
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNTYTYNKGIISGEEFFGQRARLFKKLDTILNKFSKTHLNLEEYNSIKKALGLSTSSIMHQ   90 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~~~~~~g~l~~~~F~~~R~~Lf~~Ld~~l~~~~~~~~~~~~~~~lk~~LglStks~vh~   90 (174)
</pre><pre>                      ....+...++.+..++++++..... ...-...+||.+-+-...+.+..+..|..         ....-+.....-.+|.
</pre><pre>T Consensus         3 ~~~l~~~T~~Ll~~~~~~~~~y~~~-r~~~~~~DF~~eVKPf~d~v~~~~~~W~~---------~a~~~i~~~~pkyih~   72 (131)
</pre><pre>T 2IM8_B            3 SQTLLEMTEQMIEVAEKGADRYQEG-KNSNHSYDFFETIKPAVEENDELAARWAE---------GALELIKVRRPKYVHK   72 (131)
</pre><pre>T ss_dssp             SCCHHHHHHHHHHHHHHHHHHHHHH-HHHCCCCCCCCCCHHHHHHHHHHHHHHHH---------HHHHHHHHTCC---CC
</pre><pre>T ss_pred             hhHHHHHHHHHHHHHHHHHHHHHHc-cCCCCCCchHHhhHHhhhHHHHHHHHHHH---------HHHHHHHhhCCCCCCH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             ccccCcccchHHHHHHHHHH
</pre><pre>Q FD01845382_026   91 WDKTGVDNIERYATYIENSA  110 (174)
</pre><pre>Q Consensus        91 w~~~~~g~Ip~~~~~i~~~~  110 (174)
</pre><pre>                            ..|....++|+.++
</pre><pre>T Consensus        73 ------~Qie~t~eNie~ls   86 (131)
</pre><pre>T 2IM8_B           73 ------EQIEAVKDNFLELV   86 (131)
</pre><pre>T ss_dssp             ------STHHHHHHHHHHHH
</pre><pre>T ss_pred             ------HHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 249
</pre><pre>>5ZVM_b pan-CoV inhibitory peptide EK1; SARS, Spike protein, S2 domain, HR1 motif, pan-Coronavirus, VIRAL PROTEIN-INHIBITOR complex; 3.3A {Human SARS coronavirus}
</pre><pre>Probab=22.33  E-value=2.6e+02  Score=18.16  Aligned_cols=22  Identities=9%  Similarity=0.199  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01845382_026   11 GKSYYEEINKILIEIQELYKNT   32 (174)
</pre><pre>Q Consensus        11 ~~~~l~~i~~~L~~ie~ly~~~   32 (174)
</pre><pre>                      +...++.++++.++|++.|.+.
</pre><pre>T Consensus        18 l~~ei~~L~evik~LN~S~IdL   39 (44)
</pre><pre>T 5ZVM_b           18 LEYEMKKLEEAIKKLEESYIDL   39 (44)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHHHTTCCCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhccH
</pre><pre>
</pre><pre>
</pre><pre>No 250
</pre><pre>>6SID_A Activity-regulated cytoskeleton associated protein 1; Arc, Gag homology, atomic resolution, capsid assembly, VIRUS LIKE PARTICLE; 1.05A {Drosophila melanogaster}
</pre><pre>Probab=22.15  E-value=3.9e+02  Score=18.59  Aligned_cols=78  Identities=10%  Similarity=0.155  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhccccC-CCCHHHHHHHHHHHHHHHHHH-HHHHHhHhcCCC-ccchHHHHhCCCcchhhhcccccCcccchHH
</pre><pre>Q FD01845382_026   26 QELYKNTYTYNKG-IISGEEFFGQRARLFKKLDTI-LNKFSKTHLNLE-EYNSIKKALGLSTSSIMHQWDKTGVDNIERY  102 (174)
</pre><pre>Q Consensus        26 e~ly~~~~~~~~g-~l~~~~F~~~R~~Lf~~Ld~~-l~~~~~~~~~~~-~~~~lk~~LglStks~vh~w~~~~~g~Ip~~  102 (174)
</pre><pre>                      .+++.+.+...++ .-...+|...-+.||.+|... +..-....+... -.+.++..|   ..           ..++.+
</pre><pre>T Consensus         8 ~~l~~~i~~r~Q~~~E~~~~fi~~~~~L~~rl~~~~~~e~~~l~~i~~~l~~~~~~~l---~~-----------~~~~t~   73 (90)
</pre><pre>T 6SID_A            8 YQIYMEFFQNKQDDHDPIDTFVIQKRALLAQLPSGRHDEETELDLLFGLLNIKYRKHI---SR-----------HSVHTF   73 (90)
</pre><pre>T ss_dssp             HHHHHHHHHCCCCTTCCHHHHHHHHHHHHTTSCTTSSCHHHHHHHHHTTSCHHHHTTC---CG-----------GGCSSH
</pre><pre>T ss_pred             HHHHHHHHhCCCCCCCCHHHHHHHHHHHHhhCCCCCCCHHHHHHHHHHhcCHHHHHHc---cc-----------ccCCCH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHh
</pre><pre>Q FD01845382_026  103 ATYIENSAKLMKLMK  117 (174)
</pre><pre>Q Consensus       103 ~~~i~~~~k~sk~lk  117 (174)
</pre><pre>                      .+.+.....+-..++
</pre><pre>T Consensus        74 ~~L~~~~~~~e~~~~   88 (90)
</pre><pre>T 6SID_A           74 KDLLEQGRIIEHNNQ   88 (90)
</pre><pre>T ss_dssp             HHHHHHHHHHHHC--
</pre><pre>T ss_pred             HHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre>