<pre>Query         WP_029789863.1/2180-2277 WP_029789863.1 RHS repeat-associated core domain-containing protein [Vibrio harveyi]
</pre><pre>Match_columns 88
</pre><pre>No_of_seqs    116 out of 525
</pre><pre>Neff          6.87218
</pre><pre>Searched_HMMs 86581
</pre><pre>Date          Mon Feb 26 20:54:12 2024
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -o ../results/5142420.hhr -oa3m ../results/5142420.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF14436.10 ; EndoU_bacteria ;   98.0 1.3E-05 1.5E-10   52.5   4.1   50   22-85     80-142 (143)
</pre><pre>  2 PF04365.17 ; BrnT_toxin ; Ribo  95.6    0.33 3.8E-06   28.3   7.5   51   36-86     17-76  (79)
</pre><pre>  3 PF18810.5 ; PBECR2 ; phage-Bar  94.6    0.58 6.7E-06   30.3   7.0   53   33-85     45-108 (132)
</pre><pre>  4 3U97_A Ribonuclease toxin BrnT  93.8     1.4 1.6E-05   28.6   7.5   50   36-86     54-108 (114)
</pre><pre>  5 PF14076.10 ; DUF4258 ; Domain   87.8     5.4 6.2E-05   22.4   6.9   48   36-83     16-74  (75)
</pre><pre>  6 PF15534.10 ; Ntox35 ; Bacteria  87.2       8 9.3E-05   23.8   7.5   50   31-83     23-75  (76)
</pre><pre>  7 7VD7_A toxin; Toxin-Antitoxin   87.1     7.5 8.6E-05   23.3   6.9   51   36-86     22-84  (94)
</pre><pre>  8 3NZL_A DNA-binding protein SAT  82.4     2.1 2.5E-05   27.4   2.0   22   27-48     14-35  (83)
</pre><pre>  9 PF16557.9 ; CUTL ; CUT1-like D  81.9       2 2.4E-05   26.6   1.7   22   27-48      6-27  (72)
</pre><pre> 10 PF12997.11 ; DUF3881 ; Domain   79.8      26  0.0003   26.4   7.2   58   26-84      6-74  (285)
</pre><pre> 11 PF19215.4 ; CoV_NSP15_C ; Coro  73.1      16 0.00018   26.0   4.2   22   63-84    132-154 (155)
</pre><pre> 12 5YRX_A Nucleoid-associated pro  58.3      45 0.00052   22.2   4.1   55   25-81      6-60  (139)
</pre><pre> 13 3O2I_B Uncharacterized protein  48.8      44 0.00051   22.8   2.8   23   26-48    100-122 (125)
</pre><pre> 14 3CTR_A Poly(A)-specific ribonu  45.3      39 0.00046   21.5   2.2   16   26-41     13-28  (101)
</pre><pre> 15 2C1W_C ENDOU PROTEIN; NUCLEASE  45.3      81 0.00093   24.2   4.1   23   63-85    259-282 (292)
</pre><pre> 16 8ARV_B Bifunctional diguanylat  44.4 1.6E+02  0.0019   20.4   5.6   66   12-81      1-68  (275)
</pre><pre> 17 PF20786.1 ; DUF6840 ; Family o  44.2      39 0.00045   22.0   1.9   19   26-44     74-92  (92)
</pre><pre> 18 PF13037.10 ; DUF3898 ; Domain   43.8      50 0.00058   21.7   2.4   17   28-44      4-20  (90)
</pre><pre> 19 PF16137.9 ; DUF4845 ; Domain o  43.0      48 0.00056   19.8   2.2   22   26-47      3-24  (85)
</pre><pre> 20 1OTR_A protein Cue2; PROTEIN-P  41.5      34 0.00039   18.8   1.3   18   23-41     12-29  (49)
</pre><pre> 21 2JV8_A Uncharacterized protein  39.6   1E+02  0.0012   19.0   3.3   21   63-83     22-43  (73)
</pre><pre> 22 PF12636.11 ; DUF3781 ; Protein  38.6 1.5E+02  0.0018   18.3   5.2   47   31-83     24-71  (71)
</pre><pre> 23 PF15631.10 ; Imm-NTF2-2 ; NTF2  36.9 1.7E+02   0.002   18.5   4.6   39   44-83     21-80  (83)
</pre><pre> 24 7O6N_C Protein pid-3; ERH-2 PI  36.6 1.1E+02  0.0012   16.8   2.7   25   19-43      1-26  (39)
</pre><pre> 25 PF20716.1 ; DUF6828 ; Domain o  35.1      74 0.00085   21.4   2.2   20   29-48     35-54  (99)
</pre><pre> 26 PF07380.15 ; Pneumo_M2 ; Pneum  33.7      75 0.00087   20.7   2.0   15   30-44     60-74  (89)
</pre><pre> 27 PF18780.5 ; HNH_repeat ; Homin  33.2   1E+02  0.0012   15.6   2.4   15   30-44      1-15  (54)
</pre><pre> 28 PF20735.1 ; Lambda_ea8_5 ; Bac  32.0      69 0.00079   20.2   1.7   24   22-45     41-70  (86)
</pre><pre> 29 PF21796.1 ; Cac1_C ; Chromatin  31.9      74 0.00086   18.3   1.7   21   23-44     23-43  (56)
</pre><pre> 30 PF10281.13 ; Ish1 ; Putative n  30.6 1.1E+02  0.0013   15.4   2.2   14   28-41      2-15  (38)
</pre><pre> 31 6WRW_C Computationally designe  30.1 1.7E+02   0.002   18.6   3.1   48   31-83      1-50  (88)
</pre><pre> 32 PF19394.3 ; DUF5969 ; Family o  28.7 2.8E+02  0.0033   18.4   7.7   52   31-82     31-86  (90)
</pre><pre> 33 4S1T_B Uridylate-specific endo  28.3 1.9E+02  0.0022   23.7   3.8   22   63-84    324-346 (349)
</pre><pre> 34 PF02198.20 ; SAM_PNT ; Sterile  27.3 1.2E+02  0.0014   16.9   2.1   18   27-44     17-34  (85)
</pre><pre> 35 PF19848.3 ; DUF6323 ; Family o  26.6   1E+02  0.0012   21.2   1.9   24   23-46     78-101 (127)
</pre><pre> 36 PF17412.6 ; VraX ; Family of u  26.3 1.7E+02  0.0019   17.4   2.5   20   26-45     29-48  (55)
</pre><pre> 37 5YVD_A Nsp15; MERS-CoV, anti-v  26.2   2E+02  0.0023   23.4   3.7   22   63-84    318-340 (343)
</pre><pre> 38 6TMG_P ATPTG10; mitochondrial,  25.7 1.2E+02  0.0014   21.5   2.2   19   29-47     52-70  (138)
</pre><pre> 39 5SBF_A Uridylate-specific endo  25.7 2.3E+02  0.0026   23.3   3.8   22   63-84    324-346 (349)
</pre><pre> 40 PF19542.3 ; DUF6066 ; Family o  25.4      91  0.0011   23.4   1.6   56   27-86    139-205 (212)
</pre><pre> 41 6MTZ_A HepC.19480.a.B1; SSGCID  25.4 2.3E+02  0.0026   21.1   3.7   32   11-46    321-355 (355)
</pre><pre> 42 PF07262.15 ; CdiI ; CDI immuni  25.1 1.8E+02  0.0021   19.3   2.9   34    9-44    120-153 (155)
</pre><pre> 43 PF07305.16 ; DUF1454 ; Protein  24.9 4.2E+02  0.0048   19.1   6.1   50   28-85    129-179 (185)
</pre><pre> 44 3JSL_B DNA ligase; DNA ligase,  24.9 3.9E+02  0.0045   20.4   4.9   25   58-83    107-131 (318)
</pre><pre> 45 7K72_C DNA ligase A; Structura  24.4 3.9E+02  0.0045   20.7   4.8   25   58-83    126-150 (336)
</pre><pre> 46 2GKP_A hypothetical protein NM  24.3 1.8E+02   0.002   19.8   2.8   33   10-44    132-164 (167)
</pre><pre> 47 PF20636.1 ; SMN_G2-BD ; Surviv  24.3 1.1E+02  0.0013   12.3   2.2   16   30-45      3-18  (18)
</pre><pre> 48 PF15002.10 ; ERK-JNK_inhib ; E  23.6 1.4E+02  0.0016   21.7   2.2   21   21-45     71-91  (198)
</pre><pre> 49 4QQ0_A Adenylate cyclase-like   22.6 4.1E+02  0.0047   20.2   4.6   36   30-65    115-154 (280)
</pre><pre> 50 4PY9_A Putative exopolyphospha  22.2 2.1E+02  0.0025   20.8   3.0   29   10-44    307-335 (343)
</pre><pre> 51 PF06236.15 ; MelC1 ; Tyrosinas  22.2 2.2E+02  0.0026   19.3   2.8   21   63-83     61-81  (109)
</pre><pre> 52 PF05256.16 ; UPF0223 ; Unchara  21.8 1.6E+02  0.0019   19.1   2.0   13   28-40      8-20  (86)
</pre><pre> 53 4GLW_A DNA ligase; DNA ligase   21.6 5.3E+02  0.0061   19.3   5.0   25   58-83    104-128 (305)
</pre><pre> 54 3BAC_A DNA ligase; Adenylation  21.2 5.3E+02  0.0061   19.0   6.6   49   31-83     24-72  (270)
</pre><pre> 55 8CSS_G 28S ribosomal protein S  21.0 1.4E+02  0.0016   24.5   1.9   17   27-43     96-112 (396)
</pre><pre> 56 PF09133.14 ; SANTA ; SANTA (SA  21.0      75 0.00086   19.5   0.4    7   26-32     83-89  (90)
</pre><pre> 57 PF08471.14 ; Ribonuc_red_2_N ;  20.8 1.8E+02  0.0021   19.6   2.2   20   25-44     12-31  (117)
</pre><pre> 58 4Q2J_A DNA-binding protein SAT  20.7 2.1E+02  0.0024   20.8   2.6   26   23-48    111-136 (180)
</pre><pre> 59 PF09412.14 ; XendoU ; Endoribo  20.3 3.4E+02  0.0039   20.4   3.7   22   63-84    230-256 (256)
</pre><pre> 60 1SXE_A Transcriptional regulat  20.2 2.1E+02  0.0025   17.1   2.3   18   27-44     27-44  (97)
</pre><pre> 61 2M7A_A Uncharacterized protein  20.1 2.4E+02  0.0028   18.3   2.6   20   26-45     60-85  (101)
</pre><pre> 62 8Q84_O DASH complex subunit SP  20.0 2.2E+02  0.0025   22.5   2.8   23   22-44    197-219 (295)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF14436.10 ; EndoU_bacteria ; Bacterial EndoU nuclease
</pre><pre>Probab=98.04  E-value=1.3e-05  Score=52.54  Aligned_cols=50  Identities=16%  Similarity=0.314  Sum_probs=39.3  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECC---EEEEEEEEe---CCe--E---EEEe--Ec
</pre><pre>Q WP_029789863.1   22 GKSEFPQGWSDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNG---IDIRVIQER---NGD--I---VSGF--PT   85 (88)
</pre><pre>Q Consensus        22 gkt~FP~~Ws~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dG---V~I~V~~~~---~g~--i---~TAy--P~   85 (88)
</pre><pre>                      ..|.||.+|+.++|+++|.++++      .        .....+|   +.|+++++.   +|.  |   +|+|  |+
</pre><pre>T Consensus        80 ~st~FP~~ws~~~I~~~i~~a~~------~--------~~~~~~G~~~~~i~~~~~~~~~~g~~~~~~~~t~y~~P~  142 (143)
</pre><pre>T B6INW4_RHOCS/8   80 ESVFFPDRYDEGHLMCALQSAYG------R--------RHRSHDGQWEATLDDGTVVEFRSGQDGALWPVPVYVPPL  142 (143)
</pre><pre>T ss_pred             eeeeCCCCCCHHHHHHHHHHHHh------h--------ccccCCCceeEEEEEEEecccCCCCccccccccccCCCC
</pre><pre>Confidence            45899999999999999999987      1        1126678   888887773   454  9   9999  96
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF04365.17 ; BrnT_toxin ; Ribonuclease toxin, BrnT, of type II toxin-antitoxin system
</pre><pre>Probab=95.62  E-value=0.33  Score=28.25  Aligned_cols=51  Identities=20%  Similarity=0.210  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCccceeCCC-----CCeEEEEEEECCEEEEEE-EE-eCC--eEEEEeEcC
</pre><pre>Q WP_029789863.1   36 MHEISDIATDPNAIRRNGR-----GGRTITEGTRNGIDIRVI-QE-RNG--DIVSGFPTN   86 (88)
</pre><pre>Q Consensus        36 ~~ai~~v~~~P~~~~~~~~-----~g~~~~~g~~dGV~I~V~-~~-~~g--~i~TAyP~~   86 (88)
</pre><pre>                      .+.+.+++.+|........     +.++...|..+|..+.|+ +. .++  .|+||++..
</pre><pre>T Consensus        17 ~~ev~~~~~~~~~~~~~~~~~~~~~~r~~~~g~~~gr~l~vv~~~~~~~~~~iisa~~~~   76 (79)
</pre><pre>T B6INL0_RHOCS/5   17 FGDAIPALEDPDRLERIDDRFAYGEERVQTIGAAAGGILFVVTTVNEEETCRILSARRAT   76 (79)
</pre><pre>T ss_pred             HHHHHHHHhCCCceEEEccccccCCeeEEEEEEeCCeEEEEEEEEecCCEEEEEEeeeCC
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF18810.5 ; PBECR2 ; phage-Barnase-EndoU-ColicinE5/D-RelE like nuclease2
</pre><pre>Probab=94.61  E-value=0.58  Score=30.25  Aligned_cols=53  Identities=8%  Similarity=0.097  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHhCCccceeCCCCCe--------EEEEEEECCEEEEEEEE-eCC--eEEEEeEc
</pre><pre>Q WP_029789863.1   33 DKIMHEISDIATDPNAIRRNGRGGR--------TITEGTRNGIDIRVIQE-RNG--DIVSGFPT   85 (88)
</pre><pre>Q Consensus        33 ~~I~~ai~~v~~~P~~~~~~~~~g~--------~~~~g~~dGV~I~V~~~-~~g--~i~TAyP~   85 (88)
</pre><pre>                      ...+..+.+++++|+.++.......        ..+.+.+++..+.|++. .++  .+.|+||+
</pre><pre>T Consensus        45 ~~~~~~l~~~l~~PdeI~~~~~~~~~~~~~~~~~~y~k~~~~~~~~vvv~~~~~~~~v~t~~~~  108 (132)
</pre><pre>T A0A285M2N7_9HY   45 APALKQLAEAIMDPDEIWVDWEWHDVRKEWILKRRYLRSAPNLAGLLVFEWSQHGWSATTAYNA  108 (132)
</pre><pre>T ss_pred             HHHHHHHHHHHhCcceeeeeeeecccCCcEEEEEEEEeEcCCceEEEEEEEeCCeEEEEEEeec
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>3U97_A Ribonuclease toxin BrnT; RNAse Sa/RelE small ribonuclease fold, ribonuclease, BrnA, HYDROLASE; HET: MSE; 1.102A {Brucella abortus}
</pre><pre>Probab=93.79  E-value=1.4  Score=28.57  Aligned_cols=50  Identities=18%  Similarity=0.114  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCccceeCCCCCeEEEEEEECC-EEEEEEEE-eC-C--eEEEEeEcC
</pre><pre>Q WP_029789863.1   36 MHEISDIATDPNAIRRNGRGGRTITEGTRNG-IDIRVIQE-RN-G--DIVSGFPTN   86 (88)
</pre><pre>Q Consensus        36 ~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dG-V~I~V~~~-~~-g--~i~TAyP~~   86 (88)
</pre><pre>                      .+.+.+++.+|........ +++...|..++ ..+.|++. .+ +  .|+|||+..
</pre><pre>T Consensus        54 ~~ev~~~~~~~~~~~~d~e-~R~~~iG~~~~gr~l~vv~~~~~~~~~riITa~~a~  108 (114)
</pre><pre>T 3U97_A           54 FADLHFEFFLSAKVFPTKA-DRLMAIGEFNGLIIIAVIFKPVGSEALSVISMRSAS  108 (114)
</pre><pre>T ss_dssp             GGGCCHHHHHTSEEEEEET-TEEEEEEEETTTEEEEEEEEEETTTEEEEEEEEECC
</pre><pre>T ss_pred             HHHHHHHhhcccEEEeCCC-ceEEEEEEECCeEEEEEEEEEcCCCeEEEEEceeCC
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF14076.10 ; DUF4258 ; Domain of unknown function (DUF4258)
</pre><pre>Probab=87.75  E-value=5.4  Score=22.36  Aligned_cols=48  Identities=19%  Similarity=0.255  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCccceeCCC---CCeEEEEEE-ECCEEEEEEEE--eCC-----eEEEEe
</pre><pre>Q WP_029789863.1   36 MHEISDIATDPNAIRRNGR---GGRTITEGT-RNGIDIRVIQE--RNG-----DIVSGF   83 (88)
</pre><pre>Q Consensus        36 ~~ai~~v~~~P~~~~~~~~---~g~~~~~g~-~dGV~I~V~~~--~~g-----~i~TAy   83 (88)
</pre><pre>                      .+.|.+++.+|........   ...+...+. .+|-.+.|++.  .++     .|+|||
</pre><pre>T Consensus        16 ~~~i~~~i~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~vv~~~~~~~~~~~~~viT~~   74 (75)
</pre><pre>T D5BWQ5_NITHN/3   16 TRQVLDVLSKGKIIEGPVRGTTGDWECTLERFVAGDNVVVVVAIETDSAGQPVIIVTTY   74 (75)
</pre><pre>T ss_pred             HHHHHHHHhcCceeecccCCCCCCEEEEEEEcCCCCEEEEEEEEEeCCCCCeEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>PF15534.10 ; Ntox35 ; Bacterial toxin 35
</pre><pre>Probab=87.19  E-value=8  Score=23.78  Aligned_cols=50  Identities=16%  Similarity=0.290  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECCEEEEEEEE-eCC--eEEEEe
</pre><pre>Q WP_029789863.1   31 SDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNGIDIRVIQE-RNG--DIVSGF   83 (88)
</pre><pre>Q Consensus        31 s~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V~~~-~~g--~i~TAy   83 (88)
</pre><pre>                      +-++|...+..+++........+.   +.....+.|..|.|.+. .+|  .|..||
</pre><pre>T Consensus        23 ~w~~v~~~i~~vm~~G~~~~y~~~---~~k~~~~~G~~v~Vt~~~~~G~i~is~aW   75 (76)
</pre><pre>T M1MGK3_9CLOT/7   23 SRKEIYDNFVNIINENKSKLIQGN---NQLRTVINGYEVEIKAFVKDGKVASIDGQ   75 (76)
</pre><pre>T ss_pred             chHHHHHHHHHHHhcCCceeeCce---eEEEEEECCEEEEEEEEeeCCEEEEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>7VD7_A toxin; Toxin-Antitoxin complex Ribonuclease, TOXIN, TOXIN-ANTITOXIN complex; 1.43A {Salmonella typhimurium}
</pre><pre>Probab=87.05  E-value=7.5  Score=23.28  Aligned_cols=51  Identities=25%  Similarity=0.295  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhCCccceeCCC----CCeEEEEEEECC-EEEEEEEE-eC----C--eEEEEeEcC
</pre><pre>Q WP_029789863.1   36 MHEISDIATDPNAIRRNGR----GGRTITEGTRNG-IDIRVIQE-RN----G--DIVSGFPTN   86 (88)
</pre><pre>Q Consensus        36 ~~ai~~v~~~P~~~~~~~~----~g~~~~~g~~dG-V~I~V~~~-~~----g--~i~TAyP~~   86 (88)
</pre><pre>                      .+.+.+++.+|........    +.++...|..++ ..+.|++. .+    +  .|+||++..
</pre><pre>T Consensus        22 ~~ev~~~~~~~~~~~~~~~~~~~~~r~~~iG~~~~gr~l~vv~~~~~~~~~~~~~iita~~a~   84 (94)
</pre><pre>T 7VD7_A           22 FEDAVLVFDDPRHLSRQERYENGEYRWQTLGLVHGIVVILVAHSVRFESGFDVIRIISARKAD   84 (94)
</pre><pre>T ss_dssp             HHHHGGGGGSTTCEEEEEEEETTEEEEEEEEEETTTEEEEEEEEEEECSSCEEEEEEEEEECC
</pre><pre>T ss_pred             HHHHHHHhCCCcceeecchhccCChheEEEEEEcCeEEEEEEEEeccccCCcEEEEEEceeCC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>3NZL_A DNA-binding protein SATB1; alpha-helical domain, Structural Genomics, PSI-Biology, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, TRANSCRIPTION; HET: MSE; 1.199A {Homo sapiens}
</pre><pre>Probab=82.42  E-value=2.1  Score=27.37  Aligned_cols=22  Identities=27%  Similarity=0.652  Sum_probs=0.0  Template_Neff=4.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHHhCCcc
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEISDIATDPNA   48 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~~v~~~P~~   48 (88)
</pre><pre>                      |+.|+...|.+|+.+++++.+.
</pre><pre>T Consensus        14 pEqWsh~TVRnalkeLLkemnQ   35 (83)
</pre><pre>T 3NZL_A           14 PEQWSHTTVRNALKDLLKDMNQ   35 (83)
</pre><pre>T ss_dssp             GGGCCHHHHHHHHHHHTTTSCH
</pre><pre>T ss_pred             hhHccHHHHHHHHHHHHHhCCh
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>PF16557.9 ; CUTL ; CUT1-like DNA-binding domain of SATB
</pre><pre>Probab=81.93  E-value=2  Score=26.61  Aligned_cols=22  Identities=9%  Similarity=0.489  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHHhCCcc
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEISDIATDPNA   48 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~~v~~~P~~   48 (88)
</pre><pre>                      |+.|+...|.+|+.+++++.+.
</pre><pre>T Consensus         6 peqWs~~tVR~al~~LLk~~nq   27 (72)
</pre><pre>T I3KDM2_ORENI/1    6 SDQWTHATVRNALKELLRETNQ   27 (72)
</pre><pre>T ss_pred             hhHhhHHHHHHHHHHHHHhcCh
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>PF12997.11 ; DUF3881 ; Domain of unknown function, E. rectale Gene description (DUF3881)
</pre><pre>Probab=79.79  E-value=26  Score=26.42  Aligned_cols=58  Identities=17%  Similarity=0.150  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHHHhCCccceeCC-----CCCeE----EEEEEECCEEEEEEEEeCCe--EEEEeE
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISDIATDPNAIRRNG-----RGGRT----ITEGTRNGIDIRVIQERNGD--IVSGFP   84 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~v~~~P~~~~~~~-----~~g~~----~~~g~~dGV~I~V~~~~~g~--i~TAyP   84 (88)
</pre><pre>                      |+..=+.+++.+.+.+++.++.......     . +.+    ...+.-.|+.+.+.++.+|.  +.+.||
</pre><pre>T Consensus         6 f~~~~~~~~~~~li~~~~~~~~~~~~~~~~~~~~-~~~~~~~~~~~~~~G~~l~~~~~~~~~~~~~~~~P   74 (285)
</pre><pre>T R5C937_9FIRM/1    6 FSKPPKRDEISRLIERGISNPSHRAYTTDDHDEN-ALLAQFDIELGEDFGIAVCGRFDEGDQFYTEYLYP   74 (285)
</pre><pre>T ss_pred             CCCCCCHHHHHHHHHHHHhCCccceeccCCCccC-cEEEEEEEEEcCCceEEEEEEEcCCCeEEEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>PF19215.4 ; CoV_NSP15_C ; Coronavirus replicase NSP15, uridylate-specific endoribonuclease
</pre><pre>Probab=73.06  E-value=16  Score=25.96  Aligned_cols=22  Identities=9%  Similarity=0.081  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEeE
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGFP   84 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAyP   84 (88)
</pre><pre>                      .+|+..++..+= .+|.|.||||
</pre><pre>T Consensus       132 ~ID~~~v~iM~W~~dg~I~TaYp  154 (155)
</pre><pre>T B1PHK0_9ALPC/6  132 IVDCKVYRWMLWCKDHKVQTFYP  154 (155)
</pre><pre>T ss_pred             EECCEEEEEEEEEeCCeEEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>5YRX_A Nucleoid-associated protein Rv3716c; MtbRv3716, DNA binding, metal ion, DNA BINDING PROTEIN; HET: EDO; 1.9A {Mycobacterium tuberculosis H37Rv}
</pre><pre>Probab=58.33  E-value=45  Score=22.16  Aligned_cols=55  Identities=9%  Similarity=0.219  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECCEEEEEEEEeCCeEEE
</pre><pre>Q WP_029789863.1   25 EFPQGWSDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNGIDIRVIQERNGDIVS   81 (88)
</pre><pre>Q Consensus        25 ~FP~~Ws~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V~~~~~g~i~T   81 (88)
</pre><pre>                      +|+..+.-.++++.+.++.+.=......-.  .....++..+-.|+|.++.+|.|..
</pre><pre>T Consensus         6 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~l~--~~~~~~~s~~~~V~V~v~g~g~l~~   60 (139)
</pre><pre>T 5YRX_A            6 HMQPGGDMSALLAQAQQMQQKLLEAQQQLA--NSEVHGQAGGGLVKVVVKGSGEVIG   60 (139)
</pre><pre>T ss_dssp             ----------------------CHHHHTTT--TCEEEEEETTTTEEEEEETTSCEEE
</pre><pre>T ss_pred             ccCcCCCHHHHHHHHHHHHHHHHHHHHHHh--cCEEEEEeCCCeEEEEEeCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>3O2I_B Uncharacterized protein; structural genomics, PSI-2, protein structure initiative, midwest center for structural genomics, MCSG, Unknown function; HET: MSE; 2.197A {Leptospirillum rubarum}
</pre><pre>Probab=48.76  E-value=44  Score=22.78  Aligned_cols=23  Identities=17%  Similarity=0.333  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHHHhCCcc
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISDIATDPNA   48 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~v~~~P~~   48 (88)
</pre><pre>                      ||.+|+..++...|.+.+..-.+
</pre><pre>T Consensus       100 fPge~~Kn~Vk~ri~~~LgkDRS  122 (125)
</pre><pre>T 3O2I_B          100 FPGELSKTEVRRRINAFLGKDRS  122 (125)
</pre><pre>T ss_dssp             EETTCCHHHHHHHHHHHHTTCC-
</pre><pre>T ss_pred             CCCccCHHHHHHHHHHHhCCccc
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>3CTR_A Poly(A)-specific ribonuclease PARN; PARN, Protein-RNA-complex, m7G-cap, m7GTP, RNA recognition motif, RRM, Exonuclease, Hydrolase, Magnesium, Metal-binding, Nonsense-mediated mRNA decay, Nuclease; HET: MGP; 2.1A {Homo sapiens}
</pre><pre>Probab=45.31  E-value=39  Score=21.53  Aligned_cols=16  Identities=25%  Similarity=0.719  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHH
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISD   41 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~   41 (88)
</pre><pre>                      ||++|...+|......
</pre><pre>T Consensus        13 fp~~~kt~dl~~~f~~   28 (101)
</pre><pre>T 3CTR_A           13 FPKEWKTSDLYQLFSA   28 (101)
</pre><pre>T ss_dssp             CCTTCCHHHHHHHTTT
</pre><pre>T ss_pred             cCccCCHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>2C1W_C ENDOU PROTEIN; NUCLEASE, SNORNA, ENDORIBONUCLEASE, SPLICING INDEPENDENT PROCESSING; HET: PO4; 2.2A {XENOPUS LAEVIS} SCOP: d.294.1.1
</pre><pre>Probab=45.30  E-value=81  Score=24.18  Aligned_cols=23  Identities=9%  Similarity=0.323  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEeEc
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGFPT   85 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAyP~   85 (88)
</pre><pre>                      .++|..+.++.. .+..|.||||+
</pre><pre>T Consensus       259 ~l~~~~~~i~~~~~~~~i~tayp~  282 (292)
</pre><pre>T 2C1W_C          259 RLEEYELQIVVNRHGRYIGTAYPV  282 (292)
</pre><pre>T ss_dssp             EETTEEEEEEEEEETTEEEEEEEE
</pre><pre>T ss_pred             EeCCEEEEEEEEeCCCEEEeeEEe
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>8ARV_B Bifunctional diguanylate cyclase/phosphodiesterase; EAL domain, PDE, phosphodiesterase, BifA, Pseudomonas aeruginosa, c-di-GMP, HYDROLASE; 1.9A {Pseudomonas aeruginosa PAO1}
</pre><pre>Probab=44.38  E-value=1.6e+02  Score=20.36  Aligned_cols=66  Identities=8%  Similarity=-0.045  Sum_probs=0.0  Template_Neff=11.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCCHHHHHHHHHHHHhCCccceeCCCCCeEE-EEEEECCEEEEEEEE-eCCeEEE
</pre><pre>Q WP_029789863.1   12 GGHRPGTGKPGKSEFPQGWSDDKIMHEISDIATDPNAIRRNGRGGRTI-TEGTRNGIDIRVIQE-RNGDIVS   81 (88)
</pre><pre>Q Consensus        12 gGH~~g~g~~gkt~FP~~Ws~~~I~~ai~~v~~~P~~~~~~~~~g~~~-~~g~~dGV~I~V~~~-~~g~i~T   81 (88)
</pre><pre>                      |||++..  ..-...++.|..+++.+.+...+++-........  .+. ..+.+.|.++...+. .++...+
</pre><pre>T Consensus         1 ~~~~~~~--~~~~~~~~~~~~~~~~~~l~~al~~~~~~~~~qP--i~~~~~~~~~~~e~~~~~~~~~~~~~~   68 (275)
</pre><pre>T 8ARV_B            1 MGSSHHH--HHHSSGLVPRGSHELEKDLRDALQRHELHLVYQP--QVDYRDHRVVGVEALLRWQHPLHGFVP   68 (275)
</pre><pre>T ss_dssp             ----------------------CHHHHHHTTTTTTCEEEEEEE--EEETTTTEEEEEEEEEEEEETTTEEEC
</pre><pre>T ss_pred             CCCCccc--cCCCcCCCCCChhHHHHHHHHHHHcCCeeEEEEe--EEECCCCeEEEEEEEEEeeCCCCceeC
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>PF20786.1 ; DUF6840 ; Family of unknown function (DUF6840)
</pre><pre>Probab=44.16  E-value=39  Score=22.01  Aligned_cols=19  Identities=26%  Similarity=0.732  Sum_probs=0.0  Template_Neff=1.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      ||.+|+..++...|.+.+.
</pre><pre>T Consensus        74 fPge~~Kn~V~~ri~~~Lg   92 (92)
</pre><pre>T C6I0U1_9BACT/1   74 YPAEWSKNKVQGLLQDLLN   92 (92)
</pre><pre>T ss_pred             cCchhchhHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>PF13037.10 ; DUF3898 ; Domain of unknown function (DUF3898)
</pre><pre>Probab=43.77  E-value=50  Score=21.74  Aligned_cols=17  Identities=18%  Similarity=0.464  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   28 QGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        28 ~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      +.|+.+++++|..++.+
</pre><pre>T Consensus         4 E~w~~eqV~EA~~~ivE   20 (90)
</pre><pre>T Q81AM7_BACCR/2    4 EQFSTEEVMEATAQIVE   20 (90)
</pre><pre>T ss_pred             ccCCHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF16137.9 ; DUF4845 ; Domain of unknown function (DUF4845)
</pre><pre>Probab=43.04  E-value=48  Score=19.81  Aligned_cols=22  Identities=18%  Similarity=0.249  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHHHhCCc
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISDIATDPN   47 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~v~~~P~   47 (88)
</pre><pre>                      +|.-|+.-++.+++.++++.+.
</pre><pre>T Consensus         3 ~P~y~~~~~~~~~~~~~~~~~~   24 (85)
</pre><pre>T S6BSQ6_9GAMM/3    3 VPTYIENFAIKEVLANVAQDRG   24 (85)
</pre><pre>T ss_pred             cccchhhHHHHHHHHHHHhccc
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>1OTR_A protein Cue2; PROTEIN-PROTEIN COMPLEX, CELL CYCLE; NMR {Saccharomyces cerevisiae} SCOP: a.5.2.4
</pre><pre>Probab=41.51  E-value=34  Score=18.83  Aligned_cols=18  Identities=22%  Similarity=0.366  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCHHHHHHHHHH
</pre><pre>Q WP_029789863.1   23 KSEFPQGWSDDKIMHEISD   41 (88)
</pre><pre>Q Consensus        23 kt~FP~~Ws~~~I~~ai~~   41 (88)
</pre><pre>                      +..||. |+.+.|..++..
</pre><pre>T Consensus        12 ~~mFP~-~~~~~i~~~L~~   29 (49)
</pre><pre>T 1OTR_A           12 MDMFPA-ISKSKLQVHLLE   29 (49)
</pre><pre>T ss_dssp             HHHCSS-SCHHHHHHHHHH
</pre><pre>T ss_pred             HHHCCC-CCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>2JV8_A Uncharacterized protein NE1242; solution structure, NESG, STRUCTURAL GENOMICS, UNKNOWN FUNCTION, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium; NMR {Nitrosomonas europaea ATCC 19718}
</pre><pre>Probab=39.61  E-value=1e+02  Score=19.04  Aligned_cols=21  Identities=24%  Similarity=0.382  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEe
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGF   83 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAy   83 (88)
</pre><pre>                      .+||..|.|... .+|...|+.
</pre><pre>T Consensus        22 ~IDG~~l~v~~~~~~G~y~s~~   43 (73)
</pre><pre>T 2JV8_A           22 TINGKEISYVHDAVKNKWSSRY   43 (73)
</pre><pre>T ss_dssp             EETTEECCCCCCSSSCCCCCTT
</pre><pre>T ss_pred             EECCEEeeEEEeCCCCeEEeCc
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>PF12636.11 ; DUF3781 ; Protein of unknown function (DUF3781)
</pre><pre>Probab=38.58  E-value=1.5e+02  Score=18.30  Aligned_cols=47  Identities=15%  Similarity=0.092  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHhCCcc-ceeCCCCCeEEEEEEECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   31 SDDKIMHEISDIATDPNA-IRRNGRGGRTITEGTRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        31 s~~~I~~ai~~v~~~P~~-~~~~~~~g~~~~~g~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      ++ ++++-+..++.+|+. ....|.  .|.+.  .+++.|.|--. +-.|+||.
</pre><pre>T Consensus        24 ~~-div~~~k~~I~~~~~~i~rkGK--N~Yv~--~~~i~iTiNa~-s~~iITah   71 (71)
</pre><pre>T D3DZN9_METRM/4   24 DV-DPVEYCILKIKDKDAVIEEKGK--NYYVT--VDDCIITVNKH-SFTIITAH   71 (71)
</pre><pre>T ss_pred             CC-CHHHHHHHHHhcCCCeEEEcCc--cEEEe--eCCEEEEEecC-CceEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>PF15631.10 ; Imm-NTF2-2 ; NTF2 fold immunity protein
</pre><pre>Probab=36.92  E-value=1.7e+02  Score=18.47  Aligned_cols=39  Identities=21%  Similarity=0.205  Sum_probs=0.0  Template_Neff=6.300
</pre><pre>
</pre><pre>Q ss_pred             hCCccceeCCCCCeEEEEEEECCEEE--------------------EEEEE-eCCeEEEEe
</pre><pre>Q WP_029789863.1   44 TDPNAIRRNGRGGRTITEGTRNGIDI--------------------RVIQE-RNGDIVSGF   83 (88)
</pre><pre>Q Consensus        44 ~~P~~~~~~~~~g~~~~~g~~dGV~I--------------------~V~~~-~~g~i~TAy   83 (88)
</pre><pre>                      +.|-.+...++ ..|.+.|+......                    .+.++ .||+|++-+
</pre><pre>T Consensus        21 q~Pf~v~~~~d-~~W~V~Gs~~~~~~~~~~~~~~~~~~~~~~gG~~~i~I~k~dG~Il~~~   80 (83)
</pre><pre>T G8TDE4_NIAKG/4   21 EKPFIARLENN-KVWIVEGTLKGDSVYNSKPNPNSRSIFIVVGGVAHIEIQKSDCKILQVS   80 (83)
</pre><pre>T ss_pred             cCCEEEEEcCC-CEEEEEeeCCCCcccCCCCCCCCceeeEEeCcEEEEEEecCCCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>7O6N_C Protein pid-3; ERH-2 PID-3 complex piRNA processing PETISCO, PROTEIN BINDING; 2.17A {Caenorhabditis elegans}
</pre><pre>Probab=36.61  E-value=1.1e+02  Score=16.77  Aligned_cols=25  Identities=12%  Similarity=0.262  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCC-CCHHHHHHHHHHHH
</pre><pre>Q WP_029789863.1   19 GKPGKSEFPQG-WSDDKIMHEISDIA   43 (88)
</pre><pre>Q Consensus        19 g~~gkt~FP~~-Ws~~~I~~ai~~v~   43 (88)
</pre><pre>                      |..+|..|... ++.+.|..|+..||
</pre><pre>T Consensus         1 G~~~~~~~~~~~f~~~~i~~aV~~VL   26 (39)
</pre><pre>T 7O6N_C            1 GPDSMWTFDKVLFNSEDIKDSVFKVL   26 (39)
</pre><pre>T ss_pred             CCccccccceeeeCHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>PF20716.1 ; DUF6828 ; Domain of unknown function (DUF6828)
</pre><pre>Probab=35.06  E-value=74  Score=21.35  Aligned_cols=20  Identities=10%  Similarity=0.425  Sum_probs=0.0  Template_Neff=3.700
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHHHHHhCCcc
</pre><pre>Q WP_029789863.1   29 GWSDDKIMHEISDIATDPNA   48 (88)
</pre><pre>Q Consensus        29 ~Ws~~~I~~ai~~v~~~P~~   48 (88)
</pre><pre>                      ||+|..|+.++..-+..+..
</pre><pre>T Consensus        35 SWdD~~V~~aLp~Yl~s~~~   54 (99)
</pre><pre>T A0A1Q9DPK1_SYM   35 SWDDAAVKAALPEYLKSSGE   54 (99)
</pre><pre>T ss_pred             CCChHHHHHHHHHHHhCCHH
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>PF07380.15 ; Pneumo_M2 ; Pneumovirus M2 protein
</pre><pre>Probab=33.74  E-value=75  Score=20.67  Aligned_cols=15  Identities=7%  Similarity=0.503  Sum_probs=0.0  Template_Neff=1.800
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   30 WSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        30 Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      |+.+++++|+...++
</pre><pre>T Consensus        60 WTsq~Lida~q~fLq   74 (89)
</pre><pre>T M22_HRSVB/1-89   60 WTPKNLLDATQQFLQ   74 (89)
</pre><pre>T ss_pred             cCchHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>PF18780.5 ; HNH_repeat ; Homing endonuclease associated repeat
</pre><pre>Probab=33.20  E-value=1e+02  Score=15.65  Aligned_cols=15  Identities=40%  Similarity=0.616  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   30 WSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        30 Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      ++++++++++..+++
</pre><pre>T Consensus         1 ~t~e~l~~~l~~~~~   15 (54)
</pre><pre>T A0A1I6L2K7_9EU    1 LSRDDIIQEIQRVAA   15 (54)
</pre><pre>T ss_pred             CCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>PF20735.1 ; Lambda_ea8_5 ; Bacteriophage lambda ea8.5-related domain
</pre><pre>Probab=31.95  E-value=69  Score=20.16  Aligned_cols=24  Identities=0%  Similarity=0.105  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCC------CCHHHHHHHHHHHHhC
</pre><pre>Q WP_029789863.1   22 GKSEFPQG------WSDDKIMHEISDIATD   45 (88)
</pre><pre>Q Consensus        22 gkt~FP~~------Ws~~~I~~ai~~v~~~   45 (88)
</pre><pre>                      +...|+++      ++.+++.++|.+++.+
</pre><pre>T Consensus        41 a~~~~~~g~~~~~f~s~~e~~d~Ik~v~~~   70 (86)
</pre><pre>T A0A0K2FIU9_9CA   41 SMKVYKSNEDKSPFCNVREMTDTVQNYYHE   70 (86)
</pre><pre>T ss_pred             HHhhcCCCCCCCCCCCHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF21796.1 ; Cac1_C ; Chromatin assembly factor 1 subunit Cac1, C-terminal domain
</pre><pre>Probab=31.90  E-value=74  Score=18.26  Aligned_cols=21  Identities=29%  Similarity=0.495  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   23 KSEFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        23 kt~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      +..|| +.+...|.+.|.+++.
</pre><pre>T Consensus        23 ~~~fp-~~sK~~Ik~tl~~~A~   43 (56)
</pre><pre>T A0A8H6EC10_ASP   23 KKRFP-KVSKDVLKDTLNSVAI   43 (56)
</pre><pre>T ss_pred             HHHCC-CCCHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF10281.13 ; Ish1 ; Putative nuclear envelope organisation protein
</pre><pre>Probab=30.63  E-value=1.1e+02  Score=15.45  Aligned_cols=14  Identities=14%  Similarity=0.304  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHH
</pre><pre>Q WP_029789863.1   28 QGWSDDKIMHEISD   41 (88)
</pre><pre>Q Consensus        28 ~~Ws~~~I~~ai~~   41 (88)
</pre><pre>                      ++|+++++...+..
</pre><pre>T Consensus         2 ~~Ws~~~L~~wL~~   15 (38)
</pre><pre>T Q75BC7_ASHGO/2    2 SSWPLDSLRSWLEV   15 (38)
</pre><pre>T ss_pred             CCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>6WRW_C Computationally designed protein 2DS25.5; Complex, Beta sheet, Receptor, DE NOVO PROTEIN; HET: NAG, MAN, BMA; 2.84A {Homo sapiens}
</pre><pre>Probab=30.10  E-value=1.7e+02  Score=18.57  Aligned_cols=48  Identities=17%  Similarity=0.304  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECCEEEEEEEE--eCCeEEEEe
</pre><pre>Q WP_029789863.1   31 SDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNGIDIRVIQE--RNGDIVSGF   83 (88)
</pre><pre>Q Consensus        31 s~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V~~~--~~g~i~TAy   83 (88)
</pre><pre>                      ++++|..||.+.++..-+.   ..  .-...-.+-.|.+.|++.  ..|+-+|-|
</pre><pre>T Consensus         1 deeeiqkaieellrkgvse---ee--aaiiivqrfnvavvvvvqderqgkhisey   50 (88)
</pre><pre>T 6WRW_C            1 DEEEIQKAIEELLRKGVSE---EE--AAIIIVQRFNVAVVVVVQDERQGKHISEY   50 (88)
</pre><pre>T ss_dssp             CHHHHHHHHHHHHHTTCCH---HH--HHHHHHHHHCCSEEEECSSHHHHHHHHHH
</pre><pre>T ss_pred             CHHHHHHHHHHHHHcCCCH---HH--HheEEEeeccEEEEEEEccccccccHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>PF19394.3 ; DUF5969 ; Family of unknown function (DUF5969)
</pre><pre>Probab=28.67  E-value=2.8e+02  Score=18.40  Aligned_cols=52  Identities=12%  Similarity=0.211  Sum_probs=0.0  Template_Neff=3.400
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHhCCcccee--CCCCCeEEEEEEECCEEEEEEEE--eCCeEEEE
</pre><pre>Q WP_029789863.1   31 SDDKIMHEISDIATDPNAIRR--NGRGGRTITEGTRNGIDIRVIQE--RNGDIVSG   82 (88)
</pre><pre>Q Consensus        31 s~~~I~~ai~~v~~~P~~~~~--~~~~g~~~~~g~~dGV~I~V~~~--~~g~i~TA   82 (88)
</pre><pre>                      |+-.=...++.++.+|+....  ....+-...++.+.|++.+|-..  .++.+++|
</pre><pre>T Consensus        31 S~~RQ~ALlK~v~~D~~tv~~~as~~~g~~~ar~~fRGveFEve~~~~~~~~~I~a   86 (90)
</pre><pre>T A0A2W2G0T8_9AC   31 STFYQRTLLKRVVLDPSTLAEDADAGEGITVNRSRYQGVEFELHTRADGADREIVA   86 (90)
</pre><pre>T ss_pred             CHHHHHHHHHHHhcCccceeeccccCCceeEEEEEEeceEEEEEEEecCcceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>4S1T_B Uridylate-specific endoribonuclease; HCoV-229e, endoribonuclease, trimeric nsp15, double mutation, HYDROLASE; HET: PO4; 2.504A {Human coronavirus 229E} SCOP: d.294.1.0, l.1.1.1, c.66.1.0
</pre><pre>Probab=28.28  E-value=1.9e+02  Score=23.66  Aligned_cols=22  Identities=9%  Similarity=0.167  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEeE
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGFP   84 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAyP   84 (88)
</pre><pre>                      .+|...++..+= +||.|.|+||
</pre><pre>T Consensus       324 ~ID~k~vr~MvW~kd~~v~T~Yp  346 (349)
</pre><pre>T 4S1T_B          324 IIDNKPWRWMLWCKDNAVATFYP  346 (349)
</pre><pre>T ss_dssp             EETTEEEEEEEEEETTEEEEEEE
</pre><pre>T ss_pred             EECCeEEeEEEEEcCCeeEeeee
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF02198.20 ; SAM_PNT ; Sterile alpha motif (SAM)/Pointed domain
</pre><pre>Probab=27.33  E-value=1.2e+02  Score=16.88  Aligned_cols=18  Identities=11%  Similarity=0.330  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      |..|+.+++...+..+..
</pre><pre>T Consensus        17 p~~W~~~~v~~wl~~~~~   34 (85)
</pre><pre>T FLI1_HUMAN/114   17 PTLWTQEHVRQWLEWAIK   34 (85)
</pre><pre>T ss_pred             hhhCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>PF19848.3 ; DUF6323 ; Family of unknown function (DUF6323)
</pre><pre>Probab=26.58  E-value=1e+02  Score=21.18  Aligned_cols=24  Identities=13%  Similarity=0.163  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCHHHHHHHHHHHHhCC
</pre><pre>Q WP_029789863.1   23 KSEFPQGWSDDKIMHEISDIATDP   46 (88)
</pre><pre>Q Consensus        23 kt~FP~~Ws~~~I~~ai~~v~~~P   46 (88)
</pre><pre>                      |.+.+...+|+++++++.+..+.+
</pre><pre>T Consensus        78 Kne~~d~isDdeLi~~m~~~f~~~  101 (127)
</pre><pre>T A0A2Y9BLM3_9FI   78 KNATLDRMTDDEVLTFMKEQYETV  101 (127)
</pre><pre>T ss_pred             HHhcccCCCHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF17412.6 ; VraX ; Family of unknown function
</pre><pre>Probab=26.29  E-value=1.7e+02  Score=17.43  Aligned_cols=20  Identities=15%  Similarity=0.230  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHHHhC
</pre><pre>Q WP_029789863.1   26 FPQGWSDDKIMHEISDIATD   45 (88)
</pre><pre>Q Consensus        26 FP~~Ws~~~I~~ai~~v~~~   45 (88)
</pre><pre>                      ||++.++.++...+..+..+
</pre><pre>T Consensus        29 ~~~d~s~~ev~~~L~ll~~d   48 (55)
</pre><pre>T VRAX_STAEQ/1-5   29 CDETFNKYEIYQLLSLLEND   48 (55)
</pre><pre>T ss_pred             eCCCCCHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>5YVD_A Nsp15; MERS-CoV, anti-viral, endoribonuclease, HYDROLASE; HET: GOL; 2.7A {Middle East respiratory syndrome-related coronavirus} SCOP: d.294.1.0, c.66.1.0
</pre><pre>Probab=26.23  E-value=2e+02  Score=23.44  Aligned_cols=22  Identities=14%  Similarity=0.244  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEeE
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGFP   84 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAyP   84 (88)
</pre><pre>                      .+|...++..+= +||.|.|+||
</pre><pre>T Consensus       318 ~ID~k~vr~MvW~kd~~v~T~Yp  340 (343)
</pre><pre>T 5YVD_A          318 PIDLTMIEFMLWCKDGQVQTFYP  340 (343)
</pre><pre>T ss_dssp             EETTEEEEEEEEEETTEEEEEEE
</pre><pre>T ss_pred             EECCceecEEEEEeCCeEEeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>6TMG_P ATPTG10; mitochondrial, ATP synthase, membrane region, lipids, MEMBRANE PROTEIN; HET: LMT, PEE, PC1, CDL; 2.8A {Toxoplasma gondii (strain ATCC 50853 / GT1)}
</pre><pre>Probab=25.74  E-value=1.2e+02  Score=21.54  Aligned_cols=19  Identities=11%  Similarity=0.448  Sum_probs=0.0  Template_Neff=3.200
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHHHHHhCCc
</pre><pre>Q WP_029789863.1   29 GWSDDKIMHEISDIATDPN   47 (88)
</pre><pre>Q Consensus        29 ~Ws~~~I~~ai~~v~~~P~   47 (88)
</pre><pre>                      +|+|..|+.++..-+..+.
</pre><pre>T Consensus        52 SWdD~~Vl~aLp~YL~s~~   70 (138)
</pre><pre>T 6TMG_P           52 SWDDADVLKALPQYLNCRG   70 (138)
</pre><pre>T ss_dssp             CCCCHHHHHHHHHHTTCCS
</pre><pre>T ss_pred             CCCcHHHHHHHHHHhcCCH
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>5SBF_A Uridylate-specific endoribonuclease; SGC - Diamond I04-1 fragment screening, PanDDA, XChemExplorer, nsp15, nendoU, sars-cov-2, sars, covid, covid19, HYDROLASE; HET: CIT; 1.64A {Severe acute respiratory syndrome coronavirus 2}
</pre><pre>Probab=25.74  E-value=2.3e+02  Score=23.28  Aligned_cols=22  Identities=18%  Similarity=0.350  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE-eCCeEEEEeE
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE-RNGDIVSGFP   84 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~-~~g~i~TAyP   84 (88)
</pre><pre>                      .+|...++..+= .+|.|.|+||
</pre><pre>T Consensus       324 ~ID~k~vr~MvW~~~~~V~T~Yp  346 (349)
</pre><pre>T 5SBF_A          324 TIDYTEISFMLWCKDGHVETFYP  346 (349)
</pre><pre>T ss_dssp             EETTEEEEEEEEEETTEEEEEEE
</pre><pre>T ss_pred             EECCeEecEEEEEeCCeeeeeeE
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF19542.3 ; DUF6066 ; Family of unknown function (DUF6066)
</pre><pre>Probab=25.44  E-value=91  Score=23.45  Aligned_cols=56  Identities=21%  Similarity=0.399  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHH------HHHhCCccceeCCCCCeEEEEEEECCEEEEE---EEE--eCCeEEEEeEcC
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEIS------DIATDPNAIRRNGRGGRTITEGTRNGIDIRV---IQE--RNGDIVSGFPTN   86 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~------~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V---~~~--~~g~i~TAyP~~   86 (88)
</pre><pre>                      |++|.+.+......      +++=.|...|....    .-.|.++||.+++   .|.  .+|+|+-+.|.+
</pre><pre>T Consensus       139 p~~~~~~~~~r~~~~~~lR~q~Vfrp~~~W~l~~----kggg~~~GV~~~v~a~rV~~~rtG~vl~~~~~~  205 (212)
</pre><pre>T A0A085WP66_9DE  139 PEGWNGGMFARAFSSKAVRLSVVFTPLDIWSLPK----PGGGKIFGVAARLEAVLVTEGRTGDHLGLWLDG  205 (212)
</pre><pre>T ss_pred             CCCcccchhhHHhhcCcEEEEEEEeecceeecCC----CCCCceeeEEEEEEEEEEecCCCCcEEEEEecC
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>6MTZ_A HepC.19480.a.B1; SSGCID, Structural Genomics, Helicobacter pylori, uncharacterized protein, Seattle Structural Genomics Center for Infectious Disease, UNKNOWN FUNCTION; HET: IOD, EDO; 2.15A {Helicobacter pylori (strain G27)}
</pre><pre>Probab=25.38  E-value=2.3e+02  Score=21.12  Aligned_cols=32  Identities=28%  Similarity=0.474  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCC---CHHHHHHHHHHHHhCC
</pre><pre>Q WP_029789863.1   11 GGGHRPGTGKPGKSEFPQGW---SDDKIMHEISDIATDP   46 (88)
</pre><pre>Q Consensus        11 ~gGH~~g~g~~gkt~FP~~W---s~~~I~~ai~~v~~~P   46 (88)
</pre><pre>                      +|||....|.    .+|..+   +-+++++.+.+.++++
</pre><pre>T Consensus       321 GGGH~~AaG~----~i~~~~~~~~~~~~~~~~~~~i~~~  355 (355)
</pre><pre>T 6MTZ_A          321 GGGHRNASGG----KIDGFRESFNYRDIKEQIEEIFNNA  355 (355)
</pre><pre>T ss_dssp             CEEETTEEEE----ECTTCCCCSCHHHHHHHHHHHHHTC
</pre><pre>T ss_pred             CCCCccccCc----cccCCchhcCHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>PF07262.15 ; CdiI ; CDI immunity protein
</pre><pre>Probab=25.05  E-value=1.8e+02  Score=19.25  Aligned_cols=34  Identities=6%  Similarity=-0.116  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1    9 ATGGGHRPGTGKPGKSEFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus         9 ~~~gGH~~g~g~~gkt~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      ...||...-  ......+|.+-++++|-++|+++++
</pre><pre>T Consensus       120 ~~~~g~~~~--~~~~i~l~~~~~~~eLG~av~~al~  153 (155)
</pre><pre>T Q9K0T5_NEIMB/9  120 VKLEAWDAI--DADDVILSLDNSPEEIGAGLKLALS  153 (155)
</pre><pre>T ss_pred             CCCCcceeC--CCCeEEEeCCCCHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF07305.16 ; DUF1454 ; Protein of unknown function (DUF1454)
</pre><pre>Probab=24.89  E-value=4.2e+02  Score=19.12  Aligned_cols=50  Identities=12%  Similarity=0.096  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECCEEEEEEEE-eCCeEEEEeEc
</pre><pre>Q WP_029789863.1   28 QGWSDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNGIDIRVIQE-RNGDIVSGFPT   85 (88)
</pre><pre>Q Consensus        28 ~~Ws~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V~~~-~~g~i~TAyP~   85 (88)
</pre><pre>                      ++++.+++.+.+.+++.+.........        +++|++..+... ..|-..++-|+
</pre><pre>T Consensus       129 P~ls~~e~~~~l~dLl~~~~~~~~~s~--------~~g~vkYs~~~s~~~Gl~F~vep~  179 (185)
</pre><pre>T YIIQ_ECOLI/8-1  129 PLMTKTQSQKKLQSLLTAGKNKRYYTE--------TEGALRYVVADNGEKGLTFAVEPI  179 (185)
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHHHccCCCCceEE--------EECCEEEEEEeCCCceEEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>3JSL_B DNA ligase; DNA ligase, NAD+-dependent, Staphylococcus aureus, DNA damage, DNA repair, DNA replication, Ligase, Magnesium, Manganese, Metal-binding, NAD; HET: SO4; 1.8A {Staphylococcus aureus} SCOP: d.142.2.2
</pre><pre>Probab=24.86  E-value=3.9e+02  Score=20.40  Aligned_cols=25  Identities=20%  Similarity=0.355  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   58 TITEGTRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        58 ~~~~g~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      +.+.-.+||+.+.+.+. +|..+.||
</pre><pre>T Consensus       107 ~~~e~KlDGvs~~l~y~-~G~lv~a~  131 (318)
</pre><pre>T 3JSL_B          107 YMCELKIDGLAVSLKYV-DGYFVQGL  131 (318)
</pre><pre>T ss_dssp             EEEEEEECSEEEEEEEE-TTEEEEEE
</pre><pre>T ss_pred             EEEEEecCceEEEEEEE-CCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>7K72_C DNA ligase A; Structural Genomics, Seattle Structural Genomics Center for Infectious Disease, SSGCID, LIGASE; HET: NAD, MPD; 2.05A {Mycobacterium tuberculosis (strain ATCC 25618 / H37Rv)} SCOP: d.142.2.0
</pre><pre>Probab=24.41  E-value=3.9e+02  Score=20.67  Aligned_cols=25  Identities=12%  Similarity=0.303  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   58 TITEGTRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        58 ~~~~g~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      +.+.-.+||+.+.+.+. +|+.+.|+
</pre><pre>T Consensus       126 ~~~e~K~DGvs~~l~y~-~G~lv~a~  150 (336)
</pre><pre>T 7K72_C          126 YLCELKIDGVALSLVYR-EGRLTRAS  150 (336)
</pre><pre>T ss_dssp             EEEEEEECSEEEEEEEE-TTEEEEEE
</pre><pre>T ss_pred             EEEEEEeCcceEEEEEE-CCEEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>2GKP_A hypothetical protein NMB0488; structural genomics, APC83854, hypothetical protein, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, UNKNOWN; HET: MSE; 1.35A {Neisseria meningitidis} SCOP: d.351.1.1
</pre><pre>Probab=24.32  E-value=1.8e+02  Score=19.78  Aligned_cols=33  Identities=6%  Similarity=-0.123  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   10 TGGGHRPGTGKPGKSEFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        10 ~~gGH~~g~g~~gkt~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      ..||...-..  ....+|.+.+++++-++|++++.
</pre><pre>T Consensus       132 ~~~g~~~~~~--~~i~l~~~~s~e~LG~av~~al~  164 (167)
</pre><pre>T 2GKP_A          132 KLEAWDAIDA--DDVILSLDNSPEEIGAGLKLALS  164 (167)
</pre><pre>T ss_dssp             ETTEEEECSS--CCEEEETTSCHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCcceecCC--CeEEEeCCCCHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>PF20636.1 ; SMN_G2-BD ; Survival Motor Neuron, Gemin2-binding domain
</pre><pre>Probab=24.28  E-value=1.1e+02  Score=12.28  Aligned_cols=16  Identities=13%  Similarity=0.520  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHhC
</pre><pre>Q WP_029789863.1   30 WSDDKIMHEISDIATD   45 (88)
</pre><pre>Q Consensus        30 Ws~~~I~~ai~~v~~~   45 (88)
</pre><pre>                      |++..++++..+.+..
</pre><pre>T Consensus         3 wdd~~l~~~~~~~~~~   18 (18)
</pre><pre>T A0A0D8XRC6_DIC    3 WDDTELIKMYEHAVSA   18 (18)
</pre><pre>T ss_pred             ccHHHHHHHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>PF15002.10 ; ERK-JNK_inhib ; ERK and JNK pathways, inhibitor
</pre><pre>Probab=23.65  E-value=1.4e+02  Score=21.68  Aligned_cols=21  Identities=43%  Similarity=0.722  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCHHHHHHHHHHHHhC
</pre><pre>Q WP_029789863.1   21 PGKSEFPQGWSDDKIMHEISDIATD   45 (88)
</pre><pre>Q Consensus        21 ~gkt~FP~~Ws~~~I~~ai~~v~~~   45 (88)
</pre><pre>                      ++ ..||   .++.+.+|+..|++|
</pre><pre>T Consensus        71 ~~-~~fp---~d~~~~dals~vlEN   91 (198)
</pre><pre>T T1EZX4_HELRO/4   71 PG-DPFP---SDDNIKEVFSLVLEN   91 (198)
</pre><pre>T ss_pred             CC-CCCC---CChHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>4QQ0_A Adenylate cyclase-like protein; BON-domain, type III secretion, membrane protein, STRUCTURAL PROTEIN; HET: SO4; 2.0A {Chlamydia trachomatis}
</pre><pre>Probab=22.58  E-value=4.1e+02  Score=20.23  Aligned_cols=36  Identities=14%  Similarity=0.228  Sum_probs=0.0  Template_Neff=8.100
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHH-HHhCCccceeCCC---CCeEEEEEEEC
</pre><pre>Q WP_029789863.1   30 WSDDKIMHEISD-IATDPNAIRRNGR---GGRTITEGTRN   65 (88)
</pre><pre>Q Consensus        30 Ws~~~I~~ai~~-v~~~P~~~~~~~~---~g~~~~~g~~d   65 (88)
</pre><pre>                      |++++|.+++.+ ++.+|......-.   +|.+...|.+.
</pre><pre>T Consensus       115 ~~de~i~~~v~~~L~~~~~~~~v~v~~~~~G~v~L~G~v~  154 (280)
</pre><pre>T 4QQ0_A          115 IDDEAVWQEMNILLSKNPEFKGISMQSPEPGIFVISGYLK  154 (280)
</pre><pre>T ss_dssp             EEHHHHHHHHHHHHHTSGGGTEEEEEEEETTEEEEEEEES
</pre><pre>T ss_pred             CChHHHHHHHHHHHhhCCccCCCceecCcCcEEEEEEEeC
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>4PY9_A Putative exopolyphosphatase-related protein; Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, EXOPOLYPHOSPHATASE-RELATED PROTEIN, UNKNOWN FUNCTION; HET: PO4, MSE; 2.25A {Bacteroides fragilis}
</pre><pre>Probab=22.24  E-value=2.1e+02  Score=20.78  Aligned_cols=29  Identities=24%  Similarity=0.416  Sum_probs=0.0  Template_Neff=11.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCCCCCCCCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   10 TGGGHRPGTGKPGKSEFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        10 ~~gGH~~g~g~~gkt~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      .+|||....|.    .||  .+-+++++.+.+.++
</pre><pre>T Consensus       307 ~GGGH~~AaG~----~~~--~~~~~~~~~i~~~l~  335 (343)
</pre><pre>T 4PY9_A          307 NGGGHLNASGG----EFY--GTMEEAVKVFEQALE  335 (343)
</pre><pre>T ss_dssp             SCEECSSEEEE----EES--SCHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCCCccCce----Eee--cCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>PF06236.15 ; MelC1 ; Tyrosinase co-factor MelC1
</pre><pre>Probab=22.18  E-value=2.2e+02  Score=19.31  Aligned_cols=21  Identities=19%  Similarity=0.495  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      .|||+.+.|..+.+|..+|+.
</pre><pre>T Consensus        61 ~IDGr~LhVmr~adG~y~S~v   81 (109)
</pre><pre>T K4RFH5_STRDJ/1   61 FVDGVELHVMRNADGSWISVV   81 (109)
</pre><pre>T ss_pred             EECCEEeEEEECCCCCEEecc
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>PF05256.16 ; UPF0223 ; Uncharacterised protein family (UPF0223)
</pre><pre>Probab=21.76  E-value=1.6e+02  Score=19.11  Aligned_cols=13  Identities=23%  Similarity=0.560  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHH
</pre><pre>Q WP_029789863.1   28 QGWSDDKIMHEIS   40 (88)
</pre><pre>Q Consensus        28 ~~Ws~~~I~~ai~   40 (88)
</pre><pre>                      .+||.++|++.|.
</pre><pre>T Consensus         8 ~~WsteEii~Vi~   20 (86)
</pre><pre>T YFDD_LACLA/4-8    8 LSWSTTEMTEVLS   20 (86)
</pre><pre>T ss_pred             CCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>4GLW_A DNA ligase; DNA ligase A, inhibitor, LIGASE-LIGASE INHIBITOR complex; HET: NMN, 0XT, SO4; 2.0A {Streptococcus pneumoniae} SCOP: d.142.2.2
</pre><pre>Probab=21.60  E-value=5.3e+02  Score=19.32  Aligned_cols=25  Identities=32%  Similarity=0.469  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             EEEEEEECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   58 TITEGTRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        58 ~~~~g~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      |.+.=.+||+.+.+.+. +|.++.+|
</pre><pre>T Consensus       104 ~~~e~K~DG~~~~l~~~-~g~l~~~~  128 (305)
</pre><pre>T 4GLW_A          104 YICELKIDGLSISLTYE-KGILVAGV  128 (305)
</pre><pre>T ss_dssp             EEEEEEECSEEEEEEEE-TTEEEEEE
</pre><pre>T ss_pred             EEEEEeeCceEEEEEEe-CCeEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>3BAC_A DNA ligase; Adenylation Domain, DNA damage, DNA repair, DNA replication, Ligase, NAD; HET: 3B9, PO4; 3.0A {Haemophilus influenzae}
</pre><pre>Probab=21.22  E-value=5.3e+02  Score=18.96  Aligned_cols=49  Identities=14%  Similarity=0.194  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CHHHHHHHHHHHHhCCccceeCCCCCeEEEEEEECCEEEEEEEEeCCeEEEEe
</pre><pre>Q WP_029789863.1   31 SDDKIMHEISDIATDPNAIRRNGRGGRTITEGTRNGIDIRVIQERNGDIVSGF   83 (88)
</pre><pre>Q Consensus        31 s~~~I~~ai~~v~~~P~~~~~~~~~g~~~~~g~~dGV~I~V~~~~~g~i~TAy   83 (88)
</pre><pre>                      +.+++.+....+............   +.++=.+||+.+.+.+. +|+++.+|
</pre><pre>T Consensus        24 ~~~~~~~~~~~~~~~~~~~~~~~~---~~~e~K~DG~r~~l~~~-~g~~~~~~   72 (270)
</pre><pre>T 3BAC_A           24 SDAEFNAFVKRIEDRLILLPKPLT---FCCEPKLDGLAVSILYV-NGELTQAA   72 (270)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHHSSSCCSSCE---EEEEEEESSEEEEEEEE-TTEEEEEE
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHhccCCCCce---EEEEEecCCeeEEEEEE-CCEEEEee
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>8CSS_G 28S ribosomal protein S9, mitochondrial; Ribonucleoprotein complex Mitochondria Biogenesis, RIBOSOME; HET: MG, ATP, GDP, SAM, NAD, MA6; 2.36A {Homo sapiens}
</pre><pre>Probab=21.00  E-value=1.4e+02  Score=24.49  Aligned_cols=17  Identities=18%  Similarity=0.429  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHH
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEISDIA   43 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~~v~   43 (88)
</pre><pre>                      |+++|.++|-.||+.++
</pre><pre>T Consensus        96 pe~ftq~did~aI~YLf  112 (396)
</pre><pre>T 8CSS_G           96 PETFTQEDIDRAIAYLF  112 (396)
</pre><pre>T ss_dssp             GGGCCHHHHHHHHHHHS
</pre><pre>T ss_pred             cccCCHHHHHHHHHHHC
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>PF09133.14 ; SANTA ; SANTA (SANT Associated)
</pre><pre>Probab=20.97  E-value=75  Score=19.50  Aligned_cols=7  Identities=57%  Similarity=1.450  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCH
</pre><pre>Q WP_029789863.1   26 FPQGWSD   32 (88)
</pre><pre>Q Consensus        26 FP~~Ws~   32 (88)
</pre><pre>                      ||++|.+
</pre><pre>T Consensus        83 FP~~Wk~   89 (90)
</pre><pre>T M4F308_BRARP/6   83 FPYHWKD   89 (90)
</pre><pre>T ss_pred             CCCchhc
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>PF08471.14 ; Ribonuc_red_2_N ; Class II vitamin B12-dependent ribonucleotide reductase
</pre><pre>Probab=20.83  E-value=1.8e+02  Score=19.59  Aligned_cols=20  Identities=15%  Similarity=0.076  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   25 EFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        25 ~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      +||+.||..-+.-+...-++
</pre><pre>T Consensus        12 e~P~~WS~~A~~i~a~kyfr   31 (117)
</pre><pre>T D5BSP1_PUNMI/3   12 EIPEQFSQVATDILAQKYFR   31 (117)
</pre><pre>T ss_pred             EccCccCHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>4Q2J_A DNA-binding protein SATB1; DNA binding protein; 2.603A {Mus musculus}
</pre><pre>Probab=20.72  E-value=2.1e+02  Score=20.79  Aligned_cols=26  Identities=23%  Similarity=0.489  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCHHHHHHHHHHHHhCCcc
</pre><pre>Q WP_029789863.1   23 KSEFPQGWSDDKIMHEISDIATDPNA   48 (88)
</pre><pre>Q Consensus        23 kt~FP~~Ws~~~I~~ai~~v~~~P~~   48 (88)
</pre><pre>                      ....|+.|+.++|.+++..++.....
</pre><pre>T Consensus       111 ~~~~p~~~~~~~Vr~~Ll~LL~~~~q  136 (180)
</pre><pre>T 4Q2J_A          111 EDLPPEQWSHTTVRNALKDLLKDMNQ  136 (180)
</pre><pre>T ss_dssp             TTSCGGGCCHHHHHHHHHHHHTSSCH
</pre><pre>T ss_pred             ccCCchhccHHHHHHHHHHHHHHcch
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF09412.14 ; XendoU ; Endoribonuclease XendoU
</pre><pre>Probab=20.33  E-value=3.4e+02  Score=20.40  Aligned_cols=22  Identities=5%  Similarity=0.024  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             EECCEEEEEEEE---eCC--eEEEEeE
</pre><pre>Q WP_029789863.1   63 TRNGIDIRVIQE---RNG--DIVSGFP   84 (88)
</pre><pre>Q Consensus        63 ~~dGV~I~V~~~---~~g--~i~TAyP   84 (88)
</pre><pre>                      .++|..+.+...   .++  .|.||||
</pre><pre>T Consensus       230 ~l~g~~~~i~~~~~~~~~~~~i~tayP  256 (256)
</pre><pre>T E5S926_TRISP/7  230 KFRSCTGNIFTYSLQRRNKSYLSTAYF  256 (256)
</pre><pre>T ss_pred             EECCEEEEEEEEEeccCCceEEEEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>1SXE_A Transcriptional regulator ERG; alpha helical, TRANSCRIPTION, SIGNALING PROTEIN; NMR {Homo sapiens} SCOP: a.60.1.1, l.1.1.1
</pre><pre>Probab=20.20  E-value=2.1e+02  Score=17.10  Aligned_cols=18  Identities=22%  Similarity=0.379  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   27 PQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        27 P~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      |..|+.+++...+..+.+
</pre><pre>T Consensus        27 p~~Ws~~~v~~wl~~~~~   44 (97)
</pre><pre>T 1SXE_A           27 PTLWSTDHVRQWLEWAVK   44 (97)
</pre><pre>T ss_dssp             TTSCCHHHHHHHHHHHHH
</pre><pre>T ss_pred             hhhCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>2M7A_A Uncharacterized protein; bacteriophage, zinc binding, CHCC motif, UNKNOWN FUNCTION; HET: ZN; NMR {Escherichia coli}
</pre><pre>Probab=20.13  E-value=2.4e+02  Score=18.29  Aligned_cols=20  Identities=10%  Similarity=0.097  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             CCCC------CCHHHHHHHHHHHHhC
</pre><pre>Q WP_029789863.1   26 FPQG------WSDDKIMHEISDIATD   45 (88)
</pre><pre>Q Consensus        26 FP~~------Ws~~~I~~ai~~v~~~   45 (88)
</pre><pre>                      |+++      ++.+++.++|.+++.+
</pre><pre>T Consensus        60 ~~~g~~~~~f~~~~e~~daIk~v~~~   85 (101)
</pre><pre>T 2M7A_A           60 YQKDNSCNLFNTLTEARDAIKDAYES   85 (101)
</pre><pre>T ss_dssp             HCSCCCCSTTSCTHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccCCCCCCCCCCHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>8Q84_O DASH complex subunit SPC34; Kinetochore, microtubule, error correction, chromosome segregation, CELL CYCLE;{Saccharomyces cerevisiae}
</pre><pre>Probab=20.03  E-value=2.2e+02  Score=22.50  Aligned_cols=23  Identities=17%  Similarity=0.437  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCCCCCHHHHHHHHHHHHh
</pre><pre>Q WP_029789863.1   22 GKSEFPQGWSDDKIMHEISDIAT   44 (88)
</pre><pre>Q Consensus        22 gkt~FP~~Ws~~~I~~ai~~v~~   44 (88)
</pre><pre>                      +||.|..+++-+.++.++..+++
</pre><pre>T Consensus       197 r~t~~~~d~dve~lL~~aekL~~  219 (295)
</pre><pre>T 8Q84_O          197 RKTIFVEDFPTDLILKVMAEVTD  219 (295)
</pre><pre>T ss_dssp             EEEEESTTSCHHHHHHHHHHHHH
</pre><pre>T ss_pred             ccceecCCCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre>