<pre>Query         FD01848239_02317 type IV secretion protein Rhs
</pre><pre>Match_columns 99
</pre><pre>No_of_seqs    118 out of 695
</pre><pre>Neff          7.14392
</pre><pre>Searched_HMMs 86581
</pre><pre>Date          Mon Feb 26 21:12:03 2024
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -o ../results/1612204.hhr -oa3m ../results/1612204.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF14412.10 ; AHH ; A nuclease   99.5 1.2E-14 1.4E-19   97.1   4.7   90    1-98     17-131 (139)
</pre><pre>  2 PF09533.14 ; DUF2380 ; Predict  96.6   0.009   1E-07   43.7   4.7   73    1-94    100-178 (189)
</pre><pre>  3 PF05120.16 ; GvpG ; Gas vesicl  83.2      13 0.00015   23.1   5.7   37   61-97     31-67  (72)
</pre><pre>  4 PF21648.1 ; M1E1E6-like ; Inte  81.6     8.7  0.0001   25.7   4.7   37   59-95     64-100 (104)
</pre><pre>  5 1GZS_D SOPE; TOXIN/CELL CYCLE,  80.4      31 0.00036   25.4   7.4   84   15-98     64-163 (165)
</pre><pre>  6 3FY6_D Integron cassette prote  75.9      15 0.00018   25.5   4.7   37   59-95     73-109 (126)
</pre><pre>  7 PF07487.17 ; SopE_GEF ; SopE G  74.2      41 0.00047   23.8   6.5   77   15-98     35-134 (134)
</pre><pre>  8 PF15635.10 ; Tox-GHH2 ; GHH si  72.7       3 3.4E-05   27.1   0.6   10    2-11      2-11  (105)
</pre><pre>  9 PF13391.10 ; HNH_2 ; HNH endon  69.2     2.7 3.1E-05   22.2  -0.1   36    1-36     11-49  (62)
</pre><pre> 10 2JOK_A Putative G-nucleotide e  66.3      67 0.00078   24.2   6.4   78   15-98     69-168 (186)
</pre><pre> 11 6AZ3_O 60S ribosomal protein L  63.3      90   0.001   24.9   7.0   65   28-98    174-259 (305)
</pre><pre> 12 5T5H_u 60S ribosomal protein L  62.9      67 0.00077   24.9   6.1   65   28-98    169-254 (254)
</pre><pre> 13 6TH6_BQ 50S ribosomal protein   62.4      66 0.00077   23.8   5.9   65   28-98    126-195 (201)
</pre><pre> 14 PF05766.16 ; NinG ; Bacterioph  57.5      28 0.00032   24.8   3.2   74    1-97    109-191 (197)
</pre><pre> 15 7P9K_B DUF262 domain-containin  56.1      13 0.00015   30.4   1.5   39    1-39    471-519 (587)
</pre><pre> 16 PF19551.3 ; DUF6074 ; Family o  51.5      92  0.0011   19.5   5.9   39   60-98     33-71  (81)
</pre><pre> 17 8HL4_L18P 50S ribosomal protei  50.8 1.5E+02  0.0017   21.7   6.4   64   28-98    125-193 (193)
</pre><pre> 18 6XOR_A Protein swallow; coiled  48.0 1.1E+02  0.0013   19.5   4.6   40   58-97     28-67  (71)
</pre><pre> 19 PF11867.12 ; T1RH-like_C ; Typ  47.9 1.3E+02  0.0015   23.1   5.6   42   57-98    198-239 (334)
</pre><pre> 20 8A3D_G 60S ribosomal protein L  47.1 2.2E+02  0.0025   22.5   6.7   65   28-98    168-250 (297)
</pre><pre> 21 4V8P_BM 60S RIBOSOMAL PROTEIN   46.1 1.6E+02  0.0018   23.5   5.9   65   28-98    168-257 (301)
</pre><pre> 22 6ZU5_LD0 uL18; Microsporidia,   45.0 2.2E+02  0.0026   22.0   7.3   65   28-98    144-225 (266)
</pre><pre> 23 3BHP_B UPF0291 protein ynzC; N  44.4 1.2E+02  0.0013   18.6   5.1   39   61-99      4-45  (60)
</pre><pre> 24 1VQ8_N 50S ribosomal protein L  44.0 1.2E+02  0.0014   22.3   4.7   58   28-98    125-183 (187)
</pre><pre> 25 6ZJ3_Ld Ribosomal protein uL18  43.4 2.4E+02  0.0027   21.9   6.4   65   28-98    166-248 (260)
</pre><pre> 26 2LSE_A Four Helix Bundle Prote  42.9 1.5E+02  0.0017   19.5   4.5   37   61-99      6-42  (101)
</pre><pre> 27 7YH8_C L-19437; heterochiral,   39.8 1.4E+02  0.0016   18.1   4.8   37   57-99     25-61  (62)
</pre><pre> 28 PF15188.10 ; CCDC-167 ; Coiled  37.9 1.7E+02   0.002   18.7   5.0   38   61-99     14-51  (84)
</pre><pre> 29 PF14357.10 ; DUF4404 ; Domain   37.8 1.4E+02  0.0016   18.8   3.8   37   63-99      1-37  (85)
</pre><pre> 30 3FXD_A Protein IcmQ; 4 helix b  37.7 1.5E+02  0.0017   18.0   4.8   43   57-99      4-46  (57)
</pre><pre> 31 7R81_G1 60S ribosomal protein   36.4 2.1E+02  0.0024   22.9   5.3   65   28-98    171-253 (301)
</pre><pre> 32 PF10520.13 ; Lipid_desat ; Lip  36.3      37 0.00043   24.2   1.1   21    1-21     34-54  (178)
</pre><pre> 33 8FJG_A H12; De novo protein, H  36.1 2.1E+02  0.0024   19.1   5.5   43   57-99     57-100 (109)
</pre><pre> 34 PF15500.10 ; Ntox1 ; Putative   35.9   2E+02  0.0024   19.9   4.4   35   60-97     47-94  (109)
</pre><pre> 35 1GW3_A APOA-I; HIGH DENSITY LI  34.9 1.3E+02  0.0015   16.4   5.1   40   59-99      3-44  (46)
</pre><pre> 36 6NZ1_C Design construct XXA_GV  34.5 2.2E+02  0.0026   19.0   6.5   43   48-97      1-45  (98)
</pre><pre> 37 6DM9_D DHD15_extended_B; Compu  34.1   2E+02  0.0023   18.4   5.4   44   56-99     17-60  (78)
</pre><pre> 38 PF09677.14 ; TrbI_Ftype ; Type  34.0 2.1E+02  0.0025   18.7   5.0   37   63-99     31-67  (103)
</pre><pre> 39 7QCA_LD0 60S ribosomal protein  33.2 3.7E+02  0.0043   21.2   7.2   65   28-98    172-253 (291)
</pre><pre> 40 6DMP_B Designed orthogonal pro  32.3 2.1E+02  0.0024   18.3   3.8   33   61-97     24-56  (82)
</pre><pre> 41 6DM9_C DHD15_extended_A; Compu  31.5 2.3E+02  0.0026   18.2   5.4   44   56-99     17-60  (78)
</pre><pre> 42 PF16305.9 ; DUF4947 ; Domain o  31.5 3.5E+02   0.004   20.3   5.3   35   59-93     88-122 (167)
</pre><pre> 43 PF19903.3 ; DUF6376 ; Family o  31.4 2.8E+02  0.0033   19.3   5.2   42   58-99     68-110 (116)
</pre><pre> 44 5FBM_A DNA-binding protein HU;  31.0 2.1E+02  0.0025   17.8   4.0   32   68-99      5-37  (99)
</pre><pre> 45 5TN2_D HTH-type transcriptiona  30.1 1.5E+02  0.0018   16.8   2.8   36   48-89      1-36  (47)
</pre><pre> 46 8AZW_h uL18 (60S ribosomal pro  29.8 3.2E+02  0.0037   21.6   5.3   65   28-98    167-249 (301)
</pre><pre> 47 3HE5_B SYNZIP2; heterodimeric   29.0   2E+02  0.0023   16.8   4.4   36   60-99      5-40  (52)
</pre><pre> 48 7C70_B Sugar ABC transporter,   28.7 4.1E+02  0.0047   20.2   6.1   42   57-98    374-415 (423)
</pre><pre> 49 PF20136.3 ; DUF6526 ; Family o  28.6   2E+02  0.0024   20.7   3.7   30   65-94    111-140 (144)
</pre><pre> 50 PF21616.1 ; Topo-V_HhH_2_2nd ;  28.2 2.2E+02  0.0026   17.1   3.8   34   56-89     23-57  (58)
</pre><pre> 51 2HEP_A UPF0291 protein ynzC; S  27.6 2.9E+02  0.0034   18.2   5.1   39   61-99      4-45  (85)
</pre><pre> 52 2YXY_A Hypothetical conserved   27.5 2.3E+02  0.0026   19.8   3.6   30   67-98      4-33  (115)
</pre><pre> 53 6P4X_B Glucokinase-1; hexokina  27.2 3.7E+02  0.0043   22.0   5.4   41   56-99      7-47  (500)
</pre><pre> 54 PF21106.1 ; YtxK_like ; YtxK N  26.9 1.8E+02  0.0021   18.2   3.0   30   60-89     44-73  (80)
</pre><pre> 55 PF05979.16 ; DUF896 ; Bacteria  26.9 2.5E+02  0.0029   17.2   4.7   37   63-99      2-41  (61)
</pre><pre> 56 PF04690.17 ; YABBY ; YABBY pro  26.4 2.7E+02  0.0031   19.9   4.0   32   57-90    113-144 (156)
</pre><pre> 57 7KDF_B NUF2 isoform 1,NUF2 iso  26.0 2.5E+02  0.0028   20.7   3.9   37   60-96    178-215 (215)
</pre><pre> 58 5UDO_D A118 serine integrase;   25.7 4.3E+02  0.0049   19.5   5.5   37   61-97    224-260 (328)
</pre><pre> 59 PF06570.15 ; DUF1129 ; Protein  25.5 3.9E+02  0.0046   19.0   5.4   39   60-98      3-41  (201)
</pre><pre> 60 5CW4_B Protein FAM175B; Metal   24.5 3.5E+02   0.004   21.3   4.6   42   57-98    226-267 (289)
</pre><pre> 61 2CAZ_B VACUOLAR PROTEIN SORTIN  24.4 4.3E+02  0.0049   19.4   4.7   37   63-99     54-90  (155)
</pre><pre> 62 PF08855.14 ; DUF1825 ; Domain   24.1 3.6E+02  0.0041   18.0   5.4   36   64-99     68-103 (103)
</pre><pre> 63 PF11460.12 ; DUF3007 ; Protein  22.9 3.9E+02  0.0045   18.0   4.2   33   51-91     60-95  (96)
</pre><pre> 64 3J79_R 60S ribosomal protein u  22.5 5.9E+02  0.0068   20.3   5.6   65   28-98    170-249 (294)
</pre><pre> 65 6KSR_A Phosphotransferase; Hex  22.5 2.7E+02  0.0032   22.2   3.8   37   60-99      2-38  (467)
</pre><pre> 66 PF18175.5 ; HU-CCDC81_bac_2 ;   22.5 2.5E+02  0.0029   15.7   3.8   30   70-99      4-33  (69)
</pre><pre> 67 2F6M_D Vacuolar protein sortin  22.3   4E+02  0.0047   18.0   5.1   35   63-97     37-71  (109)
</pre><pre> 68 5MK9_A MalE1; carbohydrate bin  22.3 5.2E+02   0.006   19.3   5.1   38   61-98    341-378 (379)
</pre><pre> 69 5UOI_A HHH_rd1_0142; de novo d  22.2 2.6E+02   0.003   15.7   3.4   25   66-90      2-27  (43)
</pre><pre> 70 PF09650.14 ; PHA_gran_rgn ; Pu  22.1 2.5E+02  0.0029   17.5   3.0   22   78-99      8-29  (87)
</pre><pre> 71 8BTR_LF Ribosomal protein L5;   21.7 6.6E+02  0.0076   20.3   6.9   65   28-98    176-256 (297)
</pre><pre> 72 6S7O_E Dolichyl-diphosphooligo  21.5 5.5E+02  0.0064   22.2   5.6   37   61-97    559-595 (607)
</pre><pre> 73 6QLF_Y Inner kinetochore subun  21.4 6.4E+02  0.0074   20.0   5.6   40   57-97    195-238 (238)
</pre><pre> 74 1SF9_A yfhH hypothetical prote  21.3 3.4E+02  0.0039   19.4   3.6   27   70-98     25-51  (128)
</pre><pre> 75 6RM3_LD0 uL18; Microsporidia,   21.2 5.4E+02  0.0063   20.5   5.1   65   28-98    173-254 (295)
</pre><pre> 76 PF06738.16 ; ThrE ; Putative t  21.1 4.3E+02  0.0049   19.6   4.4   29   65-93     72-100 (238)
</pre><pre> 77 PF11527.12 ; ARL2_Bind_BART ;   21.0 3.8E+02  0.0044   17.2   4.3   30   56-88     47-76  (118)
</pre><pre> 78 3OIQ_B DNA polymerase alpha ca  20.9 2.1E+02  0.0025   15.5   2.0   15   58-72     11-25  (36)
</pre><pre> 79 PF15207.10 ; TMEM240 ; TMEM240  20.7      56 0.00064   24.5  -0.2   10    1-10     56-65  (179)
</pre><pre> 80 1YWW_A Hypothetical protein PA  20.7 3.5E+02  0.0041   16.7   3.6   39   57-95     46-85  (85)
</pre><pre> 81 4UA8_A Carbohydrate ABC transp  20.3   3E+02  0.0035   20.7   3.5   37   63-99    360-399 (400)
</pre><pre> 82 6LMJ_A A104R; Complex, dsDNA,   20.2   4E+02  0.0046   17.1   4.0   33   64-99     18-50  (110)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF14412.10 ; AHH ; A nuclease family of the HNH/ENDO VII superfamily with conserved AHH
</pre><pre>Probab=99.54  E-value=1.2e-14  Score=97.13  Aligned_cols=90  Identities=17%  Similarity=0.216  Sum_probs=68.4  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhhhcCH-------------------HHHhcCCcCCCccceeecCCCCCC--CCCCCcccCCCcCCCChHHH
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEFRNHP-------------------FLQRTNIDINGATNMTYLPVADGI--HPTKVIHADFKLDGLPHSDY   59 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~~~~~-------------------~l~~~g~din~~~Ngi~LP~~~~~--~~~~~~H~G~~~~~~~H~~Y   59 (99)
</pre><pre>                      +|+|||||.+.....                   ++...|||+|  .|+++||.....  ..+.+.|.|      .|+.|
</pre><pre>T Consensus        17 ~~~HHiip~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~g~~~~--~N~i~LP~~~~~~~~~~~~~H~g------~H~~Y   88 (139)
</pre><pre>T Q2SED4_HAHCH/1   17 NQVHHIVNISSVRNGIEEIAKDLPDLRVLIVNGLLDEKYNINYK--DNSLILPTQRSASAKTGLPSHYG------SHPSY   88 (139)
</pre><pre>T ss_pred             cCceeeecHHHHHhchhhhhcCCchHhHHHHcchHHHhcCCCCC--CCeeecCCchHHHhhhCCCcccC------CChHH
</pre><pre>Confidence            589999999873322                   7788889999  999999986554  356789999      99999
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHH----HHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   60 NKKM----GGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        60 ~~~V----~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ++.|    .+.|+.+.........+..++...+.++..+|+..
</pre><pre>T Consensus        89 ~~~v~~~~~~~l~~~~~~~~~~~~~~~~~~~~l~~l~~~l~~~  131 (139)
</pre><pre>T Q2SED4_HAHCH/1   89 SKDILQHVKQALQPYKKIANEMKKNKPHSKPDPVELKKKLEDA  131 (139)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHchhcCcccCCCHHHHHHHHHHH
</pre><pre>Confidence            9999    99999988776554455555666666665555543
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>PF09533.14 ; DUF2380 ; Predicted lipoprotein of unknown function (DUF2380)
</pre><pre>Probab=96.56  E-value=0.009  Score=43.69  Aligned_cols=73  Identities=19%  Similarity=0.298  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhhhcCHHHHhcCCcCCCccceeecCCCCCCCCCCCcccCCCcCCCChHH-----HHHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEFRNHPFLQRTNIDINGATNMTYLPVADGIHPTKVIHADFKLDGLPHSD-----YNKKMGGILDALEERAI   75 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~~~~~~l~~~g~din~~~Ngi~LP~~~~~~~~~~~H~G~~~~~~~H~~-----Y~~~V~~~Ld~i~~~~~   75 (99)
</pre><pre>                      ++.|||+|++..-...|+..|+||+.  -.+.+|..        .|..      .|..     ||+.-++-+++-..   
</pre><pre>T Consensus       100 ~e~HHifPQ~~~~~~~F~~~GInIhd--ytv~i~~~--------~H~~------IH~G~~gg~WN~~W~~fi~~n~~---  160 (189)
</pre><pre>T E3FZI5_STIAD/5  100 LIKHHLFPQEARLATWFRANGINIHE--WTMLVPEQ--------VHLR------VHRGARGGLWNEAWRQYYEANSA---  160 (189)
</pre><pre>T ss_pred             cccccccchHHHHHHHHHHCCCcccc--eeeecCHH--------Hhhh------hccccccccHHHHHHHHHHHcCC---
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             hcCC-CHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   76 LDGW-DNNRATKEALNLQSD   94 (99)
</pre><pre>Q Consensus        76 ~~~~-~~~~~~~~i~~l~~~   94 (99)
</pre><pre>                        +. |++++.+...+|+.+
</pre><pre>T Consensus       161 --~a~T~~eI~~~a~~L~~r  178 (189)
</pre><pre>T E3FZI5_STIAD/5  161 --RPVSREELLSKAFELALR  178 (189)
</pre><pre>T ss_pred             --CCCCHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>PF05120.16 ; GvpG ; Gas vesicle protein G
</pre><pre>Probab=83.22  E-value=13  Score=23.13  Aligned_cols=37  Identities=14%  Similarity=0.100  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      ..|+++|.++...+..+.+|.++..+.-..|...++.
</pre><pre>T Consensus        31 ~~i~~~L~eL~~~~e~GeIseeEy~~~e~~Ll~rl~~   67 (72)
</pre><pre>T Q2JJS5_SYNJB/2   31 ENLQKELTALQIQLDLGEIDEETYARREEEILLALEA   67 (72)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>PF21648.1 ; M1E1E6-like ; Integron cassette protein
</pre><pre>Probab=81.65  E-value=8.7  Score=25.74  Aligned_cols=37  Identities=8%  Similarity=-0.044  Sum_probs=0.0  Template_Neff=6.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   59 YNKKMGGILDALEERAILDGWDNNRATKEALNLQSDT   95 (99)
</pre><pre>Q Consensus        59 Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~l   95 (99)
</pre><pre>                      |-..+...|..|.+..+.+.++.+++.+++++|.+-+
</pre><pre>T Consensus        64 yP~~l~~~le~lW~~~~~~~l~~ee~Q~~l~ela~WI  100 (104)
</pre><pre>T A0A091AID3_9PR   64 PPAILNDLLEHVWEAWRNGYLKDESVDQELHAVEEWL  100 (104)
</pre><pre>T ss_pred             CchHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>1GZS_D SOPE; TOXIN/CELL CYCLE, COMPLEX (TOXIN-CELL CYCLE PROTEIN), SOPE, CDC42, SALMONELLA TYPHIMURIUM, GEF, TOXIN, GTP- BINDING, LIPOPROTEIN, PRENYLATION; HET: SO4; 2.3A {HOMO SAPIENS} SCOP: a.168.1.1
</pre><pre>Probab=80.37  E-value=31  Score=25.37  Aligned_cols=84  Identities=14%  Similarity=0.135  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHhcC-----CcCCCcc-ceeecCCCCCC-----CCCCCcccCCCcCCCCh---HHHHHHHHHHHHHHHHHHHhc--C
</pre><pre>Q FD01848239_023   15 PFLQRTN-----IDINGAT-NMTYLPVADGI-----HPTKVIHADFKLDGLPH---SDYNKKMGGILDALEERAILD--G   78 (99)
</pre><pre>Q Consensus        15 ~~l~~~g-----~din~~~-Ngi~LP~~~~~-----~~~~~~H~G~~~~~~~H---~~Y~~~V~~~Ld~i~~~~~~~--~   78 (99)
</pre><pre>                      ++|+++|     +.+.+.. |||+.|...+.     ..--+.+..|+....++   ..+..++++.+-......-..  .
</pre><pre>T Consensus        64 ~yl~eiG~~A~~~GL~g~~Kngvf~P~GaGanpf~t~v~ss~~~k~p~~~~n~~q~~~~~~~Ae~~i~~~V~~l~~e~g~  143 (165)
</pre><pre>T 1GZS_D           64 PFLQEIGEAAKNAGLPGTTKNDVFTPSGAGANPFITPLISSANSKYPRMFINQHQQASFKIYAEKIIMTEVAPLFNECAM  143 (165)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTCCEEEETTEEEETTCSCCTTHHHHHHHHHHHCGGGGSSHHHHHHHHHHHHHHHHHHHGGGGTTCSS
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCCCCCCCCeeCCCCCCCCCccHHHccHHHhhCHHHhcCHHhHHHHHHHHHHHHHHHHHHHhhhcCC
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             CCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   79 WDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        79 ~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      .++.+....+..+.+.....
</pre><pre>T Consensus       144 ~~P~~F~~~L~~Ia~ky~~~  163 (165)
</pre><pre>T 1GZS_D          144 PTPQQFQLILENIANKYIQN  163 (165)
</pre><pre>T ss_dssp             CCHHHHHHHHHHHHHHHHSC
</pre><pre>T ss_pred             CCHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>3FY6_D Integron cassette protein; Novel, Integron cassette protein, Vibrio cholerae, Oyster pond, Woodshole, USA, UNKNOWN FUNCTION, Structural Genomics, PSI-2, Protein; HET: MSE; 2.1A {Vibrio cholerae}
</pre><pre>Probab=75.92  E-value=15  Score=25.53  Aligned_cols=37  Identities=11%  Similarity=-0.066  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   59 YNKKMGGILDALEERAILDGWDNNRATKEALNLQSDT   95 (99)
</pre><pre>Q Consensus        59 Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~l   95 (99)
</pre><pre>                      |-..+...|..|.+..+.+.++.+++.+++++|.+-+
</pre><pre>T Consensus        73 yP~~l~~~le~lW~~~~~~~l~~ee~Q~~L~ela~WI  109 (126)
</pre><pre>T 3FY6_D           73 PPEITQNLFQHAWLEWRKGALDNDEVTRELELVAQWV  109 (126)
</pre><pre>T ss_dssp             CBTTHHHHHHHHHHHHHTTSSCHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CchhHHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>PF07487.17 ; SopE_GEF ; SopE GEF domain
</pre><pre>Probab=74.19  E-value=41  Score=23.79  Aligned_cols=77  Identities=6%  Similarity=-0.027  Sum_probs=0.0  Template_Neff=5.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHhcC-------CcCCCccceeecCCCCCC-----------CCCCCcccCCCcCCCCh---HHHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   15 PFLQRTN-------IDINGATNMTYLPVADGI-----------HPTKVIHADFKLDGLPH---SDYNKKMGGILDALEER   73 (99)
</pre><pre>Q Consensus        15 ~~l~~~g-------~din~~~Ngi~LP~~~~~-----------~~~~~~H~G~~~~~~~H---~~Y~~~V~~~Ld~i~~~   73 (99)
</pre><pre>                      ++|+++|       .+... .|||+.|+..+.           ...++.+..      .+   ..+..++++++-.....
</pre><pre>T Consensus        35 ~yl~eiG~aA~~~GL~g~~-K~gvf~P~GaGAnpf~t~v~~~~~~~~~~~~~------~~~~~~~~~~~a~~~i~~~v~~  107 (134)
</pre><pre>T Q7P1B7_CHRVO/1   35 GYLRELGEAARAAGLPGEN-KHGVFIPSGDGASPFVNYLLIPVQKEFGHRLR------NESQQRLFRDFARQLSMELVAP  107 (134)
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCCCcc-cCCeeCCCCCCCCchhHHHHHHHHHHhhHhhc------CHHHHHHhHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHhc--CCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   74 AILD--GWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        74 ~~~~--~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      .-..  ..++.+....++.+.+.-+..
</pre><pre>T Consensus       108 ~~~e~~~~~p~~F~~~L~~i~~~~~~~  134 (134)
</pre><pre>T Q7P1B7_CHRVO/1  108 HAAERGWEPPAAFAARLEAVGRPWLAQ  134 (134)
</pre><pre>T ss_pred             HHHHhCCCCHHHHHHHHHHHHhHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF15635.10 ; Tox-GHH2 ; GHH signature containing HNH/Endo VII superfamily nuclease toxin  2
</pre><pre>Probab=72.67  E-value=3  Score=27.09  Aligned_cols=10  Identities=20%  Similarity=0.315  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CccccCChhh
</pre><pre>Q FD01848239_023    2 NRHHIIPYEF   11 (99)
</pre><pre>Q Consensus         2 qaHHIIP~~~   11 (99)
</pre><pre>                      +.|||||...
</pre><pre>T Consensus         2 t~hHivpd~~   11 (105)
</pre><pre>T G4T2Y2_META2/2    2 TPDHLIDVKS   11 (105)
</pre><pre>T ss_pred             CCCceeeHHH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>PF13391.10 ; HNH_2 ; HNH endonuclease
</pre><pre>Probab=69.16  E-value=2.7  Score=22.20  Aligned_cols=36  Identities=14%  Similarity=0.098  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhh--hcCHHHHhcC-CcCCCccceeecCCC
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEF--RNHPFLQRTN-IDINGATNMTYLPVA   36 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~--~~~~~l~~~g-~din~~~Ngi~LP~~   36 (99)
</pre><pre>                      .++.||+|...  ...+.....+ .++++..|++.|...
</pre><pre>T Consensus        11 ~~~ahi~~~~~~~~~~~~~~~~~~~~~~~~~N~i~l~~~   49 (62)
</pre><pre>T Q5KQA0_CRYNJ/1   11 CTASHIVPASRPDIYDCFYGDGGGLPMFRPSAGLLLRDD   49 (62)
</pre><pre>T ss_pred             eeEEEEecccchhhhhhhcCCCCCCCCCChHHhEEECch
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>2JOK_A Putative G-nucleotide exchange factor; Guanine nucleotide exchange factor, Burkholderia pseudomallei, Type III secretion, SopE, SopE2, CELL INVASION, SIGNALING PROTEIN; NMR {Burkholderia pseudomallei} SCOP: a.168.1.1
</pre><pre>Probab=66.27  E-value=67  Score=24.19  Aligned_cols=78  Identities=12%  Similarity=0.111  Sum_probs=0.0  Template_Neff=4.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHhcC-----CcCCCcc-ceeecCCCCCC-----------CCCCCcccCCCcCCCCh---HHHHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   15 PFLQRTN-----IDINGAT-NMTYLPVADGI-----------HPTKVIHADFKLDGLPH---SDYNKKMGGILDALEERA   74 (99)
</pre><pre>Q Consensus        15 ~~l~~~g-----~din~~~-Ngi~LP~~~~~-----------~~~~~~H~G~~~~~~~H---~~Y~~~V~~~Ld~i~~~~   74 (99)
</pre><pre>                      ++|+++|     +.+.+.. |||+.|...+.           ...++.+..      ++   ..+..++++++-......
</pre><pre>T Consensus        69 ~yl~eIG~aA~~aGL~g~~Kngvf~P~GaGAnpf~t~v~~~~~~k~p~~~~------n~~q~~~~~~yA~q~i~~~V~~l  142 (186)
</pre><pre>T 2JOK_A           69 AFIEALGDAARATGLPGADKQGVFTPSGAGTNPLYTEIRLRADTLMGAELA------ARPEYRELQPYARQQAIDLVANA  142 (186)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTCSCEEETTEEECCSCCCCSSHHHHHHHHHHHCCSHHH------HSGGGTTCSHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHcCCCCcccCCccCCCCCCCCCchHHHhHHHHHHchhhhh------cchhHHhhHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             Hhc--CCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   75 ILD--GWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        75 ~~~--~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      -..  ..++......+..|.+..+..
</pre><pre>T Consensus       143 ~~e~~~~~p~~Fr~~L~~Ia~ky~~~  168 (186)
</pre><pre>T 2JOK_A          143 LPAERSNTLVEFRQTVQTLEATYRRA  168 (186)
</pre><pre>T ss_dssp             CTTHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ChhcCCCCHHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6AZ3_O 60S ribosomal protein L5, putative; Leishmania, ribosome, aminoglycoside, paromomycin, RIBOSOME-ANTIBIOTIC complex; HET: OMU, OMC, PAR, OMG, A2M, MG; 2.5A {Leishmania donovani}
</pre><pre>Probab=63.30  E-value=90  Score=24.93  Aligned_cols=65  Identities=11%  Similarity=-0.031  Sum_probs=0.0  Template_Neff=6.700
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC-------------cccCCCcCCCCh-HHHHHHHHHH-------HHHHHHHHHhcCCCHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV-------------IHADFKLDGLPH-SDYNKKMGGI-------LDALEERAILDGWDNNRATK   86 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~-------------~H~G~~~~~~~H-~~Y~~~V~~~-------Ld~i~~~~~~~~~~~~~~~~   86 (99)
</pre><pre>                      ++||.+|.+...-+++.             .=.|      .| ..|...+.+.       ...-...+...+++++.+.+
</pre><pre>T Consensus       174 d~Gl~ip~~~~~~p~~~~e~~~~~~e~~r~rI~G------~hi~~y~~~l~~~~~~~~e~~~~~fs~~~~~~~~~~~~~~  247 (305)
</pre><pre>T 6AZ3_O          174 DGGMAVPHRPNRFPGYNKEKSSLDAKVHRDRIFG------KHVAEYLKQVKEEASSNPDEKCVQFSRYMAAKVLPESIEG  247 (305)
</pre><pre>T ss_dssp             TTTCBCCCCTTSSTTBCTTTCCBCHHHHHHHHTT------HHHHHHHHHHHHHTTSCC----CTTHHHHTTTCCTTTTHH
</pre><pre>T ss_pred             hcCccccCCCCcCCCCCcCCCcCChHHHHHHHcc------ccHHHHHHHHHhhhcCChhHhHHHHHHHHhcCCChHhHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhc
</pre><pre>Q FD01848239_023   87 EALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        87 ~i~~l~~~lR~~   98 (99)
</pre><pre>                      .+.+.-..+|+.
</pre><pre>T Consensus       248 ~~~~~~~~I~~~  259 (305)
</pre><pre>T 6AZ3_O          248 MYKKAHAAIRAD  259 (305)
</pre><pre>T ss_dssp             HHHHHHHHHHSC
</pre><pre>T ss_pred             HHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>5T5H_u 60S ribosomal protein L5; Trypanosoma cruzi, ribosomal subunit, assmebly, multiply fragmented rRNAs, 2.5 angstrom cryo-EM structure, RIBOSOME; HET: 5MC, OMU, OMC, OMG, A2M, MG, 7MG; 2.54A {Trypanosoma cruzi}
</pre><pre>Probab=62.91  E-value=67  Score=24.87  Aligned_cols=65  Identities=14%  Similarity=0.018  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC-------------cccCCCcCCCCh-HHHHHHHHHH-------HHHHHHHHHhcCCCHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV-------------IHADFKLDGLPH-SDYNKKMGGI-------LDALEERAILDGWDNNRATK   86 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~-------------~H~G~~~~~~~H-~~Y~~~V~~~-------Ld~i~~~~~~~~~~~~~~~~   86 (99)
</pre><pre>                      ++|+.+|.+...-++..             .=.|      .| .+|...+.+.       .......+...+++++.+.+
</pre><pre>T Consensus       169 d~Gl~ip~~~~~~p~~~~e~~~~~~~~~r~rI~g------~hi~~y~~~l~~~~~~~~~~~~~~f~~~~~~~~~~~~~~~  242 (254)
</pre><pre>T 5T5H_u          169 DGGLAVPHRPNRFPGYDKEKDSLNAKAHRDRILG------LHVAEYLKQVREEASSNPEESTCQFSKYIKDKVVPNELEG  242 (254)
</pre><pre>T ss_pred             hCCccccCCCCcCCCCCccccccChHHHHHHHHH------HHHHHHHHHHHHhhcCChHHhHHHHHHHHhcCCCHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhc
</pre><pre>Q FD01848239_023   87 EALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        87 ~i~~l~~~lR~~   98 (99)
</pre><pre>                      .+.+.-..+|+.
</pre><pre>T Consensus       243 ~~~~~~~~I~~~  254 (254)
</pre><pre>T 5T5H_u          243 MYKKAHAAIRAD  254 (254)
</pre><pre>T ss_pred             HHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>6TH6_BQ 50S ribosomal protein L18; T. kodakarensis, ac4C, Ribosome, cryo-EM; HET: ZN, 5MU, 6MZ, 5MC, 4AC, OMU, 4SU, MA6, OMC, OMG, LHH, A2M, LV2; 2.55A {Thermococcus kodakarensis (strain ATCC BAA-918 / JCM 12380 / KOD1)}
</pre><pre>Probab=62.38  E-value=66  Score=23.77  Aligned_cols=65  Identities=11%  Similarity=0.163  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCCcccCCCcCCCCh-HHHHHHHHH-H---HHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKVIHADFKLDGLPH-SDYNKKMGG-I---LDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~~H~G~~~~~~~H-~~Y~~~V~~-~---Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ++||.+|.....-+....-.|      .| .+|...+.+ .   ...-...+...+++++.+.+.+.+.-..+++.
</pre><pre>T Consensus       126 d~Gl~i~~~~~~~p~~~ri~G------~hi~~y~~~l~~~~~~~~~~~fs~~~~~~~~~~~~~~~~~~~~~~I~~~  195 (201)
</pre><pre>T 6TH6_BQ         126 DAGLNVPHSEEIYPEDYRING------EHIANYAKALKEEDEALYRKQFSGYLVKGLEPEKLPEHFEEVKAKIIEK  195 (201)
</pre><pre>T ss_pred             hcCccCCCCCCcChhhhhcch------hhHHHHHHHchhhCHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>PF05766.16 ; NinG ; Bacteriophage Lambda NinG protein
</pre><pre>Probab=57.47  E-value=28  Score=24.80  Aligned_cols=74  Identities=9%  Similarity=0.059  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhh-hcCHHHHhcCCcCCCccceeecCCCCCCCCCCCcccCCCcCCCCh---HHHHHHHHHHH-----HHHH
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEF-RNHPFLQRTNIDINGATNMTYLPVADGIHPTKVIHADFKLDGLPH---SDYNKKMGGIL-----DALE   71 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~-~~~~~l~~~g~din~~~Ngi~LP~~~~~~~~~~~H~G~~~~~~~H---~~Y~~~V~~~L-----d~i~   71 (99)
</pre><pre>                      +++|||+|... .         -.-.+..|++.|=..        .|..      .|   ..|...+...+     +.|.
</pre><pre>T Consensus       109 ~~~~Hi~~~~~~~---------~~~~~~~N~~~lC~~--------Ch~~------~~~~~~~~~~~l~~~~g~~~~~~l~  165 (197)
</pre><pre>T A0A2U2GVX8_YER  109 YHAGHYLTVGANP---------ELRFNEDNCHRQCAP--------CNNH------LSGNIEKYTPNLIAKIGQVRFDILM  165 (197)
</pre><pre>T ss_pred             ceeccccCcCCCh---------hhcCChhhcHHhCHH--------HHhh------ccCCHHhhHHHHHHHHCHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   72 ERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        72 ~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      .......++.+++.+.+..+...+++
</pre><pre>T Consensus       166 ~~~~~~k~~~~~~~~~~~~~~~~~~~  191 (197)
</pre><pre>T A0A2U2GVX8_YER  166 GPHEMTNYRRDDYIRIRDEYRAKTKA  191 (197)
</pre><pre>T ss_pred             CCCcccCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>7P9K_B DUF262 domain-containing protein; Restriction endonuclease, Phage defence protein, DNA BINDING PROTEIN; HET: SO4, GOL; 2.12A {Escherichia fergusonii (strain ATCC 35469 / DSM 13698 / CCUG 18766 / IAM 14443 / JCM 21226 / LMG 7866 / NBRC 102419 / NCTC 12128 / CDC 0568-73)}
</pre><pre>Probab=56.10  E-value=13  Score=30.41  Aligned_cols=39  Identities=18%  Similarity=0.221  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhhhcCHHHHhcCCcC----------CCccceeecCCCCCC
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEFRNHPFLQRTNIDI----------NGATNMTYLPVADGI   39 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~~~~~~l~~~g~di----------n~~~Ngi~LP~~~~~   39 (99)
</pre><pre>                      +..|||+|++......+...++.-          |.-.|++.|+.....
</pre><pre>T Consensus       471 ~~ieHI~P~~~~~~~~~~~~~~~~~~~~~~~~~~n~iGNl~lL~~~~N~  519 (587)
</pre><pre>T 7P9K_B          471 FHVDHIYPKGLFTRNKLAKVGVPAEQLDELIEASNKLPNLQLLEGTINN  519 (587)
</pre><pre>T ss_dssp             EEEEESSCGGGCCHHHHHHHTCCTTSHHHHHHHHTBGGGEEEEESSCCT
</pre><pre>T ss_pred             CccccccchhhccchhHHhcCCCHHHHHHHHHHhhcchhHHccCCcccc
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>PF19551.3 ; DUF6074 ; Family of unknown function (DUF6074)
</pre><pre>Probab=51.50  E-value=92  Score=19.49  Aligned_cols=39  Identities=15%  Similarity=0.152  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ..+....++.+.......+++++.+..++..+....+.+
</pre><pre>T Consensus        33 ~~~w~~~~~~l~~~l~~~G~~~~~i~~ev~~f~~aV~~e   71 (81)
</pre><pre>T A0A0K2DBD8_9HY   33 EAKWRGLLKRHRRLRRELGMAEEAIDADLLAFAAAVRNL   71 (81)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHcCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>8HL4_L18P 50S ribosomal protein L18; Sulfolobus acidocaldarius ribosome small subunit, RIBOSOME; HET: GNP, UNK;{Sulfolobus acidocaldarius DSM 639}
</pre><pre>Probab=50.76  E-value=1.5e+02  Score=21.71  Aligned_cols=64  Identities=20%  Similarity=0.201  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCCcccCCCcCCCCh-HHHHHHHHH---H-HHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKVIHADFKLDGLPH-SDYNKKMGG---I-LDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~~H~G~~~~~~~H-~~Y~~~V~~---~-Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      +.|+.+|.+. .-+....=.|      .| ..|...+.+   . ...-...+...+++++.+.+.+.+.-..+++.
</pre><pre>T Consensus       125 d~Gl~i~~~~-~~p~~~ri~g------~hi~~y~~~l~~~~~~~~~~~fs~~~~~~~~~~~~~~~~~~~~~~i~~~  193 (193)
</pre><pre>T 8HL4_L18P       125 DAGLKVPVGD-LGKLKDRVNG------SHISAYAQKLKNENQELYNKLFSSYIQRGLDPVLLPQHFEEVLNKIKEN  193 (193)
</pre><pre>T ss_pred             hcCccccccc-cCCchHhccC------ccHHHHHHHHhhhCHHHHHHHHHHHHHcCCChhhHHHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>6XOR_A Protein swallow; coiled coil, self-association domain, RNA BINDING PROTEIN; NMR {Drosophila melanogaster}
</pre><pre>Probab=48.03  E-value=1.1e+02  Score=19.53  Aligned_cols=40  Identities=15%  Similarity=0.136  Sum_probs=0.0  Template_Neff=3.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   58 DYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        58 ~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      .|++++..+|++-....-.-.-+-+.++.++.+..+.||.
</pre><pre>T Consensus        28 ~yndYLQ~kLdeK~~e~~~lr~Nfe~lR~eLs~ck~KLkr   67 (71)
</pre><pre>T 6XOR_A           28 GFNEYVQERLDRITDDFVKMKDNFETLRTELSEAQQKLRR   67 (71)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>PF11867.12 ; T1RH-like_C ; Type I restriction enzyme HindI endonuclease subunit-like, C-terminal
</pre><pre>Probab=47.91  E-value=1.3e+02  Score=23.08  Aligned_cols=42  Identities=17%  Similarity=0.099  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      |.+.....++|.++.+.+..+..+..+..+++..+.+++...
</pre><pre>T Consensus       198 p~~y~~~serl~~li~~~~~~~~~~~e~l~el~~l~~~~~~~  239 (334)
</pre><pre>T W8RWQ4_9RHOB/6  198 VVQNQKFSERLTNAIARYHNRSVDALQVIQELIGMAKDLSAE  239 (334)
</pre><pre>T ss_pred             hhhhhhHHHHHHHHHHHHhhCCCcHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>8A3D_G 60S ribosomal protein L5; Ribosome, human, TRANSLATION, rna modifications, water, magnesium, potassium, homoharringtonine; HET: K, HMT, 1MA, 6MZ, EPE, 5MC, OMU, OMC, PSU, OMG, SPM, A2M, UR3, MG; 1.67A {Homo sapiens}
</pre><pre>Probab=47.06  E-value=2.2e+02  Score=22.51  Aligned_cols=65  Identities=11%  Similarity=0.050  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCC-------------CcccCCCcCCCCh-HHHHHHHH----HHHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTK-------------VIHADFKLDGLPH-SDYNKKMG----GILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~-------------~~H~G~~~~~~~H-~~Y~~~V~----~~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      .+||.+|.+...-++.             ..=.|      .| ..|...+.    +.-......+...+++++.+.+.+.
</pre><pre>T Consensus       168 d~Gl~ip~~~~~~p~~~~e~~~y~~~~~r~ri~g------~hi~~y~~~l~~~d~~~~~~~fs~~~~~~~~~~~~~~~f~  241 (297)
</pre><pre>T 8A3D_G          168 DGGLSIPHSTKRFPGYDSESKEFNAEVHRKHIMG------QNVADYMRYLMEEDEDAYKKQFSQYIKNSVTPDMMEEMYK  241 (297)
</pre><pre>T ss_dssp             HTTCBCCCCSTTSTTEETTTTEECHHHHHHHHTT------HHHHHHHHHHHHHCHHHHHHHTHHHHHTTCCGGGHHHHHH
</pre><pre>T ss_pred             cCCCcCCCCCccCCCCCCCCcccCHHHHHHHhhc------chHHHHHHHhhhcCHHHHHHHHHHHHhcCCChHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhc
</pre><pre>Q FD01848239_023   90 NLQSDTRKN   98 (99)
</pre><pre>Q Consensus        90 ~l~~~lR~~   98 (99)
</pre><pre>                      ++-..++..
</pre><pre>T Consensus       242 ~~~~~I~~~  250 (297)
</pre><pre>T 8A3D_G          242 KAHAAIREN  250 (297)
</pre><pre>T ss_dssp             HHHHHHHHC
</pre><pre>T ss_pred             HHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>4V8P_BM 60S RIBOSOMAL PROTEIN L5; RIBOSOME, EUKARYOTIC INITIATION FACTOR 6, EIF6, 60S, TRANSLATION, LARGE RIBOSOMAL SUBUNIT, RRNA, RIBOSOMAL PROTEIN; HET: MG, ZN; 3.52A {TETRAHYMENA THERMOPHILA}
</pre><pre>Probab=46.10  E-value=1.6e+02  Score=23.50  Aligned_cols=65  Identities=17%  Similarity=0.182  Sum_probs=0.0  Template_Neff=6.900
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC------------------cccCCCcCCCCh-HHHHHHHHHH----HHHHHHHHHh--cCCCHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV------------------IHADFKLDGLPH-SDYNKKMGGI----LDALEERAIL--DGWDNN   82 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~------------------~H~G~~~~~~~H-~~Y~~~V~~~----Ld~i~~~~~~--~~~~~~   82 (99)
</pre><pre>                      ++|+.+|.+...-++..                  .=.|      .| .+|...+.+.    ...-...+..  .+++++
</pre><pre>T Consensus       168 d~Gl~ip~~~~~~p~~~~~~~~~es~~y~~e~~r~rI~g------~hi~~y~~~l~~~~~~~~~~~fs~~~~~~~~~~~~  241 (301)
</pre><pre>T 4V8P_BM         168 DGGINIPHSESRFPGYVRASDEGESSKYKPEDHKARIFG------KHIDAYMKHLKGQSNEAFQKQFSKWSKTLEAAKVD  241 (301)
</pre><pre>T ss_pred             cCCeeccCCCccCCCcccCCCCCCCccCChHHHHHHHhc------cHHHHHHHHhccCCHHHHHHHHHHHHHHhhcCCch
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   83 RATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        83 ~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      .+.+.+.+.-..+|+.
</pre><pre>T Consensus       242 ~~~~~~~~~~~~i~~~  257 (301)
</pre><pre>T 4V8P_BM         242 SVEKLFTKVHAEIRKN  257 (301)
</pre><pre>T ss_pred             hHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>6ZU5_LD0 uL18; Microsporidia, Pathogen, Ribosome, Hibernation, Genome Compaction; HET: AMP, MG; 2.9A {Paranosema locustae}
</pre><pre>Probab=44.99  E-value=2.2e+02  Score=22.02  Aligned_cols=65  Identities=15%  Similarity=0.041  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC------------cccCCCcCCCCh-HHHHHHHH----HHHHHHHHHHHhcCCCHHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV------------IHADFKLDGLPH-SDYNKKMG----GILDALEERAILDGWDNNRATKEALN   90 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~------------~H~G~~~~~~~H-~~Y~~~V~----~~Ld~i~~~~~~~~~~~~~~~~~i~~   90 (99)
</pre><pre>                      +.|+.+|.+...-++..            .-.|      .| .+|...+.    +.-..-...+...+++++.+.+.+.+
</pre><pre>T Consensus       144 d~Gl~ip~~~~~~p~~~e~~~~~~~~~r~ri~g------~hi~~y~~~l~~~d~~~~~~~fs~~~~~~~~~~~~~~~~~~  217 (266)
</pre><pre>T 6ZU5_LD0        144 DGGLVMPHSLKRVPGYVSEEEFDSEVFRNKLFG------KILAGYMKEMMENYPEKYKKTFQEYIKKGINPDDLENIYEN  217 (266)
</pre><pre>T ss_pred             hCCccccCCCcccCCCCCCcccChHHHHHHHcc------chHHHHHHHHHhhCHHHHHHHHHHHHHcCCChhhHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhc
</pre><pre>Q FD01848239_023   91 LQSDTRKN   98 (99)
</pre><pre>Q Consensus        91 l~~~lR~~   98 (99)
</pre><pre>                      .-..++..
</pre><pre>T Consensus       218 ~~~~i~~~  225 (266)
</pre><pre>T 6ZU5_LD0        218 AFKKIRED  225 (266)
</pre><pre>T ss_pred             HHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>3BHP_B UPF0291 protein ynzC; NESG, SR384, O31818, UPF0291 protein ynzC, Structural Genomics, PSI-2, Protein Structure Initiative, Northeast Structural Genomics Consortium; HET: MSE; 2.01A {Bacillus subtilis} SCOP: a.2.21.1
</pre><pre>Probab=44.43  E-value=1.2e+02  Score=18.63  Aligned_cols=39  Identities=8%  Similarity=0.049  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHH--HHHH-HHHHHHhcC
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATK--EALN-LQSDTRKNL   99 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~--~i~~-l~~~lR~~L   99 (99)
</pre><pre>                      +.+.++++.+....+..++|+++..+  .+.+ .+..+|..+
</pre><pre>T Consensus         4 ~~~i~RINeLakk~K~~~LT~eE~~Eq~~LRkeYl~~~R~~~   45 (60)
</pre><pre>T 3BHP_B            4 NAKIARINELAAKAKAGVITEEEKAEQQKLRQEYLKGFRSSM   45 (60)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTTCCCHHHHHHHHHHHHHHHHTTHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>1VQ8_N 50S ribosomal protein L18P; ribosome 50S, PROTEIN-PROTEIN COMPLEX, RNA-RNA COMPLEX, PROTEIN-RNA COMPLEX, PEPTIDYL TRANSFERASE REACTION, RIBOSOME; HET: SPS, 1MA, NA, SR, UR3, MG, OMU, OMG, CL, PSU; 2.2A {Haloarcula marismortui} SCOP: c.55.4.1
</pre><pre>Probab=43.97  E-value=1.2e+02  Score=22.30  Aligned_cols=58  Identities=10%  Similarity=0.111  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCCcccCCCcCCCCh-HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKVIHADFKLDGLPH-SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~~H~G~~~~~~~H-~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ++|+.+|.+...-+....=.|      .| .+|...+...      .+ ..+++++++.+.++++-..+++.
</pre><pre>T Consensus       125 d~Gl~i~~~~~~~p~~~ri~g------~hi~~y~~~l~~~------~f-~~~~~~~~~~~~~~~~~~~i~~~  183 (187)
</pre><pre>T 1VQ8_N          125 DAGLDIPHNDDVLADWQRTRG------AHIAEYDEQLEEP------LY-SGDFDAADLPEHFDELRETLLDG  183 (187)
</pre><pre>T ss_dssp             HTTCBCCCCGGGSCCHHHHHT------HHHHHHHHSCSSC------SS-SSCSCSSCCHHHHHHHHHHHHSS
</pre><pre>T ss_pred             ccCCCCCCCCcccCCcccccc------chHHHHHHhcCch------Hh-cCCCCHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>6ZJ3_Ld Ribosomal protein uL18; Single particle cryo-EM RNA modifications, RIBOSOME; HET: 1MA, 6MZ, MIA, B8N, 5MC, OMU, B8H, MA6, OMC, PSU, OMG, JMH, A2M, UR3, JMC, 7MG; 3.15A {Euglena gracilis}
</pre><pre>Probab=43.35  E-value=2.4e+02  Score=21.93  Aligned_cols=65  Identities=15%  Similarity=0.104  Sum_probs=0.0  Template_Neff=7.300
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC-------------cccCCCcCCCCh-HHHHHHHH----HHHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV-------------IHADFKLDGLPH-SDYNKKMG----GILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~-------------~H~G~~~~~~~H-~~Y~~~V~----~~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      ++|+.+|.+...-++..             .-.|      .| .+|...+.    +.-..-...+...+++++.+.+...
</pre><pre>T Consensus       166 d~Gl~ip~~~~~~p~~~~e~~~~~~~~~r~rI~g------~hi~~y~~~l~~~d~~~y~~~fs~~~~~~~~~~~~~~~~~  239 (260)
</pre><pre>T 6ZJ3_Ld         166 DGGVNVPHSMKRFPGYNRDKGEMDSETLRERIFA------GHVAEYQKMLIREEPEKYQEVYSQYIAKGVKPGEVEDMWA  239 (260)
</pre><pre>T ss_pred             hCCcccccccccCCCCCCCCCCCCHHHHHHHHHH------hhHHHHHHHHhhcCHHHHHHHHHHHHHcCCChhhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhc
</pre><pre>Q FD01848239_023   90 NLQSDTRKN   98 (99)
</pre><pre>Q Consensus        90 ~l~~~lR~~   98 (99)
</pre><pre>                      +.-..+|+.
</pre><pre>T Consensus       240 ~~~~~I~~~  248 (260)
</pre><pre>T 6ZJ3_Ld         240 NCHASIRAN  248 (260)
</pre><pre>T ss_pred             HHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>2LSE_A Four Helix Bundle Protein; Structural Genomics, NORTHEAST STRUCTURAL GENOMICS CONSORTIUM, NESG, PSI-Biology, Protein Structure Initiative, DE NOVO PROTEIN; NMR {artificial gene}
</pre><pre>Probab=42.90  E-value=1.5e+02  Score=19.48  Aligned_cols=37  Identities=19%  Similarity=0.236  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +.+-++|..|.+..-  +-.+++++++|+++..+.+..|
</pre><pre>T Consensus         6 kklleklekildevt--dgapdearerieklakdvkdel   42 (101)
</pre><pre>T 2LSE_A            6 KKLLEKLEKILDEVT--DGAPDEARERIEKLAKDVKDEL   42 (101)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHH--HTSCGGGHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             HHHHHHHHHHHHHHh--cCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>7YH8_C L-19437; heterochiral, miniprotein, alpha helix, D-protein, D-peptide, mirror-image, de novo, designer, DE NOVO PROTEIN; HET: DLY, DLE, DHI, DAS, DPN, DTR, DTY, DAR, DIL, DAL, DGL, DTH; 2.2A {synthetic construct}
</pre><pre>Probab=39.77  E-value=1.4e+02  Score=18.05  Aligned_cols=37  Identities=16%  Similarity=0.246  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |......++.|+...+...      +++...+.+++..+|.+|
</pre><pre>T Consensus        25 pelariarevlerarklgn------eeaarfvlelierlrrel   61 (62)
</pre><pre>T 7YH8_C           25 PELARIAREVLERARKLGN------EEAARFVLELIERLRREL   61 (62)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHTC------HHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHhCc------HHHHHHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>PF15188.10 ; CCDC-167 ; Coiled-coil domain-containing protein 167
</pre><pre>Probab=37.92  E-value=1.7e+02  Score=18.73  Aligned_cols=38  Identities=11%  Similarity=0.049  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      .....+|+.+..+.....++++ -++.+++-...+++.+
</pre><pre>T Consensus        14 ~~~~~rle~le~~Lr~~~Ls~e-~R~~Le~E~~~lk~~l   51 (84)
</pre><pre>T G3WU53_SARHA/3   14 SLCRRNLEELNIRLRREELSSE-ARRALEKEKNNLTSKA   51 (84)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhcccCCHH-HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF14357.10 ; DUF4404 ; Domain of unknown function (DUF4404)
</pre><pre>Probab=37.80  E-value=1.4e+02  Score=18.83  Aligned_cols=37  Identities=11%  Similarity=0.056  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   63 MGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +++.|.++........--+++.++.+..+.+++...|
</pre><pre>T Consensus         1 l~~~L~~L~~eL~~~~~~d~~~r~~L~~l~~~i~~~l   37 (85)
</pre><pre>T D2R1W8_PIRSD/5    1 LKATVAELEAELSQVDSLDDETRALLQSAADDISRAL   37 (85)
</pre><pre>T ss_pred             CHHHHHHHHHHHhcCCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>3FXD_A Protein IcmQ; 4 helix bundle, helix-turn-helix, UNKNOWN FUNCTION; 2.1A {Legionella pneumophila}
</pre><pre>Probab=37.68  E-value=1.5e+02  Score=18.00  Aligned_cols=43  Identities=16%  Similarity=0.169  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      ..+.+.....|..+......+.|+..-..+.+.+=+.++|..+
</pre><pre>T Consensus         4 ~~~~e~~~~il~aLD~~i~~GpWe~S~FLr~i~K~L~~ird~~   46 (57)
</pre><pre>T 3FXD_A            4 QLSDEQKETILKALNDAIEKGPWDKSNFLRVIGKKLIAIRDRF   46 (57)
</pre><pre>T ss_dssp             CCCHHHHHHHHHHHHHHHHHSCTTSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hhcHHHHHHHHHHHHHHHHHCCccccHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>7R81_G1 60S ribosomal protein L5; Protein Synthesis Inhibitor, Cytosolic Ribosome, Polysomal Ribosome, Peptidyl Transferase Center, TRANSLATION; HET: SPD, 3HE, MG; 2.7A {Neurospora crassa}
</pre><pre>Probab=36.44  E-value=2.1e+02  Score=22.87  Aligned_cols=65  Identities=12%  Similarity=0.186  Sum_probs=0.0  Template_Neff=6.800
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCC-------------CcccCCCcCCCCh-HHHHHHHHH----HHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTK-------------VIHADFKLDGLPH-SDYNKKMGG----ILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~-------------~~H~G~~~~~~~H-~~Y~~~V~~----~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      ++|+.+|.....-++.             ..=.|      .| .+|...+.+    ....-...+...+++++.+.+...
</pre><pre>T Consensus       171 d~Gl~ip~~~~~~p~~~~es~~~~~e~~r~rI~G------~hi~~y~~~l~~~d~~~~~~~fs~~~k~~~~~~~~~~~~~  244 (301)
</pre><pre>T 7R81_G1         171 DGGIFIPHSENRFPGYDMESEELDAETLKKYIFG------GHVAEYMETLADDDEERYKSQFNRYIEDDLEADGLEDLYA  244 (301)
</pre><pre>T ss_pred             cCCceecccCccCCCCCCCCcccCHHHHHHHHcc------ccHHHHHHHhcccCHHHHHHHHHHHHhcCCCcccHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhc
</pre><pre>Q FD01848239_023   90 NLQSDTRKN   98 (99)
</pre><pre>Q Consensus        90 ~l~~~lR~~   98 (99)
</pre><pre>                      +.-..+|+.
</pre><pre>T Consensus       245 ~~~~~I~~~  253 (301)
</pre><pre>T 7R81_G1         245 EAHAAIRED  253 (301)
</pre><pre>T ss_pred             HHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>PF10520.13 ; Lipid_desat ; Lipid desaturase domain
</pre><pre>Probab=36.28  E-value=37  Score=24.19  Aligned_cols=21  Identities=29%  Similarity=0.672  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChhhhcCHHHHhcC
</pre><pre>Q FD01848239_023    1 YNRHHIIPYEFRNHPFLQRTN   21 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~~~~~~~l~~~g   21 (99)
</pre><pre>                      +|.||..|.+....++++..|
</pre><pre>T Consensus        34 f~~HH~~P~~~~~~~~~~~~~   54 (178)
</pre><pre>T E9HN67_DAPPU/1   34 FREHHIDPTAITRHDFIETNG   54 (178)
</pre><pre>T ss_pred             ccccCCCchhhccccHHHhch
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>8FJG_A H12; De novo protein, Hallucination, repeat proteins, central pockets, pseudocyclic proteins; 2.13A {synthetic construct}
</pre><pre>Probab=36.13  E-value=2.1e+02  Score=19.11  Aligned_cols=43  Identities=14%  Similarity=0.104  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhc-CCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILD-GWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~-~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      ..|-..+.+++.+-......+ .++.++.++.+..+++++-..+
</pre><pre>T Consensus        57 ekyfekmeekikeefeklkkdpsvtledfkkklkeildemleai  100 (109)
</pre><pre>T 8FJG_A           57 EKYFEKMEEKIKEEFEKLKKDPSVTLEDFKKKLKEILDEMLEAI  100 (109)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHCTTCCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHccCCCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF15500.10 ; Ntox1 ; Putative RNase-like toxin, toxin_1
</pre><pre>Probab=35.90  E-value=2e+02  Score=19.94  Aligned_cols=35  Identities=17%  Similarity=0.251  Sum_probs=0.0  Template_Neff=3.800
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHH-------------HHHHHHHHHHh
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATK-------------EALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~-------------~i~~l~~~lR~   97 (99)
</pre><pre>                      +..++++|++|.....   .+++.+.+             ++..|+++|..
</pre><pre>T Consensus        47 np~L~~~l~~ie~~~~---~dPe~kA~~~~~~~~~~~~~~~~~~L~q~Le~   94 (109)
</pre><pre>T Q1QPZ6_NITHX/6   47 DDALRAEMETLEAKRK---LAAKLEADGIPDRNFAKGLNKEAAKLQQKLET   94 (109)
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHh---hCHHHHhcCCCchhhhhcchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>1GW3_A APOA-I; HIGH DENSITY LIPOPROTEINS, KEY IN VIVO COFACTOR FOR THE ENZYME LECITHIN-CHOLESTEROL TRANSFERASE, CHOLESTEROL EFFLUX, RECEPTOR BINDING; NMR {Homo sapiens} SCOP: j.39.1.1
</pre><pre>Probab=34.95  E-value=1.3e+02  Score=16.42  Aligned_cols=40  Identities=8%  Similarity=-0.038  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhcCCCH--HHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   59 YNKKMGGILDALEERAILDGWDN--NRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        59 Y~~~V~~~Ld~i~~~~~~~~~~~--~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |.+.++++|..=.+..+.. +.+  +++.+.|..=..++|+.|
</pre><pre>T Consensus         3 ~~eel~~~l~~~~e~lr~~-l~P~~eel~~~l~~~~~el~~~l   44 (46)
</pre><pre>T 1GW3_A            3 LGEEMRDRARAHVDALRTH-LAPYSDELRQRLAARLEALKENG   44 (46)
</pre><pre>T ss_dssp             CCSSTTTTTTTHHHHHHHH-TSSTTHHHHHHHHHGGGTSSCSS
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHH-hHHHHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>6NZ1_C Design construct XXA_GVDQ; homotrimer, helix, DE NOVO PROTEIN; 1.9A {synthetic construct}
</pre><pre>Probab=34.46  E-value=2.2e+02  Score=19.04  Aligned_cols=43  Identities=12%  Similarity=0.044  Sum_probs=0.0  Template_Neff=1.600
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCCCh--HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   48 DFKLDGLPH--SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        48 G~~~~~~~H--~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      |      +|  ..|-.++-++|.++.++...+ -|.+.+.+.+.-+....+.
</pre><pre>T Consensus         1 G------SHMGTkyLk~~L~~Lke~LErLeKN-PseD~iVe~~r~IVe~n~~   45 (98)
</pre><pre>T 6NZ1_C            1 G------SHMGTEDLKYSLERLREILERLEEN-PSEKQIVEAIRAIVENNAQ   45 (98)
</pre><pre>T ss_dssp             -------CCCCHHHHHHHHHHHHHHHHHHHHS-CCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             C------CccchHHHHHHHHHHHHHHHHHHHC-CCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>6DM9_D DHD15_extended_B; Computational design, heterodimer, coiled-coil, DE NOVO PROTEIN; HET: FME; 2.25A {synthetic construct}
</pre><pre>Probab=34.07  E-value=2e+02  Score=18.42  Aligned_cols=44  Identities=11%  Similarity=0.104  Sum_probs=0.0  Template_Neff=1.500
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   56 HSDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        56 H~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |-.--+.+.+.|..+.+....-.-.++++.+.+.+++..+|.-|
</pre><pre>T Consensus        17 ~~~im~E~~Eim~r~~~LL~KARGADE~V~~~ir~ii~RirEii   60 (78)
</pre><pre>T 6DM9_D           17 SKRLLDEMAEIMRRIKKLLKKARGADEKVLDELRKIIERIRELL   60 (78)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>PF09677.14 ; TrbI_Ftype ; Type-F conjugative transfer system protein (TrbI_Ftype)
</pre><pre>Probab=33.98  E-value=2.1e+02  Score=18.66  Aligned_cols=37  Identities=11%  Similarity=0.106  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   63 MGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +.+-+++........++|+++..+....+...+.+.|
</pre><pre>T Consensus        31 ~~~lv~~~~~~l~~~~ls~~e~~~~~~~f~~~l~~~i   67 (103)
</pre><pre>T Q2G6C0_NOVAD/2   31 LSRILGDFIEAEARAGRPPEETRLRVQAYLKAVEASV   67 (103)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHcCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>7QCA_LD0 60S ribosomal protein L5; Microsporidia, RIBOSOME; HET: K, MG; 2.78A {Spraguea lophii 42_110}
</pre><pre>Probab=33.24  E-value=3.7e+02  Score=21.23  Aligned_cols=65  Identities=12%  Similarity=0.040  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC------------cccCCCcCCCCh-HHHHHHHH----HHHHHHHHHHHhcCCCHHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV------------IHADFKLDGLPH-SDYNKKMG----GILDALEERAILDGWDNNRATKEALN   90 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~------------~H~G~~~~~~~H-~~Y~~~V~----~~Ld~i~~~~~~~~~~~~~~~~~i~~   90 (99)
</pre><pre>                      ++|+.+|.+...-++..            .=.|      .| .+|...+.    +.-..-...+...+++++.+.+.+.+
</pre><pre>T Consensus       172 d~Gl~ip~~~~~~p~~~e~~~~~~~~~r~rI~g------~hi~~y~~~l~~~d~~~~~~~fs~~~~~~~~~~~~~~~~~~  245 (291)
</pre><pre>T 7QCA_LD0        172 DGGIKIPYSPKIFPGYNKDEEFDQQKLRDRIFG------KEVANYMQLLKKEDETKFKKQFSEYIKQNINAEDLEKMYEN  245 (291)
</pre><pre>T ss_pred             cCCCcCCCCCccCCCCCCCccccHHHHHHHHhc------chHHHHHHHhhhhCHHHHHHHHHHHHHcCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhc
</pre><pre>Q FD01848239_023   91 LQSDTRKN   98 (99)
</pre><pre>Q Consensus        91 l~~~lR~~   98 (99)
</pre><pre>                      .-..+++.
</pre><pre>T Consensus       246 ~~~~i~~~  253 (291)
</pre><pre>T 7QCA_LD0        246 LFNSIVTK  253 (291)
</pre><pre>T ss_pred             HHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>6DMP_B Designed orthogonal protein DHD13_XAAA_B; De Novo protein, Computational design, coiled coil; NMR {synthetic construct}
</pre><pre>Probab=32.33  E-value=2.1e+02  Score=18.32  Aligned_cols=33  Identities=18%  Similarity=0.267  Sum_probs=0.0  Template_Neff=1.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      +.+-.+|+.|.......    ++++++..+++.++|.
</pre><pre>T Consensus        24 qelhrrle~ivr~~gss----eea~k~~~kileeire   56 (82)
</pre><pre>T 6DMP_B           24 QELHRRLEEIVRQSGSS----EEAKKEAKKILEEIRE   56 (82)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHTTC----HHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHhcCCCc----HHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>6DM9_C DHD15_extended_A; Computational design, heterodimer, coiled-coil, DE NOVO PROTEIN; HET: FME; 2.25A {synthetic construct}
</pre><pre>Probab=31.52  E-value=2.3e+02  Score=18.17  Aligned_cols=44  Identities=14%  Similarity=0.137  Sum_probs=0.0  Template_Neff=1.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   56 HSDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        56 H~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |-.--+.+.+.|..+.+....-.-.++++.+.+.+++..+|+-|
</pre><pre>T Consensus        17 hi~im~E~~Eim~r~~~LL~KARGADE~V~~~ir~ii~RirEii   60 (78)
</pre><pre>T 6DM9_C           17 HIEIMREILELLQKMEELLEKARGADEDVAKTIKELLRRLKEII   60 (78)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHhCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>PF16305.9 ; DUF4947 ; Domain of unknown function (DUF4947)
</pre><pre>Probab=31.49  E-value=3.5e+02  Score=20.30  Aligned_cols=35  Identities=9%  Similarity=-0.101  Sum_probs=0.0  Template_Neff=3.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   59 YNKKMGGILDALEERAILDGWDNNRATKEALNLQS   93 (99)
</pre><pre>Q Consensus        59 Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~   93 (99)
</pre><pre>                      +..+....++.|...+..+.+|.+++++.+-...+
</pre><pre>T Consensus        88 ~~~l~~~E~~~ik~l~q~gklT~eQarAF~~~a~D  122 (167)
</pre><pre>T Q8DRZ2_STRMU/5   88 QNGFKEKELENINELINSGSLTEEQGKAFYAVAVD  122 (167)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF19903.3 ; DUF6376 ; Family of unknown function (DUF6376)
</pre><pre>Probab=31.44  E-value=2.8e+02  Score=19.27  Aligned_cols=42  Identities=10%  Similarity=0.071  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHhcCCCHHHHHH-HHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   58 DYNKKMGGILDALEERAILDGWDNNRATK-EALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        58 ~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~-~i~~l~~~lR~~L   99 (99)
</pre><pre>                      .||+.+...++........+.++++.+.. .|-+-++++..-+
</pre><pre>T Consensus        68 ~~Ne~l~~~Id~~l~~i~~G~~~~~~le~sei~qti~~i~~l~  110 (116)
</pre><pre>T A0A2W1LIS0_9BA   68 SYNATLLAEVNTYLEQLEKNTINLGDLQNSPMAQTLNEITGTL  110 (116)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHcCCCCHHHHhcChHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>5FBM_A DNA-binding protein HU; Histone-like protein, DNA binding, Dimerization, DNA BINDING PROTEIN; 1.9A {Streptococcus mutans serotype c (strain ATCC 700610 / UA159)} SCOP: l.1.1.1, a.55.1.0
</pre><pre>Probab=31.01  E-value=2.1e+02  Score=17.77  Aligned_cols=32  Identities=9%  Similarity=0.016  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHhc-CCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   68 DALEERAILD-GWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        68 d~i~~~~~~~-~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +++.+..... +++.+++.+.+..+.+.++..|
</pre><pre>T Consensus         5 ~~l~~~ia~~~~l~~~~~~~vl~~l~~~i~~~l   37 (99)
</pre><pre>T 5FBM_A            5 QDLIAKVAEATELTKKDSAAAVDAVFSAVSSYL   37 (99)
</pre><pre>T ss_dssp             HHHHHHHHHHHCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>5TN2_D HTH-type transcriptional regulator SinR; biofilm formation, multimerization domain, TRANSCRIPTION; NMR {Bacillus subtilis (strain 168)}
</pre><pre>Probab=30.09  E-value=1.5e+02  Score=16.75  Aligned_cols=36  Identities=14%  Similarity=0.223  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             CCCcCCCChHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   48 DFKLDGLPHSDYNKKMGGILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        48 G~~~~~~~H~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      |      ++..+...+....-.+.......+++++++++.+.
</pre><pre>T Consensus         1 ~------~~~~~~~~ld~EWi~Li~eA~~~Gitkeei~~fl~   36 (47)
</pre><pre>T 5TN2_D            1 G------SHMEYDGQLDSEWEKLVRDAMTSGVSKKQFREFLD   36 (47)
</pre><pre>T ss_dssp             C------CCSCCSSCCCHHHHHHHHHHHHTTCCHHHHHHHHH
</pre><pre>T ss_pred             C------CccccccccCHHHHHHHHHHHHCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>8AZW_h uL18 (60S ribosomal protein L5); Plant, 80S, rRNA Modifications, ribosome; HET: K, 1MA, 5MC, UY1, SPD, OMU, OMC, PSU, OMG, SPM, A2M, UR3, MG; 2.14A {Nicotiana tabacum}
</pre><pre>Probab=29.80  E-value=3.2e+02  Score=21.63  Aligned_cols=65  Identities=11%  Similarity=0.082  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCC-------------CcccCCCcCCCCh-HHHHHHHHH----HHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTK-------------VIHADFKLDGLPH-SDYNKKMGG----ILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~-------------~~H~G~~~~~~~H-~~Y~~~V~~----~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      +.||.+|.+...-++.             ..=.|      .| .+|...+.+    ....-...+...+++++.+.+...
</pre><pre>T Consensus       167 d~Gl~ip~~~~~~p~~~~e~~~~~~~~~r~~i~g------~hi~~y~~~l~eed~e~~kkqFs~~ik~~i~~~~~~~~~~  240 (301)
</pre><pre>T 8AZW_h          167 DGGLDIPHSEKRFAGFSKDSKQLDADVHRKYIYG------GHVATYMKTLTEDEPEKYQSHFSEYIKQGLEADDLEEMYK  240 (301)
</pre><pre>T ss_pred             hCCCCCCCCCcccCCCCCCCCcCCHHHHHHHHhh------ccHHHHHhhccccCHHHHHHHHHHHHHcCCChhhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHhc
</pre><pre>Q FD01848239_023   90 NLQSDTRKN   98 (99)
</pre><pre>Q Consensus        90 ~l~~~lR~~   98 (99)
</pre><pre>                      ..-..+|+.
</pre><pre>T Consensus       241 ~~~~~I~~~  249 (301)
</pre><pre>T 8AZW_h          241 KVHAAIRAD  249 (301)
</pre><pre>T ss_pred             HHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>3HE5_B SYNZIP2; heterodimeric coiled-coil, DE NOVO PROTEIN; 1.75A {artificial gene}
</pre><pre>Probab=29.00  E-value=2e+02  Score=16.78  Aligned_cols=36  Identities=11%  Similarity=0.203  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +.+++.++..+.+.    ++..+.-.+-+++++..+|.++
</pre><pre>T Consensus         5 naylrkkiarlkkd----nlqlerdeqnlekiianlrdei   40 (52)
</pre><pre>T 3HE5_B            5 NAYLRKKIARLKKD----NLQLERDEQNLEKIIANLRDEI   40 (52)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHH----HHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHh----cchhHhHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>7C70_B Sugar ABC transporter, periplasmic sugar-binding protein; Conformational dynamics, substrate-binding protein, Induced-fit mechanism, Two-step ligand binding, Venus Fly-trap mechanism, SUGAR BINDING PROTEIN; HET: PEG, SO4, BGC, EDO, PGO; 1.63A {Thermus thermophilus HB8}
</pre><pre>Probab=28.74  E-value=4.1e+02  Score=20.25  Aligned_cols=42  Identities=2%  Similarity=-0.153  Sum_probs=0.0  Template_Neff=12.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ..+...+...+..+......+..+++++.+.+++.+++....
</pre><pre>T Consensus       374 ~~~~~~~~~~~~~~~~~~~~g~~t~~~~~~~~~~~~~~~~~~  415 (423)
</pre><pre>T 7C70_B          374 GGVENLAVQHLGMAWDLVAQGRLTREALKDLMDKASAAINQA  415 (423)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHTTCCCHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHHHHCCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>PF20136.3 ; DUF6526 ; Family of unknown function (DUF6526)
</pre><pre>Probab=28.56  E-value=2e+02  Score=20.74  Aligned_cols=30  Identities=20%  Similarity=0.273  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHhcCCCHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   65 GILDALEERAILDGWDNNRATKEALNLQSD   94 (99)
</pre><pre>Q Consensus        65 ~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~   94 (99)
</pre><pre>                      ++|-++.++...+++++++++++|.+..-+
</pre><pre>T Consensus       111 ~El~~L~~~a~~e~l~~~~IK~~I~~Wr~D  140 (144)
</pre><pre>T A0A0B4R8B5_9BA  111 AEVVGLAERAVIEELSADDIKKSINMWRAD  140 (144)
</pre><pre>T ss_pred             hHHHHHHHHHHHcCCCHHHHHHHHHhhccc
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>PF21616.1 ; Topo-V_HhH_2_2nd ; Topoisomerase V, second (HhH)2 tandem domain
</pre><pre>Probab=28.17  E-value=2.2e+02  Score=17.08  Aligned_cols=34  Identities=9%  Similarity=-0.001  Sum_probs=0.0  Template_Neff=1.200
</pre><pre>
</pre><pre>Q ss_pred             h-HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   56 H-SDYNKKMGGILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        56 H-~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      | ..-.-.....|++|.+.+..+.++.+....+|+
</pre><pre>T Consensus        23 hfesiagilatdle~i~rmyeeg~ls~eay~aav~   57 (58)
</pre><pre>T F1SVL0_METKA/3   23 HFESIAGILATDLEEIERMYEEGRLSEEAYRAAVE   57 (58)
</pre><pre>T ss_pred             HHHHHHHHHhcCHHHHHHHHHhCCCCHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>2HEP_A UPF0291 protein ynzC; SR384, AUTOSTRUCTURE, Northeast Structural Genomics Consortium, PSI-1, Protein Structure Initiative, NESG, STRUCTURAL GENOMICS, UNKNOWN FUNCTION; NMR {Bacillus subtilis} SCOP: a.2.21.1
</pre><pre>Probab=27.63  E-value=2.9e+02  Score=18.20  Aligned_cols=39  Identities=8%  Similarity=0.049  Sum_probs=0.0  Template_Neff=4.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHH--HHHH-HHHHHHhcC
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATK--EALN-LQSDTRKNL   99 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~--~i~~-l~~~lR~~L   99 (99)
</pre><pre>                      +.+.++++.+.......++|+++..+  .+.. .++.+|..+
</pre><pre>T Consensus         4 ~eli~rIN~Lakk~K~~gLT~eE~~Eq~~LR~~Yi~~~r~~~   45 (85)
</pre><pre>T 2HEP_A            4 NAKIARINELAAKAKAGVITEEEKAEQQKLRQEYLKGFRSSM   45 (85)
</pre><pre>T ss_dssp             SHHHHHHHHHHHHHHHHCCCHHHHHHHHHHHHHHHHHCC---
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhcCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>2YXY_A Hypothetical conserved protein, GK0453; GK0453, alpha and beta proteins (a+b) class, STRUCTURAL GENOMICS, UNKNOWN FUNCTION, NPPSFA, National Project on Protein; 2.2A {Geobacillus kaustophilus}
</pre><pre>Probab=27.48  E-value=2.3e+02  Score=19.82  Aligned_cols=30  Identities=17%  Similarity=0.196  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   67 LDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        67 Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      +.....++.  .+|++++.++|..|.++.|+.
</pre><pre>T Consensus         4 ~~~m~kryS--qMt~~EL~~Ei~~L~e~~~kA   33 (115)
</pre><pre>T 2YXY_A            4 IKGEQKRYS--EMTKEELQQEIAMLTEKARKA   33 (115)
</pre><pre>T ss_dssp             ---CCCCGG--GCCHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             chhhHHhHh--cCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>6P4X_B Glucokinase-1; hexokinase, transferase, actin atpase, glycolysis; HET: PO4; 3.59A {Saccharomyces cerevisiae (strain ATCC 204508 / S288c)}
</pre><pre>Probab=27.23  E-value=3.7e+02  Score=21.97  Aligned_cols=41  Identities=10%  Similarity=0.082  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   56 HSDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        56 H~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |..=...+.+.+++|.+.+.   ++.+++.+-+..+.++++.+|
</pre><pre>T Consensus         7 ~~~~~~~~~~~~~~i~~~~~---ls~~~l~~i~~~l~~~~~~gl   47 (500)
</pre><pre>T 6P4X_B            7 HKATERAVIQAVDQICDDFE---VTPEKLDELTAYFIEQMEKGL   47 (500)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHC---CCHHHHHHHHHHHHHHHHHHT
</pre><pre>T ss_pred             hHhhHHHHHHHHHHHHHhhC---CCHHHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>PF21106.1 ; YtxK_like ; YtxK N-terminal helical domain
</pre><pre>Probab=26.93  E-value=1.8e+02  Score=18.23  Aligned_cols=30  Identities=13%  Similarity=0.034  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHHHHH
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATKEAL   89 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~   89 (99)
</pre><pre>                      +..+.++|..+........++++++++.++
</pre><pre>T Consensus        44 ~~e~~~~L~~~~~~~~~~~~~~ee~Rka~Q   73 (80)
</pre><pre>T I3ECH4_BACMM/7   44 SEITVKRLRKKYKEINLEKYSNEEIRKAYQ   73 (80)
</pre><pre>T ss_pred             CHHHHHHHHHHHHhcCHhhCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>PF05979.16 ; DUF896 ; Bacterial protein of unknown function (DUF896)
</pre><pre>Probab=26.90  E-value=2.5e+02  Score=17.18  Aligned_cols=37  Identities=14%  Similarity=0.181  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHhcCCCHHHHHH--HHHH-HHHHHHhcC
</pre><pre>Q FD01848239_023   63 MGGILDALEERAILDGWDNNRATK--EALN-LQSDTRKNL   99 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~~~~~~~~~~~--~i~~-l~~~lR~~L   99 (99)
</pre><pre>                      +.++++.+.......++|+++..+  .+.+ .++.+|..+
</pre><pre>T Consensus         2 ~i~RINeLakk~K~~~LT~eE~~Eq~~LR~~Yl~~~r~~~   41 (61)
</pre><pre>T R7FCB8_9FIRM/4    2 KLARINELAKKSRTEGLTTEEKQEQEALRNEYRRAVVGNL   41 (61)
</pre><pre>T ss_pred             hHHHHHHHHHHHhcCCCCHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>PF04690.17 ; YABBY ; YABBY protein
</pre><pre>Probab=26.40  E-value=2.7e+02  Score=19.89  Aligned_cols=32  Identities=16%  Similarity=0.186  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHH
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALN   90 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~   90 (99)
</pre><pre>                      ..|+.+|++.+..+.....  +++..++...+.+
</pre><pre>T Consensus       113 saYn~Fik~ei~~ik~~~p--~i~hke~f~~aa~  144 (156)
</pre><pre>T M1A5M2_SOLTU/9  113 SAYNRFIKEEIQRIKASNP--DISHREAFSTAAK  144 (156)
</pre><pre>T ss_pred             hHHHHHHHHHHHHHHHhCC--CCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>7KDF_B NUF2 isoform 1,NUF2 isoform 1; Stu2, tension sensing, Ndc80, kinetochore, CELL CYCLE; HET: SO4; 2.72A {Saccharomyces cerevisiae}
</pre><pre>Probab=26.01  E-value=2.5e+02  Score=20.69  Aligned_cols=37  Identities=5%  Similarity=-0.024  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhc-CCCHHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILD-GWDNNRATKEALNLQSDTR   96 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~-~~~~~~~~~~i~~l~~~lR   96 (99)
</pre><pre>                      .+.++++++++....... .-.-+++.+++..|.++||
</pre><pre>T Consensus       178 ~~~l~~~i~~l~~~~~~~~~~~~~~l~~~~~~L~~~L~  215 (215)
</pre><pre>T 7KDF_B          178 IKQLQKDFEVEVKEIEIEYSLLSGHINKYMNEMLEYMQ  215 (215)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHTCC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>5UDO_D A118 serine integrase; site-specific recombination, coiled-coil, RECOMBINATION; 2.541A {Listeria innocua}
</pre><pre>Probab=25.71  E-value=4.3e+02  Score=19.48  Aligned_cols=37  Identities=8%  Similarity=-0.012  Sum_probs=0.0  Template_Neff=11.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      ..++.+++.+.+....+.++.+.+.+.+..+..++.+
</pre><pre>T Consensus       224 ~~~~~~~~~l~~~~~~~~~~~~~~~~~~~~l~~~~~~  260 (328)
</pre><pre>T 5UDO_D          224 KIEHAKKKRLFDLYINGSYEVSELDSMMNDIDAQINY  260 (328)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHTTCSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHhcCCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF06570.15 ; DUF1129 ; Protein of unknown function (DUF1129)
</pre><pre>Probab=25.49  E-value=3.9e+02  Score=19.02  Aligned_cols=39  Identities=8%  Similarity=0.073  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ++.-++-++++....+..++++++..+.+.++.+.+.+.
</pre><pre>T Consensus         3 ~~~n~~~~~~l~~~L~~~~~~~~~~ee~l~~~~~~l~ea   41 (201)
</pre><pre>T Q38ZQ5_LATSS/2    3 SKKNADYLFKFKRALKEENVSDEKQAEMLAEMLPEMLEA   41 (201)
</pre><pre>T ss_pred             CHHHHHHHHHHHHHHHHCCCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>5CW4_B Protein FAM175B; Metal dependent enzyme, METAL BINDING PROTEIN; HET: MSE, GOL; 2.543A {Camponotus floridanus}
</pre><pre>Probab=24.50  E-value=3.5e+02  Score=21.34  Aligned_cols=42  Identities=12%  Similarity=0.011  Sum_probs=0.0  Template_Neff=7.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ..+.+.+.++|..+.+......-.-.++.++|..|..+++..
</pre><pre>T Consensus       226 ~~~~~~l~~~L~~l~~~v~~sE~~l~~L~~ev~~L~~~l~~~  267 (289)
</pre><pre>T 5CW4_B          226 MLIQKAAEHHLMSLIPKVCESDLEVAELEKQVHELKIKIATQ  267 (289)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>2CAZ_B VACUOLAR PROTEIN SORTING-ASSOCIATED PROTEIN VPS28; PROTEIN TRANSPORT, ESCRT, MVB, MULTIVESICULAR BODIES, ENDOSOME, LYSOSOME, PH DOMAIN, PROTEIN SORTING, VESICLE TRAFFICKING, UBIQUITIN, PHOSPHOINOSITIDE; 3.6A {SACCHAROMYCES CEREVISIAE} SCOP: a.2.17.2
</pre><pre>Probab=24.39  E-value=4.3e+02  Score=19.37  Aligned_cols=37  Identities=22%  Similarity=0.195  Sum_probs=0.0  Template_Neff=5.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   63 MGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      |-..|+.+++.+..+.++.++......+|++..+..+
</pre><pre>T Consensus        54 II~tlE~LEkAyvrd~I~~~eYt~~C~rLI~Qykt~~   90 (155)
</pre><pre>T 2CAZ_B           54 IVITLDHVEKAYLKDSIDDTQYTNTVDKLLKQFKVYL   90 (155)
</pre><pre>T ss_dssp             HHHHHHHHHHHHTTTSSCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHhccCCCHHHhHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>PF08855.14 ; DUF1825 ; Domain of unknown function (DUF1825)
</pre><pre>Probab=24.14  E-value=3.6e+02  Score=18.02  Aligned_cols=36  Identities=11%  Similarity=0.056  Sum_probs=0.0  Template_Neff=6.600
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   64 GGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        64 ~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +..++.+......-++++.++.+.+......+|+.|
</pre><pre>T Consensus        68 ~~~~~~~~~~~~~~G~~~~~~~~~m~~~l~~mr~~~  103 (103)
</pre><pre>T K9TWP3_CHRTP/1   68 QMTVEQLRTQLNQFGITPQQMFEQMHNTLERMKAEL  103 (103)
</pre><pre>T ss_pred             HHHHHHHHHHHHHCCCCHHHHHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>PF11460.12 ; DUF3007 ; Protein of unknown function (DUF3007)
</pre><pre>Probab=22.91  E-value=3.9e+02  Score=18.03  Aligned_cols=33  Identities=15%  Similarity=0.151  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             cCCCCh---HHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHH
</pre><pre>Q FD01848239_023   51 LDGLPH---SDYNKKMGGILDALEERAILDGWDNNRATKEALNL   91 (99)
</pre><pre>Q Consensus        51 ~~~~~H---~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l   91 (99)
</pre><pre>                      .++++|   .+|.+.|      +.++.+  .+++++..+-..++
</pre><pre>T Consensus        60 k~MTy~qQl~~Ye~av------l~kRle--els~eE~~~L~~e~   95 (96)
</pre><pre>T B0C6F9_ACAM1/1   60 GNMTYNQQREDYENAL------LEKRLQ--EMTPEELEALQASI   95 (96)
</pre><pre>T ss_pred             CCCCHHHHHHHHHHHH------HHHHHH--hCCHHHHHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>3J79_R 60S ribosomal protein uL18; emetine, RIBOSOME-INHIBITOR complex; HET: MG; 3.2A {Plasmodium falciparum}
</pre><pre>Probab=22.53  E-value=5.9e+02  Score=20.31  Aligned_cols=65  Identities=14%  Similarity=0.142  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCC----------CCcccCCCcCCCCh-HHHHHHHHHH-H---HHHHHHHHhcCCCHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPT----------KVIHADFKLDGLPH-SDYNKKMGGI-L---DALEERAILDGWDNNRATKEALNLQ   92 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~----------~~~H~G~~~~~~~H-~~Y~~~V~~~-L---d~i~~~~~~~~~~~~~~~~~i~~l~   92 (99)
</pre><pre>                      +.|+.+|.+...-++          +..-.|      .| .+|...+.+. -   ..-...+...+++++.+.+...+.-
</pre><pre>T Consensus       170 d~Gl~ip~~~~~~p~~~~~y~~e~~r~rI~G------~hi~~y~~~l~~~d~~~y~~~fs~~~k~~~~~~~~~~~~~~~~  243 (294)
</pre><pre>T 3J79_R          170 DGGLNIPHGNNRFPGSKNEFNPEQLRKNILG------IHVAEYMKTLQEEDNDKYKTHFNDYLKNNIGADDIEQMYLNAH  243 (294)
</pre><pre>T ss_dssp             HTTCBCCCCGGGSSCC---CCHHHHHHHHHT------HHHHHHHHHHHHSCHHHHHHHTHHHHHHTCCTTHHHHHHHHHH
</pre><pre>T ss_pred             hCCccccCCCccCCCCCcCCCHHHHHHHhhc------ccHHHHHHhhchhChHHHHHHHHHHHHcCCChhHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHhc
</pre><pre>Q FD01848239_023   93 SDTRKN   98 (99)
</pre><pre>Q Consensus        93 ~~lR~~   98 (99)
</pre><pre>                      ..+|+.
</pre><pre>T Consensus       244 ~~I~~~  249 (294)
</pre><pre>T 3J79_R          244 EKIRQN  249 (294)
</pre><pre>T ss_dssp             HHHHHC
</pre><pre>T ss_pred             HHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>6KSR_A Phosphotransferase; Hexokinase, BIOSYNTHETIC PROTEIN, TRANSFERASE; HET: GLA; 1.37A {Eimeria tenella} SCOP: c.55.1.0
</pre><pre>Probab=22.51  E-value=2.7e+02  Score=22.19  Aligned_cols=37  Identities=5%  Similarity=0.080  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   60 NKKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        60 ~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +..+.+.|+++...+.   ++.+++.+-.+.+.++++.+|
</pre><pre>T Consensus         2 ~~~~~~~~~~~~~~~~---l~~~~l~~i~~~~~~~~~~~l   38 (467)
</pre><pre>T 6KSR_A            2 NNSLEQKAEQVLAPLR---LSKEKLQDLSKTFSDELLRGL   38 (467)
</pre><pre>T ss_dssp             CTHHHHHHHHHHGGGS---CCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ChhHHHHHHHHHHhcC---CCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>PF18175.5 ; HU-CCDC81_bac_2 ; CCDC81-like prokaryotic HU domain 2
</pre><pre>Probab=22.51  E-value=2.5e+02  Score=15.69  Aligned_cols=30  Identities=10%  Similarity=0.020  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   70 LEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        70 i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +..-....+++..++...+..+.+.++..|
</pre><pre>T Consensus         4 ~~~ia~~~~~~~~~~~~~l~~~~~~i~~~l   33 (69)
</pre><pre>T F0P1J6_WEEVC/5    4 INALMRKQSMSQEKALETIQNQVRFWQEHL   33 (69)
</pre><pre>T ss_pred             HHHHHHHcCCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>2F6M_D Vacuolar protein sorting-associated protein VPS28; endosomes, trafficking complex, Vps23, Vps28, Vacuole Protein Sorting, ESCRT protein complexes, Endosomal Sorting Complex Required for; HET: DDQ; 2.1A {Saccharomyces cerevisiae} SCOP: a.2.17.2
</pre><pre>Probab=22.32  E-value=4e+02  Score=18.00  Aligned_cols=35  Identities=20%  Similarity=0.181  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   63 MGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      |-..|+.+++.+..+.+++++......+|+...+.
</pre><pre>T Consensus        37 ii~tle~LEka~~rd~I~~~eY~~~C~~Li~qyk~   71 (109)
</pre><pre>T 2F6M_D           37 IVITLDHVEKAYLKDSIDDTQYTNTVDKLLKQFKV   71 (109)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHTTSSCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHhccCCCHHHhHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>5MK9_A MalE1; carbohydrate binding protein, lactobacillus casei, sugar binding protein; HET: GLC; 0.919A {Lactobacillus casei}
</pre><pre>Probab=22.30  E-value=5.2e+02  Score=19.26  Aligned_cols=38  Identities=8%  Similarity=-0.039  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ..+...+.......-.+..+++++.+.+.+..+++.+.
</pre><pre>T Consensus       341 ~~~~~~~~~~~~~~~~g~~t~~~~~~~l~~~~~~~~~~  378 (379)
</pre><pre>T 5MK9_A          341 ATFWNDAAPLINGAYTGSIPASQYSTKLDTFVKNISKA  378 (379)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHHHSSSCGGGHHHHHHHHHHHHCCC
</pre><pre>T ss_pred             hHHhhhhHHHHHHHhcCCCChHHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>5UOI_A HHH_rd1_0142; de novo design, all-alpha, helix-helix-helix, mini protein, DE NOVO PROTEIN; NMR {Escherichia coli}
</pre><pre>Probab=22.15  E-value=2.6e+02  Score=15.72  Aligned_cols=25  Identities=12%  Similarity=0.136  Sum_probs=0.0  Template_Neff=1.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHhc-CCCHHHHHHHHHH
</pre><pre>Q FD01848239_023   66 ILDALEERAILD-GWDNNRATKEALN   90 (99)
</pre><pre>Q Consensus        66 ~Ld~i~~~~~~~-~~~~~~~~~~i~~   90 (99)
</pre><pre>                      +-++|..+.+.. +++++++++++++
</pre><pre>T Consensus         2 kweeiaerlreefninpeeareavek   27 (43)
</pre><pre>T 5UOI_A            2 KWEEIAERLREEFNINPEEAREAVEK   27 (43)
</pre><pre>T ss_dssp             CSHHHHHHHHHHCCCCHHHHHHHHHH
</pre><pre>T ss_pred             cHHHHHHHHHHHhCCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>PF09650.14 ; PHA_gran_rgn ; Putative polyhydroxyalkanoic acid system protein (PHA_gran_rgn)
</pre><pre>Probab=22.12  E-value=2.5e+02  Score=17.53  Aligned_cols=22  Identities=5%  Similarity=0.022  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   78 GWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        78 ~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +++.++++++++.+.+++.+..
</pre><pre>T Consensus         8 ~l~~~ea~~ri~~~~~~l~~~~   29 (87)
</pre><pre>T Q2S0D4_SALRD/2    8 SLGLDDGRAAVGRVADKLETEL   29 (87)
</pre><pre>T ss_pred             CCCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>8BTR_LF Ribosomal protein L5; Ribosome, Translation, Giardia, Eukaryote, Eukaryotic, Macromolecule, tRNA; 3.25A {Giardia lamblia ATCC 50803}
</pre><pre>Probab=21.70  E-value=6.6e+02  Score=20.25  Aligned_cols=65  Identities=14%  Similarity=0.055  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCCC-----------cccCCCcCCCCh-HHHHHHH----HHHHHHHHHHHHhcCCCHHHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTKV-----------IHADFKLDGLPH-SDYNKKM----GGILDALEERAILDGWDNNRATKEALNL   91 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~~-----------~H~G~~~~~~~H-~~Y~~~V----~~~Ld~i~~~~~~~~~~~~~~~~~i~~l   91 (99)
</pre><pre>                      ++|+.+|.+...-++..           .=.|      .| .+|...+    .+....-...+...+++++.+.+...+.
</pre><pre>T Consensus       176 d~Gl~ip~~~~~fp~~~~e~y~~e~~r~rI~G------~hi~~y~~~l~~~d~~~y~~~fs~~~~~~~~~~~~~~~~~~~  249 (297)
</pre><pre>T 8BTR_LF         176 DAGLNIPHSMKRFPGYSKDGFDSSALRDRIMG------KHVGDYMAELKAEDEETYRKRFSKYIAAKVEPLRLEAMYTKA  249 (297)
</pre><pre>T ss_pred             HcCCccccCCccCCCCCCCCCCHHHHHHHHcc------ccHHHHHHHhhhcCHHHHHHHHHHHHHcCCChhhHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHhc
</pre><pre>Q FD01848239_023   92 QSDTRKN   98 (99)
</pre><pre>Q Consensus        92 ~~~lR~~   98 (99)
</pre><pre>                      ...+|+.
</pre><pre>T Consensus       250 ~~~I~~~  256 (297)
</pre><pre>T 8BTR_LF         250 HAAIRAN  256 (297)
</pre><pre>T ss_pred             HHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>6S7O_E Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1; N-glycosylation, Oligosaccharyltransferase, OSTA, TRANSFERASE; HET: NAG, KZE, MAN, EGY, BMA, KZB; 3.5A {Homo sapiens}
</pre><pre>Probab=21.54  E-value=5.5e+02  Score=22.16  Aligned_cols=37  Identities=8%  Similarity=0.034  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   61 KKMGGILDALEERAILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        61 ~~V~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      ..+.+.|.++..++..+.++.+...+.+.++.+++++
</pre><pre>T Consensus       559 ~~~~~~l~~~~~~y~~gkisr~~y~kl~~~~~k~i~k  595 (607)
</pre><pre>T 6S7O_E          559 KELVLKSAVEAERLVAGKLKKDTYIENEKLISGKRQE  595 (607)
</pre><pre>T ss_dssp             HHHHHHTTSSSCCSHHHHHHHHHHHHHHHHHHHHCC-
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHcCCCCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>6QLF_Y Inner kinetochore subunit NKP1; inner kinetochore, CCAN, DNA BINDING PROTEIN; 3.45A {Saccharomyces cerevisiae}
</pre><pre>Probab=21.36  E-value=6.4e+02  Score=19.98  Aligned_cols=40  Identities=18%  Similarity=0.185  Sum_probs=0.0  Template_Neff=4.700
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHH----HHhcCCCHHHHHHHHHHHHHHHHh
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEER----AILDGWDNNRATKEALNLQSDTRK   97 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~----~~~~~~~~~~~~~~i~~l~~~lR~   97 (99)
</pre><pre>                      ..|...|...|.++...    .+..+.+|+ .++.+.+|++.++.
</pre><pre>T Consensus       195 ~~Y~~~l~~~LkE~~~llE~~lK~~~~~p~-k~~~L~~ii~~l~~  238 (238)
</pre><pre>T 6QLF_Y          195 ATYNGKLVVALEEMKLLLEEAVKTFGNSPE-KREKIKKILSELKK  238 (238)
</pre><pre>T ss_dssp             HTHHHHHHHHHHHHHHHHHTTTSSSCCSSH-HHHHHHHHHHHTTC
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHHHHHHHcCCCHH-HHHHHHHHHHHhcC
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>1SF9_A yfhH hypothetical protein; Structural Genomics, unknown function, PSI, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; 1.71A {Bacillus subtilis} SCOP: b.34.15.1, l.1.1.1
</pre><pre>Probab=21.34  E-value=3.4e+02  Score=19.44  Aligned_cols=27  Identities=22%  Similarity=0.225  Sum_probs=0.0  Template_Neff=4.200
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhcCCCHHHHHHHHHHHHHHHHhc
</pre><pre>Q FD01848239_023   70 LEERAILDGWDNNRATKEALNLQSDTRKN   98 (99)
</pre><pre>Q Consensus        70 i~~~~~~~~~~~~~~~~~i~~l~~~lR~~   98 (99)
</pre><pre>                      ..++++  .+|++++.++|..|.++.|+.
</pre><pre>T Consensus        25 m~kryS--qMt~~EL~~EI~~Lke~~~kA   51 (128)
</pre><pre>T 1SF9_A           25 MEKRYS--QMTPHELNTEIALLSEKARKA   51 (128)
</pre><pre>T ss_dssp             HHHHHH--TCCHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             hhhhHh--cCCHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>6RM3_LD0 uL18; Microsporidia, Ribosome, Intracellular Parasite; HET: MG; 3.4A {Vairimorpha necatrix}
</pre><pre>Probab=21.21  E-value=5.4e+02  Score=20.50  Aligned_cols=65  Identities=15%  Similarity=0.029  Sum_probs=0.0  Template_Neff=7.100
</pre><pre>
</pre><pre>Q ss_pred             cceeecCCCCCCCCCC------------CcccCCCcCCCCh-HHHHHHHHH--HH--HHHHHHHHhcCCCHHHHHHHHHH
</pre><pre>Q FD01848239_023   28 TNMTYLPVADGIHPTK------------VIHADFKLDGLPH-SDYNKKMGG--IL--DALEERAILDGWDNNRATKEALN   90 (99)
</pre><pre>Q Consensus        28 ~Ngi~LP~~~~~~~~~------------~~H~G~~~~~~~H-~~Y~~~V~~--~L--d~i~~~~~~~~~~~~~~~~~i~~   90 (99)
</pre><pre>                      ++||.+|.+...-+++            ..-.|      .| .+|...+.+  ..  ..-...+...+++++.+.+.+.+
</pre><pre>T Consensus       173 d~Gl~ip~~~~~~p~~~e~~~y~~~~~r~ri~g------~hi~~y~~~l~~~d~~~y~~~fs~~~~~~~~~~~~~~~~~~  246 (295)
</pre><pre>T 6RM3_LD0        173 DAGVYIPHSESKFIGYEKNKPFKADELRDRIFC------KHVSEYMKTLKENDEEKFKIQFSDYIKKNIEPDNIAGIYQN  246 (295)
</pre><pre>T ss_pred             cCCcccccCCcccCCCCCCCCCCHHHHHHHhch------hhHHHHHHHhcccCHHHHHHHHHHHHHhcCCcccHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHhc
</pre><pre>Q FD01848239_023   91 LQSDTRKN   98 (99)
</pre><pre>Q Consensus        91 l~~~lR~~   98 (99)
</pre><pre>                      .-..+++.
</pre><pre>T Consensus       247 ~~~~I~~~  254 (295)
</pre><pre>T 6RM3_LD0        247 ALDGIIKN  254 (295)
</pre><pre>T ss_pred             HHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>PF06738.16 ; ThrE ; Putative threonine/serine exporter
</pre><pre>Probab=21.15  E-value=4.3e+02  Score=19.64  Aligned_cols=29  Identities=14%  Similarity=0.099  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHhcCCCHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   65 GILDALEERAILDGWDNNRATKEALNLQS   93 (99)
</pre><pre>Q Consensus        65 ~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~   93 (99)
</pre><pre>                      ++++++.++...++++.+++.++++++..
</pre><pre>T Consensus        72 ~~v~~l~~~~~~~~i~~~~a~~~L~~i~~  100 (238)
</pre><pre>T R7G2W2_9FIRM/1   72 DYLNNLCRYVCQHTPDEKEMREKYLEVIN  100 (238)
</pre><pre>T ss_pred             HHHHHHHHHHHhCCCCHHHHHHHHHHHHh
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>PF11527.12 ; ARL2_Bind_BART ; The ARF-like 2 binding protein BART
</pre><pre>Probab=20.96  E-value=3.8e+02  Score=17.24  Aligned_cols=30  Identities=10%  Similarity=0.251  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             hHHHHHHHHHHHHHHHHHHHhcCCCHHHHHHHH
</pre><pre>Q FD01848239_023   56 HSDYNKKMGGILDALEERAILDGWDNNRATKEA   88 (99)
</pre><pre>Q Consensus        56 H~~Y~~~V~~~Ld~i~~~~~~~~~~~~~~~~~i   88 (99)
</pre><pre>                      |..|...|.+.|+.......   ++.++..+.+
</pre><pre>T Consensus        47 ~~~y~~l~e~~l~~fl~~~g---~~~~~~~~~~   76 (118)
</pre><pre>T A7SHQ3_NEMVE/4   47 HKNYKEMVDSLLSSFVKDVG---ITEEQFAVAC   76 (118)
</pre><pre>T ss_pred             HHHHHHHHHHHHHHHHHHhC---CCHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>3OIQ_B DNA polymerase alpha catalytic subunit A; OB fold, dimer, dimeric complex, PROTEIN BINDING; 2.4A {Saccharomyces cerevisiae}
</pre><pre>Probab=20.90  E-value=2.1e+02  Score=15.53  Aligned_cols=15  Identities=27%  Similarity=0.594  Sum_probs=0.0  Template_Neff=1.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   58 DYNKKMGGILDALEE   72 (99)
</pre><pre>Q Consensus        58 ~Y~~~V~~~Ld~i~~   72 (99)
</pre><pre>                      .|.+.|.+.||+.+.
</pre><pre>T Consensus        11 ry~ndvqdllddven   25 (36)
</pre><pre>T 3OIQ_B           11 RYANDVQDLLDDVEN   25 (36)
</pre><pre>T ss_dssp             HHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>PF15207.10 ; TMEM240 ; TMEM240 family
</pre><pre>Probab=20.71  E-value=56  Score=24.48  Aligned_cols=10  Identities=40%  Similarity=0.866  Sum_probs=0.0  Template_Neff=3.500
</pre><pre>
</pre><pre>Q ss_pred             CCccccCChh
</pre><pre>Q FD01848239_023    1 YNRHHIIPYE   10 (99)
</pre><pre>Q Consensus         1 ~qaHHIIP~~   10 (99)
</pre><pre>                      .|+||+||.+
</pre><pre>T Consensus        56 h~v~~VIPY~   65 (179)
</pre><pre>T W5NYK5_SHEEP/1   56 HHVHYVIPYD   65 (179)
</pre><pre>T ss_pred             cceEEEecCC
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>1YWW_A Hypothetical protein PA4738; structural genomics, YjbJ, PSI, Protein Structure Initiative, Northeast Structural Genomics Consortium, NESG, UNKNOWN FUNCTION; NMR {Pseudomonas aeruginosa} SCOP: a.60.11.0
</pre><pre>Probab=20.68  E-value=3.5e+02  Score=16.69  Aligned_cols=39  Identities=21%  Similarity=0.324  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHHHHHHHhc-CCCHHHHHHHHHHHHHHH
</pre><pre>Q FD01848239_023   57 SDYNKKMGGILDALEERAILD-GWDNNRATKEALNLQSDT   95 (99)
</pre><pre>Q Consensus        57 ~~Y~~~V~~~Ld~i~~~~~~~-~~~~~~~~~~i~~l~~~l   95 (99)
</pre><pre>                      .+....++.+.+++....... +.+.+++.++|..+.++|
</pre><pre>T Consensus        46 ~~d~~~~~G~~~~l~g~iq~~~g~~~e~a~~~i~~~~~~~   85 (85)
</pre><pre>T 1YWW_A           46 DDDLQAADGHAEYLVGKLQERYGWSKERAEQEVRDFSDRL   85 (85)
</pre><pre>T ss_dssp             HHHHSSSCCSHHHHHHHHHHHTCCCHHHHHHHHHHHHHTC
</pre><pre>T ss_pred             hhHHHHhCCcHHHHHHHHHHHHCCCHHHHHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>4UA8_A Carbohydrate ABC transporter substrate-binding protein, CUT1 family (TC 3.A.1.1.-); solute-binding protein maltotriose ABC, TRANSPORT PROTEIN; HET: PEG, EDO, GLC; 1.54A {Eubacterium rectale DSM 17629}
</pre><pre>Probab=20.34  E-value=3e+02  Score=20.68  Aligned_cols=37  Identities=5%  Similarity=0.057  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHHh---cCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   63 MGGILDALEERAIL---DGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        63 V~~~Ld~i~~~~~~---~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      +...+..+......   +..+++++.+.+.+..+++.+.|
</pre><pre>T Consensus       360 ~~~~~~~~~~~~~~~~~g~~t~e~~~~~l~~~~~~~~~~l  399 (400)
</pre><pre>T 4UA8_A          360 YWDAATPFGDAFQNGAEGQITKDNAAQKTEDFNTQLNDSL  399 (400)
</pre><pre>T ss_dssp             HHHHHHHHHHTTSTTCTTCCCTTTHHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             hhhcchhHhHHHhhccCCccChhhHHHHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>6LMJ_A A104R; Complex, dsDNA, ASFV, DNA BINDING PROTEIN, DNA BINDING PROTEIN-DNA complex; 2.8A {African swine fever virus}
</pre><pre>Probab=20.17  E-value=4e+02  Score=17.11  Aligned_cols=33  Identities=6%  Similarity=-0.002  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             HHHHHHHHHHHHhcCCCHHHHHHHHHHHHHHHHhcC
</pre><pre>Q FD01848239_023   64 GGILDALEERAILDGWDNNRATKEALNLQSDTRKNL   99 (99)
</pre><pre>Q Consensus        64 ~~~Ld~i~~~~~~~~~~~~~~~~~i~~l~~~lR~~L   99 (99)
</pre><pre>                      .+..+.|.++..   ++..++...+..+.+.+++.|
</pre><pre>T Consensus        18 ~~l~~~ia~~~~---~s~~~v~~vl~~~~~~i~~~L   50 (110)
</pre><pre>T 6LMJ_A           18 QELYSLVAADTQ---LNKALIERIFTSQQKIIQNAL   50 (110)
</pre><pre>T ss_dssp             HHHHHHHHHHTT---CCHHHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             HHHHHHHHHHhC---CCHHHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre>