<pre>Query         MBL3828645.1/108-193
</pre><pre>Match_columns 57
</pre><pre>No_of_seqs    100 out of 108
</pre><pre>Neff          4.00333
</pre><pre>Searched_HMMs 93885
</pre><pre>Date          Fri Jul 11 01:11:24 2025
</pre><pre>Command       hhsearch -cpu 8 -i ../results/full.a3m -d /cluster/toolkit/production/databases/hh-suite/mmcif70/pdb70 -d /cluster/toolkit/production/databases/hh-suite/pfama/pfama -o ../results/7784730.hhr -oa3m ../results/7784730.a3m -p 20 -Z 250 -loc -z 1 -b 1 -B 250 -ssm 2 -sc 1 -seq 1 -dbstrlen 10000 -norealign -maxres 32000 -contxt /cluster/toolkit/production/bioprogs/tools/hh-suite-build-new/data/context_data.crf 
</pre><pre>
</pre><pre> No Hit                             Prob E-value P-value  Score    SS Cols Query HMM  Template HMM
</pre><pre>  1 PF08774.16 ; VRR_NUC ; VRR-NUC  98.1 2.5E-05 2.6E-10   47.1   5.2   50    5-55     64-121 (127)
</pre><pre>  2 5Y7Q_A Fanconi-associated nucl  95.2     0.1 1.1E-06   40.7   5.0   45    6-56    527-571 (580)
</pre><pre>  3 4QBO_A Nuclease; nuclease, HYD  95.1    0.19   2E-06   29.0   5.0   43    6-56     33-75  (92)
</pre><pre>  4 4REC_A Fanconi-associated nucl  93.8    0.26 2.8E-06   38.4   4.6   46    6-57    589-634 (647)
</pre><pre>  5 PF14281.11 ; PDDEXK_4 ; PD-(D/  93.7    0.17 1.8E-06   33.3   2.9   37    4-48     72-108 (192)
</pre><pre>  6 4QBN_B Nuclease; Nuclease, HYD  93.2     0.8 8.6E-06   26.1   4.9   44    6-56     34-78  (93)
</pre><pre>  7 9GY0_A Fanconi-associated nucl  90.7     1.2 1.3E-05   38.0   5.1   45    6-56    963-1007(1021)
</pre><pre>  8 PF20713.2 ; DUF6826 ; Domain o  88.4     2.5 2.7E-05   26.8   4.2   43    4-47     38-86  (94)
</pre><pre>  9 4QBL_C VRR-NUC; Nuclease, HYDR  87.5     4.5 4.8E-05   26.2   5.0   44    6-56     69-115 (145)
</pre><pre> 10 PF01646.21 ; Herpes_UL24 ; Her  85.3     1.5 1.6E-05   31.2   2.2   23    4-30     58-80  (174)
</pre><pre> 11 6OKH_B Uncharacterized protein  84.3     5.6   6E-05   25.6   4.3   48    1-48     81-133 (190)
</pre><pre> 12 PF15516.11 ; BpuSI_N ; BpuSI N  84.1     6.5   7E-05   26.3   4.6   43    4-51     43-93  (152)
</pre><pre> 13 1GEF_A HOLLIDAY JUNCTION RESOL  82.9     6.1 6.5E-05   23.9   3.8   35    6-48     32-69  (123)
</pre><pre> 14 3IJM_A uncharacterized restric  80.6     6.8 7.2E-05   25.1   3.6   37    1-42     63-99  (151)
</pre><pre> 15 3DNX_A uncharacterized protein  78.5     3.3 3.6E-05   26.9   1.8   38    4-48     41-79  (153)
</pre><pre> 16 3OT2_B Uncharacterized protein  78.3      20 0.00021   22.6   5.1   56    2-57     76-140 (187)
</pre><pre> 17 8TI9_D Shedu protein SduA; She  76.1      15 0.00016   25.4   4.5   40    1-46     92-150 (229)
</pre><pre> 18 2WCW_D HJC; TYPE II RESTRICTIO  75.8      18 0.00019   22.1   4.3   36    5-48     39-77  (139)
</pre><pre> 19 1OB8_A HOLLIDAY-JUNCTION RESOL  72.8      17 0.00018   22.9   3.8   44    6-57     38-86  (135)
</pre><pre> 20 PF15649.11 ; Tox-REase-7 ; Res  72.4      11 0.00012   22.7   2.8   47    1-57     28-76  (88)
</pre><pre> 21 PF06319.17 ; MmcB-like ; DNA r  71.2     3.7 3.9E-05   26.7   0.6   38    4-48     42-80  (148)
</pre><pre> 22 1WDJ_A hypothetical protein TT  70.7      13 0.00014   23.4   3.0   42    1-42     80-130 (187)
</pre><pre> 23 PF01870.23 ; Hjc ; Archaeal ho  61.3      43 0.00046   18.3   4.2   37    4-48     24-63  (87)
</pre><pre> 24 3S1S_A restriction endonucleas  60.8      24 0.00025   29.1   3.4   49    4-57     57-111 (878)
</pre><pre> 25 PF05685.17 ; Uma2 ; Putative r  60.1      54 0.00058   21.5   4.4   49    1-49     69-124 (185)
</pre><pre> 26 8GLV_Jo Radial spoke protein 1  56.4      79 0.00084   23.0   5.0   48    5-53    262-315 (346)
</pre><pre> 27 PF14082.11 ; SduA_C ; Shedu pr  56.2      24 0.00026   23.0   2.3   25    4-36     43-67  (170)
</pre><pre> 28 3LZ8_B Putative chaperone DnaJ  53.8      84 0.00089   22.4   4.7   47    5-52    269-318 (329)
</pre><pre> 29 PF04411.17 ; PDDEXK_7 ; PD-(D/  53.2      48 0.00051   21.4   3.3   43    1-46     70-121 (149)
</pre><pre> 30 PF18743.6 ; AHJR-like ; REase_  53.0      84  0.0009   19.1   4.8   43    6-56     45-87  (123)
</pre><pre> 31 8TI8_C Shedu protein SduA; She  52.9      68 0.00072   24.2   4.4   39    4-48    248-305 (382)
</pre><pre> 32 PF10847.13 ; DUF2656 ; Protein  50.5      30 0.00031   24.6   2.1   15   26-40     49-63  (141)
</pre><pre> 33 PF18742.6 ; DpnII-MboI ; REase  50.2      40 0.00042   22.4   2.6   29   21-49     71-101 (148)
</pre><pre> 34 PF19986.4 ; DUF6422 ; Family o  47.1      42 0.00045   22.2   2.3   17   32-48      3-19  (94)
</pre><pre> 35 6JZB_A Chaperone protein DnaJ;  43.5 1.6E+02  0.0018   21.7   5.0   43    5-48    313-360 (385)
</pre><pre> 36 PF10069.14 ; DICT ; Sensory do  43.4      49 0.00052   20.6   2.1   29   29-57     46-74  (121)
</pre><pre> 37 PF10117.14 ; McrBC ; McrBC 5-m  42.1      70 0.00075   22.3   2.9   36    5-48    262-303 (318)
</pre><pre> 38 PF20472.3 ; PDDEXK_11 ; PD-(D/  41.7 1.1E+02  0.0012   20.7   3.7   41    1-47     53-93  (155)
</pre><pre> 39 1KNV_A Bse634I restriction end  40.7 1.1E+02  0.0012   23.7   4.0   36    6-49    145-184 (293)
</pre><pre> 40 PF03838.19 ; RecU ; Recombinat  40.6 1.3E+02  0.0014   20.2   3.8   44    6-57     54-103 (161)
</pre><pre> 41 8PS4_A Shedu effector protein;  40.5 1.2E+02  0.0012   23.8   4.1   39    1-46    266-323 (411)
</pre><pre> 42 5XKT_A Urease accessory protei  36.2      66  0.0007   19.8   1.9   26    1-26    136-162 (200)
</pre><pre> 43 PF08011.16 ; PDDEXK_9 ; PD-(D/  35.7 1.4E+02  0.0015   16.7   3.0   33    5-47     31-63  (96)
</pre><pre> 44 PF02078.21 ; Synapsin ; Synaps  35.1 1.3E+02  0.0014   19.2   3.1   29    5-33     71-99  (102)
</pre><pre> 45 7VSR_M Protein McrC; AAA+ prot  34.9      84  0.0009   22.4   2.5   36    5-48    204-246 (310)
</pre><pre> 46 PF08722.16 ; Tn7_TnsA-like_N ;  33.7 1.3E+02  0.0014   15.6   4.3   36    5-48     26-61  (76)
</pre><pre> 47 PF21420.2 ; Tautomerase-like ;  33.2 1.6E+02  0.0017   17.6   3.0   23   26-48      2-25  (45)
</pre><pre> 48 9IAB_A PD-(D/E)XK endonuclease  32.8 1.6E+02  0.0017   18.4   3.1   37    5-48    147-183 (273)
</pre><pre> 49 PF20452.3 ; Calmod_bind_C ; Ca  32.5 1.5E+02  0.0016   16.1   2.9   28   21-48     15-47  (60)
</pre><pre> 50 6JC3_A Nucleocapsid; self-capp  31.7   2E+02  0.0021   23.7   4.2   33    4-50     93-134 (398)
</pre><pre> 51 4TKD_D Holliday junction resol  31.5 2.3E+02  0.0024   17.8   4.1   35    6-48     41-78  (141)
</pre><pre> 52 9IAC_A Similar to tr|Q3MCC8|Q3  31.4 2.2E+02  0.0024   17.8   3.6   36    5-48    153-189 (277)
</pre><pre> 53 PF21192.2 ; NMD3_OB ; NMD3 OB-  31.0      65 0.00069   19.9   1.3    9    4-12     80-88  (89)
</pre><pre> 54 3K93_A phage related exonuclea  30.1 2.8E+02   0.003   18.8   4.2   34    6-49     99-149 (223)
</pre><pre> 55 5JOQ_A Lmo2184 protein; CENTER  30.1 1.2E+02  0.0013   20.5   2.4   44    5-48     90-134 (290)
</pre><pre> 56 PF05961.16 ; Chordopox_A13L ;   29.4 1.3E+02  0.0014   19.1   2.3   34   12-46     28-64  (68)
</pre><pre> 57 3SXU_A DNA polymerase III subu  28.8 1.2E+02  0.0013   20.1   2.2   42    5-46     95-136 (150)
</pre><pre> 58 2X4L_A FERRIC-SIDEROPHORE RECE  28.4   1E+02  0.0011   21.6   2.0   44    5-48    110-163 (325)
</pre><pre> 59 PF18891.5 ; FANCL_d3 ; FANCL U  27.7      34 0.00037   21.6  -0.3   12   22-33     85-97  (98)
</pre><pre> 60 6IY6_I Aminoacyl tRNA synthase  27.4      99  0.0011   20.1   1.7   12    7-18      8-19  (215)
</pre><pre> 61 6ALL_A Fe(3+)-citrate-binding   26.3 1.1E+02  0.0012   20.9   1.8   44    5-48    114-162 (324)
</pre><pre> 62 1L6X_B Minimized B-domain of P  26.2 1.5E+02  0.0016   16.4   2.0   13   34-46     16-28  (34)
</pre><pre> 63 PF13366.11 ; PDDEXK_3 ; PD-(D/  25.8 2.6E+02  0.0028   16.7   3.6   30   26-56     71-103 (115)
</pre><pre> 64 PF04556.17 ; DpnII ; DpnII res  24.3 3.9E+02  0.0042   20.0   4.4   40    1-48    198-237 (289)
</pre><pre> 65 7OF0_0 39S ribosomal protein L  24.0 1.4E+02  0.0015   21.5   2.1   24    7-30    155-178 (188)
</pre><pre> 66 2Q8P_A Iron-regulated surface   23.6 1.8E+02  0.0019   19.4   2.4   44    5-48     60-104 (260)
</pre><pre> 67 PF22023.1 ; Pus10_THUMP_arc ;   23.5 1.2E+02  0.0013   17.9   1.5   11    3-13     68-78  (78)
</pre><pre> 68 6HZ4_N Protein McrC; AAA+ supe  23.4 3.6E+02  0.0038   19.5   3.9   45    5-57    242-302 (348)
</pre><pre> 69 6UT8_G McrBC 5-methylcytosine   23.3   2E+02  0.0022   22.0   2.9   37    5-48    338-375 (458)
</pre><pre> 70 8A22_Bb uS2m; mitoribosome, ri  23.0 1.3E+02  0.0014   21.1   1.7   14    1-14    165-178 (236)
</pre><pre> 71 PF18362.6 ; THB ; Tri-helix bu  22.8      20 0.00021   17.5  -1.7   15   34-48      4-18  (34)
</pre><pre> 72 PF15608.11 ; PELOTA_1 ; PELOTA  22.7 1.5E+02  0.0016   19.0   1.8   14    4-17     36-49  (81)
</pre><pre> 73 PF13101.11 ; DUF3945 ; Protein  22.5 1.4E+02  0.0015   16.7   1.5   17   30-46     20-36  (50)
</pre><pre> 74 PF11496.13 ; HDA2-3 ; Class II  22.4 1.7E+02  0.0018   19.5   2.0   13   35-47     11-23  (274)
</pre><pre> 75 5YET_A Uncharacterized protein  22.2 2.5E+02  0.0026   21.5   3.1   29    6-34    133-161 (399)
</pre><pre> 76 7EEB_H Cation channel sperm-as  22.1      76 0.00081   27.9   0.6   50    8-57    280-329 (985)
</pre><pre> 77 3R5T_A Ferric vibriobactin ABC  21.8      91 0.00097   21.5   0.8   27    5-32     86-112 (305)
</pre><pre> 78 1ZDB_A MINI PROTEIN A DOMAIN,   21.6   2E+02  0.0021   16.4   2.0   13   34-46     21-33  (38)
</pre><pre> 79 2FZL_A DNA repair protein RAD2  21.4 1.7E+02  0.0018   18.1   1.8   15   35-49     35-49  (219)
</pre><pre> 80 PF13439.11 ; Glyco_transf_4 ;   21.3 1.4E+02  0.0015   16.4   1.3   11    4-14     70-80  (166)
</pre><pre> 81 PF07832.16 ; Bse634I ; Cfr10I/  20.9 1.1E+02  0.0012   23.7   1.1   43    6-49    133-176 (285)
</pre><pre> 82 1CFR_A RESTRICTION ENDONUCLEAS  20.8 2.1E+02  0.0022   22.1   2.6   40    6-49    133-176 (285)
</pre><pre> 83 PF09563.15 ; RE_LlaJI ; LlaJI   20.7 1.9E+02  0.0021   21.2   2.3   39    5-48    265-310 (362)
</pre><pre> 84 3LHS_A Ferrichrome ABC transpo  20.6 2.1E+02  0.0023   19.3   2.3   44    5-48     85-130 (296)
</pre><pre> 85 1ZP7_B Recombination protein U  20.6 5.1E+02  0.0054   18.4   4.2   35    6-48     87-128 (206)
</pre><pre> 86 PF18569.6 ; Thioredoxin_16 ; T  20.5 1.3E+02  0.0014   18.6   1.3   13    6-18      4-16  (92)
</pre><pre> 87 9E5C_3m Chaperone protein DNAJ  20.4 5.1E+02  0.0055   18.1   5.3   44    7-51    270-319 (319)
</pre><pre>
</pre><pre>No 1
</pre><pre>>PF08774.16 ; VRR_NUC ; VRR-NUC domain
</pre><pre>Probab=98.07  E-value=2.5e-05  Score=47.14  Aligned_cols=50  Identities=14%  Similarity=0.096  Sum_probs=38.4  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCC--------CCchhhEEEeeCCCCCCCHHHHHHHHHHhCCcccce
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPT--------QDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLV   55 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~--------qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~   55 (57)
</pre><pre>                      .||++++..... +.        .++--.+||||+++|.+++.|+..+..+.....++.
</pre><pre>T Consensus        64 ~PDl~~~~~~~~-~~~~~~~~~~~~~~~~~iEvK~~~~~~s~~Q~~~~~~l~~~g~~~~  121 (127)
</pre><pre>T FAN1_DICDI/958   64 MPDLLLWKLNNN-NDDDIDDKNNNNSSIKFVEVKGTGDRLRDQQRIWIDMLISFGCDVE  121 (127)
</pre><pre>T ss_pred             CCcEEeEeCCCC-CCcccccccCCCceEEEEEecCCCCCCCHHHHHHHHHHHHcCCcEE
</pre><pre>Confidence            499999987643 22        123577899999999999999999998887655443
</pre><pre>
</pre><pre>
</pre><pre>No 2
</pre><pre>>5Y7Q_A Fanconi-associated nuclease 1 homolog; Nuclease, HYDROLASE-DNA complex; 2.7A {Pseudomonas aeruginosa (strain ATCC 15692 / DSM 22644 / CIP 104116 / JCM 14847 / LMG 12228 / 1C / PRS 101 / PAO1)}
</pre><pre>Probab=95.17  E-value=0.1  Score=40.71  Aligned_cols=45  Identities=20%  Similarity=0.217  Sum_probs=0.0  Template_Neff=8.400
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||+++.+..++..      ..||||-|+|.+++.|+..-.....-.-.+.+
</pre><pre>T Consensus       527 PDL~lw~~~~~~~------~fvEVK~p~D~Ls~~Q~~wl~~l~~~G~~~~v  571 (580)
</pre><pre>T 5Y7Q_A          527 PDLIQFWPAQRRY------RMVEVKGPGDRLQDNQLRWLQFCREREMPVAV  571 (580)
</pre><pre>T ss_dssp             CSEEEEETTTTEE------EEEEEECTTCCCCHHHHHHHHHHHHTTCCEEE
</pre><pre>T ss_pred             CcEEEEEcCCCeE------EEEEEECCCCCCCHHHHHHHHHHHHCCCCEEE
</pre><pre>
</pre><pre>
</pre><pre>No 3
</pre><pre>>4QBO_A Nuclease; nuclease, HYDROLASE; 1.3A {Streptococcus phage P9} SCOP: c.52.1.35, l.1.1.1
</pre><pre>Probab=95.10  E-value=0.19  Score=29.03  Aligned_cols=43  Identities=23%  Similarity=0.234  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||+++...       ... ..||+|.+++.+++.|..--..+........+
</pre><pre>T Consensus        33 pDl~~~~~-------~g~-~~iE~K~~~~~~~~~Q~~~~~~~~~~g~~~~v   75 (92)
</pre><pre>T 4QBO_A           33 PDRIVVMN-------TGT-FFVEVKAPGKKPRPSQVAMHKKIKEAGQHVWV   75 (92)
</pre><pre>T ss_dssp             CSEEEEET-------TEE-EEEEECCTTCCCCHHHHHHHHHHHHHTCCEEE
</pre><pre>T ss_pred             CcEEEEec-------CCe-EEEEEeCCCCCCCHHHHHHHHHHHHCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 4
</pre><pre>>4REC_A Fanconi-associated nuclease 1; HJC, TPR, SAP, structure specific nuclease, FANCID2, nucleus, Hydrolase-DNA complex; 2.2A {Homo sapiens}
</pre><pre>Probab=93.82  E-value=0.26  Score=38.40  Aligned_cols=46  Identities=22%  Similarity=0.253  Sum_probs=0.0  Template_Neff=9.100
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      ||+++.+...+      --..||||=|+|.+|+.|+..-.....-.-.+.+.
</pre><pre>T Consensus       589 PDL~~~~~~~~------~~~fvEVK~p~D~ls~~Q~~wl~~l~~~G~~~~v~  634 (647)
</pre><pre>T 4REC_A          589 PALVVWNSQSR------HFKLVEVKGPNDRLSHKQMIWLAELQKLGAEVEVC  634 (647)
</pre><pre>T ss_dssp             CSEEEECTTTC------CEEEEEEECTTCCCCHHHHHHHHHHHHTTCCEEEE
</pre><pre>T ss_pred             CCEEEEecCCC------eEEEEEEcCCCCCCCHHHHHHHHHHHHcCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 5
</pre><pre>>PF14281.11 ; PDDEXK_4 ; PD-(D/E)XK nuclease superfamily
</pre><pre>Probab=93.66  E-value=0.17  Score=33.29  Aligned_cols=37  Identities=30%  Similarity=0.376  Sum_probs=0.0  Template_Neff=9.400
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +|+|++|       -..+...-+||.|+ +-..+..|.+.|.+.+
</pre><pre>T Consensus        72 ~riDi~i-------~~~~~~~ivIEnKi-~s~~~~~QL~~Y~~~~  108 (192)
</pre><pre>T Q0PC95_CAMJE/2   72 RRIDILI-------TLSDKRYIIIENKI-YAKDQKNQLKDYINFV  108 (192)
</pre><pre>T ss_pred             CceeEEE-------EeCCCEEEEEEecC-CccccchHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 6
</pre><pre>>4QBN_B Nuclease; Nuclease, HYDROLASE; HET: SO4; 1.85A {Salmonella phage SETP3} SCOP: c.52.1.35
</pre><pre>Probab=93.15  E-value=0.8  Score=26.14  Aligned_cols=44  Identities=23%  Similarity=0.252  Sum_probs=0.0  Template_Neff=10.100
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCC-CCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPD-KMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D-~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||+++.....       .-..+|+|.+++ .++..|+.--..+........+
</pre><pre>T Consensus        34 pDl~~~~~~~-------~~~~iE~K~~~~~~~~~~Q~~~~~~~~~~g~~~~v   78 (93)
</pre><pre>T 4QBN_B           34 PDLLVILPRG-------VIWFVEVKKDENTKPDPHQLREHERFRKRGANVFV   78 (93)
</pre><pre>T ss_dssp             CSEEEEETTT-------EEEEEEECSSTTCCCCHHHHHHHHHHHHTTCEEEE
</pre><pre>T ss_pred             CcEEEEEcCc-------eEEEEEEecCCCCCCCHHHHHHHHHHHHCCCEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 7
</pre><pre>>9GY0_A Fanconi-associated nuclease 1; FAN1 R507H, PCNA, DNA BINDING PROTEIN; 3.42A {Homo sapiens}
</pre><pre>Probab=90.65  E-value=1.2  Score=38.03  Aligned_cols=45  Identities=24%  Similarity=0.307  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||+++.+....      --.+||||-|+|.++..|+..-.....-.-.+.+
</pre><pre>T Consensus       963 PDL~lw~~~~~------~~~fVEVK~p~DrLs~~Q~~wl~~L~~~G~~v~V 1007 (1021)
</pre><pre>T 9GY0_A          963 PDLVVWNSQSR------HFKLVEVKGPNDRLSHKQMIWLAELQKLGAEVEV 1007 (1021)
</pre><pre>T ss_dssp             CSEEEEETTTT------EEEEEEEECTTCCCCHHHHHHHHHHHHTTCCEEE
</pre><pre>T ss_pred             CceEEEeCCCC------eEEEEEEECCCCccCHHHHHHHHHHHHCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 8
</pre><pre>>PF20713.2 ; DUF6826 ; Domain of unknown function (DUF6826)
</pre><pre>Probab=88.44  E-value=2.5  Score=26.85  Aligned_cols=43  Identities=26%  Similarity=0.305  Sum_probs=0.0  Template_Neff=6.500
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEee--CC----CCCCCHHHHHHHHHH
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIK--FP----PDKMDIKQERAYIRI   47 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~K--Fp----~D~l~~~Q~~~Y~~I   47 (57)
</pre><pre>                      +.||++++..... ...+|+-.|.|+|  =+    -+.-..+|--+|..+
</pre><pre>T Consensus        38 ~~PD~~f~~k~~~-~~~~~vv~igElK~~~~~~~~~~~~~~GQl~~Yl~~   86 (94)
</pre><pre>T A0A2Z6RAH8_9GL   38 RKPDFVILSKQDP-LDPLNVVAVGEIRVIKAQHGIFSNADIGHAVSFGEK   86 (94)
</pre><pre>T ss_pred             CCCCEEEEeCCCC-CCcceEEEEEEEeeecCcCCCCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 9
</pre><pre>>4QBL_C VRR-NUC; Nuclease, HYDROLASE; HET: MSE; 2.0A {Psychrobacter sp.} SCOP: c.52.1.35
</pre><pre>Probab=87.48  E-value=4.5  Score=26.18  Aligned_cols=44  Identities=20%  Similarity=0.081  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchh--hEEEee-CCCCCCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIK--QVVEIK-FPPDKMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~--~vVE~K-Fp~D~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||++++..       ...-  ..||+| -++..+++.|+.--..+......+.+
</pre><pre>T Consensus        69 pDl~~~~~-------~~~~~~~~iE~K~~~~~~~s~~Q~~~~~~~~~~G~~~~v  115 (145)
</pre><pre>T 4QBL_C           69 PDLIIDIA-------RGAYHGLRIEIKKDGNSYATPAQKERIEMLAKEGYCAVV  115 (145)
</pre><pre>T ss_dssp             CSEEECCC-------BTTBSCEEEEEESSTTCCCCHHHHHHHHHHHHTTCEEEE
</pre><pre>T ss_pred             CcEEEEec-------CCceeEEEEEEEeCCCCCCCHHHHHHHHHHHHCCCeEEE
</pre><pre>
</pre><pre>
</pre><pre>No 10
</pre><pre>>PF01646.21 ; Herpes_UL24 ; Herpes virus proteins UL24 and UL76
</pre><pre>Probab=85.29  E-value=1.5  Score=31.23  Aligned_cols=23  Identities=39%  Similarity=0.509  Sum_probs=0.0  Template_Neff=5.300
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEee
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIK   30 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~K   30 (57)
</pre><pre>                      ||||.|++-.....+    +=-|||.|
</pre><pre>T Consensus        58 RipDCI~l~~~~~~~----~CyiiElK   80 (174)
</pre><pre>T UL24_HHV8P/14-   58 RIADCVILVSGGHQP----VCYVVELK   80 (174)
</pre><pre>T ss_pred             CCCCEEEEEEcCCCC----EEEEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 11
</pre><pre>>6OKH_B Uncharacterized protein; SSGCID, Structural Genomics, Seattle Structural Genomics Center for Infectious Disease, uncharacterized protein, UNKNOWN FUNCTION; 1.85A {Leptospira borgpetersenii serovar Hardjo-bovis (strain JB197)}
</pre><pre>Probab=84.26  E-value=5.6  Score=25.58  Aligned_cols=48  Identities=10%  Similarity=0.124  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCC-----CCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDP-----TKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~-----~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +..+.||+.++...     ...........+|||..++......+.+.....+
</pre><pre>T Consensus        81 ~~~~~PDv~v~~~~~~~~~~~~~~~~~p~lvvEvvs~~~~~~d~~~K~~~y~~  133 (190)
</pre><pre>T 6OKH_B           81 HNVVQPDLIFISKARNTIIREKRIEGAPDWIAEILSEGNAYHDLKTKKRLYEK  133 (190)
</pre><pre>T ss_dssp             TEEECCSEEEEEGGGGGGBCSSSEESCCSEEEEECCGGGCHCCCCCHHHHHHH
</pre><pre>T ss_pred             CCeecCCEEEEeCCcchhhhhcccccCCcEEEEecCCCchhhcHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 12
</pre><pre>>PF15516.11 ; BpuSI_N ; BpuSI N-terminal domain
</pre><pre>Probab=84.06  E-value=6.5  Score=26.27  Aligned_cols=43  Identities=26%  Similarity=0.269  Sum_probs=0.0  Template_Neff=7.700
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCC----HHHHHHHHHHh----CCc
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMD----IKQERAYIRIA----GDQ   51 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~----~~Q~~~Y~~IA----G~~   51 (57)
</pre><pre>                      .+||.++....     ..+...+||+|=++..+.    ..|...|..-.    +.+
</pre><pre>T Consensus        43 ~r~D~vl~~~~-----~~~~~~vIE~K~~~~~l~~~~~~~Qa~sYa~~l~~~~~~~   93 (152)
</pre><pre>T A0A1L8QX79_9EN   43 LEMDYVIQNII-----TGKYLCVVEVKRTPSDIHSARYQFQAMSYVQMNANESEKP   93 (152)
</pre><pre>T ss_pred             CeecEEEEEcc-----CCeEEEEEEEeCCCCCcCChHHHHHHHHHHHHccccCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 13
</pre><pre>>1GEF_A HOLLIDAY JUNCTION RESOLVASE; Holliday junction resolvase, HJC, HYDROLASE; HET: SO4; 2.0A {Pyrococcus furiosus} SCOP: c.52.1.18
</pre><pre>Probab=82.90  E-value=6.1  Score=23.87  Aligned_cols=35  Identities=17%  Similarity=0.192  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCC---CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPD---KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D---~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ||+++.++        |-..+||+|....   .++..|.+....+|
</pre><pre>T Consensus        32 ~Dli~~~~--------~~~~~iEvK~~~~~~~~~~~~~~~~l~~~~   69 (123)
</pre><pre>T 1GEF_A           32 VDLVAGNG--------KKYLCIEVKVTKKDHLYVGKRDMGRLIEFS   69 (123)
</pre><pre>T ss_dssp             CSEEEECS--------SCEEEEEEEEESSSCEEECHHHHHHHHHHH
</pre><pre>T ss_pred             CCEEEEeC--------CeEEEEEEEEecCCceecCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 14
</pre><pre>>3IJM_A uncharacterized restriction endonuclease-like fold superfamily protein; DUF820, Cyanobacteria, PD(D/E)xK superfamily, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG; HET: MSE, GOL; 1.7A {Spirosoma linguale}
</pre><pre>Probab=80.55  E-value=6.8  Score=25.08  Aligned_cols=37  Identities=16%  Similarity=0.156  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQER   42 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~   42 (57)
</pre><pre>                      +..+.||++++.....     ...-+|||-.++......+.+
</pre><pre>T Consensus        63 ~~~~~PDi~vv~~~~~-----~p~lviEV~s~~~~~~d~~~k   99 (151)
</pre><pre>T 3IJM_A           63 YSSPVPDVLLYDHQTE-----EAKVIIEVCQNSGLKHDTSKI   99 (151)
</pre><pre>T ss_dssp             TTSCCCSEEEEETTTT-----EEEEEEEEECGGGHHHHHHHH
</pre><pre>T ss_pred             CCccCCCEEEEeCCCC-----ceEEEEEEeccccccccHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 15
</pre><pre>>3DNX_A uncharacterized protein SPO1766; structural genomics, APC88088, protein of unknown function, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics; HET: MSE; 1.94A {Silicibacter pomeroyi}
</pre><pre>Probab=78.47  E-value=3.3  Score=26.92  Aligned_cols=38  Identities=24%  Similarity=0.212  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCC-CCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFP-PDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp-~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ++||++..+..      ..+ ..+|+|=. .|.++..|..+|...+
</pre><pre>T Consensus        41 ~~~Dvv~~~~~------~~i-~~vEvK~s~~d~~~~~q~~~~~~~~   79 (153)
</pre><pre>T 3DNX_A           41 LRVDVMGLGPK------GEI-WVIECKSSRADFQADAKWQGYLEWC   79 (153)
</pre><pre>T ss_dssp             CCEEEEEECTT------CCE-EEEEECSSHHHHHHTTTGGGGGGGC
</pre><pre>T ss_pred             CeeeEEEEcCC------CcE-EEEEEeeCHHHHHHHHhHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 16
</pre><pre>>3OT2_B Uncharacterized protein; STRUCTURAL GENOMICS, JOINT CENTER FOR STRUCTURAL GENOMICS, JCSG, PROTEIN STRUCTURE INITIATIVE, PSI-BIOLOGY, UNKNOWN FUNCTION; HET: EDO, MSE; 1.96A {Anabaena variabilis}
</pre><pre>Probab=78.27  E-value=20  Score=22.60  Aligned_cols=56  Identities=11%  Similarity=0.045  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             CccCCcEEEEcCCCCCCCCCch---------hhEEEeeCCCCCCCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1    2 MVRRPDVVIVNDPTKPPTQDNI---------KQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus         2 ~~RrPDViiV~d~~~~p~qdNi---------~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      ..+.||+.++..........+.         .-+|||.-+++...+-..+.-...+-+-..+.+.
</pre><pre>T Consensus        76 ~~~~PDv~v~~~~~~~~~~~~~~~~~~~~~P~lviEv~s~~~~~~d~~~K~~~Y~~~gv~~~~~i  140 (187)
</pre><pre>T 3OT2_B           76 HSTVPDISVFIWSRIPREENGEIANIFLIAPDWTIEILSPDQSQTKVTKNILHCLKHGTQMGWLI  140 (187)
</pre><pre>T ss_dssp             EEECCSEEEEEGGGSCBCTTSSBCSCCCSCCSEEEEECCTTCCHHHHHHHHHHHHHTTCSEEEEE
</pre><pre>T ss_pred             ccccccEEEEEcccCCccCCcchhcccccCCcEEEEEeCCCCcHHHHHHHHHHHHHhCCcEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 17
</pre><pre>>8TI9_D Shedu protein SduA; Shedu, DUF4263, Bacterial defense systems, Nuclease, Anti-plasmid defense system, PD-(D/E)XK nuclease, Whirly domain, Two-component signaling, DNA;{Bacillus cereus B4264}
</pre><pre>Probab=76.08  E-value=15  Score=25.37  Aligned_cols=40  Identities=18%  Similarity=0.179  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCC-------------------HHHHHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMD-------------------IKQERAYIR   46 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~-------------------~~Q~~~Y~~   46 (57)
</pre><pre>                      +.-++||.+++.      ..++--.|||+|-|...+-                   -.|-..|..
</pre><pre>T Consensus        92 ~~~~~pDf~~~~------~~~~~~~iVEiK~p~~~l~~~~~~r~~~~~~s~~~~~ai~Qi~~y~~  150 (229)
</pre><pre>T 8TI9_D           92 DSGKLVDFLYAN------PFSKDAVLIAIKTPSTPLITPTEYRTGVYSAHKDLTGAVTQVLTYKT  150 (229)
</pre><pre>T ss_dssp             TTCCHHHHHHHC------TTCSSCEEEEECCTTCCCBCSSCSBTTBCCBCHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCceeeEEEec------CCCCeEEEEEEeCCCCCCCCCcccCCCccccCHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 18
</pre><pre>>2WCW_D HJC; TYPE II RESTRICTION ENDONUCLEASE, HYDROLASE, DNA BINDING PROTEIN, HOLLIDAY JUNCTION RESOLVASE; HET: ACT; 1.58A {ARCHAEOGLOBUS FULGIDUS} SCOP: c.52.1.0
</pre><pre>Probab=75.83  E-value=18  Score=22.14  Aligned_cols=36  Identities=19%  Similarity=0.330  Sum_probs=0.0  Template_Neff=10.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCC---CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPD---KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D---~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||++..+        ++...+||+|-...   .++..|.....+.+
</pre><pre>T Consensus        39 ~~Dii~~~--------~~~~~~iEvK~~~~~~~~~~~~~~~~l~~~~   77 (139)
</pre><pre>T 2WCW_D           39 CPDIVAGN--------GRTYLAIEVKMRKELPLYLSADEVEQLVTFA   77 (139)
</pre><pre>T ss_dssp             CCSEEEEC--------SSCEEEEEEEECSSSCEEECHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEc--------CCcEEEEEEEecCCCCeecCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 19
</pre><pre>>1OB8_A HOLLIDAY-JUNCTION RESOLVASE; HYDROLASE, ENZYME, HOMOLOGOUS RECOMBINATION, HOLLIDAY JUNCTION RESOLVING ENZYME, NUCLEASE, ARCHAEA, THERMOPHILE; HET: SO4, EDO; 1.8A {SULFOLOBUS SOLFATARICUS} SCOP: c.52.1.18
</pre><pre>Probab=72.77  E-value=17  Score=22.89  Aligned_cols=44  Identities=20%  Similarity=0.177  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCC---CCCHHHHHHHHHHhCCcc--cceeC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPD---KMDIKQERAYIRIAGDQT--KLVEL   57 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D---~l~~~Q~~~Y~~IAG~~~--k~~~l   57 (57)
</pre><pre>                      ||++.+.+..        .-+||+|-...   .++..|.+.....|....  ...++
</pre><pre>T Consensus        38 ~Dli~~~~~~--------~~~iEvK~~~~~~~~~~~~q~~~l~~~~~~~g~~~~~~l   86 (135)
</pre><pre>T 1OB8_A           38 PDIFATKGNT--------LLSIECKSTWENKVKVKEHQVRKLLDFLSMFTMKGVPLI   86 (135)
</pre><pre>T ss_dssp             CSEEEEETTE--------EEEEEEEEESSSEEEECHHHHHHHHHHHHTSSSEEEEEE
</pre><pre>T ss_pred             CcEEEEeCCe--------EEEEEEEecCCCceecCHHHHHHHHHHHHHcCCCceEEE
</pre><pre>
</pre><pre>
</pre><pre>No 20
</pre><pre>>PF15649.11 ; Tox-REase-7 ; Restriction endonuclease fold toxin 7
</pre><pre>Probab=72.44  E-value=11  Score=22.69  Aligned_cols=47  Identities=28%  Similarity=0.252  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCC--CCCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPD--KMDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D--~l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      +..|+||.+          ..+...|.|+|--..  .--..|.++|...|.....-..|
</pre><pre>T Consensus        28 ~~~~iPD~~----------~~~~~~i~evKnv~~~~~~~t~Qi~~~~~~a~~~g~~~~l   76 (88)
</pre><pre>T D5UX20_TSUPD/3   28 GGYRIPDFL----------DEPNKQLTEVKNVNAISRRDDKQITDEANWAQENGYTMTL   76 (88)
</pre><pre>T ss_pred             CceecCCcc----------cCCCCeEEEEEcccccCCCccHHHHHHHHHHHHhCCcEEE
</pre><pre>
</pre><pre>
</pre><pre>No 21
</pre><pre>>PF06319.17 ; MmcB-like ; DNA repair protein MmcB-like
</pre><pre>Probab=71.16  E-value=3.7  Score=26.67  Aligned_cols=38  Identities=26%  Similarity=0.282  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCH-HHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDI-KQERAYIRIA   48 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~-~Q~~~Y~~IA   48 (57)
</pre><pre>                      +++|+|.++..       +.-..||||=..+++.. .|...|...+
</pre><pre>T Consensus        42 ~~~Dvv~~~~~-------~~~~~iEiK~s~~d~~~~~q~~~y~~~~   80 (148)
</pre><pre>T A6WV78_BRUA4/8   42 RRADLMAVNRK-------GEIWIVEIKSSIEDWKADHKWPEYRAYC   80 (148)
</pre><pre>T ss_pred             cEEEEEEEeCC-------CcEEEEEEEecHHHHHHhhcHHHHHhhC
</pre><pre>
</pre><pre>
</pre><pre>No 22
</pre><pre>>1WDJ_A hypothetical protein TT1808; Structural genomics, RIKEN Structural Genomics/Proteomics Initiative, RSGI, unknown function; 2.0A {Thermus thermophilus} SCOP: c.52.1.27
</pre><pre>Probab=70.71  E-value=13  Score=23.44  Aligned_cols=42  Identities=17%  Similarity=0.209  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCC---------CCCCCCchhhEEEeeCCCCCCCHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPT---------KPPTQDNIKQVVEIKFPPDKMDIKQER   42 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~---------~~p~qdNi~~vVE~KFp~D~l~~~Q~~   42 (57)
</pre><pre>                      +..+.||++++....         .........-+|||.-+.+...+..++
</pre><pre>T Consensus        80 ~~~~~PDi~v~~~~~~~~~~~~~~~~~~~~~p~lvvEi~s~~~~~~d~~~K  130 (187)
</pre><pre>T 1WDJ_A           80 GSILSPDAAFVERGAWEALSEAEREGFPPLAPKAVFEVRSASQDPEELRAK  130 (187)
</pre><pre>T ss_dssp             SCEECCSEEEEEHHHHHTSCHHHHHSSCBSCCSEEEEECCTTSCHHHHHHH
</pre><pre>T ss_pred             CCEEcCcEEEEehhhHHhhcHHHhcCCCCCCCcEEEEeCCCCCCHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 23
</pre><pre>>PF01870.23 ; Hjc ; Archaeal holliday junction resolvase (hjc)
</pre><pre>Probab=61.28  E-value=43  Score=18.30  Aligned_cols=37  Identities=11%  Similarity=0.132  Sum_probs=0.0  Template_Neff=10.800
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCC---CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPD---KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D---~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ..+|++..++        +...+||+|-..+   .++..|.....+.+
</pre><pre>T Consensus        24 ~~~Dli~~~~--------~~~~~vevK~~~~~~~~~~~~~~~~~~~~~   63 (87)
</pre><pre>T HJC_METJA/10-9   24 KGVDLIAGRK--------GEVLIFECKTSSKTKFYINKEDIEKLISFS   63 (87)
</pre><pre>T ss_pred             CCCcEEEEEC--------CEEEEEEEEecCCCceecCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 24
</pre><pre>>3S1S_A restriction endonuclease BpuSI; PD--(D/E)xk catalytic motif, gamma-N6m-adenosine methyltransferase, S-adenosyl-methionine binding, HYDROLASE, TRANSFERASE; HET: MSE, IOD, SAH, EDO; 2.35A {Bacillus pumilus}
</pre><pre>Probab=60.79  E-value=24  Score=29.14  Aligned_cols=49  Identities=29%  Similarity=0.316  Sum_probs=0.0  Template_Neff=9.200
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHH----HHHHHHHHhCCc--ccceeC
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIK----QERAYIRIAGDQ--TKLVEL   57 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~----Q~~~Y~~IAG~~--~k~~~l   57 (57)
</pre><pre>                      ++||.++....++.|.     -+||.|-++-.++..    |...|..-.|..  ..+.+|
</pre><pre>T Consensus        57 ~~~D~vl~~~~~~~~~-----~vIEaK~~~~~~~~~~~~~Qa~~Ya~~~~~~~~~~~~~l  111 (878)
</pre><pre>T 3S1S_A           57 IEADFVIKNKQTKKYL-----LIVEVKRTKSQVSSTRYRLQAQSYVREANIKVEQHYYCL  111 (878)
</pre><pre>T ss_dssp             EEEEEEEEETTTCCEE-----EEEEECSSHHHHTCHHHHHHHHHHHHHHGGGEEEEEEEE
</pre><pre>T ss_pred             ceeeEEEEECCCCcEE-----EEEEEecChhhhcChhHHHHHHHHHHHcccccCccEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 25
</pre><pre>>PF05685.17 ; Uma2 ; Putative restriction endonuclease
</pre><pre>Probab=60.10  E-value=54  Score=21.51  Aligned_cols=49  Identities=16%  Similarity=0.127  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCC-----CCC--CCCCchhhEEEeeCCCCCCCHHHHHHHHHHhC
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDP-----TKP--PTQDNIKQVVEIKFPPDKMDIKQERAYIRIAG   49 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~-----~~~--p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG   49 (57)
</pre><pre>                      ...+.||+.++...     ...  .....-.-+|||.-+.+...-..+...-..+|
</pre><pre>T Consensus        69 ~~~~~PDi~v~~~~~~~~~~~~~~~~~~~p~lvvEv~s~~~~~~~~~k~~~Y~~~g  124 (185)
</pre><pre>T P73158_SYNY3/4   69 FHGRIPDVSVIDRHVWRSDPGDYRALTEPVQLAVEVVSSNWETDYFDKLDEYQRLG  124 (185)
</pre><pre>T ss_pred             CceeeCcEEEEchhHccCCCCcccccCCCCcEEEEEecCCchhhHHHHHHHHHHcC
</pre><pre>
</pre><pre>
</pre><pre>No 26
</pre><pre>>8GLV_Jo Radial spoke protein 16; cilia, microtubule, dynein, MOTOR PROTEIN; HET: GTP, MG, GDP; 3.1A {Chlamydomonas reinhardtii}
</pre><pre>Probab=56.42  E-value=79  Score=22.99  Aligned_cols=48  Identities=15%  Similarity=0.164  Sum_probs=0.0  Template_Neff=9.500
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCC------CCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCccc
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPP------TQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTK   53 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p------~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k   53 (57)
</pre><pre>                      .|.-.++-...+.|      ...||---++|+||. .+++.|++...+++.....
</pre><pre>T Consensus       262 ~~~~~~~i~g~G~p~~~~~~~rGdL~v~~~v~~P~-~l~~~~~~~l~~~~~~~~~  315 (346)
</pre><pre>T 8GLV_Jo         262 RPGSSLTVPGEGMPLPATPSARGNLVIEIDLLFPT-HLTETQKMLLRSAFFLPPP  315 (346)
</pre><pre>T ss_pred             cCCCeEEeCCCcCCCCCCCCCCCcEEEEEEEEcCC-CCCHHHHHHHHHHcCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 27
</pre><pre>>PF14082.11 ; SduA_C ; Shedu protein SduA, C-terminal
</pre><pre>Probab=56.25  E-value=24  Score=23.01  Aligned_cols=25  Identities=24%  Similarity=0.339  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCC
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKM   36 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l   36 (57)
</pre><pre>                      |.||.++..        ++--.+||+|-|...+
</pre><pre>T Consensus        43 ~~pDf~~~~--------~~~~~ivElK~p~~~l   67 (170)
</pre><pre>T Q0APR0_MARMM/2   43 PVVDFLMTF--------TDYTALVEIKKPSTPI   67 (170)
</pre><pre>T ss_pred             cccceEEee--------CCceEEEEeCCCCCcc
</pre><pre>
</pre><pre>
</pre><pre>No 28
</pre><pre>>3LZ8_B Putative chaperone DnaJ; STRUCTURE GENOMICS, Structural Genomics, PSI-2, Protein Structure Initiative, Midwest Center for Structural Genomics, MCSG, Chaperone; 2.9A {Klebsiella pneumoniae subsp. pneumoniae}
</pre><pre>Probab=53.77  E-value=84  Score=22.36  Aligned_cols=47  Identities=13%  Similarity=0.191  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCC---CCchhhEEEeeCCCCCCCHHHHHHHHHHhCCcc
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPT---QDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQT   52 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~---qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~   52 (57)
</pre><pre>                      .|.-.++-...+.|.   ..||---+.|+||.. ++.+|++..++|++.+.
</pre><pre>T Consensus       269 ~~~~~~~~~g~G~p~~~~rGdL~v~~~v~~P~~-~~~~~~~~l~~~~~~~~  318 (329)
</pre><pre>T 3LZ8_B          269 QAGQRLRIKGKGLVSKTHTGDLFAVIKIVMPTK-PDEKARELWQQLAAAEA  318 (329)
</pre><pre>T ss_dssp             CTTCEEEEEEEEEEC--CEEEEEEEEEECCCSS-CCHHHHHHHHHHHHHTT
</pre><pre>T ss_pred             CCCcEEEEcCCCCCcCCCCccEEEEEEEEcCCC-CCHHHHHHHHHHHHhhc
</pre><pre>
</pre><pre>
</pre><pre>No 29
</pre><pre>>PF04411.17 ; PDDEXK_7 ; PD-(D/E)XK nuclease superfamily
</pre><pre>Probab=53.17  E-value=48  Score=21.37  Aligned_cols=43  Identities=14%  Similarity=0.201  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCC---------HHHHHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMD---------IKQERAYIR   46 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~---------~~Q~~~Y~~   46 (57)
</pre><pre>                      +.-++||++|.....+.+.   .--+++.||..+.-.         -.|..+|..
</pre><pre>T Consensus        70 ~~~~~PDi~i~~~~~~~~~---~~~vlDaKy~~~~~~~~~~~~~~~~~~l~~Y~~  121 (149)
</pre><pre>T Q3ABD9_CARHZ/3   70 TSDQKPDTVLCLNKKGKIN---RMYIFDAKYRVEINNEEIGPLEEDINAMHRYRD  121 (149)
</pre><pre>T ss_pred             CcCCCCcEEEEEEECCeEE---EEEEEeeccccccCCCCCCCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 30
</pre><pre>>PF18743.6 ; AHJR-like ; REase_AHJR-like
</pre><pre>Probab=52.96  E-value=84  Score=19.08  Aligned_cols=43  Identities=21%  Similarity=0.181  Sum_probs=0.0  Template_Neff=9.800
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      ||+|.+        .++...+||+|-.....+..|.....+.+........
</pre><pre>T Consensus        45 ~Dli~~--------~~~~~v~vEvK~~~~~~~~~~l~~~~~~~~~~~~~~~   87 (123)
</pre><pre>T A0A252DYT9_9NO   45 LDMIAI--------CDHENVVVEVKSKPTLTKSSYLTALAKAVNNQPGWRF   87 (123)
</pre><pre>T ss_pred             ccEEEE--------cCCceEEEEEEcCCcccchHHHHHHHHHHhCCCCcEE
</pre><pre>
</pre><pre>
</pre><pre>No 31
</pre><pre>>8TI8_C Shedu protein SduA; Shedu, DUF4263, Bacterial defense systems, Nuclease, Anti-plasmid defense system, PD-(D/E)XK nuclease, Whirly domain, Two-component signaling, DNA;{Bacillus cereus B4264}
</pre><pre>Probab=52.89  E-value=68  Score=24.19  Aligned_cols=39  Identities=21%  Similarity=0.231  Sum_probs=0.0  Template_Neff=8.700
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCC-------------------HHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMD-------------------IKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~-------------------~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .+||.++.+.      .++--.|||||-|...+-                   -.|-..|....
</pre><pre>T Consensus       248 ~~~Dfl~~~~------~~~~~~lVEIK~P~~~l~~~~~~r~~~~~~s~~l~~ai~Qv~~Y~~~l  305 (382)
</pre><pre>T 8TI8_C          248 KLVDFLYANP------FSKDAVLIEIKTPSTPLITPTEYRTGVYSAHKDLTGAVTQVLTYKTTL  305 (382)
</pre><pre>T ss_dssp             CHHHHHHTCS------SCSCCEEEECCCTTCCCBCSSCSBTTBCCBCHHHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             ceeeEEEEcC------CCCcEEEEEecCCCCCcCCCCCCCCCccccCHHHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 32
</pre><pre>>PF10847.13 ; DUF2656 ; Protein of unknown function (DUF2656)
</pre><pre>Probab=50.55  E-value=30  Score=24.63  Aligned_cols=15  Identities=27%  Similarity=0.417  Sum_probs=0.0  Template_Neff=4.200
</pre><pre>
</pre><pre>Q ss_pred             EEEeeCCCCCCCHHH
</pre><pre>Q MBL3828645.1/1   26 VVEIKFPPDKMDIKQ   40 (57)
</pre><pre>Q Consensus        26 vVE~KFp~D~l~~~Q   40 (57)
</pre><pre>                      +|||-|+.+.+++.|
</pre><pre>T Consensus        49 ~veI~f~~~~~sP~~   63 (141)
</pre><pre>T U5QL37_9CYAN/9   49 LVEVLFAADTHPPAR   63 (141)
</pre><pre>T ss_pred             EEEEEeccCCCCHHH
</pre><pre>
</pre><pre>
</pre><pre>No 33
</pre><pre>>PF18742.6 ; DpnII-MboI ; REase_DpnII-MboI
</pre><pre>Probab=50.17  E-value=40  Score=22.43  Aligned_cols=29  Identities=21%  Similarity=0.168  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CchhhEEEeeCCCC--CCCHHHHHHHHHHhC
</pre><pre>Q MBL3828645.1/1   21 DNIKQVVEIKFPPD--KMDIKQERAYIRIAG   49 (57)
</pre><pre>Q Consensus        21 dNi~~vVE~KFp~D--~l~~~Q~~~Y~~IAG   49 (57)
</pre><pre>                      +++..+||+||..+  .++..+.+-+..|.+
</pre><pre>T Consensus        71 ~~~~~~IEvK~~~~~~~~~~~~~qi~~D~~~  101 (148)
</pre><pre>T D8PIY8_9BACT/1   71 KEEGIVVELKKTRRSLDTKAIGDQLLIDIGR  101 (148)
</pre><pre>T ss_pred             CccCEEEEEEEcCCCCChhHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 34
</pre><pre>>PF19986.4 ; DUF6422 ; Family of unknown function (DUF6422)
</pre><pre>Probab=47.06  E-value=42  Score=22.19  Aligned_cols=17  Identities=24%  Similarity=0.167  Sum_probs=0.0  Template_Neff=4.800
</pre><pre>
</pre><pre>Q ss_pred             CCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1   32 PPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus        32 p~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +.+.|+++|++++++-|
</pre><pre>T Consensus         3 ~~~~LTd~Q~kAl~eaa   19 (94)
</pre><pre>T A0A1G9A3Y1_9AC    3 TYQDLTAKQKSALESAA   19 (94)
</pre><pre>T ss_pred             CcccCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 35
</pre><pre>>6JZB_A Chaperone protein DnaJ; cochaperone activity, zinc finger, tetramer, STRUCTURAL PROTEIN; 2.754A {Streptococcus pneumoniae}
</pre><pre>Probab=43.46  E-value=1.6e+02  Score=21.66  Aligned_cols=43  Identities=14%  Similarity=0.141  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCC-----CchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQ-----DNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~q-----dNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .|.-+++-...+.|..     .||---+.|.|| +.+++.|++..++++
</pre><pre>T Consensus       313 ~~g~~~~i~g~G~p~~~~~~~GdL~v~~~v~~p-~~~~~~~~~~l~~~~  360 (385)
</pre><pre>T 6JZB_A          313 QTGKKFRLRSKGAPSLRGGAVGDQYVTVNVVTP-TGLNDRQKVALKEFA  360 (385)
</pre><pre>T ss_dssp             CTTCEEEETTCSCCCTTSCCCCCEEEEEEEECC-CSCCHHHHHHHHHHH
</pre><pre>T ss_pred             CCCCEEEECCCCCCCCCCCCCccEEEEEEEEcC-CCCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 36
</pre><pre>>PF10069.14 ; DICT ; Sensory domain in DIguanylate Cyclases and Two-component system
</pre><pre>Probab=43.39  E-value=49  Score=20.60  Aligned_cols=29  Identities=17%  Similarity=0.352  Sum_probs=0.0  Template_Neff=9.000
</pre><pre>
</pre><pre>Q ss_pred             eeCCCCCCCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1   29 IKFPPDKMDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus        29 ~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      .+|+....-..+.+.|.++|.....+.++
</pre><pre>T Consensus        46 ~~~~~~~~~~~~~~ry~~la~~~~~v~v~   74 (121)
</pre><pre>T K9U4U9_CHRTP/6   46 ASFQRERFYRQEAHRYKRLARQTNQVYVL   74 (121)
</pre><pre>T ss_pred             EecCChhhhhHHHHHHHHHHhhCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 37
</pre><pre>>PF10117.14 ; McrBC ; McrBC 5-methylcytosine restriction system component
</pre><pre>Probab=42.07  E-value=70  Score=22.32  Aligned_cols=36  Identities=19%  Similarity=0.331  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCC------CHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKM------DIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l------~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +||+|+.++        +...|++.|+-.-..      ......-|+.+|
</pre><pre>T Consensus       262 ~PDivi~~~--------~~~~I~DAKYk~~~~~~~~~~~~~~~DiyQ~~~  303 (318)
</pre><pre>T Q6LZ73_METMP/6  262 KPDIYAETS--------SKKYIMDAKWKILNSDKNKNYGISQNDMYQLLS  303 (318)
</pre><pre>T ss_pred             cCeEEEEeC--------CcEEEEEccccccCCccccCCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 38
</pre><pre>>PF20472.3 ; PDDEXK_11 ; PD-(D/E)XK nuclease superfamily domain
</pre><pre>Probab=41.72  E-value=1.1e+02  Score=20.74  Aligned_cols=41  Identities=10%  Similarity=0.323  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRI   47 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~I   47 (57)
</pre><pre>                      |.-++||.+|...      ..+-.-+||.||-...=|.+++-.|.-+
</pre><pre>T Consensus        53 g~~~~~Df~i~~~------~~~~~~iIE~K~Q~~~GSvdEKlp~~~~   93 (155)
</pre><pre>T A0A418LY79_9BA   53 GHQGKTEFLLQSE------EYGLQIRIECKWQQSSGSVDEKFPYLYL   93 (155)
</pre><pre>T ss_pred             CCCCceeEEEEeC------CcCCeEEEEEEEEecCCChhHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 39
</pre><pre>>1KNV_A Bse634I restriction endonuclease; Restriction endonuclease, apo-enzyme, HYDROLASE; 2.17A {Geobacillus stearothermophilus} SCOP: c.52.1.7
</pre><pre>Probab=40.73  E-value=1.1e+02  Score=23.74  Aligned_cols=36  Identities=22%  Similarity=0.439  Sum_probs=0.0  Template_Neff=5.800
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHH----HHHHHhC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQER----AYIRIAG   49 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~----~Y~~IAG   49 (57)
</pre><pre>                      ||.|||.-+......+..        +-..++.....    +|..+.|
</pre><pre>T Consensus       145 PD~Viv~~~~~~~~~~~~--------~i~~l~~~~l~~L~~~y~~l~G  184 (293)
</pre><pre>T 1KNV_A          145 PDLLIIRQKDLIKSEYNL--------PINKLTHENIDVALTLFKDIEG  184 (293)
</pre><pre>T ss_dssp             CSEEEEECGGGCCGGGGS--------CCCSCCHHHHHHHHTHHHHHTT
</pre><pre>T ss_pred             CCEEEEEcccccccccCC--------CcccCCHHHHHHHHHHHHHHcc
</pre><pre>
</pre><pre>
</pre><pre>No 40
</pre><pre>>PF03838.19 ; RecU ; Recombination protein U
</pre><pre>Probab=40.64  E-value=1.3e+02  Score=20.17  Aligned_cols=44  Identities=7%  Similarity=-0.087  Sum_probs=0.0  Template_Neff=8.300
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCC------CCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDK------MDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~------l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      ||++.+.+....        .+|+|-....      +++.|.+.-.+++.......++
</pre><pre>T Consensus        54 ~D~~~~~~g~~~--------~iE~K~~~~~~~~~~~i~~~Q~~~l~~~~~~g~~~~v~  103 (161)
</pre><pre>T R5YL25_9FIRM/1   54 VDYIGAVQGIPV--------CFDAKECATDTFPMMNIHEHQVKFMKDFEKQGGVSFII  103 (161)
</pre><pre>T ss_pred             CCEEEEECCEEE--------EEEccccCCCccChhhCcHHHHHHHHHHHHCCCeEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 41
</pre><pre>>8PS4_A Shedu effector protein; SduA, Shedu, Prokaryotic immune system, IMMUNE SYSTEM; 2.95A {Escherichia coli KTE10}
</pre><pre>Probab=40.45  E-value=1.2e+02  Score=23.78  Aligned_cols=39  Identities=31%  Similarity=0.425  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCC-CCCCC------------------HHHHHHHHH
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFP-PDKMD------------------IKQERAYIR   46 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp-~D~l~------------------~~Q~~~Y~~   46 (57)
</pre><pre>                      +.-|+||.+++..       ++--.|||+|=| +..+-                  -.|-+.|..
</pre><pre>T Consensus       266 ~~~~~~Dfll~~~-------~g~~~iIEIK~P~~~~l~~~~~yR~~~~ps~els~ai~Qv~~Y~~  323 (411)
</pre><pre>T 8PS4_A          266 KTNRFIDIALVDA-------NGNLDIIEVKKPFDDKILRKTPYRDNYIPTSELSGGIMQAEKYIF  323 (411)
</pre><pre>T ss_dssp             CEEEEEEEEEEET-------TCBEEEEEECCCCTTSSSCSSCBTTBCCCCHHHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCcceEEEEEEcC-------CCCEEEEEEcCcCCCccccCCCCCCCCCCCHHHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 42
</pre><pre>>5XKT_A Urease accessory protein UreG; SIMIBI Class GTPase, HYDROLASE; HET: GNP; 1.8A {Klebsiella pneumoniae subsp. rhinoscleromatis SB3432}
</pre><pre>Probab=36.19  E-value=66  Score=19.85  Aligned_cols=26  Identities=19%  Similarity=0.382  Sum_probs=0.0  Template_Neff=11.700
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCC-CCchhhE
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPT-QDNIKQV   26 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~-qdNi~~v   26 (57)
</pre><pre>                      +.++.||+++++..+..+. ..+++.+
</pre><pre>T Consensus       136 ~~~~~~~iivl~k~d~~~~~~~~~~~~  162 (200)
</pre><pre>T 5XKT_A          136 PGITRSDFLVINKTDLAPYVGASLKVM  162 (200)
</pre><pre>T ss_dssp             HHHHHSSEEEEECGGGHHHHTCCHHHH
</pre><pre>T ss_pred             CcccccCEEEEECcccccccCCCHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 43
</pre><pre>>PF08011.16 ; PDDEXK_9 ; PD-(D/E)XK nuclease superfamily
</pre><pre>Probab=35.65  E-value=1.4e+02  Score=16.70  Aligned_cols=33  Identities=18%  Similarity=0.118  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHH
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRI   47 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~I   47 (57)
</pre><pre>                      +.|+++.        ..+-.-++|+|+..+.  ..-.++..+|
</pre><pre>T Consensus        31 ~~di~i~--------~~~~~~iiE~K~~~~~--~~~~~a~~Qi   63 (96)
</pre><pre>T Q2FP75_METHJ/4   31 RIDLTVK--------TRTGIWIFEFKVSDIP--PENTSALEQI   63 (96)
</pre><pre>T ss_pred             eecEEEE--------cCCEEEEEEEEecCCC--cchhHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 44
</pre><pre>>PF02078.21 ; Synapsin ; Synapsin, N-terminal domain
</pre><pre>Probab=35.14  E-value=1.3e+02  Score=19.19  Aligned_cols=29  Identities=28%  Similarity=0.491  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCC
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPP   33 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~   33 (57)
</pre><pre>                      +||-|++......+.++-...|+-+.|.+
</pre><pre>T Consensus        71 ~PDfVLiRq~~~~~~~d~rnil~GL~~ag   99 (102)
</pre><pre>T E9G692_DAPPU/1   71 RPDFVLIRQNMRDAGEDYKSIVLGLKFGG   99 (102)
</pre><pre>T ss_pred             CCcEEEEeccccCCCCCchhhcceeecCC
</pre><pre>
</pre><pre>
</pre><pre>No 45
</pre><pre>>7VSR_M Protein McrC; AAA+ protein, GTPase, endonuclease, McrBC, stalkless mutant, DNA BINDING PROTEIN; HET: GNP;{Escherichia coli (strain K12)}
</pre><pre>Probab=34.94  E-value=84  Score=22.41  Aligned_cols=36  Identities=14%  Similarity=0.252  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCC-------CCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPP-------DKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~-------D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +||+++-..        +...|++.||-.       +.....+..-|+-+|
</pre><pre>T Consensus       204 ~PDi~i~~~--------~~~~ViDaKYK~~~~~~~~~~~~~~~~D~yQ~~a  246 (310)
</pre><pre>T 7VSR_M          204 ETDITIRSS--------EKILIVDAKYYKSIFSRRMGTEKFHSQNLYQLMN  246 (310)
</pre><pre>T ss_dssp             ECSEEEECS--------SCEEEEEECCCHHHHCC----CCCCCTHHHHHHH
</pre><pre>T ss_pred             cccEEEEeC--------CeEEEEEeeeChhHHHhhcCCccccCchHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 46
</pre><pre>>PF08722.16 ; Tn7_TnsA-like_N ; TnsA-like endonuclease N terminal
</pre><pre>Probab=33.72  E-value=1.3e+02  Score=15.60  Aligned_cols=36  Identities=19%  Similarity=0.156  Sum_probs=0.0  Template_Neff=11.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||+++.......        ++|+|-..+..+...+..+..+.
</pre><pre>T Consensus        26 ~pD~~v~~~~g~~--------~~~vk~~~~~~~~~~~~~~~~~~   61 (76)
</pre><pre>T A6T2H6_JANMA/6   26 FPDLFVVGNGRAA--------FIEVKSHTDPDLCEAIERGDAIR   61 (76)
</pre><pre>T ss_pred             cccEEEEEcCCcE--------EEEEccCCCcchHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 47
</pre><pre>>PF21420.2 ; Tautomerase-like ; Tautomerase-like
</pre><pre>Probab=33.18  E-value=1.6e+02  Score=17.61  Aligned_cols=23  Identities=22%  Similarity=0.408  Sum_probs=0.0  Template_Neff=2.700
</pre><pre>
</pre><pre>Q ss_pred             EEEeeCCCCC-CCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1   26 VVEIKFPPDK-MDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus        26 vVE~KFp~D~-l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ++||=..+|. ++.+|++++.+=|
</pre><pre>T Consensus         2 MlEVl~~~e~pl~~e~krAfa~eA   25 (45)
</pre><pre>T A9W9V0_CHLAA/2    2 MLEVFYSGDRPPDRTRKQAFAAEA   25 (45)
</pre><pre>T ss_pred             ceEEEEcCCCCCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 48
</pre><pre>>9IAB_A PD-(D/E)XK endonuclease-like domain-containing protein; Cas4 family protein, IMMUNE SYSTEM, deoxyribonuclease, hydrolase; HET: MES, SF4, SO4; 1.57A {Chroococcidiopsis thermalis}
</pre><pre>Probab=32.77  E-value=1.6e+02  Score=18.42  Aligned_cols=37  Identities=24%  Similarity=0.355  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ++|.|+..       .++--.+|+.|......+..|...|...+
</pre><pre>T Consensus       147 ~iD~v~~~-------~~~~~~iiD~Kt~~~~~~~~Ql~~Y~~~~  183 (273)
</pre><pre>T 9IAB_A          147 AMDCVLQE-------EDGTLHIVDFKTGKSDFDRRQAYIYLLAA  183 (273)
</pre><pre>T ss_dssp             CCSEEEEC-------TTSCEEEEEEECTTCCCCHHHHHHHHHHH
</pre><pre>T ss_pred             EeeEEEEc-------CCCeEEEEECCCCCCchhHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 49
</pre><pre>>PF20452.3 ; Calmod_bind_C ; Calmodulin binding protein C-terminal domain
</pre><pre>Probab=32.50  E-value=1.5e+02  Score=16.15  Aligned_cols=28  Identities=7%  Similarity=0.228  Sum_probs=0.0  Template_Neff=8.800
</pre><pre>
</pre><pre>Q ss_pred             CchhhEEEeeCCC-----CCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1   21 DNIKQVVEIKFPP-----DKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus        21 dNi~~vVE~KFp~-----D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +++-.|+.++|.+     |.|++.|+..-.+..
</pre><pre>T Consensus        15 n~v~~v~~~~~~g~~~~~~~l~~~qk~~v~~l~   47 (60)
</pre><pre>T A0A3B6PHL1_WHE   15 SPIGEILAARIGDQTCSLQQLHQEHKAQVNQLA   47 (60)
</pre><pre>T ss_pred             CCCceEEEEEECCEEcCHHhCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 50
</pre><pre>>6JC3_A Nucleocapsid; self-capping helical structure complex, NUCLEAR PROTEIN, NUCLEAR PROTEIN-RNA complex; 4.8A {Avian avulavirus 1}
</pre><pre>Probab=31.74  E-value=2e+02  Score=23.73  Aligned_cols=33  Identities=21%  Similarity=0.509  Sum_probs=0.0  Template_Neff=5.000
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCCCCCCchhhEEEe--------eC-CCCCCCHHHHHHHHHHhCC
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKPPTQDNIKQVVEI--------KF-PPDKMDIKQERAYIRIAGD   50 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~p~qdNi~~vVE~--------KF-p~D~l~~~Q~~~Y~~IAG~   50 (57)
</pre><pre>                      +.||+.|.              +|||        +| +....++.+.++|..||++
</pre><pre>T Consensus        93 ~~pd~~i~--------------ivei~~~~~~~~~~~~R~~~~~~~~~~~~~~a~~  134 (398)
</pre><pre>T 6JC3_A           93 KQNEATLA--------------VLEIDGFTNGVPQFNNRSGVSEERAQRFMMIAGS  134 (398)
</pre><pre>T ss_dssp             TCTTSCBC--------------CCBCCCCCSSSSCCCCCSSCCTTHHHHHHHHHTC
</pre><pre>T ss_pred             cCCCcEEE--------------EEEEecCCCCceEEecCCCChHHHHHHHHHHHhc
</pre><pre>
</pre><pre>
</pre><pre>No 51
</pre><pre>>4TKD_D Holliday junction resolvase Hjc; HYDROLASE; 2.01A {Sulfolobus solfataricus} SCOP: c.52.1.18
</pre><pre>Probab=31.46  E-value=2.3e+02  Score=17.80  Aligned_cols=35  Identities=23%  Similarity=0.423  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCC---CCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDK---MDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~---l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ||+|..++.        ...+||+|-....   ++..|.....+.|
</pre><pre>T Consensus        41 ~Dii~~~~~--------~~~~iEvK~~~~~~~~~~~~~~~~l~~~~   78 (141)
</pre><pre>T 4TKD_D           41 PDIIALKNG--------VIILIEMKSRKDGKIYVRREQAEGIIEFA   78 (141)
</pre><pre>T ss_dssp             CSEEEEETT--------EEEEEEEEECSSSEEEECHHHHHHHHHHH
</pre><pre>T ss_pred             CCEEEEECC--------EEEEEEEEeeCCCceeecHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 52
</pre><pre>>9IAC_A Similar to tr|Q3MCC8|Q3MCC8_ANAVT Hypothetical protein; Cas4 family protein, IMMUNE SYSTEM, deoxyribonuclease, hydrolase; HET: EDO, SF4; 2.52A {Microcystis aeruginosa}
</pre><pre>Probab=31.42  E-value=2.2e+02  Score=17.82  Aligned_cols=36  Identities=22%  Similarity=0.370  Sum_probs=0.0  Template_Neff=12.800
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEc-CCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVN-DPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~-d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ++|.|+.. +..        -.+|+.|......+..|...|...+
</pre><pre>T Consensus       153 ~iD~i~~~~~~~--------~~ivD~Kt~~~~~~~~Ql~~Y~~~~  189 (277)
</pre><pre>T 9IAC_A          153 AFDCIFREADDT--------IHILDLKTGQSNFDRRQAHVYLLAA  189 (277)
</pre><pre>T ss_dssp             CCSEEEECTTSC--------EEEEEEECTTCCCCHHHHHHHHHHH
</pre><pre>T ss_pred             EeeEEEEeCCCe--------EEEEEeecCCCChhHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 53
</pre><pre>>PF21192.2 ; NMD3_OB ; NMD3 OB-fold domain
</pre><pre>Probab=31.03  E-value=65  Score=19.92  Aligned_cols=9  Identities=44%  Similarity=0.811  Sum_probs=0.0  Template_Neff=7.400
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEc
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVN   12 (57)
</pre><pre>Q Consensus         4 RrPDViiV~   12 (57)
</pre><pre>                      +.||||+|+
</pre><pre>T Consensus        80 ~~pdvilVk   88 (89)
</pre><pre>T S3CVJ6_OPHP1/3   80 TVPEVVLVK   88 (89)
</pre><pre>T ss_pred             CCCcEEEEe
</pre><pre>
</pre><pre>
</pre><pre>No 54
</pre><pre>>3K93_A phage related exonuclease; phage related exonuclease, Structural Genomics, Joint Center for Structural Genomics, JCSG, Protein Structure Initiative, PSI-2, HYDROLASE; HET: PEG, MSE; 2.15A {Haemophilus somnus 129PT}
</pre><pre>Probab=30.14  E-value=2.8e+02  Score=18.82  Aligned_cols=34  Identities=18%  Similarity=0.161  Sum_probs=0.0  Template_Neff=9.700
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCH-----------------HHHHHHHHHhC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDI-----------------KQERAYIRIAG   49 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~-----------------~Q~~~Y~~IAG   49 (57)
</pre><pre>                      || +++.....         ++|+|.+.+...-                 .|-..|.-++|
</pre><pre>T Consensus        99 pD-~~~~~~~~---------ivEiK~~~~~~~~~~~~~~~~~~~~~~~y~~Q~q~qm~vtg  149 (223)
</pre><pre>T 3K93_A           99 CD-IYVPSRKL---------IIDTKCSWDIGSHPFFTDEAQEKAKKAGYDIQMQGYMWLWD  149 (223)
</pre><pre>T ss_dssp             CS-EEEGGGTE---------EEEEEECSSTTTSCCSHHHHHHHHHHHTHHHHHHHHHHHHT
</pre><pre>T ss_pred             cc-EEEcCCCe---------EEEEeCCCccccCcccchhHHHHhhcCCchHHHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 55
</pre><pre>>5JOQ_A Lmo2184 protein; CENTER FOR STRUCTURAL GENOMICS OF INFECTIOUS DISEASES, CSGID, NIAID, NATIONAL INSTITUTE OF ALLERGY AND INFECTIOUS DISEASES; HET: CIT; 1.99A {Listeria monocytogenes serovar 1/2a (strain ATCC BAA-679 / EGD-e)}
</pre><pre>Probab=30.09  E-value=1.2e+02  Score=20.53  Aligned_cols=44  Identities=11%  Similarity=0.191  Sum_probs=0.0  Template_Neff=11.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhE-EEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQV-VEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~v-VE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||+||..........+.++.. +.+-.-+..--....+.++.||
</pre><pre>T Consensus        90 ~PDlIi~~~~~~~~~~~~L~~~~i~v~~~~~~~~~~~~~~~~~lg  134 (290)
</pre><pre>T 5JOQ_A           90 KPDMVLSTKTLEADLKSGFEGADLEADFLDFTSIASMQTEIKNLG  134 (290)
</pre><pre>T ss_dssp             CCSEEEEEGGGHHHHHHHHHHTTCCEEEECCSSHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEecccccHHHHhccccCCCeEEEecCCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 56
</pre><pre>>PF05961.16 ; Chordopox_A13L ; Chordopoxvirus A13L protein
</pre><pre>Probab=29.45  E-value=1.3e+02  Score=19.09  Aligned_cols=34  Identities=15%  Similarity=0.198  Sum_probs=0.0  Template_Neff=4.300
</pre><pre>
</pre><pre>Q ss_pred             cCCCCCCCCCchhhEEEee---CCCCCCCHHHHHHHHH
</pre><pre>Q MBL3828645.1/1   12 NDPTKPPTQDNIKQVVEIK---FPPDKMDIKQERAYIR   46 (57)
</pre><pre>Q Consensus        12 ~d~~~~p~qdNi~~vVE~K---Fp~D~l~~~Q~~~Y~~   46 (57)
</pre><pre>                      ..+..+|..++...-.|.+   | .|.|+++|-++.-+
</pre><pre>T Consensus        28 ~~~~~~~~~~~~~~~~~~~~~~f-vD~L~~dhi~s~yr   64 (68)
</pre><pre>T PG139_VACCW/1-   28 NSQNPNPSPNLNSPPPEPKNTKF-VNNLEKDHISSLYN   64 (68)
</pre><pre>T ss_pred             CCCCCCCCCCCCCCCCCCCCCcc-cccCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 57
</pre><pre>>3SXU_A DNA polymerase III subunit chi; DNA replication, Chi binds to SSB and Psi, TRANSFERASE; 1.85A {Escherichia coli} SCOP: c.128.1.1
</pre><pre>Probab=28.81  E-value=1.2e+02  Score=20.14  Aligned_cols=42  Identities=17%  Similarity=0.174  Sum_probs=0.0  Template_Neff=8.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHH
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIR   46 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~   46 (57)
</pre><pre>                      .+|++|..++.-++...+..+++|+=-..+..-..-|+-|+.
</pre><pre>T Consensus        95 ~~~~linl~~~~~~~~~~f~ri~eiv~~~~~~~~~aR~r~k~  136 (150)
</pre><pre>T 3SXU_A           95 RRDILISLRTSFADFATAFTEVVDFVPYEDSLKQLARERYKA  136 (150)
</pre><pre>T ss_dssp             CCSEEEECCSSCCGGGGGCSEEEEEECSSGGGHHHHHHHHHH
</pre><pre>T ss_pred             cCCEEEECCcccchhhhhCCEEEEecCCCHHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 58
</pre><pre>>2X4L_A FERRIC-SIDEROPHORE RECEPTOR PROTEIN; TRANSPORT; 1.5A {STREPTOMYCES COELICOLOR}
</pre><pre>Probab=28.39  E-value=1e+02  Score=21.61  Aligned_cols=44  Identities=11%  Similarity=0.133  Sum_probs=0.0  Template_Neff=10.400
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCC--------CCCCCCchhhEEEeeCCCC--CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPT--------KPPTQDNIKQVVEIKFPPD--KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~--------~~p~qdNi~~vVE~KFp~D--~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||+||+....        ......-|+.++.+-.-..  .--+.-.+.+..||
</pre><pre>T Consensus       110 ~PDlIi~~~~~~~~~~~~~~~~~~~~L~~~~pvv~~~~~~~~~~~~~~~~~~lg  163 (325)
</pre><pre>T 2X4L_A          110 APEVLITTTFDTAGTLWSVPEESKDKVAKLAPSVAISVFDRQLTQPLQRMWELA  163 (325)
</pre><pre>T ss_dssp             CCSEEEEEECSTTCCCTTSCGGGHHHHHHHSCEEEEECSSSCTHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEcccCCCCcccCCChHHHHHHHHhCCEEEeecCccccHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 59
</pre><pre>>PF18891.5 ; FANCL_d3 ; FANCL UBC-like domain 3
</pre><pre>Probab=27.66  E-value=34  Score=21.59  Aligned_cols=12  Identities=25%  Similarity=0.794  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             chhhEEEe-eCCC
</pre><pre>Q MBL3828645.1/1   22 NIKQVVEI-KFPP   33 (57)
</pre><pre>Q Consensus        22 Ni~~vVE~-KFp~   33 (57)
</pre><pre>                      ||+.|.|+ +||.
</pre><pre>T Consensus        85 NL~~lL~~~~fP~   97 (98)
</pre><pre>T D2A687_TRICA/1   85 ELLKILSLEKFPQ   97 (98)
</pre><pre>T ss_pred             HHHHHhCCccCCC
</pre><pre>
</pre><pre>
</pre><pre>No 60
</pre><pre>>6IY6_I Aminoacyl tRNA synthase complex-interacting multifunctional protein 2; Aminoacyl-tRNA transferase Glutathion-S transferase domain, TRANSLATION; HET: PO4; 3.6A {Homo sapiens}
</pre><pre>Probab=27.44  E-value=99  Score=20.08  Aligned_cols=12  Identities=50%  Similarity=1.066  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCCCCCC
</pre><pre>Q MBL3828645.1/1    7 DVVIVNDPTKPP   18 (57)
</pre><pre>Q Consensus         7 DViiV~d~~~~p   18 (57)
</pre><pre>                      |+||..||+.||
</pre><pre>T Consensus         8 ~~vi~~~p~~pP   19 (215)
</pre><pre>T 6IY6_I            8 DIVINANPASPP   19 (215)
</pre><pre>T ss_dssp             EEEEEECTTSCC
</pre><pre>T ss_pred             eEEEEcCCCCCC
</pre><pre>
</pre><pre>
</pre><pre>No 61
</pre><pre>>6ALL_A Fe(3+)-citrate-binding protein yfmC; structural genomics, NIAID, national institute of allergy and infectious diseases, PERIPLASMIC LIGAND BINDING PROTEIN, SUBSTRATE BINDING; HET: PE3; 2.47A {Bacillus anthracis str. 'Ames Ancestor'}
</pre><pre>Probab=26.33  E-value=1.1e+02  Score=20.95  Aligned_cols=44  Identities=14%  Similarity=0.247  Sum_probs=0.0  Template_Neff=11.100
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCCC-----CHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKM-----DIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l-----~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +||+|+..........+.++.++.+=+-....     -+...+..+.|+
</pre><pre>T Consensus       114 ~PDlIi~~~~~~~~~~~~l~~~~~~v~~~~~~~~~~~~~~~~~~~~~lg  162 (324)
</pre><pre>T 6ALL_A          114 KPDLIITASFRGKAIKNELEQIAPTVMFDPSTSNNDHFAEMTETFKQIA  162 (324)
</pre><pre>T ss_dssp             CCSEEEEEHHHHTTTHHHHHTTSCEEEECTTTTTSCHHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEecCcccHHHHHHHhhhCCEEEeCCCCCCcchHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 62
</pre><pre>>1L6X_B Minimized B-domain of Protein A Z34C; IgG1 fc, protein A, fc complex, b-domain, IMMUNE SYSTEM; HET: BMA, MAN, GAL, FUL, NAG; 1.65A {Homo sapiens} SCOP: k.13.1.1
</pre><pre>Probab=26.21  E-value=1.5e+02  Score=16.43  Aligned_cols=13  Identities=23%  Similarity=0.360  Sum_probs=0.0  Template_Neff=4.200
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHH
</pre><pre>Q MBL3828645.1/1   34 DKMDIKQERAYIR   46 (57)
</pre><pre>Q Consensus        34 D~l~~~Q~~~Y~~   46 (57)
</pre><pre>                      ..|++.|+.+|..
</pre><pre>T Consensus        16 pnLn~~Qr~~fI~   28 (34)
</pre><pre>T 1L6X_B           16 PNLNEEQRNAKIK   28 (34)
</pre><pre>T ss_dssp             TTCCHHHHHHHHH
</pre><pre>T ss_pred             CCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 63
</pre><pre>>PF13366.11 ; PDDEXK_3 ; PD-(D/E)XK nuclease superfamily
</pre><pre>Probab=25.75  E-value=2.6e+02  Score=16.67  Aligned_cols=30  Identities=20%  Similarity=0.248  Sum_probs=0.0  Template_Neff=9.900
</pre><pre>
</pre><pre>Q ss_pred             EEEeeCCCCCCCH---HHHHHHHHHhCCccccee
</pre><pre>Q MBL3828645.1/1   26 VVEIKFPPDKMDI---KQERAYIRIAGDQTKLVE   56 (57)
</pre><pre>Q Consensus        26 vVE~KFp~D~l~~---~Q~~~Y~~IAG~~~k~~~   56 (57)
</pre><pre>                      +||+|-. +.+++   .|-..|.+.+|-+.-+.+
</pre><pre>T Consensus        71 ivevk~~-~~~~~~~~~ql~~yl~~~~~~~G~li  103 (115)
</pre><pre>T B3E6N8_TRIL1/4   71 ILELKAV-KSLLPEHQAQLLNYLKASGLPVGLLI  103 (115)
</pre><pre>T ss_pred             EEEEEeh-hcCCHHHHHHHHHHHHHhCCCEEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 64
</pre><pre>>PF04556.17 ; DpnII ; DpnII restriction endonuclease
</pre><pre>Probab=24.33  E-value=3.9e+02  Score=19.98  Aligned_cols=40  Identities=20%  Similarity=0.075  Sum_probs=0.0  Template_Neff=7.800
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      |.-|++|.||.+        .+-.-+||+||-..+=|..++.++.-+.
</pre><pre>T Consensus       198 ~~~k~~DfvI~~--------~~~~i~IE~n~~~t~GSk~~~~~~~~~~  237 (289)
</pre><pre>T D6ZH53_MOBCV/3  198 KSSKRFDFVVCT--------DTTIFGIETNFYGSGGSKLNETARSYKQ  237 (289)
</pre><pre>T ss_pred             CCCCceeEEEec--------CCeEEEEEEeeccCCCcHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 65
</pre><pre>>7OF0_0 39S ribosomal protein L32, mitochondrial; Mitochondria, Ribosome, Biogenesis, GTPase, NSUN4, MTERF4; HET: MG; 2.2A {Homo sapiens}
</pre><pre>Probab=23.96  E-value=1.4e+02  Score=21.48  Aligned_cols=24  Identities=25%  Similarity=0.447  Sum_probs=0.0  Template_Neff=6.400
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCCCCCCCCCchhhEEEee
</pre><pre>Q MBL3828645.1/1    7 DVVIVNDPTKPPTQDNIKQVVEIK   30 (57)
</pre><pre>Q Consensus         7 DViiV~d~~~~p~qdNi~~vVE~K   30 (57)
</pre><pre>                      ||+++-..+......+=+++|||+
</pre><pre>T Consensus       155 ev~v~Y~~e~~~~~~~~k~ive~~  178 (188)
</pre><pre>T 7OF0_0          155 ETVVLYTGETPSEQDQGKRIIERD  178 (188)
</pre><pre>T ss_dssp             CEEEECTTCCCCSTTTTSEEEECS
</pre><pre>T ss_pred             cEEEeeCCCCCchhcCCcEEEeeC
</pre><pre>
</pre><pre>
</pre><pre>No 66
</pre><pre>>2Q8P_A Iron-regulated surface determinant E; helical backbone metal receptor superfamily, METAL TRANSPORT; HET: HEM, MSE; 1.95A {Staphylococcus aureus subsp. aureus}
</pre><pre>Probab=23.59  E-value=1.8e+02  Score=19.41  Aligned_cols=44  Identities=16%  Similarity=0.104  Sum_probs=0.0  Template_Neff=10.900
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhE-EEeeCCCCCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQV-VEIKFPPDKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~v-VE~KFp~D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||+||+......-..+.|+.. +.+-+-...--+...+..+.++
</pre><pre>T Consensus        60 ~PDlIi~~~~~~~~~~~~l~~~gi~v~~~~~~~~~~~~~~~~~lg  104 (260)
</pre><pre>T 2Q8P_A           60 KPTHVLSVSTIKDEMQPFYKQLNMKGYFYDFDSLKGMQKSITQLG  104 (260)
</pre><pre>T ss_dssp             CCSEEEEEGGGHHHHHHHHHHHTSCCEEECCSSHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEeccccchhhHHHHHHCCCeEEEeccccHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 67
</pre><pre>>PF22023.1 ; Pus10_THUMP_arc ; Pus10 THUMP domain, archaeal
</pre><pre>Probab=23.49  E-value=1.2e+02  Score=17.95  Aligned_cols=11  Identities=55%  Similarity=0.896  Sum_probs=0.0  Template_Neff=8.000
</pre><pre>
</pre><pre>Q ss_pred             ccCCcEEEEcC
</pre><pre>Q MBL3828645.1/1    3 VRRPDVVIVND   13 (57)
</pre><pre>Q Consensus         3 ~RrPDViiV~d   13 (57)
</pre><pre>                      ..+||++|+-|
</pre><pre>T Consensus        68 ~~~PDv~i~vD   78 (78)
</pre><pre>T E1RIL6_METP4/7   68 FKRPDVVAVCN   78 (78)
</pre><pre>T ss_pred             CCCCCEEEEEC
</pre><pre>
</pre><pre>
</pre><pre>No 68
</pre><pre>>6HZ4_N Protein McrC; AAA+ superfamily, restriction enzyme, DNA BINDING PROTEIN; HET: GNP, GDP; 3.6A {Escherichia coli (strain K12)}
</pre><pre>Probab=23.37  E-value=3.6e+02  Score=19.52  Aligned_cols=45  Identities=13%  Similarity=0.163  Sum_probs=0.0  Template_Neff=10.000
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCC------------CCCCHHHHHHHHHHhCCc----ccceeC
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPP------------DKMDIKQERAYIRIAGDQ----TKLVEL   57 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~------------D~l~~~Q~~~Y~~IAG~~----~k~~~l   57 (57)
</pre><pre>                      +||+++..        .+-..|++.||-.            ..=+-.|.-+|..-.+.+    .+..++
</pre><pre>T Consensus       242 ~PDi~l~~--------~~~~~vlDaKYk~~~~~~~~~~~~~~~~diyQ~~~Y~~~~~~~~~~~~~~~Ll  302 (348)
</pre><pre>T 6HZ4_N          242 ETDITIRS--------SEKILIVDAKYYKSIFSRRMGTEKFHSQNLYQLMNYLWSLKPENGENIGGLLI  302 (348)
</pre><pre>T ss_dssp             ECSEEEEC--------SSCEEEECCCCCHHHHCC----CCCCCTTHHHHHHHHHHCCCSSCSCEEEEEE
</pre><pre>T ss_pred             cCcEEEEc--------CCeEEEEEEecChhhhhhhcCCccccchhHHHHHHHHHhcCccCCCCceEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 69
</pre><pre>>6UT8_G McrBC 5-methylcytosine restriction system component; Endonuclease, AAA protein, GTPase, Methylation-dependent restriction, DNA BINDING PROTEIN; HET: GSP, GDP;{Thermococcus gammatolerans}
</pre><pre>Probab=23.32  E-value=2e+02  Score=22.00  Aligned_cols=37  Identities=14%  Similarity=0.091  Sum_probs=0.0  Template_Neff=9.300
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCC-CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPD-KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D-~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +||+++..+.       +...|++.|+-.. .-+..+..-|+-+|
</pre><pre>T Consensus       338 ~PDivi~~~~-------~~~~ViDaKyK~~~~~~~~~~DlyQ~~a  375 (458)
</pre><pre>T 6UT8_G          338 KPDYVVRKGN-------TPVVVLDAKYRELKERIPSSDMLRQLYV  375 (458)
</pre><pre>T ss_dssp             CCSEEEESTT-------SCCEEECCCCCCCGGGSCCHHHHHHHHH
</pre><pre>T ss_pred             eCeEEEEeCC-------ccEEEEEeCCcccccCCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 70
</pre><pre>>8A22_Bb uS2m; mitoribosome, ribosome, mitochondria, polytomella, fragmentation, rrna, evolution, translation; HET: K, ATP, MG; 2.91A {Polytomella magna}
</pre><pre>Probab=22.98  E-value=1.3e+02  Score=21.08  Aligned_cols=14  Identities=36%  Similarity=0.622  Sum_probs=0.0  Template_Neff=8.900
</pre><pre>
</pre><pre>Q ss_pred             CCccCCcEEEEcCC
</pre><pre>Q MBL3828645.1/1    1 GMVRRPDVVIVNDP   14 (57)
</pre><pre>Q Consensus         1 G~~RrPDViiV~d~   14 (57)
</pre><pre>                      ++.+.||+||+-++
</pre><pre>T Consensus       165 ~~~~~Pdlvii~~~  178 (236)
</pre><pre>T 8A22_Bb         165 GALSKLALVICLDP  178 (236)
</pre><pre>T ss_pred             CccccCCEEEEeCC
</pre><pre>
</pre><pre>
</pre><pre>No 71
</pre><pre>>PF18362.6 ; THB ; Tri-helix bundle domain
</pre><pre>Probab=22.80  E-value=20  Score=17.53  Aligned_cols=15  Identities=20%  Similarity=0.273  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1   34 DKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus        34 D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +.+...+..+|++||
</pre><pre>T Consensus         4 ~~l~~~~~~~~e~~~   18 (34)
</pre><pre>T H2LNL0_ORYLA/2    4 SILSKAPPSAFEKIA   18 (34)
</pre><pre>T ss_pred             HHHhcCCCChHHHHc
</pre><pre>
</pre><pre>
</pre><pre>No 72
</pre><pre>>PF15608.11 ; PELOTA_1 ; PELOTA RNA binding domain
</pre><pre>Probab=22.66  E-value=1.5e+02  Score=19.03  Aligned_cols=14  Identities=36%  Similarity=0.548  Sum_probs=0.0  Template_Neff=5.200
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCCCCC
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDPTKP   17 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~~~~   17 (57)
</pre><pre>                      |.|+.|+|++++.+
</pre><pre>T Consensus        36 R~P~~ilv~~~~~~   49 (81)
</pre><pre>T A8MM96_ALKOO/7   36 RVPWKILVRDYDNP   49 (81)
</pre><pre>T ss_pred             CCCcEEEEcCCCCc
</pre><pre>
</pre><pre>
</pre><pre>No 73
</pre><pre>>PF13101.11 ; DUF3945 ; Protein of unknown function (DUF3945)
</pre><pre>Probab=22.54  E-value=1.4e+02  Score=16.74  Aligned_cols=17  Identities=0%  Similarity=0.137  Sum_probs=0.0  Template_Neff=7.000
</pre><pre>
</pre><pre>Q ss_pred             eCCCCCCCHHHHHHHHH
</pre><pre>Q MBL3828645.1/1   30 KFPPDKMDIKQERAYIR   46 (57)
</pre><pre>Q Consensus        30 KFp~D~l~~~Q~~~Y~~   46 (57)
</pre><pre>                      ++.|-.||+.|++++..
</pre><pre>T Consensus        20 ~i~gv~Lt~~Q~~~L~~   36 (50)
</pre><pre>T I8XB01_9BACE/2   20 RIKEADLSEGKQMDFYG   36 (50)
</pre><pre>T ss_pred             eeCCeeCCHHHHHHHhC
</pre><pre>
</pre><pre>
</pre><pre>No 74
</pre><pre>>PF11496.13 ; HDA2-3 ; Class II histone deacetylase complex subunits 2 and 3
</pre><pre>Probab=22.39  E-value=1.7e+02  Score=19.51  Aligned_cols=13  Identities=23%  Similarity=0.220  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHHH
</pre><pre>Q MBL3828645.1/1   35 KMDIKQERAYIRI   47 (57)
</pre><pre>Q Consensus        35 ~l~~~Q~~~Y~~I   47 (57)
</pre><pre>                      .+++.|++.|.++
</pre><pre>T Consensus        11 ~ls~~q~~~Y~~~   23 (274)
</pre><pre>T Q6FSK5_CANGA/2   11 TMSLYQKELTDQI   23 (274)
</pre><pre>T ss_pred             CCCHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 75
</pre><pre>>5YET_A Uncharacterized protein R354; MIMIVIRE, Cas4-like, nuclease, R354, NUCLEAR PROTEIN; 2.806A {Acanthamoeba polyphaga mimivirus}
</pre><pre>Probab=22.25  E-value=2.5e+02  Score=21.54  Aligned_cols=29  Identities=24%  Similarity=0.399  Sum_probs=0.0  Template_Neff=8.600
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPD   34 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D   34 (57)
</pre><pre>                      ||-++.......-..+....++|||.|..
</pre><pre>T Consensus       133 pDgi~~~~~~~~~~~~~~~~llEIK~p~~  161 (399)
</pre><pre>T 5YET_A          133 PDGICSKKANTGGLSKLVGRLLEIKFPFS  161 (399)
</pre><pre>T ss_dssp             CSEEECSBCTTSSEETTTTCEEEEECCSS
</pre><pre>T ss_pred             CCeEEecCCcCCCcccceeeEEEEECCCc
</pre><pre>
</pre><pre>
</pre><pre>No 76
</pre><pre>>7EEB_H Cation channel sperm-associated protein subunit epsilon; ion channel, membrane protein, calcium channel, protein complex, PROTEIN TRANSPORT; HET: BMA, 9Z9, NAG; 2.9A {Mus musculus}
</pre><pre>Probab=22.14  E-value=76  Score=27.94  Aligned_cols=50  Identities=14%  Similarity=0.179  Sum_probs=0.0  Template_Neff=7.200
</pre><pre>
</pre><pre>Q ss_pred             EEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCcccceeC
</pre><pre>Q MBL3828645.1/1    8 VVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQTKLVEL   57 (57)
</pre><pre>Q Consensus         8 ViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~~k~~~l   57 (57)
</pre><pre>                      +|++.+..--.|+||.....|+++|+..++++++-.-..+|=-.+.+.+|
</pre><pre>T Consensus       280 avlvs~~~~f~T~DsF~T~~~l~vp~~~l~~~~~~~Vssaaf~~~~I~~L  329 (985)
</pre><pre>T 7EEB_H          280 ALLATDQETFQTNDSFQTWTRVRAPPGILSDAQRHSLRDVIIFDQGTLFL  329 (985)
</pre><pre>T ss_dssp             EEEECSSCEEEESSSSSSCEECEECTTSSCHHHHTTEEEEEECSSCEEEE
</pre><pre>T ss_pred             EEEeeCCCeEEeCCCCcceEEEEeCCCcCCHHhcceeeeEEEeCCcEEEE
</pre><pre>
</pre><pre>
</pre><pre>No 77
</pre><pre>>3R5T_A Ferric vibriobactin ABC transporter, periplasmic ferric vibriobactin-binding protein; Iron-vibriobactin transport protein, Periplasmic transport protein, Iron-siderophore binding, METAL TRANSPORT; HET: VBN; 1.45A {Vibrio cholerae} SCOP: c.92.2.0, l.1.1.1
</pre><pre>Probab=21.83  E-value=91  Score=21.51  Aligned_cols=27  Identities=26%  Similarity=0.465  Sum_probs=0.0  Template_Neff=10.600
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCC
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFP   32 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp   32 (57)
</pre><pre>                      +||+||+......-...++++|-++ .|
</pre><pre>T Consensus        86 ~PDlIi~~~~~~~~~~~~~~~l~~~-~p  112 (305)
</pre><pre>T 3R5T_A           86 QPDLIVVSMIGADSARDQIPLLQAI-AP  112 (305)
</pre><pre>T ss_dssp             CCSEEEEESSSTTCCGGGHHHHHTT-SC
</pre><pre>T ss_pred             CCCEEEEeCCCCcchhhHHHHHHhh-CC
</pre><pre>
</pre><pre>
</pre><pre>No 78
</pre><pre>>1ZDB_A MINI PROTEIN A DOMAIN, Z38; IGG BINDING DOMAIN, PROTEIN A MIMIC; NMR {synthetic construct} SCOP: k.13.1.1
</pre><pre>Probab=21.56  E-value=2e+02  Score=16.36  Aligned_cols=13  Identities=23%  Similarity=0.360  Sum_probs=0.0  Template_Neff=4.100
</pre><pre>
</pre><pre>Q ss_pred             CCCCHHHHHHHHH
</pre><pre>Q MBL3828645.1/1   34 DKMDIKQERAYIR   46 (57)
</pre><pre>Q Consensus        34 D~l~~~Q~~~Y~~   46 (57)
</pre><pre>                      ..|++.|+.+|..
</pre><pre>T Consensus        21 pnLne~Qr~~fI~   33 (38)
</pre><pre>T 1ZDB_A           21 PNLNEEQRNAKIK   33 (38)
</pre><pre>T ss_dssp             SSSCHHHHHHHHH
</pre><pre>T ss_pred             CCCCHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 79
</pre><pre>>2FZL_A DNA repair protein RAD25, XPB; XPB, nucleotide excision repair, dna repair, DNA BINDING PROTEIN; 2.9A {Archaeoglobus fulgidus} SCOP: c.37.1.19
</pre><pre>Probab=21.43  E-value=1.7e+02  Score=18.12  Aligned_cols=15  Identities=7%  Similarity=0.093  Sum_probs=0.0  Template_Neff=12.100
</pre><pre>
</pre><pre>Q ss_pred             CCCHHHHHHHHHHhC
</pre><pre>Q MBL3828645.1/1   35 KMDIKQERAYIRIAG   49 (57)
</pre><pre>Q Consensus        35 ~l~~~Q~~~Y~~IAG   49 (57)
</pre><pre>                      .+++.|++.|..+..
</pre><pre>T Consensus        35 ~l~~~~~~~y~~~~~   49 (219)
</pre><pre>T 2FZL_A           35 PLAEDERVEYEKREK   49 (219)
</pre><pre>T ss_dssp             ECCHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 80
</pre><pre>>PF13439.11 ; Glyco_transf_4 ; Glycosyltransferase Family 4
</pre><pre>Probab=21.29  E-value=1.4e+02  Score=16.36  Aligned_cols=11  Identities=9%  Similarity=0.135  Sum_probs=0.0  Template_Neff=13.600
</pre><pre>
</pre><pre>Q ss_pred             cCCcEEEEcCC
</pre><pre>Q MBL3828645.1/1    4 RRPDVVIVNDP   14 (57)
</pre><pre>Q Consensus         4 RrPDViiV~d~   14 (57)
</pre><pre>                      ++||+|++..+
</pre><pre>T Consensus        70 ~~~d~i~~~~~   80 (166)
</pre><pre>T L0L008_METHD/3   70 YNYDIIHAHGH   80 (166)
</pre><pre>T ss_pred             CCCCEEEecCh
</pre><pre>
</pre><pre>
</pre><pre>No 81
</pre><pre>>PF07832.16 ; Bse634I ; Cfr10I/Bse634I restriction endonuclease
</pre><pre>Probab=20.86  E-value=1.1e+02  Score=23.66  Aligned_cols=43  Identities=12%  Similarity=0.187  Sum_probs=0.0  Template_Neff=5.900
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCch-hhEEEeeCCCCCCCHHHHHHHHHHhC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNI-KQVVEIKFPPDKMDIKQERAYIRIAG   49 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi-~~vVE~KFp~D~l~~~Q~~~Y~~IAG   49 (57)
</pre><pre>                      ||.|||..+...-..+|. ...+=-+++.-+.+... .+|..+.|
</pre><pre>T Consensus       133 PD~viv~~~~~~~~~~~~~~~~~~~~i~~ls~~~~~-~~y~~l~G  176 (285)
</pre><pre>T T2CX_CITFR/1-2  133 PDFSIIDIRGRREELKSMLKDISFSNISLSTISEID-NLYKNFID  176 (285)
</pre><pre>T ss_pred             CCEEEEEcCCcchHHHHhhhhhhccchhhCCHHHHH-HHHHHHhh
</pre><pre>
</pre><pre>
</pre><pre>No 82
</pre><pre>>1CFR_A RESTRICTION ENDONUCLEASE; RESTRICTION ENDONUCLEASE, RESTRICTION ENZYME; 2.15A {Citrobacter freundii} SCOP: c.52.1.7
</pre><pre>Probab=20.81  E-value=2.1e+02  Score=22.15  Aligned_cols=40  Identities=10%  Similarity=0.204  Sum_probs=0.0  Template_Neff=6.000
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCCCCCHHHHH----HHHHHhC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPDKMDIKQER----AYIRIAG   49 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D~l~~~Q~~----~Y~~IAG   49 (57)
</pre><pre>                      ||.|||.-++..+...+...+.++.=    ++.....    +|..+.|
</pre><pre>T Consensus       133 PD~Viv~~~~~~~~~~~~~~~~~i~~----l~~~~l~~l~~~y~~l~G  176 (285)
</pre><pre>T 1CFR_A          133 PDFSIIDIRGRREELKSMLKDISFSN----ISLSTISEIDNLYKNFID  176 (285)
</pre><pre>T ss_dssp             CSEEEEECTTCHHHHHHHTTTCCSTT----CCHHHHHHHHHGGGGGTT
</pre><pre>T ss_pred             CCEEEEEcCCcchhhHhccccCCccc----CCHHHHHHHHHHHHHhch
</pre><pre>
</pre><pre>
</pre><pre>No 83
</pre><pre>>PF09563.15 ; RE_LlaJI ; LlaJI restriction endonuclease
</pre><pre>Probab=20.73  E-value=1.9e+02  Score=21.16  Aligned_cols=39  Identities=10%  Similarity=0.112  Sum_probs=0.0  Template_Neff=9.600
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCCCC-------CCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPPDK-------MDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~D~-------l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      +||+++..     ........|+..|+-.-.       -......-|+.+|
</pre><pre>T Consensus       265 ~PDivi~~-----~~~~~~~~IiDaKYk~~~~~~~~~~~~~~~~DiyQ~~~  310 (362)
</pre><pre>T Q81H81_BACCR/5  265 RPDIIKTY-----IDIQRYFLILDAKYYNIRFKEGDLKGNPGVNDIAKQLL  310 (362)
</pre><pre>T ss_pred             CCeEEEEE-----eCCccEEEEEEcCccceeecCCCccCCCChHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 84
</pre><pre>>3LHS_A Ferrichrome ABC transporter lipoprotein; siderophore, iron, receptor, lipoprotein, binding protein, TRANSPORT PROTEIN; HET: GOL, SF8; 1.3A {Staphylococcus aureus subsp. aureus strain} SCOP: c.92.2.0, l.1.1.1
</pre><pre>Probab=20.59  E-value=2.1e+02  Score=19.27  Aligned_cols=44  Identities=14%  Similarity=0.244  Sum_probs=0.0  Template_Neff=11.200
</pre><pre>
</pre><pre>Q ss_pred             CCcEEEEcCCCCCCCCCchhhEEEeeCCC--CCCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    5 RPDVVIVNDPTKPPTQDNIKQVVEIKFPP--DKMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         5 rPDViiV~d~~~~p~qdNi~~vVE~KFp~--D~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      .||+||............++...-+-+-.  ..--+.-.+.+..|+
</pre><pre>T Consensus        85 ~PDlii~~~~~~~~~~~~l~~~~~~v~~~~~~~~~~~~~~~~~~lg  130 (296)
</pre><pre>T 3LHS_A           85 KPDLIIADSSRHKGINKELNKIAPTLSLKSFDGDYKQNINSFKTIA  130 (296)
</pre><pre>T ss_dssp             CCSEEEEETTTTTTTHHHHHHHSCEEEECSTTCCHHHHHHHHHHHH
</pre><pre>T ss_pred             CCCEEEEeCccchhHHHHHHHhCCEEEEecCCCCHHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 85
</pre><pre>>1ZP7_B Recombination protein U; recombination, DNA-binding protein, resolvase, DNA BINDING PROTEIN; 2.25A {Bacillus subtilis} SCOP: c.52.1.28
</pre><pre>Probab=20.58  E-value=5.1e+02  Score=18.37  Aligned_cols=35  Identities=11%  Similarity=-0.042  Sum_probs=0.0  Template_Neff=7.900
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCCCCCchhhEEEeeCCCC-------CCCHHHHHHHHHHh
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPPTQDNIKQVVEIKFPPD-------KMDIKQERAYIRIA   48 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p~qdNi~~vVE~KFp~D-------~l~~~Q~~~Y~~IA   48 (57)
</pre><pre>                      ||++.+.+        +.--.+|+|-...       .+++.|.+.-.+++
</pre><pre>T Consensus        87 ~D~i~~~~--------g~~~~~E~K~t~~~~~~~~~~i~~~Q~~~L~~~~  128 (206)
</pre><pre>T 1ZP7_B           87 TDYNGIYK--------GRYIDFEAKETKNKTSFPLQNFHDHQIEHMKQVK  128 (206)
</pre><pre>T ss_dssp             CSEEEEET--------TEEEEEEEEECCCSSEEEGGGSCHHHHHHHHHHH
</pre><pre>T ss_pred             CCEEEEEC--------CeEEEEEEeeCCCCCccchhhCCHHHHHHHHHHH
</pre><pre>
</pre><pre>
</pre><pre>No 86
</pre><pre>>PF18569.6 ; Thioredoxin_16 ; Thioredoxin-like domain
</pre><pre>Probab=20.53  E-value=1.3e+02  Score=18.55  Aligned_cols=13  Identities=54%  Similarity=0.971  Sum_probs=0.0  Template_Neff=7.600
</pre><pre>
</pre><pre>Q ss_pred             CcEEEEcCCCCCC
</pre><pre>Q MBL3828645.1/1    6 PDVVIVNDPTKPP   18 (57)
</pre><pre>Q Consensus         6 PDViiV~d~~~~p   18 (57)
</pre><pre>                      .|+||..||+.+|
</pre><pre>T Consensus         4 ~diVI~a~P~~PP   16 (92)
</pre><pre>T F7ELU9_MONDO/1    4 KDIVINANPTVPP   16 (92)
</pre><pre>T ss_pred             eEEEEEcCCCChh
</pre><pre>
</pre><pre>
</pre><pre>No 87
</pre><pre>>9E5C_3m Chaperone protein DNAJ, putative; flagella, microtubule, MOTOR PROTEIN; HET: ZN, MG, GDP, GTP, ATP, ADP; 3.2A {Trypanosoma brucei brucei TREU927}
</pre><pre>Probab=20.44  E-value=5.1e+02  Score=18.07  Aligned_cols=44  Identities=16%  Similarity=0.206  Sum_probs=0.0  Template_Neff=10.700
</pre><pre>
</pre><pre>Q ss_pred             cEEEEcCCCCC------CCCCchhhEEEeeCCCCCCCHHHHHHHHHHhCCc
</pre><pre>Q MBL3828645.1/1    7 DVVIVNDPTKP------PTQDNIKQVVEIKFPPDKMDIKQERAYIRIAGDQ   51 (57)
</pre><pre>Q Consensus         7 DViiV~d~~~~------p~qdNi~~vVE~KFp~D~l~~~Q~~~Y~~IAG~~   51 (57)
</pre><pre>                      +++++.....+      ....||---+.|.||. .++..|++.+++|...+
</pre><pre>T Consensus       270 ~~~~i~g~G~p~~~~~~~~~Gdl~v~~~v~~P~-~l~~~~~~~l~~~~~~~  319 (319)
</pre><pre>T 9E5C_3m         270 YSRRVVGQGLPRNDEPSNPRGDLIVECDTTFPG-FLTLEQKSELSRILDAK  319 (319)
</pre><pre>T ss_pred             cEEEECCCCCccCCCCCCCCCCEEEEEEEECCc-CCCHHHHHHHHHHhccC
</pre><pre>
</pre>
